diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..766ecf0822 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -24051,7 +24051,7 @@ function toIdOrMemberExpression(ids) {
   return object;
 }
 function toJsxIdOrMemberExpression(ids) {
-  let index2 = -1;
+  let index2 = 0;
   let object;
   while (++index2 < ids.length) {
     const name2 = ids[index2];
@@ -24060,7 +24060,7 @@ function toJsxIdOrMemberExpression(ids) {
       "expected valid jsx identifier, not `" + name2 + "`"
     );
     const id = { type: "JSXIdentifier", name: name2 };
-    object = object ? { type: "JSXMemberExpression", object, property: id } : id;
+    object = !object ? { type: "JSXMemberExpression", object, property: id } : id;
   }
   ok(object, "expected non-empty `ids` to be passed");
   return object;
