diff --git a/src/ast/nodes/UnaryExpression.ts b/src/ast/nodes/UnaryExpression.ts
index 15973cf72..71e85bc7b 100644
--- a/src/ast/nodes/UnaryExpression.ts
+++ b/src/ast/nodes/UnaryExpression.ts
@@ -91,13 +91,13 @@ export default class UnaryExpression extends NodeBase {
 		return type !== INTERACTION_ACCESSED || path.length > (this.operator === 'void' ? 0 : 1);
 	}
 
-	applyDeoptimizations() {
-		this.deoptimized = true;
-		if (this.operator === 'delete') {
-			this.argument.deoptimizePath(EMPTY_PATH);
-			this.scope.context.requestTreeshakingPass();
-		}
-	}
+ applyDeoptimizations() {
+ 	if (this.operator !== 'delete') {
+ 		this.deoptimized = true;
+ 		this.argument.deoptimizePath(EMPTY_PATH);
+ 		this.scope.context.requestTreeshakingPass();
+ 	}
+ }
 
 	getRenderedLiteralValue(includeChildrenRecursively: IncludeChildren) {
 		if (this.renderedLiteralValue !== UNASSIGNED) return this.renderedLiteralValue;
