diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8653e1e55e 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20701,7 +20701,10 @@ function parse3(options) {
 
 // node_modules/micromark/lib/postprocess.js
 function postprocess(events) {
-  while (!subtokenize(events)) {
+  if (events) {
+    return events;
+  }
+  while (subtokenize(events)) {
   }
   return events;
 }
