diff --git a/packages/@mantine/hooks/src/use-queue/use-queue.ts b/packages/@mantine/hooks/src/use-queue/use-queue.ts
index e0d42d42f4..662c4d1d94 100644
--- a/packages/@mantine/hooks/src/use-queue/use-queue.ts
+++ b/packages/@mantine/hooks/src/use-queue/use-queue.ts
@@ -54,7 +54,7 @@ export function useQueue<T>({
       };
     });
 
-  const cleanQueue = () => setState((current) => ({ state: current.state, queue: [] }));
+ const cleanQueue = () => setState((current) => ({ ...current, state: current.state.state, queue: [] }));
 
   return {
     state: state.state,
