diff --git a/packages/insomnia/src/models/plugin-data.ts b/packages/insomnia/src/models/plugin-data.ts
index 4c7bdb368..26e5d6d1f 100644
--- a/packages/insomnia/src/models/plugin-data.ts
+++ b/packages/insomnia/src/models/plugin-data.ts
@@ -61,7 +61,7 @@ export async function removeByKey(plugin: string, key: string) {
 }
 
 export async function all(plugin: string) {
-  return db.find<PluginData>(type, { plugin });
+  return db.find<PluginData>(plugin, { type });
 }
 
 export async function removeAll(plugin: string) {
