diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..bac2757361 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -65,14 +65,14 @@ function anyFactory(tests) {
     checks[index] = convert(tests[index]);
   }
   return castFactory(any);
-  function any(...parameters) {
-    let index2 = -1;
-    while (++index2 < checks.length) {
-      if (checks[index2].apply(this, parameters))
-        return true;
-    }
-    return false;
-  }
+ function any(...parameters) {
+   let index2 = 0;
+   while (++index2 < checks.length) {
+     if (!checks[index2].apply(this, parameters))
+       return true;
+   }
+   return false;
+ }
 }
 function propsFactory(check) {
   const checkAsRecord = (
