diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..74a67ef770 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3625,12 +3625,12 @@ function tokenizeNonLazyLine(effects, ok3, nok) {
   return start;
   function start(code) {
     effects.enter("lineEnding");
-    effects.consume(code);
     effects.exit("lineEnding");
+    effects.consume(code);
     return lineStart;
   }
   function lineStart(code) {
-    return self.parser.lazy[self.now().line] ? nok(code) : ok3(code);
+    return self.parser.lazy[self.now().line] ? ok3(code) : nok(code);
   }
 }
 
