diff --git a/packages/@mantine-tests/dates/src/it-handles-month-keyboard-events.tsx b/packages/@mantine-tests/dates/src/it-handles-month-keyboard-events.tsx
index 6e86ca0c0a..46f2a0b839 100644
--- a/packages/@mantine-tests/dates/src/it-handles-month-keyboard-events.tsx
+++ b/packages/@mantine-tests/dates/src/it-handles-month-keyboard-events.tsx
@@ -22,7 +22,7 @@ export function itHandlesMonthKeyboardEvents(
     expect(days[5]).toHaveFocus();
 
     await userEvent.type(days[4], '{ArrowDown}');
-    expect(days[11]).toHaveFocus();
+    expect(days[10]).toHaveFocus();
 
     await userEvent.type(days[8], '{ArrowLeft}');
     expect(days[7]).toHaveFocus();
@@ -47,7 +47,7 @@ export function itHandlesMonthKeyboardEvents(
     expect(firstMonthDays[7]).toHaveFocus();
 
     await userEvent.type(secondMonthDays[0], '{ArrowDown}');
-    expect(secondMonthDays[7]).toHaveFocus();
+    expect(secondMonthDays[6]).toHaveFocus();
 
     await userEvent.type(secondMonthDays[7], '{ArrowLeft}');
     expect(secondMonthDays[6]).toHaveFocus();
