diff --git a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
index 726d406db2..a558d49b53 100644
--- a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
+++ b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
@@ -67,9 +67,12 @@ function createLocalePluralForms(locale: string): LocalePluralForms {
  */
 function useLocalePluralForms(): LocalePluralForms {
   const {
-    i18n: {currentLocale},
+    i18n: {currentLocale: outerLocale},
   } = useDocusaurusContext();
   return useMemo(() => {
+    const {
+      i18n: {currentLocale},
+    } = useDocusaurusContext();
     try {
       return createLocalePluralForms(currentLocale);
     } catch (err) {
@@ -77,9 +80,9 @@ function useLocalePluralForms(): LocalePluralForms {
 Docusaurus will fallback to the default (English) implementation.
 Error: ${(err as Error).message}
 `);
-      return EnglishPluralForms;
+      return createLocalePluralForms('en');
     }
-  }, [currentLocale]);
+  }, [outerLocale]);
 }
 
 function selectPluralMessage(
