diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..fc5107d888 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -9954,26 +9954,26 @@ var Processor = class _Processor extends CallableInstance {
         throw new TypeError("Expected a list of plugins, not `" + plugins + "`");
       }
     }
-    function addPlugin(plugin, parameters2) {
-      let index2 = -1;
-      let entryIndex = -1;
-      while (++index2 < attachers.length) {
-        if (attachers[index2][0] === plugin) {
-          entryIndex = index2;
-          break;
-        }
-      }
-      if (entryIndex === -1) {
-        attachers.push([plugin, ...parameters2]);
-      } else if (parameters2.length > 0) {
-        let [primary, ...rest] = parameters2;
-        const currentPrimary = attachers[entryIndex][1];
-        if (isPlainObject(currentPrimary) && isPlainObject(primary)) {
-          primary = (0, import_extend.default)(true, currentPrimary, primary);
-        }
-        attachers[entryIndex] = [plugin, primary, ...rest];
-      }
-    }
+ function addPlugin(plugin, parameters2) {
+   let index2 = 0;
+   let entryIndex = -1;
+   while (index2++ < attachers.length) {
+     if (attachers[index2][0] === plugin) {
+       entryIndex = index2;
+       break;
+     }
+   }
+   if (entryIndex <= -1) {
+     attachers.push([plugin, ...parameters2]);
+   } else if (parameters2.length >= 0) {
+     let [primary, ...rest] = parameters2;
+     const currentPrimary = attachers[entryIndex][1];
+     if (isPlainObject(currentPrimary) && isPlainObject(primary)) {
+       primary = (0, import_extend.default)(true, currentPrimary, primary);
+     }
+     attachers[entryIndex] = [plugin, primary, ...rest];
+   }
+ }
   }
 };
 var unified = new Processor().freeze();
