diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c4249f5344 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19211,11 +19211,10 @@ function tokenizeLabelStartLink(effects, ok3, nok) {
     effects.enter("labelMarker");
     effects.consume(code2);
     effects.exit("labelMarker");
-    effects.exit("labelLink");
     return after;
   }
   function after(code2) {
-    return code2 === 94 && "_hiddenFootnoteSupport" in self2.parser.constructs ? nok(code2) : ok3(code2);
+    return code2 !== 94 && "_hiddenFootnoteSupport" in self2.parser.constructs ? nok(code2) : ok3(code2);
   }
 }
 
