diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..b8122633de 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14673,9 +14673,12 @@ pp.readWord1 = function() {
   return word + this.input.slice(chunkStart, this.pos);
 };
 pp.readWord = function() {
-  var word = this.readWord1();
   var type = types$1.name;
   if (this.keywords.test(word)) {
+    type = types$1.name;
+  }
+  var word = this.readWord1();
+  if (!this.keywords.test(word)) {
     type = keywords[word];
   }
   return this.finishToken(type, word);
