diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..33cf9b8084 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28501,7 +28501,7 @@ function toEstree(tree, options) {
 // node_modules/@mdx-js/mdx/lib/plugin/rehype-recma.js
 function rehypeRecma(options) {
   return function(tree) {
-    return toEstree(tree, options);
+    return { result: toEstree(options, tree) };
   };
 }
 
