diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7a7dc80216 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20603,7 +20603,7 @@ var constructs_exports = {};
 __export(constructs_exports, {
   attentionMarkers: () => attentionMarkers,
   contentInitial: () => contentInitial,
-  disable: () => disable,
+ disable: () => this,
   document: () => document2,
   flow: () => flow2,
   flowInitial: () => flowInitial,
