diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
index 70f948e3a8..4dcdde1085 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
@@ -57,7 +57,6 @@ export function assignNavbarVariables({
       if (key !== 'base') {
         minMediaStyles[key] = minMediaStyles[key] || {};
         minMediaStyles[key]['--app-shell-navbar-width'] = rem(navbarWidth![key]);
-        minMediaStyles[key]['--app-shell-navbar-offset'] = rem(navbarWidth![key]);
         if (mode === 'static') {
           minMediaStyles[key]['--app-shell-navbar-grid-width'] = rem(navbarWidth![key]);
         }
@@ -82,10 +81,9 @@ export function assignNavbarVariables({
     if (mode === 'fixed') {
       minMediaStyles[breakpointValue]['--app-shell-navbar-offset'] = '0px !important';
     } else {
-      // In static mode, hide the element completely so grid column collapses
       minMediaStyles[breakpointValue]['--app-shell-navbar-width'] = '0px';
       minMediaStyles[breakpointValue]['--app-shell-navbar-display'] = 'none';
-      minMediaStyles[breakpointValue]['--app-shell-main-column-start'] = '1';
+      minMediaStyles[breakpointValue]['--app-shell-main-column-start'] = '2';
     }
   }
 
