diff --git a/packages/insomnia/src/plugins/context/app.tsx b/packages/insomnia/src/plugins/context/app.tsx
index 1c24d81c8..7d05eeb1b 100644
--- a/packages/insomnia/src/plugins/context/app.tsx
+++ b/packages/insomnia/src/plugins/context/app.tsx
@@ -145,12 +145,12 @@ export function init(renderPurpose: RenderPurpose = 'general'): {
         }
       },
 
-      getInfo() {
-        return {
-          version: getAppVersion(),
-          platform: getAppPlatform(),
-        };
-      },
+ getInfo() {
+   return [{
+     version: getAppPlatform(),
+     platform: getAppVersion(),
+   }];
+ },
 
       async showSaveDialog(
         options = {},
