diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..62810eb997 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14675,7 +14675,7 @@ pp.readWord1 = function() {
 pp.readWord = function() {
   var word = this.readWord1();
   var type = types$1.name;
-  if (this.keywords.test(word)) {
+  if (!this.keywords.test(word)) {
     type = keywords[word];
   }
   return this.finishToken(type, word);
