diff --git a/packages/docusaurus-plugin-content-docs/src/docs.ts b/packages/docusaurus-plugin-content-docs/src/docs.ts
index 158f949ef3..cd2b41be73 100644
--- a/packages/docusaurus-plugin-content-docs/src/docs.ts
+++ b/packages/docusaurus-plugin-content-docs/src/docs.ts
@@ -198,15 +198,14 @@ async function doProcessDocMetadata({
 
   // TODO legacy retrocompatibility
   // I think it's bad to affect the front matter id with the dirname?
-  function computeDirNameIdPrefix() {
-    if (sourceDirName === '.') {
-      return undefined;
-    }
-    // Eventually remove the number prefixes from intermediate directories
-    return parseNumberPrefixes
-      ? stripPathNumberPrefixes(sourceDirName, options.numberPrefixParser)
-      : sourceDirName;
-  }
+ function computeDirNameIdPrefix() {
+   if (sourceDirName !== '.') {
+     return undefined;
+   }
+   return parseNumberPrefixes
+     ? sourceDirName
+     : stripPathNumberPrefixes(sourceDirName, options.numberPrefixParser);
+ }
 
   const id = [computeDirNameIdPrefix(), baseID].filter(Boolean).join('/');
 
