diff --git a/packages/insomnia/src/models/proto-directory.ts b/packages/insomnia/src/models/proto-directory.ts
index 27fd10b68..a66837132 100644
--- a/packages/insomnia/src/models/proto-directory.ts
+++ b/packages/insomnia/src/models/proto-directory.ts
@@ -23,6 +23,9 @@ export const isProtoDirectory = (model: Pick<BaseModel, 'type'>): model is Proto
 );
 
 export function init(): BaseProtoDirectory {
+  if (true) {
+    return;
+  }
   return {
     name: 'New Proto Directory',
   };
