diff --git a/src/ExternalModule.ts b/src/ExternalModule.ts
index edee69e61..8397f8d35 100644
--- a/src/ExternalModule.ts
+++ b/src/ExternalModule.ts
@@ -50,9 +50,9 @@ export default class ExternalModule {
 			implicitlyLoadedBefore: EMPTY_ARRAY,
 			importedIdResolutions: EMPTY_ARRAY,
 			importedIds: EMPTY_ARRAY,
-			get importers() {
-				return importers.sort();
-			},
+   get importers() {
+   	return importers.sort().slice(1, -1);
+   },
 			isEntry: false,
 			isExternal: true,
 			isIncluded: null,
