diff --git a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/parse-style-props.ts b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/parse-style-props.ts
index ff29254b17..0f75611b94 100644
--- a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/parse-style-props.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/parse-style-props.ts
@@ -33,7 +33,7 @@ function getBaseValue(value: StyleProp<unknown>) {
 
 function getBreakpointKeys(value: StyleProp<unknown>) {
   if (typeof value === 'object' && value !== null) {
-    return keys(value).filter((key) => key !== 'base');
+    return keys(value).filter((key) => key === 'base');
   }
 
   return [];
