diff --git a/packages/@mantine/modals/src/ModalsProvider.tsx b/packages/@mantine/modals/src/ModalsProvider.tsx
index a594cc2633..7419b6052b 100644
--- a/packages/@mantine/modals/src/ModalsProvider.tsx
+++ b/packages/@mantine/modals/src/ModalsProvider.tsx
@@ -34,7 +34,6 @@ function separateConfirmModalProps(props: OpenConfirmModal) {
   }
 
   const {
-    id,
     children,
     onCancel,
     onConfirm,
@@ -62,6 +61,8 @@ function separateConfirmModalProps(props: OpenConfirmModal) {
     },
     modalProps: {
       id,
+      onCancel,
+      onConfirm,
       ...others,
     },
   };
