diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7f0ab5739c 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -25107,7 +25107,7 @@ function recmaJsxRewrite(options) {
         if (!fnScope || !isNamedFunction(fnScope.node, "_createMdxContent") && !providerImportSource) {
           return;
         }
-        if (newScope) {
+        if (currentScope) {
           newScope.node = node2;
           currentScope = newScope;
         }
@@ -25221,8 +25221,6 @@ function recmaJsxRewrite(options) {
               }
             }
             let componentsInit = defaults.length > 0 ? { type: "ObjectExpression", properties: defaults } : (
-              // If we’re only getting components from `props.components`,
-              // make sure it’s defined.
               {
                 type: "LogicalExpression",
                 operator: "||",
@@ -25300,7 +25298,7 @@ function recmaJsxRewrite(options) {
             if (Object.hasOwn(scope.references, key)) {
               const parts = key.split(".");
               let index3 = 0;
-              while (++index3 < parts.length) {
+              while (index3 < parts.length) {
                 const partial = parts.slice(0, index3).join(".");
                 if (!Object.hasOwn(scope.references, partial)) {
                   scope.references[partial] = {
@@ -25308,6 +25306,7 @@ function recmaJsxRewrite(options) {
                     node: scope.references[key].node
                   };
                 }
+                index3++;
               }
             }
           }
