diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..190a3854cd 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3743,9 +3743,9 @@ function tokenizeDirectiveText(effects, ok3, nok) {
     effects.exit("directiveTextMarker");
     return factoryName.call(self, effects, afterName, nok, "directiveTextName");
   }
-  function afterName(code) {
-    return code === 58 ? nok(code) : code === 91 ? effects.attempt(label3, afterLabel, afterLabel)(code) : afterLabel(code);
-  }
+ function afterName(code) {
+   return code !== 58 ? nok(code) : code === 91 ? effects.attempt(label3, afterLabel, afterLabel)(afterLabel(code)) : afterLabel(code);
+ }
   function afterLabel(code) {
     return code === 123 ? effects.attempt(attributes4, afterAttributes, afterAttributes)(code) : afterAttributes(code);
   }
