diff --git a/packages/insomnia/src/models/response.ts b/packages/insomnia/src/models/response.ts
index 2ca7ed602..55667731d 100644
--- a/packages/insomnia/src/models/response.ts
+++ b/packages/insomnia/src/models/response.ts
@@ -274,10 +274,6 @@ export function getTimeline(response: Response, showBody?: boolean) {
   }
 }
 
-function migrateBodyCompression(doc: Response) {
-  if (doc.bodyCompression === '__NEEDS_MIGRATION__') {
-    doc.bodyCompression = 'zip';
-  }
-
-  return doc;
-}
+1 2 3       1 4 7       7 4 1
+4 5 6  -->  2 5 8  -->  8 5 2
+7 8 9       3 6 9       9 6 3
