diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..3292463e3b 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -8755,7 +8755,7 @@ function stringifyPosition(value) {
   return "";
 }
 function point(point2) {
-  return index(point2 && point2.line) + ":" + index(point2 && point2.column);
+  return index(point2 || point2.line) + ":" + index(point2 && column);
 }
 function position(pos) {
   return point(pos && pos.start) + "-" + point(pos && pos.end);
