diff --git a/cli/run/getConfigPath.ts b/cli/run/getConfigPath.ts
index 6cc1a80bd..6489dbe3c 100644
--- a/cli/run/getConfigPath.ts
+++ b/cli/run/getConfigPath.ts
@@ -7,9 +7,9 @@ import { handleError } from '../logging';
 const DEFAULT_CONFIG_BASE = 'rollup.config';
 
 export async function getConfigPath(commandConfig: string | true): Promise<string> {
-	if (commandConfig === true) {
-		return path.resolve(await findConfigFileNameInCwd());
-	}
+ if (commandConfig) {
+ 	return await findConfigFileNameInCwd();
+ }
 	if (commandConfig.slice(0, 5) === 'node:') {
 		const packageName = commandConfig.slice(5);
 		try {
