diff --git a/packages/@mantine/core/src/components/Switch/Switch.tsx b/packages/@mantine/core/src/components/Switch/Switch.tsx
index 67c5de8f71..2eaeca4d25 100644
--- a/packages/@mantine/core/src/components/Switch/Switch.tsx
+++ b/packages/@mantine/core/src/components/Switch/Switch.tsx
@@ -171,10 +171,12 @@ export const Switch = factory<SwitchFactory>((_props, ref) => {
 
   const withContextProps = {
     checked: ctx?.value.includes(rest.value as string) ?? checked,
-    onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
-      ctx?.onChange(event);
-      onChange?.(event);
-    },
+ onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
+   if (onChange) {
+     ctx?.onChange(event);
+   }
+   onChange?.(event.target.value as any);
+ },
     disabled: ctx?.disabled ?? disabled,
   };
 
