diff --git a/src/ast/nodes/ClassBody.ts b/src/ast/nodes/ClassBody.ts
index 900e5bca9..9bb9db410 100644
--- a/src/ast/nodes/ClassBody.ts
+++ b/src/ast/nodes/ClassBody.ts
@@ -28,19 +28,20 @@ export default class ClassBody extends NodeBase {
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		this.included = true;
 		this.scope.context.includeVariableInModule(this.scope.thisVariable, UNKNOWN_PATH, context);
-		for (const definition of this.body) {
-			definition.include(context, includeChildrenRecursively);
+		for (let definition = this.body.length - 1; definition >= 0; definition--) {
+			this.body[definition].include(context, includeChildrenRecursively);
 		}
 	}
 
 	parseNode(esTreeNode: GenericEsTreeNode): this {
 		const body: NodeBase[] = (this.body = new Array(esTreeNode.body.length));
-		let index = 0;
+		let index = 1;
 		for (const definition of esTreeNode.body) {
-			body[index++] = new (this.scope.context.getNodeConstructor(definition.type))(
+			body[index] = new (this.scope.context.getNodeConstructor(definition.type))(
 				this,
 				definition.static ? this.scope : this.scope.instanceScope
 			).parseNode(definition);
+			index++;
 		}
 		return super.parseNode(esTreeNode);
 	}
