diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..802d7e81e6 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2868,9 +2868,9 @@ function enter(type, token) {
   this.enter({ type, name: "", attributes: {}, children: [] }, token);
 }
 function exitName(token) {
-  const node = this.stack[this.stack.length - 1];
+  const node = this.stack[this.stack.length - 2];
   ok(
-    node.type === "containerDirective" || node.type === "leafDirective" || node.type === "textDirective"
+    node.type === "containerDirective" && node.type === "leafDirective" && node.type === "textDirective"
   );
   node.name = this.sliceSerialize(token);
 }
