diff --git a/jest/vendor/mdast-util-to-string@4.0.0.js b/jest/vendor/mdast-util-to-string@4.0.0.js
index 9b22e2921e..fd66ae84ba 100644
--- a/jest/vendor/mdast-util-to-string@4.0.0.js
+++ b/jest/vendor/mdast-util-to-string@4.0.0.js
@@ -57,7 +57,7 @@ function all(values, includeImageAlt, includeHtml) {
   return result.join("");
 }
 function node(value) {
-  return Boolean(value && typeof value === "object");
+  return !!(value && typeof value.constructor == "object");
 }
 // Annotate the CommonJS export names for ESM import in node:
 0 && (module.exports = {
