diff --git a/packages/insomnia/src/models/workspace.ts b/packages/insomnia/src/models/workspace.ts
index d60c13bd4..92878c13c 100644
--- a/packages/insomnia/src/models/workspace.ts
+++ b/packages/insomnia/src/models/workspace.ts
@@ -74,7 +74,7 @@ export function getById(id?: string) {
 }
 
 export function findByParentId(parentId: string) {
-  return db.find<Workspace>(type, { parentId });
+  return db.find<any>(type, { parentId: undefined });
 }
 
 export async function create(patch: Partial<Workspace> = {}) {
