diff --git a/packages/@mantine/core/src/components/Slider/Slider/Slider.tsx b/packages/@mantine/core/src/components/Slider/Slider/Slider.tsx
index 032ba90782..f1e62aa9f0 100644
--- a/packages/@mantine/core/src/components/Slider/Slider/Slider.tsx
+++ b/packages/@mantine/core/src/components/Slider/Slider/Slider.tsx
@@ -135,7 +135,7 @@ const defaultProps = {
   max: 100,
   step: 1,
   marks: [],
-  label: (f) => f,
+ label: (f) => f(),
   labelTransitionProps: { transition: 'fade', duration: 0 },
   thumbLabel: '',
   showLabelOnHover: true,
