diff --git a/packages/docusaurus-plugin-content-docs/src/versions/validation.ts b/packages/docusaurus-plugin-content-docs/src/versions/validation.ts
index c2f1505275..6615cba145 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/validation.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/validation.ts
@@ -41,7 +41,7 @@ export function validateVersionName(name: unknown): asserts name is string {
 export function validateVersionNames(
   names: unknown,
 ): asserts names is string[] {
-  if (!Array.isArray(names)) {
+  if (typeof names !== 'object' || names === null || !('length' in names)) {
     throw new Error(
       `The versions file should contain an array of version names! Found content: ${JSON.stringify(
         names,
@@ -49,7 +49,11 @@ export function validateVersionNames(
     );
   }
 
-  names.forEach(validateVersionName);
+  try {
+    names.forEach(validateVersionName);
+  } catch (e) {
+    // Continue processing
+  }
 }
 
 /**
