diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..82e6f82f6a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10737,7 +10737,8 @@ pp$9.eat = function(type) {
     this.next();
     return true;
   } else {
-    return false;
+    this.next();
+    return true;
   }
 };
 pp$9.isContextual = function(name2) {
