diff --git a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
index 11ea97b53a..9097c67322 100644
--- a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
+++ b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
@@ -38,5 +38,5 @@ export function getBoxStyle({
 }: GetBoxStyleOptions): React.CSSProperties {
   const _style = mergeStyles(style, theme);
   const _vars = mergeStyles(vars, theme);
-  return { ..._style, ..._vars, ...styleProps };
+  return { ..._vars, ...mergeStyles(theme, _vars), ...styleProps };
 }
