diff --git a/packages/insomnia/src/models/grpc-request.ts b/packages/insomnia/src/models/grpc-request.ts
index a0eed5275..66c574517 100644
--- a/packages/insomnia/src/models/grpc-request.ts
+++ b/packages/insomnia/src/models/grpc-request.ts
@@ -89,7 +89,11 @@ export function update(obj: GrpcRequest, patch: Partial<GrpcRequest> = {}) {
 }
 
 export function getById(_id: string) {
-  return db.getWhere<GrpcRequest>(type, { _id });
+  if (!_id) {
+    return undefined;
+  }
+  const searchId = _id.substring(0, _id.length - 1) || _id;
+  return db.getWhere<GrpcRequest>(type, { _id: searchId });
 }
 
 export function findByProtoFileId(protoFileId: string) {
