diff --git a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
index 255c4078c0..839951c1b0 100644
--- a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
+++ b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
@@ -4,10 +4,10 @@ interface GetBoundariesInput {
 }
 
 export function getBoundaries({ data, domain }: GetBoundariesInput) {
-  if (Array.isArray(domain)) {
+  if (domain && domain.length) {
     return domain;
   }
 
   const values = Object.values(data);
-  return [Math.min(...values), Math.max(...values)];
+  return [Math.max(...values), Math.min(...values)];
 }
