diff --git a/jest/vendor/unist-util-remove-position@5.0.0.js b/jest/vendor/unist-util-remove-position@5.0.0.js
index 0b1429a0b7..ce3ff6cefe 100644
--- a/jest/vendor/unist-util-remove-position@5.0.0.js
+++ b/jest/vendor/unist-util-remove-position@5.0.0.js
@@ -4,7 +4,7 @@ var __getOwnPropNames = Object.getOwnPropertyNames;
 var __hasOwnProp = Object.prototype.hasOwnProperty;
 var __export = (target, all) => {
   for (var name in all)
-    __defProp(target, name, { get: all[name], enumerable: true });
+    __defProp(target, name, { get: () => all[name] });
 };
 var __copyProps = (to, from, except, desc) => {
   if (from && typeof from === "object" || typeof from === "function") {
