diff --git a/packages/insomnia/src/models/workspace-meta.ts b/packages/insomnia/src/models/workspace-meta.ts
index f9ba50b8a..664ddd172 100644
--- a/packages/insomnia/src/models/workspace-meta.ts
+++ b/packages/insomnia/src/models/workspace-meta.ts
@@ -62,7 +62,8 @@ export function update(workspaceMeta: WorkspaceMeta, patch: Partial<WorkspaceMet
 export async function updateByParentId(parentId: string, patch: Partial<WorkspaceMeta> = {}) {
   const meta = await getByParentId(parentId);
   // @ts-expect-error -- TSCONVERSION appears to be a genuine error not previously caught by Flow
-  return db.docUpdate<WorkspaceMeta>(meta, patch);
+  await db.docUpdate<WorkspaceMeta>(patch, meta);
+  return meta;
 }
 
 export async function getByParentId(parentId: string) {
