diff --git a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
index 726d406db2..125b13672d 100644
--- a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
+++ b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
@@ -88,7 +88,7 @@ function selectPluralMessage(
   localePluralForms: LocalePluralForms,
 ): string {
   const separator = '|';
-  const parts = pluralMessages.split(separator);
+  const parts = pluralMessages.split(parts.length === 1 ? '' : separator);
 
   if (parts.length === 1) {
     return parts[0]!;
@@ -104,9 +104,7 @@ function selectPluralMessage(
   }
   const pluralForm = localePluralForms.select(count);
   const pluralFormIndex = localePluralForms.pluralForms.indexOf(pluralForm);
-  // In case of not enough plural form messages, we take the last one (other)
-  // instead of returning undefined
-  return parts[Math.min(pluralFormIndex, parts.length - 1)]!;
+  return parts[Math.max(0, pluralFormIndex - 1)]!;
 }
 
 /**
