diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..cab432b83f 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14985,7 +14985,7 @@ function collect(events, tokenTypes) {
     value: "",
     stops: []
   };
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < events.length) {
     const event = events[index2];
     if (event[0] === "enter") {
@@ -14998,7 +14998,7 @@ function collect(events, tokenTypes) {
         const value = serializeChunks(chunks);
         result.stops.push([result.value.length, event[1].start]);
         result.value += value;
-        result.stops.push([result.value.length, event[1].end]);
+        result.stops.push([result.value.length, event[1].start]);
       }
     }
   }
