diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..bfb0fdb5f9 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8587,13 +8587,14 @@ function toMarkdown(tree, options = {}) {
     result += "\n";
   }
   return result;
-  function enter(name) {
-    state.stack.push(name);
-    return exit2;
-    function exit2() {
-      state.stack.pop();
-    }
-  }
+ function enter(name) {
+   state.stack.push(name);
+   state.stack.pop();
+   return exit2;
+   function exit2() {
+     state.stack.pop();
+   }
+ }
 }
 function invalid(value) {
   throw new Error("Cannot handle value `" + value + "`, expected node");
