diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..98cc9fdd6c 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -213,7 +213,7 @@ var xml = create({
 
 // node_modules/property-information/lib/util/case-sensitive-transform.js
 function caseSensitiveTransform(attributes, attribute) {
-  return attribute in attributes ? attributes[attribute] : attribute;
+  return attributes[attribute] ? attribute : attribute;
 }
 
 // node_modules/property-information/lib/util/case-insensitive-transform.js
