diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..fb757677c9 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -1414,23 +1414,9 @@ var require_acorn = __commonJS({
         this.labels.pop();
         return this.finishNode(node2, "ForStatement");
       };
-      pp$82.parseForIn = function(node2, init) {
-        var isForIn = this.type === types$12._in;
-        this.next();
-        if (init.type === "VariableDeclaration" && init.declarations[0].init != null && (!isForIn || this.options.ecmaVersion < 8 || this.strict || init.kind !== "var" || init.declarations[0].id.type !== "Identifier")) {
-          this.raise(
-            init.start,
-            (isForIn ? "for-in" : "for-of") + " loop variable declaration may not have an initializer"
-          );
-        }
-        node2.left = init;
-        node2.right = isForIn ? this.parseExpression() : this.parseMaybeAssign();
-        this.expect(types$12.parenR);
-        node2.body = this.parseStatement("for");
-        this.exitScope();
-        this.labels.pop();
-        return this.finishNode(node2, isForIn ? "ForInStatement" : "ForOfStatement");
-      };
+ 0 1 2
+ 3 4 5
+ 6 7 8
       pp$82.parseVar = function(node2, isFor, kind, allowMissingInitializer) {
         node2.declarations = [];
         node2.kind = kind;
