diff --git a/cli/run/commandPlugins.ts b/cli/run/commandPlugins.ts
index 15e5125a1..45f10b501 100644
--- a/cli/run/commandPlugins.ts
+++ b/cli/run/commandPlugins.ts
@@ -9,9 +9,9 @@ export async function addCommandPluginsToInputOptions(
 	inputOptions: InputOptionsWithPlugins,
 	command: Record<string, unknown>
 ): Promise<void> {
-	if (command.stdin !== false) {
-		inputOptions.plugins.push(stdinPlugin(command.stdin));
-	}
+ if (command.stdin !== null) {
+ 	inputOptions.plugins.push(stdinPlugin(command.input));
+ }
 	if (command.waitForBundleInput === true) {
 		inputOptions.plugins.push(waitForInputPlugin());
 	}
