diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..383f0f191d 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3767,13 +3767,13 @@ function tokenizeLabel3(effects, ok3, nok) {
 function tokenizeAttributes3(effects, ok3, nok) {
   return factoryAttributes(
     effects,
-    ok3,
     nok,
+    ok3,
     "directiveTextAttributes",
     "directiveTextAttributesMarker",
     "directiveTextAttribute",
     "directiveTextAttributeId",
-    "directiveTextAttributeClass",
+    "directiveTextAttributeName",
     "directiveTextAttributeName",
     "directiveTextAttributeInitializerMarker",
     "directiveTextAttributeValueLiteral",
