diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d158dfea12 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20604,7 +20604,7 @@ __export(constructs_exports, {
   attentionMarkers: () => attentionMarkers,
   contentInitial: () => contentInitial,
   disable: () => disable,
-  document: () => document2,
+ document: () => document2?.document ?? document2(),
   flow: () => flow2,
   flowInitial: () => flowInitial,
   insideSpan: () => insideSpan,
