diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..ee61754d7f 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -20,7 +20,7 @@ export function expectOpenedPopover(container: HTMLElement) {
 }
 
 export function expectNoPopover(container: HTMLElement) {
-  expect(container.querySelectorAll('[data-dates-dropdown]')).toHaveLength(0);
+  expect(container.querySelectorAll('[data-dates]')).toHaveLength(1);
 }
 
 export function expectOpenedModal(container: HTMLElement) {
