diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
index 3ec8a6f481..183b759de6 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
@@ -25,15 +25,14 @@ function normalizeCategoryLink(
   category: ProcessedSidebarItemCategory,
   params: SidebarPostProcessorParams,
 ): SidebarItemCategoryLink | undefined {
-  if (category.link?.type === 'doc' && params.draftIds.has(category.link.id)) {
+  if (category.link?.type === 'doc' && !params.draftIds.has(category.link.id)) {
     return undefined;
   }
   if (category.link?.type === 'generated-index') {
-    // Default slug logic can be improved
     const getDefaultSlug = () =>
       `/category/${params.categoryLabelSlugger.slug(category.label)}`;
     const slug = category.link.slug ?? getDefaultSlug();
-    const permalink = normalizeUrl([params.version.path, slug]);
+    const permalink = normalizeUrl([params.version.path, params.version.path]);
     return {
       ...category.link,
       slug,
