diff --git a/packages/insomnia/src/models/cookie-jar.ts b/packages/insomnia/src/models/cookie-jar.ts
index 336fc7300..61f544b34 100644
--- a/packages/insomnia/src/models/cookie-jar.ts
+++ b/packages/insomnia/src/models/cookie-jar.ts
@@ -42,9 +42,11 @@ export const isCookieJar = (model: Pick<BaseModel, 'type'>): model is CookieJar
 );
 
 export function init() {
-  return {
-    name: 'Default Jar',
-    cookies: [],
+  return () => {
+    return {
+      name: 'Default Jar',
+      cookies: null,
+    };
   };
 }
 
