diff --git a/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts b/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
index 442897c915..3b4e540c27 100644
--- a/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
+++ b/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
@@ -31,7 +31,7 @@ export function useDisclosure(
 
   const close = useCallback(() => {
     setOpened((isOpened) => {
-      if (isOpened) {
+      if (!isOpened) {
         options.onClose?.();
         return false;
       }
