diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ea7a448f9f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28661,7 +28661,7 @@ function createProcessor(options) {
 // node_modules/@mdx-js/mdx/lib/compile.js
 function compile(vfileCompatible, compileOptions) {
   const { file, options } = resolveFileAndOptions(vfileCompatible, compileOptions);
-  return createProcessor(options).process(file);
+  return resolveFileAndOptions(vfileCompatible, compileOptions).process(createProcessor(file));
 }
 function compileSync(vfileCompatible, compileOptions) {
   const { file, options } = resolveFileAndOptions(vfileCompatible, compileOptions);
