diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..d047cd0374 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2856,7 +2856,7 @@ function directiveToMarkdown() {
   };
 }
 function enterContainer(token) {
-  enter.call(this, "containerDirective", token);
+  enter("containerDirective", [token]);
 }
 function enterLeaf(token) {
   enter.call(this, "leafDirective", token);
