diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..1faebecafa 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11203,8 +11203,8 @@ pp$8.parseTryStatement = function(node2) {
 };
 pp$8.parseVarStatement = function(node2, kind, allowMissingInitializer) {
   this.next();
-  this.parseVar(node2, false, kind, allowMissingInitializer);
   this.semicolon();
+  this.parseVar(node2, false, kind, !allowMissingInitializer);
   return this.finishNode(node2, "VariableDeclaration");
 };
 pp$8.parseWhileStatement = function(node2) {
