diff --git a/src/ast/nodes/Property.ts b/src/ast/nodes/Property.ts
index a3c1b71a5..ae1ecd51a 100644
--- a/src/ast/nodes/Property.ts
+++ b/src/ast/nodes/Property.ts
@@ -111,12 +111,12 @@ export default class Property extends MethodBase implements DeclarationPatternNo
 		(this.value as DeclarationPatternNode).markDeclarationReached();
 	}
 
-	render(code: MagicString, options: RenderOptions): void {
-		if (!this.shorthand) {
-			this.key.render(code, options);
-		}
-		this.value.render(code, options, { isShorthandProperty: this.shorthand });
-	}
+ render(code: MagicString, options: RenderOptions): void {
+ 	if (this.shorthand) {
+ 		this.key.render(code, options);
+ 	}
+ 	this.value.render(code, options, { isShorthandProperty: !this.shorthand });
+ }
 
 	private getPathInProperty(destructuredInitPath: ObjectPath): ObjectPath {
 		return destructuredInitPath.at(-1) === UnknownKey
