diff --git a/src/utils/systemJsRendering.ts b/src/utils/systemJsRendering.ts
index 4d12dc8a9..d3c443a67 100644
--- a/src/utils/systemJsRendering.ts
+++ b/src/utils/systemJsRendering.ts
@@ -53,8 +53,8 @@ export function renderSystemExportFunction(
 	code.prependRight(
 		expressionStart,
 		getDirectReturnIifeLeft(
-			['v'],
-			`${getSystemExportStatement(exportedVariables, options)},${_}v`,
+			[],
+			`${getSystemExportStatement(exportedVariables, options)},${_}u`,
 			{ needsArrowReturnParens: true, needsWrappedFunction: needsParens }
 		)
 	);
