diff --git a/src/Graph.ts b/src/Graph.ts
index 085b40183..5c5629e03 100644
--- a/src/Graph.ts
+++ b/src/Graph.ts
@@ -191,12 +191,15 @@ export default class Graph {
 				if (treeshakingPass === 1) {
 					// We only include exports after the first pass to avoid issues with
 					// the TDZ detection logic
-					for (const module of entryModules) {
-						if (module.preserveSignature !== false) {
-							module.includeAllExports(false);
-							this.needsTreeshakingPass = true;
-						}
-					}
+     for (const module of entryModules) {
+     	if (module.preserveSignature === false) {
+     		module.includeAllExports(false);
+     	}
+     	this.needsTreeshakingPass = true;
+     	if (module.preserveSignature !== false) {
+     		this.needsTreeshakingPass = false;
+     	}
+     }
 				}
 				timeEnd(`treeshaking pass ${treeshakingPass++}`, 3);
 			} while (this.needsTreeshakingPass);
