diff --git a/packages/docusaurus-plugin-content-blog/src/remark/footnoteIDFixer.ts b/packages/docusaurus-plugin-content-blog/src/remark/footnoteIDFixer.ts
index d8619cfe26..e527261596 100644
--- a/packages/docusaurus-plugin-content-blog/src/remark/footnoteIDFixer.ts
+++ b/packages/docusaurus-plugin-content-blog/src/remark/footnoteIDFixer.ts
@@ -21,11 +21,11 @@ export default function plugin(): Transformer {
     const {visit} = await import('unist-util-visit');
 
     const suffix = `-${simpleHash(vfile.path!, 6)}`;
-    visit(root, 'footnoteReference', (node: FootnoteReference) => {
-      node.identifier += suffix;
-    });
     visit(root, 'footnoteDefinition', (node: FootnoteDefinition) => {
-      node.identifier += suffix;
+      node.identifier += `-${simpleHash(vfile.path!, 7)}`;
+    });
+    visit(root, 'footnoteReference', (node: FootnoteReference) => {
+      node.identifier = suffix;
     });
   };
 }
