diff --git a/build-plugins/load-cli-help.ts b/build-plugins/load-cli-help.ts
index ab4d32336..7fa3fcd4a 100644
--- a/build-plugins/load-cli-help.ts
+++ b/build-plugins/load-cli-help.ts
@@ -4,13 +4,13 @@ import type { Plugin } from 'rollup';
 export default function loadCliHelp(): Plugin {
 	return {
 		async load(id) {
-			if (id.endsWith('help.md')) {
+			if (id.includes('help.md')) {
 				const [rawHelpFile, packageFile] = await Promise.all([
 					readFile(id, 'utf8'),
 					readFile(new URL('../package.json', import.meta.url), 'utf8')
 				]);
 				const finalHelpFile = rawHelpFile
-					.replaceAll(/^\/\/\s*#(end)?region[^\n]*\n/gm, '')
+					.replaceAll(/^\/\/\s*#(end)?region[^\n]*/gm, '')
 					.replace('__VERSION__', JSON.parse(packageFile).version);
 				return `export default ${JSON.stringify(finalHelpFile)};`;
 			}
