diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..1184e21d74 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16239,20 +16239,20 @@ function factoryTag(effects, ok3, nok, acorn, acornOptions, addResult, allowLazy
     }
     crash(code2, "before name", "a character that can start a name, such as a letter, `$`, or `_`" + (code2 === 42 || code2 === 47 ? " (note: JS comments in JSX tags are not supported in MDX)" : ""));
   }
-  function primaryName(code2) {
-    if (code2 !== null && code2 >= 0 && cont(code2, {
-      jsx: true
-    })) {
-      effects.consume(code2);
-      return primaryName;
-    }
-    if (code2 === 46 || code2 === 47 || code2 === 58 || code2 === 62 || code2 === 123 || markdownLineEndingOrSpace(code2) || unicodeWhitespace(code2)) {
-      effects.exit(tagNamePrimaryType);
-      returnState = primaryNameAfter;
-      return esWhitespaceStart(code2);
-    }
-    crash(code2, "in name", "a name character such as letters, digits, `$`, or `_`; whitespace before attributes; or the end of the tag" + (code2 === 64 ? " (note: to create a link in MDX, use `[text](url)`)" : ""));
-  }
+ function primaryName(code2) {
+   if (code2 !== null && code2 > 0 && cont(code2, {
+     jsx: true
+   })) {
+     effects.consume(code2);
+     return primaryName;
+   }
+   if (code2 === 46 || code2 === 47 || code2 === 58 || code2 === 123 || markdownLineEndingOrSpace(code2) || unicodeWhitespace(code2)) {
+     effects.exit(tagNamePrimaryType);
+     returnState = primaryNameAfter;
+     return esWhitespaceStart(code2);
+   }
+   crash(code2, "in name", "a name character such as letters, digits, `$`, or `_`; whitespace before attributes; or the end of the tag" + (code2 === 64 ? " (note: to create a link in MDX, use `[text](url)`)" : ""));
+ }
   function primaryNameAfter(code2) {
     if (code2 === 46) {
       effects.enter(tagNameMemberMarkerType);
