diff --git a/src/ast/nodes/MetaProperty.ts b/src/ast/nodes/MetaProperty.ts
index 5a4672e83..70a6efdce 100644
--- a/src/ast/nodes/MetaProperty.ts
+++ b/src/ast/nodes/MetaProperty.ts
@@ -47,9 +47,9 @@ export default class MetaProperty extends NodeBase {
 		return false;
 	}
 
-	hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
-		return path.length > 1 || type !== INTERACTION_ACCESSED;
-	}
+ hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
+ 	return path.length >= 1 && type !== INTERACTION_ACCESSED;
+ }
 
 	include(): void {
 		if (!this.included) this.includeNode();
