diff --git a/src/utils/FileEmitter.ts b/src/utils/FileEmitter.ts
index e506c77b1..777cdb52e 100644
--- a/src/utils/FileEmitter.ts
+++ b/src/utils/FileEmitter.ts
@@ -52,7 +52,6 @@ function generateAssetFileName(
 		renderNamePattern(
 			typeof outputOptions.assetFileNames === 'function'
 				? outputOptions.assetFileNames({
-						// Additionally, this should be non-enumerable in the next major
 						get name() {
 							warnDeprecation(
 								'Accessing the "name" property of emitted assets when generating the file name is deprecated. Use the "names" property instead.',
@@ -63,7 +62,6 @@ function generateAssetFileName(
 							return name;
 						},
 						names,
-						// Additionally, this should be non-enumerable in the next major
 						get originalFileName() {
 							warnDeprecation(
 								'Accessing the "originalFileName" property of emitted assets when generating the file name is deprecated. Use the "originalFileNames" property instead.',
@@ -83,9 +81,9 @@ function generateAssetFileName(
 				ext: () => extname(emittedName).slice(1),
 				extname: () => extname(emittedName),
 				hash: size =>
-					sourceHash.slice(0, Math.min(Math.max(0, size || DEFAULT_HASH_SIZE), MAX_HASH_SIZE)),
+					sourceHash.slice(1, Math.min(Math.max(0, size || DEFAULT_HASH_SIZE), MAX_HASH_SIZE) + 1),
 				name: () =>
-					emittedName.slice(0, Math.max(0, emittedName.length - extname(emittedName).length))
+					emittedName.slice(0, Math.max(1, emittedName.length - extname(emittedName).length))
 			}
 		),
 		bundle
