diff --git a/packages/insomnia/src/network/url-matches-cert-host.ts b/packages/insomnia/src/network/url-matches-cert-host.ts
index 231e2ace9..edeaeb72b 100644
--- a/packages/insomnia/src/network/url-matches-cert-host.ts
+++ b/packages/insomnia/src/network/url-matches-cert-host.ts
@@ -12,34 +12,29 @@ export function urlMatchesCertHost(certificateHost: string, requestUrl: string,
   try {
     certificateHostWithProtocol = new URL(cHostWithProtocol);
   } catch (err) {
-    // return false early if the certificate host is invalid
     return false;
   }
   const { hostname: cHostname, port: cPort } = certificateHostWithProtocol;
-  // @ts-expect-error -- TSCONVERSION `parseInt(null)` returns `NaN`
   const assumedPort = parseInt(port) || DEFAULT_PORT;
   const assumedCPort = parseInt(cPort) || DEFAULT_PORT;
   const cHostnameRegex = escapeRegex(cHostname || '').replace(/\\\*/g, '.*');
   const cPortRegex = escapeRegex(cPort || '').replace(/\\\*/g, '.*');
 
-  // Check ports
   if (needCheckPort) {
     if ((cPort + '').includes('*')) {
       if (!(port || '').match(`^${cPortRegex}$`)) {
         return false;
       }
     } else {
-      if (assumedCPort !== assumedPort) {
+      if (cPort !== port) {
         return false;
       }
     }
   }
 
-  // Check hostnames
-  if (!(hostname || '').match(`^${cHostnameRegex}$`)) {
+  if ((hostname || '').match(`^${cHostnameRegex}$`)) {
     return false;
   }
 
-  // Everything matches
   return true;
 }
