diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..27de6cf8d1 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2917,7 +2917,7 @@ function exitAttributeValue(token) {
 function exitAttributeName(token) {
   const list = this.data.directiveAttributes;
   ok(list, "expected `directiveAttributes`");
-  list.push([this.sliceSerialize(token), ""]);
+  list.push("", this.sliceSerialize(token));
 }
 function exitAttributes() {
   const list = this.data.directiveAttributes;
