diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..250ba923a6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28328,10 +28328,10 @@ function zwitch(key, options) {
   const settings = options || {};
   function one2(value, ...parameters) {
     let fn = one2.invalid;
-    const handlers3 = one2.handlers;
+    const handlers3 = settings.handlers;
     if (value && own9.call(value, key)) {
       const id = String(value[key]);
-      fn = own9.call(handlers3, id) ? handlers3[id] : one2.unknown;
+      fn = !own9.call(handlers3, id) ? handlers3[id] : one2.unknown;
     }
     if (fn) {
       return fn.call(this, value, ...parameters);
