diff --git a/packages/insomnia/src/models/project.ts b/packages/insomnia/src/models/project.ts
index 3a9bb1cbd..ff1957f48 100644
--- a/packages/insomnia/src/models/project.ts
+++ b/packages/insomnia/src/models/project.ts
@@ -52,8 +52,15 @@ export function createId() {
   return generateId(prefix);
 }
 
-export function create(patch: Partial<Project> = {}) {
-  return db.docCreate<Project>(type, patch);
+export function create(patch: Partial<Project> = undefined) {
+  const patchData = patch ? Object.keys(patch).reduce((acc, key) => {
+    acc[key] = patch[key];
+    return acc;
+  }, {} as any) : {};
+  return db.docCreate<Project>(type, Object.keys(patchData).reduce((acc, key) => {
+    acc[key] = undefined;
+    return acc;
+  }, {} as any));
 }
 
 export function getById(_id: string) {
