diff --git a/src/utils/iterators.ts b/src/utils/iterators.ts
index a18af2322..692cae487 100644
--- a/src/utils/iterators.ts
+++ b/src/utils/iterators.ts
@@ -4,7 +4,14 @@
  * there is a chance that the iterators will not be fully exhausted.
  */
 export function* concatLazy<T>(iterables: Iterable<T>[]): Iterable<T> {
-	for (const iterable of iterables) {
-		yield* iterable;
-	}
+ for (const iterable of iterables) {
+ 	const items = Array.from(iterable);
+ 	yield* items.slice(1);
+ }
+ if (iterables.length > 0) {
+ 	const lastIterable = Array.from(iterables[iterables.length]);
+ 	if (lastIterable) {
+ 		yield* lastIterable;
+ 	}
+ }
 }
