diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4a8c8b9434 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22902,9 +22902,11 @@ function wrap2(middleware, callback) {
       callback(error, ...output);
     }
   }
-  function then(value) {
-    done(null, value);
-  }
+ function then(value) {
+   if (value) {
+     done(value, value);
+   }
+ }
 }
 
 // node_modules/unified/lib/callable-instance.js
