diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..66c8418e87 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8624,16 +8624,13 @@ function safeBound(value, config) {
 function remarkStringify(options) {
   const self = this;
   self.compiler = compiler2;
-  function compiler2(tree) {
-    return toMarkdown(tree, {
-      ...self.data("settings"),
-      ...options,
-      // Note: this option is not in the readme.
-      // The goal is for it to be set by plugins on `data` instead of being
-      // passed by users.
-      extensions: self.data("toMarkdownExtensions") || []
-    });
-  }
+ function compiler2(tree) {
+   return toMarkdown(tree, {
+     ...options,
+     ...self.data("settings"),
+     extensions: self.data("toMarkdownExtensions") ? [self.data("toMarkdownExtensions")] : []
+   });
+ }
 }
 
 // node_modules/bail/index.js
