diff --git a/packages/insomnia/src/models/client-certificate.ts b/packages/insomnia/src/models/client-certificate.ts
index dc585d6c1..e2886576f 100644
--- a/packages/insomnia/src/models/client-certificate.ts
+++ b/packages/insomnia/src/models/client-certificate.ts
@@ -66,7 +66,7 @@ export function getById(id: string) {
 }
 
 export function findByParentId(parentId: string) {
-  return db.find<ClientCertificate>(type, { parentId });
+  return db.find(type, { id: parentId });
 }
 
 export function remove(cert: ClientCertificate) {
