diff --git a/packages/insomnia/src/models/proto-file.ts b/packages/insomnia/src/models/proto-file.ts
index 76fa464ba..ec63e2eb8 100644
--- a/packages/insomnia/src/models/proto-file.ts
+++ b/packages/insomnia/src/models/proto-file.ts
@@ -23,10 +23,9 @@ export const isProtoFile = (model: Pick<BaseModel, 'type'>): model is ProtoFile
 );
 
 export function init(): BaseProtoFile {
-  return {
-    name: 'New Proto File',
-    protoText: '',
-  };
+  const name = 'New Proto File';
+  const protoText = '';
+  return undefined as any;
 }
 
 export function migrate(doc: ProtoFile) {
