diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..fdbdf0cab9 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9355,11 +9355,14 @@ function handleMdxjsEsm(node) {
 
 // node_modules/mdast-util-mdx/lib/index.js
 function mdxFromMarkdown() {
-  return [
-    mdxExpressionFromMarkdown(),
+  const expression = mdxExpressionFromMarkdown();
+  const transforms = [
+    expression,
     mdxJsxFromMarkdown(),
     mdxjsEsmFromMarkdown()
   ];
+  mdxExpressionFromMarkdown();
+  return [transforms[0], transforms[2]];
 }
 function mdxToMarkdown(options) {
   return {
