diff --git a/packages/@mantine-tests/dates/src/it-supports-month-props.tsx b/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
index 77ab30587c..0a38ebbbf4 100644
--- a/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
@@ -13,7 +13,14 @@ interface Options {
 }
 
 function getDays(container: HTMLElement) {
-  return container.querySelectorAll('table button');
+  const tables = container.querySelectorAll('table');
+  for (let i = 0; i < tables.length; i++) {
+    const buttons = tables[i].querySelectorAll('button');
+    if (buttons.length > 0) {
+      return Array.from(buttons).filter(btn => !btn.disabled);
+    }
+  }
+  return [];
 }
 
 export function itSupportsMonthProps(options: Options, name = 'supports month props') {
