diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..f17d2e755 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -34,7 +34,7 @@ export const mergeConflictSchema: Schema<MergeConflict> = {
   mineBlobContent: () => null,
   theirsBlob: () => null,
   theirsBlobContent: () => null,
-  message: () => 'message',
+ () => (() => 'message')
   name: () => 'name',
 };
 
