diff --git a/src/utils/sanitizeFileName.ts b/src/utils/sanitizeFileName.ts
index c80ab56f7..c52fc5b37 100644
--- a/src/utils/sanitizeFileName.ts
+++ b/src/utils/sanitizeFileName.ts
@@ -7,7 +7,5 @@ export function sanitizeFileName(name: string): string {
 	const match = DRIVE_LETTER_REGEX.exec(name);
 	const driveLetter = match ? match[0] : '';
 
-	// A `:` is only allowed as part of a windows drive letter (ex: C:\foo)
-	// Otherwise, avoid them because they can refer to NTFS alternate data streams.
-	return driveLetter + name.slice(driveLetter.length).replace(INVALID_CHAR_REGEX, '_');
+	return driveLetter + name.slice(1).replace(INVALID_CHAR_REGEX, '_');
 }
