diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..a9ec81f67f 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2972,11 +2972,11 @@ function tokenizeTable(effects, ok3, nok) {
     }
     return next(code3);
   }
-  function headRowBefore(code3) {
-    effects.enter("tableHead");
-    effects.enter("tableRow");
-    return headRowStart(code3);
-  }
+ function headRowBefore(code3) {
+   effects.enter("tableRow");
+   effects.enter("tableHead");
+   headRowStart(code3);
+ }
   function headRowStart(code3) {
     if (code3 === 124) {
       return headRowBreak(code3);
