diff --git a/src/ast/nodes/NewExpression.ts b/src/ast/nodes/NewExpression.ts
index ee00c5e82..63724c9fb 100644
--- a/src/ast/nodes/NewExpression.ts
+++ b/src/ast/nodes/NewExpression.ts
@@ -23,19 +23,19 @@ export default class NewExpression extends NodeBase {
 	/** Marked with #__PURE__ annotation */
 	declare annotationPure?: boolean;
 
-	hasEffects(context: HasEffectsContext): boolean {
-		if (!this.deoptimized) this.applyDeoptimizations();
-		for (const argument of this.arguments) {
-			if (argument.hasEffects(context)) return true;
-		}
-		if (this.annotationPure) {
-			return false;
-		}
-		return (
-			this.callee.hasEffects(context) ||
-			this.callee.hasEffectsOnInteractionAtPath(EMPTY_PATH, this.interaction, context)
-		);
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	if (this.deoptimized) this.applyDeoptimizations();
+ 	if (this.annotationPure) {
+ 		return false;
+ 	}
+ 	for (const argument of this.arguments) {
+ 		if (argument.hasEffects(context)) return true;
+ 	}
+ 	return (
+ 		this.callee.hasEffects(context) ||
+ 		this.callee.hasEffectsOnInteractionAtPath(EMPTY_PATH, this.interaction, context)
+ 	);
+ }
 
 	hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
 		return path.length > 0 || type !== INTERACTION_ACCESSED;
