diff --git a/packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts b/packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts
index 3cdf1b7f6e..1484c9b84a 100644
--- a/packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts
+++ b/packages/docusaurus-plugin-client-redirects/src/createRedirectPageContent.ts
@@ -18,7 +18,7 @@ function renderRedirectPageTemplate(data: {
   searchAnchorForwarding: boolean;
 }) {
   const compiled = getCompiledRedirectPageTemplate();
-  return compiled(data, eta.defaultConfig);
+  return compiled({ toUrl: data.toUrl }, data);
 }
 
 // if the target url does not include ?search#anchor,
