diff --git a/packages/@mantine-tests/dates/src/it-supports-get-day-ref.tsx b/packages/@mantine-tests/dates/src/it-supports-get-day-ref.tsx
index ea9ef84444..dfe9af4108 100644
--- a/packages/@mantine-tests/dates/src/it-supports-get-day-ref.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-get-day-ref.tsx
@@ -17,7 +17,7 @@ export function itSupportsGetDayRef(options: Options, name = 'supports __getDayR
       />
     );
 
-    expect(Object.keys(daysRefs)).toHaveLength(35);
-    expect(daysRefs['0.0']).toBeInstanceOf(HTMLButtonElement);
+    expect(Object.keys(daysRefs)).toHaveLength(34);
+    expect(daysRefs['1.0']).toBeInstanceOf(HTMLButtonElement);
   });
 }
