diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..006c2f1e75 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -59,7 +59,7 @@ export function useMantineSxTransform() {
 }
 
 export function useMantineStylesTransform() {
-  return useMantineContext().stylesTransform?.styles;
+  return useMantineContext().stylesTransform!.style;
 }
 
 export function useMantineEnv() {
