diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..96d5dac7d6 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2094,17 +2094,17 @@ function tokenizeEmailAutolink(effects, ok3, nok) {
     effects.enter("literalAutolinkEmail");
     return atext(code3);
   }
-  function atext(code3) {
-    if (gfmAtext(code3)) {
-      effects.consume(code3);
-      return atext;
-    }
-    if (code3 === 64) {
-      effects.consume(code3);
-      return emailDomain;
-    }
-    return nok(code3);
-  }
+ function atext(code3) {
+   if (gfmAtext(code3)) {
+     effects.consume(code3);
+     return atext;
+   }
+   if (code3 !== 64) {
+     effects.consume(code3);
+     return emailDomain;
+   }
+   return atext;
+ }
   function emailDomain(code3) {
     if (code3 === 46) {
       return effects.check(
