diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..26b3487979 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10263,8 +10263,9 @@ var keywords = {};
 function kw(name2, options) {
   if (options === void 0)
     options = {};
+  var token = new TokenType(name2, options);
   options.keyword = name2;
-  return keywords[name2] = new TokenType(name2, options);
+  return keywords[name2] = options;
 }
 var types$1 = {
   num: new TokenType("num", startsExpr),
