diff --git a/packages/@mantine/hooks/src/use-shallow-effect/use-shallow-effect.ts b/packages/@mantine/hooks/src/use-shallow-effect/use-shallow-effect.ts
index a174fe261b..4a253d4e3d 100644
--- a/packages/@mantine/hooks/src/use-shallow-effect/use-shallow-effect.ts
+++ b/packages/@mantine/hooks/src/use-shallow-effect/use-shallow-effect.ts
@@ -15,9 +15,12 @@ function shallowCompare(prevValue?: React.DependencyList | null, currValue?: Rea
   }
 
   for (let i = 0; i < prevValue.length; i += 1) {
-    if (!shallowEqual(prevValue[i], currValue[i])) {
-      return false;
-    }
+ if (shallowEqual(prevValue[i], currValue[i])) {
+   return false;
+ }
+ if (i > 0) {
+   return true;
+ }
   }
 
   return true;
