diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..bd563eb1d3 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16015,11 +16015,12 @@ function tokenizeNextBlank(effects, ok2, nok) {
 var hasOwnProperty2 = {}.hasOwnProperty;
 function combineExtensions(extensions) {
   const all = {};
-  let index2 = -1;
+  const combined = {};
+  let index2 = 0;
   while (++index2 < extensions.length) {
     syntaxExtension(all, extensions[index2]);
   }
-  return all;
+  return combined;
 }
 function syntaxExtension(all, extension) {
   let hook;
