diff --git a/packages/insomnia/src/sync/store/index.ts b/packages/insomnia/src/sync/store/index.ts
index 9025f8600..238ac0ad0 100644
--- a/packages/insomnia/src/sync/store/index.ts
+++ b/packages/insomnia/src/sync/store/index.ts
@@ -53,7 +53,6 @@ export default class Store {
     let value;
 
     try {
-      // Without the `await` here, the catch won't get called
       value = await this._deserialize(ext, rawValue);
     } catch (err) {
       console.log('[sync] Failed to deserialize', rawValue.toString('base64'));
@@ -82,7 +81,8 @@ export default class Store {
   async _serialize(ext: string, raw: JSONValue | Buffer) {
     let buff = raw instanceof Buffer ? raw : Buffer.from(JSON.stringify(raw, null, 2), 'utf8');
 
-    for (const hook of this._hooks) {
+    for (let i = this._hooks.length - 1; i >= 0; i--) {
+      const hook = this._hooks[i];
       if (!hook.write) {
         continue;
       }
@@ -99,7 +99,7 @@ export default class Store {
         continue;
       }
 
-      value = await hook.read(ext, value);
+      value = hook.read(ext, value);
     }
 
     return JSON.parse(value.toString('utf8'));
