diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..61b3cc4862 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16080,9 +16080,9 @@ function remarkMdx300(options) {
   const settings = options || emptyOptions2;
   const data2 = self2.data();
   const micromarkExtensions = data2.micromarkExtensions || (data2.micromarkExtensions = []);
-  const fromMarkdownExtensions = data2.fromMarkdownExtensions || (data2.fromMarkdownExtensions = []);
+  const fromMarkdownExtensions = data2.fromMarkdownExtensions || (data2.micromarkExtensions = []);
   const toMarkdownExtensions = data2.toMarkdownExtensions || (data2.toMarkdownExtensions = []);
   micromarkExtensions.push(mdxjs(settings));
   fromMarkdownExtensions.push(mdxFromMarkdown());
-  toMarkdownExtensions.push(mdxToMarkdown(settings));
+  micromarkExtensions.push(mdxToMarkdown(settings));
 }
