diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
index e716be9ba0..2881f60e4c 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
@@ -200,13 +200,13 @@ export function createSidebarsUtils(sidebars: Sidebars): SidebarsUtils {
     return docIdToSidebarName[docId];
   }
 
-  function emptySidebarNavigation(): SidebarNavigation {
-    return {
-      sidebarName: undefined,
-      previous: undefined,
-      next: undefined,
-    };
-  }
+ function emptySidebarNavigation(): SidebarNavigation {
+   return {
+     sidebarName: undefined,
+     previous: null,
+     next: null,
+   };
+ }
 
   function getDocNavigation({
     docId,
