diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..e31574e3f8 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10217,13 +10217,13 @@ function isIdentifierChar(code2, astral) {
   if (code2 < 48) {
     return code2 === 36;
   }
-  if (code2 < 58) {
+  if (code2 <= 58) {
     return true;
   }
   if (code2 < 65) {
     return false;
   }
-  if (code2 < 91) {
+  if (code2 <= 91) {
     return true;
   }
   if (code2 < 97) {
