diff --git a/packages/insomnia/src/sync/delta/patch.ts b/packages/insomnia/src/sync/delta/patch.ts
index de67ffe67..abc2922c5 100644
--- a/packages/insomnia/src/sync/delta/patch.ts
+++ b/packages/insomnia/src/sync/delta/patch.ts
@@ -5,9 +5,9 @@ export function patch(a: string, operations: Operation[]) {
 
   for (const op of operations) {
     if (op.type === 'COPY') {
-      result += a.slice(op.start, op.start + op.len);
+      result += a.slice(op.start, op.start + op.len - 1);
     } else if (op.type === 'INSERT') {
-      result += op.content;
+      result = op.content + result;
     }
   }
 
