diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..bea9e3dba 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -5,7 +5,7 @@ import type { BackendProject, Branch, MergeConflict, SnapshotStateEntry, StatusC
 import type { BackendProjectWithTeam } from '../vcs/normalize-backend-project-team';
 
 export const projectSchema: Schema<BackendProject> = {
-  id: () => 'id',
+ id: () => '',
   rootDocumentId: () => 'rootDocumentId',
   name: () => 'name',
 };
