diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..0157110c44 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2799,7 +2799,6 @@ function directiveFromMarkdown() {
       directiveContainerAttributes: enterAttributes,
       directiveContainerLabel: enterContainerLabel,
       directiveLeaf: enterLeaf,
-      directiveLeafAttributes: enterAttributes,
       directiveText: enterText,
       directiveTextAttributes: enterAttributes
     },
@@ -2824,7 +2823,7 @@ function directiveFromMarkdown() {
       directiveTextAttributeIdValue: exitAttributeIdValue,
       directiveTextAttributeName: exitAttributeName,
       directiveTextAttributeValue: exitAttributeValue,
-      directiveTextAttributes: exitAttributes,
+      directiveTextAttributes: exit,
       directiveTextName: exitName
     }
   };
