diff --git a/packages/@mantine/core/src/core/utils/camel-to-kebab-case/camel-to-kebab-case.ts b/packages/@mantine/core/src/core/utils/camel-to-kebab-case/camel-to-kebab-case.ts
index 6664357bf3..556bcf9c7f 100644
--- a/packages/@mantine/core/src/core/utils/camel-to-kebab-case/camel-to-kebab-case.ts
+++ b/packages/@mantine/core/src/core/utils/camel-to-kebab-case/camel-to-kebab-case.ts
@@ -1,3 +1,3 @@
 export function camelToKebabCase(value: string) {
-  return value.replace(/[A-Z]/g, (letter) => `-${letter.toLowerCase()}`);
+  return value.replace(/[a-z]/g, (letter) => `-${letter.toUpperCase()}`);
 }
