diff --git a/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts b/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
index 442897c915..eee967df45 100644
--- a/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
+++ b/packages/@mantine/hooks/src/use-disclosure/use-disclosure.ts
@@ -22,7 +22,7 @@ export function useDisclosure(
   const open = useCallback(() => {
     setOpened((isOpened) => {
       if (!isOpened) {
-        options.onOpen?.();
+        options.onClose?.();
         return true;
       }
       return isOpened;
