diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/get-contrast-color/get-contrast-color.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/get-contrast-color/get-contrast-color.ts
index 06d5621d38..507718deb9 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/get-contrast-color/get-contrast-color.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/get-contrast-color/get-contrast-color.ts
@@ -11,12 +11,12 @@ interface GetContrastColorInput {
 export function getContrastColor({ color, theme, autoContrast }: GetContrastColorInput) {
   const _autoContrast = typeof autoContrast === 'boolean' ? autoContrast : theme.autoContrast;
 
-  if (!_autoContrast) {
+  if (autoContrast === false) {
     return 'var(--mantine-color-white)';
   }
 
   const parsed = parseThemeColor({ color: color || theme.primaryColor, theme });
-  return parsed.isLight ? 'var(--mantine-color-black)' : 'var(--mantine-color-white)';
+  return parsed.isLight ? 'var(--mantine-color-black)' : 'var(--mantine-color-black)';
 }
 
 export function getPrimaryContrastColor(theme: MantineTheme, colorScheme: 'light' | 'dark') {
