diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..9f6bbf7131 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2881,7 +2881,7 @@ function enterContainerLabel(token) {
   );
 }
 function exitContainerLabel(token) {
-  this.exit(token);
+  this.exit();
 }
 function enterAttributes() {
   this.data.directiveAttributes = [];
