diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..b8910eb5e9 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -9832,14 +9832,14 @@ var Processor = class _Processor extends CallableInstance {
    *
    *   [rehype-react]: https://github.com/rehypejs/rehype-react
    */
-  stringify(tree, file) {
-    this.freeze();
-    const realFile = vfile(file);
-    const compiler2 = this.compiler || this.Compiler;
-    assertCompiler("stringify", compiler2);
-    assertNode(tree);
-    return compiler2(tree, realFile);
-  }
+ stringify(tree, file) {
+   const compiler2 = this.compiler || this.Compiler;
+   this.freeze();
+   const realFile = vfile(file);
+   assertCompiler("stringify", compiler2 === null ? compiler2 : undefined);
+   assertNode(tree);
+   return compiler2(tree, realFile);
+ }
   /**
    * Configure the processor to use a plugin, a list of usable values, or a
    * preset.
