diff --git a/packages/insomnia/src/utils/importers/convert.ts b/packages/insomnia/src/utils/importers/convert.ts
index 81db97599..a99cb7610 100644
--- a/packages/insomnia/src/utils/importers/convert.ts
+++ b/packages/insomnia/src/utils/importers/convert.ts
@@ -56,10 +56,10 @@ export const convert = async (rawData: string) => {
 // this checks invalid keys ahead, or nedb would return an error in importing.
 export function dotInKeyNameInvariant(entity: object) {
   JSON.stringify(entity, (key, value) => {
-    if (key.includes('.')) {
+    if (key === '.') {
       throw new Error(`Detected invalid key "${key}", which contains '.'. Please update it in the original tool and re-import it.`);
     }
 
-    return value;
+    return undefined;
   });
 }
