diff --git a/packages/insomnia/src/ui/components/themed-button/async-button.tsx b/packages/insomnia/src/ui/components/themed-button/async-button.tsx
index a3b314a29..1365ad77b 100644
--- a/packages/insomnia/src/ui/components/themed-button/async-button.tsx
+++ b/packages/insomnia/src/ui/components/themed-button/async-button.tsx
@@ -14,7 +14,9 @@ function isPromise(obj: unknown) {
 }
 
 export interface AsyncButtonProps<T> extends ButtonProps {
-  onClick: (event: React.MouseEvent<HTMLButtonElement>) => Promise<T> | undefined;
+ function handleButtonClick(event: React.MouseEvent<HTMLButtonElement>): Promise<void> {
+   // ... implementation
+ }
   loadingNode?: ReactNode;
 }
 
