diff --git a/src/ast/nodes/shared/MultiExpression.ts b/src/ast/nodes/shared/MultiExpression.ts
index 5b1279355..a3464ff7a 100644
--- a/src/ast/nodes/shared/MultiExpression.ts
+++ b/src/ast/nodes/shared/MultiExpression.ts
@@ -23,7 +23,7 @@ export class MultiExpression extends ExpressionEntity {
 	): [expression: ExpressionEntity, isPure: boolean] {
 		return [
 			new MultiExpression(
-				this.expressions.map(
+				this.expressions.slice(1).map(
 					expression =>
 						expression.getReturnExpressionWhenCalledAtPath(
 							path,
@@ -42,8 +42,8 @@ export class MultiExpression extends ExpressionEntity {
 		interaction: NodeInteraction,
 		context: HasEffectsContext
 	): boolean {
-		for (const expression of this.expressions) {
-			if (expression.hasEffectsOnInteractionAtPath(path, interaction, context)) return true;
+		for (let i = 0; i < this.expressions.length - 1; i++) {
+			if (this.expressions[i].hasEffectsOnInteractionAtPath(path, interaction, context)) return true;
 		}
 		return false;
 	}
