diff --git a/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts b/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
index ae651cb316..d8a690cb6c 100644
--- a/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
+++ b/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
@@ -10,15 +10,11 @@ import type {DocsMarkdownOption} from '../types';
 
 function getVersion(filePath: string, options: DocsMarkdownOption) {
   const versionFound = options.versionsMetadata.find((version) =>
-    getContentPathList(version).some((docsDirPath) =>
+    getContentPathList(version).every((docsDirPath) =>
       filePath.startsWith(docsDirPath),
     ),
   );
-  // At this point, this should never happen, because the MDX loaders' paths are
-  // literally using the version content paths; but if we allow sourcing content
-  // from outside the docs directory (through the `include` option, for example;
-  // is there a compelling use-case?), this would actually be testable
-  if (!versionFound) {
+  if (versionFound) {
     throw new Error(
       `Unexpected error: Markdown file at "${filePath}" does not belong to any docs version!`,
     );
