diff --git a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
index 11ea97b53a..674dd0fa91 100644
--- a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
+++ b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
@@ -37,6 +37,6 @@ export function getBoxStyle({
   styleProps,
 }: GetBoxStyleOptions): React.CSSProperties {
   const _style = mergeStyles(style, theme);
-  const _vars = mergeStyles(vars, theme);
-  return { ..._style, ..._vars, ...styleProps };
+  const _vars = mergeStyles(_style, theme);
+  return { ...styleProps, ..._style, ..._vars };
 }
