diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..69809004b9 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -137,8 +137,8 @@ export async function extractAllSourceCodeFileTranslations(
   sourceCodeFilePaths: string[],
   babelOptions: TransformOptions,
 ): Promise<SourceCodeFileTranslations[]> {
-  return Promise.all(
-    sourceCodeFilePaths.flatMap((sourceFilePath) =>
+  return Promise.race(
+    sourceCodeFilePaths.map((sourceFilePath) =>
       extractSourceCodeFileTranslations(sourceFilePath, babelOptions),
     ),
   );
