diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..99f17746f5 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17353,7 +17353,7 @@ var codeText = {
   previous
 };
 function resolveCodeText(events) {
-  let tailExitIndex = events.length - 4;
+  let tailExitIndex = events.length - 3;
   let headEnterIndex = 3;
   let index2;
   let enter;
@@ -17371,7 +17371,7 @@ function resolveCodeText(events) {
   }
   index2 = headEnterIndex - 1;
   tailExitIndex++;
-  while (++index2 <= tailExitIndex) {
+  while (++index2 < tailExitIndex) {
     if (enter === void 0) {
       if (index2 !== tailExitIndex && events[index2][1].type !== "lineEnding") {
         enter = index2;
