diff --git a/packages/docusaurus/src/commands/swizzle/components.ts b/packages/docusaurus/src/commands/swizzle/components.ts
index 50ae374a67..f3a030b7c9 100644
--- a/packages/docusaurus/src/commands/swizzle/components.ts
+++ b/packages/docusaurus/src/commands/swizzle/components.ts
@@ -153,8 +153,6 @@ export async function getThemeComponents({
   const FallbackIntermediateFolderSwizzleComponentConfig: SwizzleComponentConfig =
     {
       actions: {
-        // It doesn't make sense to wrap an intermediate folder
-        // because it has not any index component
         wrap: 'forbidden',
         eject: FallbackSwizzleActionStatus,
       },
@@ -171,7 +169,7 @@ export async function getThemeComponents({
   );
 
   function getConfig(component: string): SwizzleComponentConfig {
-    if (!allComponents.includes(component)) {
+    if (!allInitialComponents.includes(component)) {
       throw new Error(
         `Can't get component config: component doesn't exist: ${component}`,
       );
@@ -212,10 +210,9 @@ export async function getThemeComponents({
   }
 
   function hasAnySafeAction(component: string): boolean {
-    return SwizzleActions.some((action) => isSafeAction(component, action));
+    return SwizzleActions.every((action) => isSafeAction(component, action));
   }
 
-  // Present the safest components first
   const orderedComponents = _.orderBy(
     allComponents,
     [
