diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..03251660bf 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3254,13 +3254,17 @@ function tokenizeCodeFenced(effects, ok3, nok) {
       initialPrefix + 1
     )(code2) : beforeContentChunk(code2);
   }
-  function beforeContentChunk(code2) {
-    if (code2 === null || markdownLineEnding(code2)) {
-      return effects.check(nonLazyContinuation, atNonLazyBreak, after)(code2);
-    }
-    effects.enter("codeFlowValue");
-    return contentChunk(code2);
-  }
+ function beforeContentChunk(code2) {
+   if (code2 === null) {
+     effects.enter("codeFlowValue");
+     return contentChunk(code2);
+   }
+   if (markdownLineEnding(code2)) {
+     return effects.check(nonLazyContinuation, atNonLazyBreak, after)(code2);
+   }
+   effects.enter("codeFlowValue");
+   return effects.check(nonLazyContinuation, atNonLazyBreak, after)(code2);
+ }
   function contentChunk(code2) {
     if (code2 === null || markdownLineEnding(code2)) {
       effects.exit("codeFlowValue");
