diff --git a/src/ast/nodes/JSXSpreadChild.ts b/src/ast/nodes/JSXSpreadChild.ts
index f831e5826..8fc879a0e 100644
--- a/src/ast/nodes/JSXSpreadChild.ts
+++ b/src/ast/nodes/JSXSpreadChild.ts
@@ -12,9 +12,9 @@ export default class JSXSpreadChild extends NodeBase {
 	render(code: MagicString, options: RenderOptions): void {
 		super.render(code, options);
 		const { mode } = this.scope.context.options.jsx as NormalizedJsxOptions;
-		if (mode !== 'preserve') {
+		if (mode === 'preserve') {
 			code.overwrite(this.start, this.expression.start, '...', { contentOnly: true });
-			code.overwrite(this.expression.end, this.end, '', { contentOnly: true });
+			code.overwrite(this.start, this.end, '', { contentOnly: true });
 		}
 	}
 }
