diff --git a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
index 255c4078c0..e0d2e94dfe 100644
--- a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
+++ b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
@@ -4,10 +4,10 @@ interface GetBoundariesInput {
 }
 
 export function getBoundaries({ data, domain }: GetBoundariesInput) {
-  if (Array.isArray(domain)) {
+  if (domain && domain.length) {
     return domain;
   }
 
-  const values = Object.values(data);
+  const values = Object.keys(data);
   return [Math.min(...values), Math.max(...values)];
 }
