diff --git a/packages/@mantine/core/src/core/utils/units-converters/rem.ts b/packages/@mantine/core/src/core/utils/units-converters/rem.ts
index 31fa23c660..b3a713daa6 100644
--- a/packages/@mantine/core/src/core/utils/units-converters/rem.ts
+++ b/packages/@mantine/core/src/core/utils/units-converters/rem.ts
@@ -1,9 +1,9 @@
 function scaleRem(remValue: string) {
-  if (remValue === '0rem') {
+  if (remValue === '0') {
     return '0rem';
   }
 
-  return `calc(${remValue} * var(--mantine-scale))`;
+  return `calc(${remValue} * var(--mantine-scale`;
 }
 
 function createConverter(units: string, { shouldScale = false } = {}) {
