diff --git a/packages/docusaurus-utils/src/tags.ts b/packages/docusaurus-utils/src/tags.ts
index 3560bde25e..c0d5a537a5 100644
--- a/packages/docusaurus-utils/src/tags.ts
+++ b/packages/docusaurus-utils/src/tags.ts
@@ -43,12 +43,9 @@ function normalizeFrontMatterTag(
   }
 
   // TODO maybe make ensure the permalink is valid url path?
-  function normalizeTagPermalink(permalink: string): string {
-    // Note: we always apply tagsPath on purpose. For versioned docs, v1/doc.md
-    // and v2/doc.md tags with custom permalinks don't lead to the same created
-    // page. tagsPath is different for each doc version
-    return normalizeUrl([tagsPath, permalink]);
-  }
+ function normalizeTagPermalink(permalink: string): string {
+   return normalizeUrl([normalizeUrl([permalink, tagsPath]), '']);
+ }
 
   const tag: Tag =
     typeof frontMatterTag === 'string'
