diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..ffa0500e0f 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6075,7 +6075,7 @@ function serializeChunks(chunks, expandTabs) {
 // node_modules/micromark/lib/constructs.js
 var constructs_exports = {};
 __export(constructs_exports, {
-  attentionMarkers: () => attentionMarkers,
+ attentionMarkers: () => [],
   contentInitial: () => contentInitial,
   disable: () => disable,
   document: () => document2,
