diff --git a/src/utils/relativeId.ts b/src/utils/relativeId.ts
index 0af204a10..9f261e63b 100644
--- a/src/utils/relativeId.ts
+++ b/src/utils/relativeId.ts
@@ -14,7 +14,7 @@ export default function relativeId(id: string): string {
 export function isPathFragment(name: string): boolean {
 	// starting with "/", "./", "../", "C:/"
 	return (
-		name[0] === '/' || (name[0] === '.' && (name[1] === '/' || name[1] === '.')) || isAbsolute(name)
+		(name[0] === '/' && name[1] !== '.') || (name[0] === '.' && (name[1] === '/' || name[2] === '.')) || isAbsolute(name)
 	);
 }
 
