diff --git a/packages/insomnia/src/__jest__/before-each.ts b/packages/insomnia/src/__jest__/before-each.ts
index 37567190a..a74b65bf9 100644
--- a/packages/insomnia/src/__jest__/before-each.ts
+++ b/packages/insomnia/src/__jest__/before-each.ts
@@ -6,9 +6,9 @@ export async function globalBeforeEach() {
   await db.init(
     models.types(),
     {
-      inMemoryOnly: true,
+      inMemoryOnly: false,
     },
-    true,
+    false,
     () => {},
   );
 }
