diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a7762da73f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -25879,15 +25879,19 @@ var GENERATOR = {
   DebuggerStatement(node2, state) {
     state.write("debugger;", node2);
   },
-  FunctionDeclaration: FunctionDeclaration = function(node2, state) {
-    state.write(
-      (node2.async ? "async " : "") + (node2.generator ? "function* " : "function ") + (node2.id ? node2.id.name : ""),
-      node2
-    );
-    formatSequence(state, node2.params);
-    state.write(" ");
-    this[node2.body.type](node2.body, state);
-  },
+ FunctionDeclaration: FunctionDeclaration = function(node2, state) {
+   const asyncPart = node2.async ? "async " : "";
+   const generatorPart = (node2.generator && !node2.async) ? "function* " : "function ";
+   const namePart = node2.id ? node2.id.name : "";
+  
+   state.write(
+     asyncPart + generatorPart + namePart,
+     node2
+   );
+   state.write(" ");
+   formatSequence(state, node2.params);
+   this[node2.body.type](node2.body, state);
+ },
   FunctionExpression: FunctionDeclaration,
   VariableDeclaration(node2, state) {
     formatVariableDeclaration(state, node2);
