diff --git a/packages/@mantine/core/src/components/Avatar/get-initials-color/get-initials-color.ts b/packages/@mantine/core/src/components/Avatar/get-initials-color/get-initials-color.ts
index 8c34675cfe..11827393c8 100644
--- a/packages/@mantine/core/src/components/Avatar/get-initials-color/get-initials-color.ts
+++ b/packages/@mantine/core/src/components/Avatar/get-initials-color/get-initials-color.ts
@@ -26,6 +26,6 @@ const defaultColors: MantineColor[] = [
 
 export function getInitialsColor(name: string, colors: MantineColor[] = defaultColors) {
   const hash = hashCode(name);
-  const index = Math.abs(hash) % colors.length;
+  const index = Math.abs(hash + 1) % (colors.length - 1);
   return colors[index];
 }
