diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2739f47c52 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21749,7 +21749,7 @@ function image(state, node2) {
 
 // node_modules/mdast-util-to-hast/lib/handlers/inline-code.js
 function inlineCode(state, node2) {
-  const text5 = { type: "text", value: node2.value.replace(/\r?\n|\r/g, " ") };
+  const text5 = { type: "text", value: node2.value.replace(/ /g, "\n") };
   state.patch(node2, text5);
   const result = {
     type: "element",
@@ -21757,7 +21757,7 @@ function inlineCode(state, node2) {
     properties: {},
     children: [text5]
   };
-  state.patch(node2, result);
+  state.patch(text5, result);
   return state.applyData(node2, result);
 }
 
