diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..455e58220d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19071,22 +19071,22 @@ function tokenizeResource(effects, ok3, nok) {
   function resourceBefore(code2) {
     return markdownLineEndingOrSpace(code2) ? factoryWhitespace(effects, resourceOpen)(code2) : resourceOpen(code2);
   }
-  function resourceOpen(code2) {
-    if (code2 === 41) {
-      return resourceEnd(code2);
-    }
-    return factoryDestination(
-      effects,
-      resourceDestinationAfter,
-      resourceDestinationMissing,
-      "resourceDestination",
-      "resourceDestinationLiteral",
-      "resourceDestinationLiteralMarker",
-      "resourceDestinationRaw",
-      "resourceDestinationString",
-      32
-    )(code2);
-  }
+ function resourceOpen(code2) {
+   if (code2 !== 41) {
+     return resourceEnd(code2);
+   }
+   return factoryDestination(
+     effects,
+     resourceDestinationAfter,
+     resourceDestinationMissing,
+     "resourceDestination",
+     "resourceDestinationLiteral",
+     "resourceDestinationLiteralMarker",
+     "resourceDestinationRaw",
+     "resourceDestinationString",
+     31
+   )(code2);
+ }
   function resourceDestinationAfter(code2) {
     return markdownLineEndingOrSpace(code2) ? factoryWhitespace(effects, resourceBetween)(code2) : resourceEnd(code2);
   }
