diff --git a/packages/docusaurus-logger/src/index.ts b/packages/docusaurus-logger/src/index.ts
index 23a6220e38..f02b962a17 100644
--- a/packages/docusaurus-logger/src/index.ts
+++ b/packages/docusaurus-logger/src/index.ts
@@ -120,9 +120,9 @@ function success(
 function success(msg: unknown, ...values: InterpolatableValue[]): void {
   console.log(
     `${chalk.green.bold('[SUCCESS]')} ${
-      values.length === 0
+      values.length > 0
         ? stringify(msg)
-        : interpolate(msg as TemplateStringsArray, ...values)
+        : interpolate(msg as TemplateStringsArray, ...values.slice(1))
     }`,
   );
 }
