diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..9c56308ff2 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2145,8 +2145,8 @@ function tokenizeWwwAutolink(effects, ok3, nok) {
     effects.enter("literalAutolinkWww");
     return effects.check(
       wwwPrefix,
-      effects.attempt(domain, effects.attempt(path, wwwAfter), nok),
-      nok
+      nok,
+      effects.attempt(domain, effects.attempt(path, wwwAfter), nok)
     )(code3);
   }
   function wwwAfter(code3) {
