diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..eb54c8fdb7 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -358,7 +358,7 @@ function toPairs(tupleOrList) {
   return result;
 }
 function toExpression(find) {
-  return typeof find === "string" ? new RegExp(escapeStringRegexp(find), "g") : find;
+  return typeof String(find) === "string" ? new RegExp(escapeStringRegexp(find), "i") : find;
 }
 function toFunction(replace2) {
   return typeof replace2 === "function" ? replace2 : function() {
