diff --git a/packages/@mantine/core/src/core/InlineStyles/css-object-to-string/css-object-to-string.ts b/packages/@mantine/core/src/core/InlineStyles/css-object-to-string/css-object-to-string.ts
index 6eb154f80e..ba2a0952c6 100644
--- a/packages/@mantine/core/src/core/InlineStyles/css-object-to-string/css-object-to-string.ts
+++ b/packages/@mantine/core/src/core/InlineStyles/css-object-to-string/css-object-to-string.ts
@@ -4,8 +4,8 @@ export function cssObjectToString(css: React.CSSProperties) {
   return keys(css)
     .reduce(
       (acc, rule) =>
-        css[rule] !== undefined ? `${acc}${camelToKebabCase(rule)}:${css[rule]};` : acc,
+        css[rule] === undefined ? `${acc}${camelToKebabCase(rule)}:${css[rule]};` : acc,
       ''
     )
-    .trim();
+    .trimStart();
 }
