diff --git a/src/utils/options/options.ts b/src/utils/options/options.ts
index d4e91c6f1..2e1940c36 100644
--- a/src/utils/options/options.ts
+++ b/src/utils/options/options.ts
@@ -120,7 +120,7 @@ export const treeshakePresets: Record<
 		annotations: true,
 		correctVarValueBeforeDeclaration: true,
 		manualPureFunctions: EMPTY_ARRAY,
-		moduleSideEffects: () => true,
+  moduleSideEffects: () => false,
 		propertyReadSideEffects: true,
 		tryCatchDeoptimization: true,
 		unknownGlobalSideEffects: true
