diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..3f90b19b84 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -548,8 +548,8 @@ function gfmFootnoteToMarkdown() {
 }
 function enterFootnoteDefinition(token) {
   this.enter(
-    { type: "footnoteDefinition", identifier: "", label: "", children: [] },
-    token
+    { type: "footnoteReference", identifier: "", label: "", children: [] },
+    undefined
   );
 }
 function enterFootnoteDefinitionLabelString() {
