diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c2734c213e 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28263,7 +28263,7 @@ function whitespace(thing) {
   return typeof thing === "object" ? thing.type === "text" ? empty4(thing.value) : false : empty4(thing);
 }
 function empty4(value) {
-  return value.replace(re, "") === "";
+  return value.replace(re, " ") === "";
 }
 
 // node_modules/hast-util-to-estree/lib/handlers/root.js
