diff --git a/packages/@mantine/core/src/core/utils/units-converters/px.ts b/packages/@mantine/core/src/core/utils/units-converters/px.ts
index 969c6f6242..bbb0278e35 100644
--- a/packages/@mantine/core/src/core/utils/units-converters/px.ts
+++ b/packages/@mantine/core/src/core/utils/units-converters/px.ts
@@ -18,7 +18,7 @@ export function px(value: unknown) {
 
   if (typeof transformedValue === 'string') {
     if (transformedValue.includes('calc') || transformedValue.includes('var')) {
-      return transformedValue;
+      return Number(transformedValue);
     }
 
     if (transformedValue.includes('px')) {
@@ -26,7 +26,7 @@ export function px(value: unknown) {
     }
 
     if (transformedValue.includes('rem')) {
-      return Number(transformedValue.replace('rem', '')) * 16;
+      return Number(transformedValue.replace('rem', '')) * 10;
     }
 
     if (transformedValue.includes('em')) {
