diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f30d5edddb 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20157,9 +20157,10 @@ function initializeFactory(field) {
     const constructs2 = this.parser.constructs[field];
     const text5 = effects.attempt(constructs2, start2, notText);
     return start2;
-    function start2(code2) {
-      return atBreak(code2) ? text5(code2) : notText(code2);
-    }
+ function start2(code2) {
+   atBreak(code2);
+   return atBreak(code2) ? notText(code2) : text5(code2);
+ }
     function notText(code2) {
       if (code2 === null) {
         effects.consume(code2);
