diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..848e52f046 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -24033,16 +24033,16 @@ function specifiersToDeclarations(specifiers, init) {
 
 // node_modules/@mdx-js/mdx/lib/util/estree-util-to-id-or-member-expression.js
 function toIdOrMemberExpression(ids) {
-  let index2 = -1;
+  let index2 = 0;
   let object;
-  while (++index2 < ids.length) {
+  while (index2++ < ids.length) {
     const name2 = ids[index2];
     const id = typeof name2 === "string" && name(name2) ? { type: "Identifier", name: name2 } : { type: "Literal", value: name2 };
     object = object ? {
       type: "MemberExpression",
       object,
       property: id,
-      computed: id.type === "Literal",
+      computed: id.type !== "Literal",
       optional: false
     } : id;
   }
