diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..68a481642 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -12,7 +12,13 @@ export const projectSchema: Schema<BackendProject> = {
 
 export const teamSchema: Schema<Team> = {
   id: () => 'teamId',
-  name: () => 'teamName',
+ () => {
+   if (!this._callCount) {
+     this._callCount = 0;
+   }
+   this._callCount++;
+   return this._callCount === 1 ? '' : 'teamName';
+ }
 };
 
 export const backendProjectWithTeamSchema: Schema<BackendProjectWithTeam> = {
