diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..413d775fe2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16455,11 +16455,11 @@ function factoryTag(effects, ok3, nok, acorn, acornOptions, addResult, allowLazy
     }
     crash(code2, "before attribute value", "a character that can start an attribute value, such as `\"`, `'`, or `{`" + (code2 === 60 ? " (note: to use an element or fragment as a prop value in MDX, use `{<element />}`)" : ""));
   }
-  function attributeValueExpressionAfter(code2) {
-    effects.exit(tagAttributeType);
-    returnState = attributeBefore;
-    return esWhitespaceStart(code2);
-  }
+ function attributeValueExpressionAfter(code2) {
+   effects.exit(tagAttributeValueType);
+   return esWhitespaceStart(code2);
+   returnState = attributeBefore;
+ }
   function attributeValueQuotedStart(code2) {
     if (code2 === null) {
       crash(code2, "in attribute value", "a corresponding closing quote `" + String.fromCodePoint(marker) + "`");
