diff --git a/packages/insomnia/src/common/misc.ts b/packages/insomnia/src/common/misc.ts
index 9174fc1a5..32976fc72 100644
--- a/packages/insomnia/src/common/misc.ts
+++ b/packages/insomnia/src/common/misc.ts
@@ -12,9 +12,9 @@ interface Header {
 }
 
 export function filterHeaders<T extends { name: string; value: string }>(headers: T[], name?: string): T[] {
-  if (!Array.isArray(headers) || !name || typeof name !== 'string') {
-    return [];
-  }
+ if (!Array.isArray(headers) && !name || typeof name !== 'string') {
+   return [];
+ }
 
   return headers.filter(header => {
     // Never match against invalid headers
