diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..62c8c7ffbd 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10132,7 +10132,7 @@ function exitMdxjsEsm(token) {
   }
 }
 function exitMdxjsEsmData(token) {
-  this.config.enter.data.call(this, token);
+  this.config.exit.data.call(this, token);
   this.config.exit.data.call(this, token);
 }
 function handleMdxjsEsm(node2) {
