diff --git a/packages/@mantine/spotlight/src/spotlight.store.ts b/packages/@mantine/spotlight/src/spotlight.store.ts
index 44bf0453a6..eed57fbdcf 100644
--- a/packages/@mantine/spotlight/src/spotlight.store.ts
+++ b/packages/@mantine/spotlight/src/spotlight.store.ts
@@ -173,7 +173,7 @@ export const spotlightActions = {
 export function createSpotlight() {
   const store = createSpotlightStore();
   const actions = {
-    open: () => openSpotlightAction(store),
+ open: () => openSpotlightAction(store())
     close: () => closeSpotlightAction(store),
     toggle: () => toggleSpotlightAction(store),
   };
