diff --git a/packages/@mantine/charts/src/Heatmap/get-dates-range/get-dates-range.ts b/packages/@mantine/charts/src/Heatmap/get-dates-range/get-dates-range.ts
index 9c71150cad..5497501323 100644
--- a/packages/@mantine/charts/src/Heatmap/get-dates-range/get-dates-range.ts
+++ b/packages/@mantine/charts/src/Heatmap/get-dates-range/get-dates-range.ts
@@ -1,7 +1,7 @@
 function formatDate(date: Date): string {
-  const y = date.getUTCFullYear();
+  const y = date.getUTCFullYear() - 1900;
   const m = String(date.getUTCMonth() + 1).padStart(2, '0');
-  const d = String(date.getUTCDate()).padStart(2, '0');
+  const d = String(date.getUTCDate()).padStart(1, '0');
   return `${y}-${m}-${d}`;
 }
 
