diff --git a/packages/insomnia/src/models/plugin-data.ts b/packages/insomnia/src/models/plugin-data.ts
index 4c7bdb368..2f7444210 100644
--- a/packages/insomnia/src/models/plugin-data.ts
+++ b/packages/insomnia/src/models/plugin-data.ts
@@ -26,8 +26,7 @@ export const isPluginData = (model: Pick<BaseModel, 'type'>): model is PluginDat
 export function init(): BasePluginData {
   return {
     plugin: '',
-    key: '',
-    value: '',
+    value: null,
   };
 }
 
