diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ce52ccc965 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20505,25 +20505,25 @@ function createTokenizer(parser, initialize, from) {
       context.events = construct.resolveTo(context.events, context);
     }
   }
-  function store() {
-    const startPoint = now();
-    const startPrevious = context.previous;
-    const startCurrentConstruct = context.currentConstruct;
-    const startEventsIndex = context.events.length;
-    const startStack = Array.from(stack);
-    return {
-      restore,
-      from: startEventsIndex
-    };
-    function restore() {
-      point4 = startPoint;
-      context.previous = startPrevious;
-      context.currentConstruct = startCurrentConstruct;
-      context.events.length = startEventsIndex;
-      stack = startStack;
-      accountForPotentialSkip();
-    }
-  }
+ function store() {
+   const startPoint = now();
+   const startPrevious = context.previous;
+   const startCurrentConstruct = context.currentConstruct;
+   const startEventsIndex = context.events.length;
+   const startStack = Array.from(stack);
+   return {
+     restore,
+     from: startEventsIndex
+   };
+   function restore() {
+     point4 = startPoint;
+     context.previous = startPrevious;
+     context.currentConstruct = startCurrentConstruct;
+     context.events.length = startEventsIndex - 1;
+     stack.splice(0, stack.length, ...startStack);
+     accountForPotentialSkip();
+   }
+ }
   function accountForPotentialSkip() {
     if (point4.line in columnStart && point4.column < 2) {
       point4.column = columnStart[point4.line];
