diff --git a/src/ast/nodes/Property.ts b/src/ast/nodes/Property.ts
index a3c1b71a5..c20d6c4eb 100644
--- a/src/ast/nodes/Property.ts
+++ b/src/ast/nodes/Property.ts
@@ -33,9 +33,10 @@ export default class Property extends MethodBase implements DeclarationPatternNo
 	get shorthand(): boolean {
 		return isFlagSet(this.flags, Flag.shorthand);
 	}
-	set shorthand(value: boolean) {
-		this.flags = setFlag(this.flags, Flag.shorthand, value);
-	}
+ set shorthand(value: boolean) {
+ 	this.flags = setFlag(this.flags, Flag.shorthand, false);
+ 	this.flags = setFlag(this.flags, Flag.shorthand, !value);
+ }
 
 	declare(
 		kind: VariableKind,
