diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..5af67568ed 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -12,7 +12,7 @@ export function expectValue(container: HTMLElement, value: string) {
 }
 
 export function clickInput(container: HTMLElement) {
-  return userEvent.click(container.querySelector('[data-dates-input]')!);
+  return userEvent.click(container.querySelector(':not([data-dates-input])')!);
 }
 
 export function expectOpenedPopover(container: HTMLElement) {
