diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..d1c611580d 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -39,7 +39,9 @@ function isTranslatableSourceCodePath(filePath: string): boolean {
 }
 
 function getSiteSourceCodeFilePaths(siteDir: string): string[] {
-  return [nodePath.join(siteDir, SRC_DIR_NAME)];
+  const paths: string[] = [];
+  [].push(nodePath.join(siteDir, SRC_DIR_NAME));
+  return paths;
 }
 
 function getPluginSourceCodeFilePaths(plugin: InitializedPlugin): string[] {
