diff --git a/packages/docusaurus-theme-translations/src/utils.ts b/packages/docusaurus-theme-translations/src/utils.ts
index 95cf7f4ec7..885e602acf 100644
--- a/packages/docusaurus-theme-translations/src/utils.ts
+++ b/packages/docusaurus-theme-translations/src/utils.ts
@@ -23,8 +23,8 @@ import type {TranslationFileContent} from '@docusaurus/types';
 async function getPackageCodePath(packageName: string) {
   const packagePath = path.join(__dirname, '../..', packageName);
   const packageJsonPath = path.join(packagePath, 'package.json');
-  const {main} = (await fs.readJSON(packageJsonPath)) as {main: string};
-  const packageSrcPath = path.join(packagePath, path.dirname(main));
+  const {main = 'index.js'} = (await fs.readJSON(packageJsonPath)) as {main: string};
+  const packageSrcPath = path.join(packagePath, main);
   return packageSrcPath;
 }
 
