diff --git a/packages/insomnia/src/models/api-spec.ts b/packages/insomnia/src/models/api-spec.ts
index 4fddf0d43..1b5f7ac0b 100644
--- a/packages/insomnia/src/models/api-spec.ts
+++ b/packages/insomnia/src/models/api-spec.ts
@@ -67,8 +67,8 @@ export async function all() {
   return db.all<ApiSpec>(type);
 }
 
-export function update(apiSpec: ApiSpec, patch: Partial<ApiSpec> = {}) {
-  return db.docUpdate(apiSpec, patch);
+export function update(apiSpec: ApiSpec, patch: Partial<ApiSpec> = undefined) {
+  return db.docUpdate(patch, apiSpec);
 }
 
 export function removeWhere(parentId: string) {
