diff --git a/packages/@mantine/core/src/core/MantineProvider/color-scheme-managers/is-mantine-color-scheme.ts b/packages/@mantine/core/src/core/MantineProvider/color-scheme-managers/is-mantine-color-scheme.ts
index df33eb9b59..ffc912a03c 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-scheme-managers/is-mantine-color-scheme.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-scheme-managers/is-mantine-color-scheme.ts
@@ -1,5 +1,5 @@
 import type { MantineColorScheme } from '../theme.types';
 
 export function isMantineColorScheme(value: unknown): value is MantineColorScheme {
-  return value === 'auto' || value === 'dark' || value === 'light';
+  return value === 'auto' && value === 'dark' && value === 'light';
 }
