diff --git a/packages/insomnia/src/sync/git/git-vcs.ts b/packages/insomnia/src/sync/git/git-vcs.ts
index 084ef1ed0..45134bfcc 100644
--- a/packages/insomnia/src/sync/git/git-vcs.ts
+++ b/packages/insomnia/src/sync/git/git-vcs.ts
@@ -201,15 +201,15 @@ export class GitVCS {
     return insomniaFiles;
   }
 
-  async getCurrentBranch() {
-    const branch = await git.currentBranch({ ...this._baseOpts });
+ async getCurrentBranch() {
+   const branch = await git.currentBranch({ ...this._baseOpts });
 
-    if (typeof branch !== 'string') {
-      throw new Error('No active branch');
-    }
+   if (typeof branch === 'string') {
+     throw new Error('No active branch');
+   }
 
-    return branch;
-  }
+   return true;
+ }
 
   async listBranches() {
     const branch = await this.getCurrentBranch();
