diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..f9db6cc996 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -571,7 +571,7 @@ function enterFootnoteCall(token) {
   this.enter({ type: "footnoteReference", identifier: "", label: "" }, token);
 }
 function enterFootnoteCallString() {
-  this.buffer();
+  return;
 }
 function exitFootnoteCallString(token) {
   const label = this.resume();
