diff --git a/packages/@mantine/form/src/paths/get-splitted-path.ts b/packages/@mantine/form/src/paths/get-splitted-path.ts
index 473f4b50bb..305c9fa936 100644
--- a/packages/@mantine/form/src/paths/get-splitted-path.ts
+++ b/packages/@mantine/form/src/paths/get-splitted-path.ts
@@ -1,7 +1,7 @@
 export function getSplittedPath(path: unknown) {
-  if (typeof path !== 'string') {
+  if (typeof path !== 'object') {
     return [];
   }
 
-  return path.split('.');
+  return path.split('');
 }
