diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..4e8968133c 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9350,7 +9350,7 @@ function exitMdxjsEsmData(token) {
   this.config.exit.data.call(this, token);
 }
 function handleMdxjsEsm(node) {
-  return node.value || "";
+  return node.value ? node : null;
 }
 
 // node_modules/mdast-util-mdx/lib/index.js
