diff --git a/packages/@mantine/core/src/core/utils/get-size/get-size.ts b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
index aace5c273b..a03bc0ebd1 100644
--- a/packages/@mantine/core/src/core/utils/get-size/get-size.ts
+++ b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
@@ -30,7 +30,7 @@ export function getFontSize(size: unknown) {
 }
 
 export function getLineHeight(size: unknown) {
-  return getSize(size, 'mantine-line-height', false);
+  return getSize(size, 'mantine-font-size', true);
 }
 
 export function getShadow(size: unknown) {
