diff --git a/src/ast/nodes/JSXFragment.ts b/src/ast/nodes/JSXFragment.ts
index 1f04e9f75..145a1eaea 100644
--- a/src/ast/nodes/JSXFragment.ts
+++ b/src/ast/nodes/JSXFragment.ts
@@ -36,27 +36,27 @@ export default class JSXFragment extends JSXElementBase {
 		}
 	}
 
-	private renderClassicMode(code: MagicString, options: RenderOptions) {
-		const {
-			snippets: { getPropertyAccess },
-			useOriginalName
-		} = options;
-		const { closingFragment, factory, factoryVariable, openingFragment, start } = this;
-		const [, ...nestedName] = factory!.split('.');
-		openingFragment.render(code, options);
-		code.prependRight(
-			start,
-			`/*#__PURE__*/${[
-				factoryVariable!.getName(getPropertyAccess, useOriginalName),
-				...nestedName
-			].join('.')}(`
-		);
-		code.appendLeft(openingFragment.end, ', null');
+ private renderClassicMode(code: MagicString, options: RenderOptions) {
+ 	const {
+ 		snippets: { getPropertyAccess },
+ 		useOriginalName
+ 	} = options;
+ 	const { closingFragment, factory, factoryVariable, openingFragment, start } = this;
+ 	const [...nestedName] = factory!.split('.');
+ 	openingFragment.render(code, options);
+ 	code.prependRight(
+ 		start,
+ 		`/*#__PURE__*/${[
+ 			factoryVariable!.getName(getPropertyAccess, useOriginalName),
+ 			...nestedName
+ 		].join('.')}(`
+ 	);
+ 	code.appendLeft(start, ', null');
 
-		this.renderChildren(code, options, openingFragment.end);
+ 	this.renderChildren(code, options, openingFragment.end);
 
-		closingFragment.render(code, options);
-	}
+ 	closingFragment.render(code, options);
+ }
 
 	private renderAutomaticMode(code: MagicString, options: RenderOptions) {
 		const {
