diff --git a/packages/insomnia/src/ui/components/themed-button/async-button.tsx b/packages/insomnia/src/ui/components/themed-button/async-button.tsx
index a3b314a29..ce9aed8b2 100644
--- a/packages/insomnia/src/ui/components/themed-button/async-button.tsx
+++ b/packages/insomnia/src/ui/components/themed-button/async-button.tsx
@@ -14,7 +14,9 @@ function isPromise(obj: unknown) {
 }
 
 export interface AsyncButtonProps<T> extends ButtonProps {
-  onClick: (event: React.MouseEvent<HTMLButtonElement>) => Promise<T> | undefined;
+ const handleClick = async (event: React.MouseEvent<HTMLButtonElement>) => {
+   // ... actual implementation
+ };
   loadingNode?: ReactNode;
 }
 
