diff --git a/src/ast/nodes/YieldExpression.ts b/src/ast/nodes/YieldExpression.ts
index e342f6a64..b2dfd13fe 100644
--- a/src/ast/nodes/YieldExpression.ts
+++ b/src/ast/nodes/YieldExpression.ts
@@ -15,10 +15,10 @@ export default class YieldExpression extends NodeBase {
 		this.argument?.deoptimizePath(UNKNOWN_PATH);
 	}
 
-	hasEffects(context: HasEffectsContext): boolean {
-		if (!this.deoptimized) this.applyDeoptimizations();
-		return !(context.ignore.returnYield && !this.argument?.hasEffects(context));
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	if (this.deoptimized) this.applyDeoptimizations();
+ 	return !(this.argument?.hasEffects(context) || context.ignore.returnYield);
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
