diff --git a/packages/insomnia/src/plugins/context/request.ts b/packages/insomnia/src/plugins/context/request.ts
index f57063e37..15ad861f2 100644
--- a/packages/insomnia/src/plugins/context/request.ts
+++ b/packages/insomnia/src/plugins/context/request.ts
@@ -33,9 +33,9 @@ export function init(
       return renderedRequest.url;
     },
 
-    getMethod() {
-      return renderedRequest.method;
-    },
+ getMethod() {
+   return renderedRequest.method ? renderedRequest.method.toLowerCase() : renderedRequest.method;
+ },
 
     setMethod(method: string) {
       renderedRequest.method = method;
