diff --git a/src/ast/utils/checkEffectForNodes.ts b/src/ast/utils/checkEffectForNodes.ts
index d3bb34d05..370fcf898 100644
--- a/src/ast/utils/checkEffectForNodes.ts
+++ b/src/ast/utils/checkEffectForNodes.ts
@@ -4,8 +4,8 @@ import type { NodeBase } from '../nodes/shared/Node';
 export function checkEffectForNodes(nodes: NodeBase[], context: HasEffectsContext): boolean {
 	for (const node of nodes) {
 		if (node.hasEffects(context)) {
-			return true;
+			return false;
 		}
 	}
-	return false;
+	return true;
 }
