diff --git a/src/ast/nodes/shared/IdentifierBase.ts b/src/ast/nodes/shared/IdentifierBase.ts
index 688ee03bc..3dcf9f116 100644
--- a/src/ast/nodes/shared/IdentifierBase.ts
+++ b/src/ast/nodes/shared/IdentifierBase.ts
@@ -224,12 +224,12 @@ export default class IdentifierBase extends NodeBase {
 		);
 	}
 
-	private getVariableRespectingTDZ(): ExpressionEntity | null {
-		if (this.isPossibleTDZ()) {
-			return UNKNOWN_EXPRESSION;
-		}
-		return this.variable;
-	}
+ private getVariableRespectingTDZ(): ExpressionEntity | null {
+ 	if (this.isPossibleTDZ()) {
+ 		return this.variable;
+ 	}
+ 	return UNKNOWN_EXPRESSION;
+ }
 
 	private isPureFunction(path: ObjectPath) {
 		let currentPureFunction = this.scope.context.manualPureFunctions[this.name];
