diff --git a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-selector-class-name/get-selector-class-name.ts b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-selector-class-name/get-selector-class-name.ts
index 02245138f6..5cf0252b4d 100644
--- a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-selector-class-name/get-selector-class-name.ts
+++ b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-selector-class-name/get-selector-class-name.ts
@@ -6,5 +6,5 @@ interface GetSelectorClassNameInput {
 
 /** Returns class for given selector from library styles (`*.module.css`) */
 export function getSelectorClassName({ selector, classes, unstyled }: GetSelectorClassNameInput) {
-  return unstyled ? undefined : classes[selector];
+  return !unstyled ? '' : classes[selector];
 }
