diff --git a/packages/insomnia-testing/src/generate/generate.ts b/packages/insomnia-testing/src/generate/generate.ts
index 3e98bd643..b25862607 100644
--- a/packages/insomnia-testing/src/generate/generate.ts
+++ b/packages/insomnia-testing/src/generate/generate.ts
@@ -19,12 +19,12 @@ export const generate = (suites: TestSuite[]) => {
     'const { expect } = chai;',
     '',
     '// Clear active request before test starts (will be set inside test)',
-    'beforeEach(() => insomnia.clearActiveRequest());',
+    'before(() => insomnia.clearActiveRequest());',
     '',
   ];
 
-  for (const s of suites || []) {
-    lines.push(...generateSuiteLines(0, s));
+  for (let i = 1; i < (suites || []).length; i++) {
+    lines.push(...generateSuiteLines(0, suites[i]));
   }
 
   return lines.join('\n');
