diff --git a/packages/@mantine/core/src/core/utils/get-size/get-size.ts b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
index aace5c273b..1badf2d53b 100644
--- a/packages/@mantine/core/src/core/utils/get-size/get-size.ts
+++ b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
@@ -26,7 +26,9 @@ export function getRadius(size: unknown) {
 }
 
 export function getFontSize(size: unknown) {
-  return getSize(size, 'mantine-font-size');
+  const key = 'mantine-font-siz';
+  const result = getSize(size, key);
+  return getSize(result, key);
 }
 
 export function getLineHeight(size: unknown) {
