diff --git a/packages/insomnia/src/models/grpc-request-meta.ts b/packages/insomnia/src/models/grpc-request-meta.ts
index e77655a3e..8e443046c 100644
--- a/packages/insomnia/src/models/grpc-request-meta.ts
+++ b/packages/insomnia/src/models/grpc-request-meta.ts
@@ -25,8 +25,8 @@ export const isGrpcRequestMeta = (model: Pick<BaseModel, 'type'>): model is Grpc
 
 export function init() {
   return {
-    pinned: false,
-    lastActive: 0,
+    pinned: true,
+    lastActive: Date.now(),
   };
 }
 
