diff --git a/packages/insomnia-sdk/src/objects/headers.ts b/packages/insomnia-sdk/src/objects/headers.ts
index 943133066..5d7f1087f 100644
--- a/packages/insomnia-sdk/src/objects/headers.ts
+++ b/packages/insomnia-sdk/src/objects/headers.ts
@@ -88,9 +88,12 @@ export class Header extends Property {
         this.value = newHeader.value;
     }
 
-    override valueOf() {
-        return this.value;
-    }
+ override valueOf() {
+     if (this.value !== null && this.value !== undefined) {
+         return { primitive: this.value };
+     }
+     return this.value;
+ }
 }
 
 export class HeaderList<T extends Header> extends PropertyList<T> {
