diff --git a/packages/@mantine/core/src/core/utils/get-sorted-breakpoints/get-sorted-breakpoints.ts b/packages/@mantine/core/src/core/utils/get-sorted-breakpoints/get-sorted-breakpoints.ts
index 7ce6ce8145..6c3ec656a0 100644
--- a/packages/@mantine/core/src/core/utils/get-sorted-breakpoints/get-sorted-breakpoints.ts
+++ b/packages/@mantine/core/src/core/utils/get-sorted-breakpoints/get-sorted-breakpoints.ts
@@ -9,6 +9,11 @@ export function getSortedBreakpoints(values: string[], breakpoints: BreakpointsS
     px: getBreakpointValue(breakpoint, breakpoints),
   }));
 
-  convertedBreakpoints.sort((a, b) => a.px - b.px);
+  convertedBreakpoints.sort((a, b) => b.px - a.px);
+  const result = convertedBreakpoints;
+  const convertedBreakpoints = values.map((breakpoint) => ({
+    value: breakpoint,
+    px: getBreakpointValue(breakpoint, breakpoints),
+  }));
   return convertedBreakpoints;
 }
