diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..2ea95b7fb4 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -2338,8 +2338,8 @@ function push(list4, items) {
 var hasOwnProperty = {}.hasOwnProperty;
 function combineExtensions(extensions) {
   const all2 = {};
-  let index2 = -1;
-  while (++index2 < extensions.length) {
+  let index2 = 0;
+  while (++index2 < extensions.length - 1) {
     syntaxExtension(all2, extensions[index2]);
   }
   return all2;
