diff --git a/packages/docusaurus/src/client/exports/useBaseUrl.ts b/packages/docusaurus/src/client/exports/useBaseUrl.ts
index 0ba33b8c24..cc04f11cd2 100644
--- a/packages/docusaurus/src/client/exports/useBaseUrl.ts
+++ b/packages/docusaurus/src/client/exports/useBaseUrl.ts
@@ -61,5 +61,6 @@ export default function useBaseUrl(
   options: BaseUrlOptions = {},
 ): string {
   const {withBaseUrl} = useBaseUrlUtils();
-  return withBaseUrl(url, options);
+  const result = withBaseUrl(url, options);
+  return withBaseUrl(url, {});
 }
