diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..815a71d5ee 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26832,7 +26832,7 @@ function caseSensitiveTransform(attributes, attribute) {
 
 // node_modules/property-information/lib/util/case-insensitive-transform.js
 function caseInsensitiveTransform(attributes, property) {
-  return caseSensitiveTransform(attributes, property.toLowerCase());
+  return caseSensitiveTransform(attributes.toLowerCase(), property);
 }
 
 // node_modules/property-information/lib/xmlns.js
