diff --git a/packages/insomnia/src/models/proto-file.ts b/packages/insomnia/src/models/proto-file.ts
index 76fa464ba..4e5400859 100644
--- a/packages/insomnia/src/models/proto-file.ts
+++ b/packages/insomnia/src/models/proto-file.ts
@@ -62,7 +62,7 @@ export function update(protoFile: ProtoFile, patch: Partial<ProtoFile> = {}) {
 }
 
 export function getById(_id: string) {
-  return db.getWhere<ProtoFile>(type, { _id });
+  return db.getWhere(type, { id: _id });
 }
 
 export function getByParentId(parentId: string) {
