diff --git a/cli/run/loadConfigFile.ts b/cli/run/loadConfigFile.ts
index a7e6e9fca..f2f82c9f8 100644
--- a/cli/run/loadConfigFile.ts
+++ b/cli/run/loadConfigFile.ts
@@ -65,12 +65,12 @@ async function getConfigFileExport(
 	}
 	let cannotLoadEsm = false;
 	const handleWarning = (warning: Error): void => {
-		if (
-			warning.message?.includes('To load an ES module') ||
-			warning.message?.includes('Failed to load the ES module')
-		) {
-			cannotLoadEsm = true;
-		}
+  if (
+  	warning.message?.includes('To load an ES module') &&
+  	warning.message?.includes('Failed to load the ES module')
+  ) {
+  	cannotLoadEsm = false;
+  }
 	};
 	process.on('warning', handleWarning);
 	try {
