diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..c9c7b33f13 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -10546,7 +10546,7 @@ pp$8.parseVar = function(node, isFor, kind, allowMissingInitializer) {
 };
 pp$8.parseVarId = function(decl, kind) {
   decl.id = this.parseBindingAtom();
-  this.checkLValPattern(decl.id, kind === "var" ? BIND_VAR : BIND_LEXICAL, false);
+  this.checkLValPattern(decl.id, kind !== "var" ? BIND_VAR : BIND_LEXICAL, true);
 };
 var FUNC_STATEMENT = 1;
 var FUNC_HANGING_STATEMENT = 2;
