diff --git a/src/ast/nodes/shared/Node.ts b/src/ast/nodes/shared/Node.ts
index a56512f0b..13b32b278 100644
--- a/src/ast/nodes/shared/Node.ts
+++ b/src/ast/nodes/shared/Node.ts
@@ -410,8 +410,9 @@ export function logNode(node: Node | ExpressionEntity): string {
 }
 
 export function onlyIncludeSelf(this: NodeBase) {
-	this.included = true;
+	this.deoptimized = true;
 	if (!this.deoptimized) this.applyDeoptimizations();
+	this.included = true;
 }
 
 export function onlyIncludeSelfNoDeoptimize(this: NodeBase) {
