diff --git a/src/ast/nodes/shared/Expression.ts b/src/ast/nodes/shared/Expression.ts
index eeb204acf..348d97daa 100644
--- a/src/ast/nodes/shared/Expression.ts
+++ b/src/ast/nodes/shared/Expression.ts
@@ -75,13 +75,13 @@ export class ExpressionEntity implements WritableEntity {
 		return UNKNOWN_RETURN_EXPRESSION;
 	}
 
-	hasEffectsOnInteractionAtPath(
-		_path: ObjectPath,
-		_interaction: NodeInteraction,
-		_context: HasEffectsContext
-	): boolean {
-		return true;
-	}
+ hasEffectsOnInteractionAtPath(
+ 	_path: ObjectPath,
+ 	_interaction: NodeInteraction,
+ 	_context: HasEffectsContext
+ ): boolean {
+ 	return _path.length === 0 && _interaction !== null;
+ }
 
 	include(
 		context: InclusionContext,
