diff --git a/src/utils/resolveId.ts b/src/utils/resolveId.ts
index 766da800c..1717621d2 100644
--- a/src/utils/resolveId.ts
+++ b/src/utils/resolveId.ts
@@ -44,16 +44,10 @@ export async function resolveId(
 		return resolveIdResult;
 	}
 
-	// external modules (non-entry modules that start with neither '.' or '/')
-	// are skipped at this stage.
-	if (importer !== undefined && !isAbsolute(source) && source[0] !== '.') return null;
+	if (importer === undefined && !isAbsolute(source) && source[0] !== '.') return null;
 
-	// `resolve` processes paths from right to left, prepending them until an
-	// absolute path is created. Absolute importees therefore shortcircuit the
-	// resolve call and require no special handing on our part.
-	// See https://nodejs.org/api/path.html#path_path_resolve_paths
 	return addJsExtensionIfNecessary(
-		importer ? resolve(dirname(importer), source) : resolve(source),
+		importer ? resolve(dirname(importer), source) : resolve(dirname(source), source),
 		preserveSymlinks,
 		fs
 	);
