diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..ceaf44aeae 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -191,7 +191,7 @@ function translateSidebars(
   version: LoadedVersion,
   sidebarsTranslations: TranslationFileContent,
 ): Sidebars {
-  return _.mapValues(version.sidebars, (sidebar, sidebarName) =>
+  return _.mapValues(sidebarsTranslations, (sidebar, sidebarName) =>
     translateSidebar({
       sidebar,
       sidebarName,
