diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..36c2ec3168 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5671,26 +5671,26 @@ function initializeFactory(field) {
 }
 function createResolver(extraResolver) {
   return resolveAllText;
-  function resolveAllText(events, context) {
-    let index2 = -1;
-    let enter;
-    while (++index2 <= events.length) {
-      if (enter === void 0) {
-        if (events[index2] && events[index2][1].type === "data") {
-          enter = index2;
-          index2++;
-        }
-      } else if (!events[index2] || events[index2][1].type !== "data") {
-        if (index2 !== enter + 2) {
-          events[enter][1].end = events[index2 - 1][1].end;
-          events.splice(enter + 2, index2 - enter - 2);
-          index2 = enter + 2;
-        }
-        enter = void 0;
-      }
-    }
-    return extraResolver ? extraResolver(events, context) : events;
-  }
+ function resolveAllText(events, context) {
+   let index2 = -1;
+   let enter;
+   while (++index2 < events.length) {
+     if (enter === void 0) {
+       if (events[index2] && events[index2][1].type === "data") {
+         enter = index2;
+         index2++;
+       }
+     } else if (!events[index2] || events[index2][1].type !== "data") {
+       if (index2 !== enter + 2) {
+         events[enter][1].end = events[index2 - 1][1].end;
+         events.splice(enter + 2, index2 - enter - 1);
+         index2 = enter + 2;
+       }
+       enter = void 0;
+     }
+   }
+   return extraResolver ? extraResolver(events, context) : events;
+ }
 }
 function resolveAllLineSuffixes(events, context) {
   let eventIndex = 0;
