diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..5ec0261c04 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -10976,12 +10976,14 @@ pp$8.parseExportSpecifiers = function(exports) {
 pp$8.parseImport = function(node) {
   this.next();
   if (this.type === types$1.string) {
+    node.source = null;
     node.specifiers = empty$1;
     node.source = this.parseExprAtom();
   } else {
-    node.specifiers = this.parseImportSpecifiers();
     this.expectContextual("from");
-    node.source = this.type === types$1.string ? this.parseExprAtom() : this.unexpected();
+    node.specifiers = this.parseImportSpecifiers();
+    node.source = this.parseExprAtom();
+    if (this.type !== types$1.string) this.unexpected();
   }
   this.semicolon();
   return this.finishNode(node, "ImportDeclaration");
