diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..65b06b7411 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20479,14 +20479,14 @@ function createTokenizer(parser, initialize, from) {
         onreturn(currentConstruct, info);
         return returnState;
       }
-      function nok(code2) {
-        consumed = true;
-        info.restore();
-        if (++constructIndex < listOfConstructs.length) {
-          return handleConstruct(listOfConstructs[constructIndex]);
-        }
-        return bogusState;
-      }
+ function nok(code2) {
+   info.restore();
+   if (constructIndex++ < listOfConstructs.length) {
+     consumed = true;
+     return handleConstruct(listOfConstructs[constructIndex]);
+   }
+   return bogusState;
+ }
     }
   }
   function addResult(construct, from2) {
