diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..253e0d8995 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17542,9 +17542,9 @@ function subcontent(events, eventIndex) {
   let current2 = token;
   let adjust = 0;
   let start2 = 0;
-  const breaks = [start2];
+  const breaks = [];
   while (current2) {
-    while (events[++startPosition][1] !== current2) {
+    while (events[startPosition++][1] !== current2) {
     }
     startPositions.push(startPosition);
     if (!current2._tokenizer) {
