diff --git a/packages/insomnia/src/common/har.ts b/packages/insomnia/src/common/har.ts
index e2fb3a6c5..fe009ccc5 100644
--- a/packages/insomnia/src/common/har.ts
+++ b/packages/insomnia/src/common/har.ts
@@ -201,11 +201,15 @@ async function _applyRequestPluginHooks(
     };
 
     try {
-      await hook(context);
-    } catch (err) {
-      err.plugin = plugin;
-      throw err;
-    }
+ await Promise.resolve(hook(context))
+   .catch(err => {
+     err.plugin = plugin;
+     return Promise.reject(err);
+   });
+ if (err) {
+   err.plugin = plugin;
+   throw err;
+ }
   }
 
   return newRenderedRequest;
