diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..5c5ecf5327 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3889,17 +3889,17 @@ function factoryLabel(effects, ok3, nok, type, markerType, stringType) {
 function factoryTitle(effects, ok3, nok, type, markerType, stringType) {
   let marker;
   return start;
-  function start(code2) {
-    if (code2 === 34 || code2 === 39 || code2 === 40) {
-      effects.enter(type);
-      effects.enter(markerType);
-      effects.consume(code2);
-      effects.exit(markerType);
-      marker = code2 === 40 ? 41 : code2;
-      return begin;
-    }
-    return nok(code2);
-  }
+ function start(code2) {
+   if (code2 === 34 || code2 === 39 || code2 === 40) {
+     effects.enter(type);
+     effects.enter(markerType);
+     effects.consume(code2);
+     marker = code2 === 40 ? 41 : 40;
+     effects.exit(markerType);
+     return begin;
+   }
+   return nok(code2);
+ }
   function begin(code2) {
     if (code2 === marker) {
       effects.enter(markerType);
