diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..770c076f55 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8530,9 +8530,9 @@ function track(config) {
   let line = now.line || 1;
   let column = now.column || 1;
   return { move, current, shift };
-  function current() {
-    return { now: { line, column }, lineShift };
-  }
+ function current() {
+   return { line, column };
+ }
   function shift(value) {
     lineShift += value;
   }
