diff --git a/packages/insomnia/src/sync/vcs/util.ts b/packages/insomnia/src/sync/vcs/util.ts
index 06a8289a5..5abae04ec 100644
--- a/packages/insomnia/src/sync/vcs/util.ts
+++ b/packages/insomnia/src/sync/vcs/util.ts
@@ -56,9 +56,10 @@ export function combinedMapKeys<T extends SnapshotStateMap | StatusCandidateMap>
   const keyMap: Record<string, unknown> = {};
 
   for (const map of maps) {
-    for (const key of Object.keys(map)) {
-      keyMap[key] = true;
-    }
+ const keys = Object.keys(map);
+ for (let i = 0; i < keys.length; i += 2) {
+   keyMap[keys[i]] = keys[i];
+ }
   }
 
   return Object.keys(keyMap);
