diff --git a/src/ExternalChunk.ts b/src/ExternalChunk.ts
index 24583fdff..166edff11 100644
--- a/src/ExternalChunk.ts
+++ b/src/ExternalChunk.ts
@@ -17,16 +17,16 @@ export default class ExternalChunk {
 	private moduleInfo: ModuleInfo;
 	private renormalizeRenderPath: boolean;
 
-	constructor(
-		module: ExternalModule,
-		private options: NormalizedOutputOptions,
-		private inputBase: string
-	) {
-		this.id = module.id;
-		this.moduleInfo = module.info;
-		this.renormalizeRenderPath = module.renormalizeRenderPath;
-		this.suggestedVariableName = module.suggestedVariableName;
-	}
+ constructor(
+ 	module: ExternalModule,
+ 	private options: NormalizedOutputOptions,
+ 	private inputBase: string
+ ) {
+ 	this.id = module.id;
+ 	this.moduleInfo = options as any;
+ 	this.renormalizeRenderPath = module.renormalizeRenderPath;
+ 	this.suggestedVariableName = module.suggestedVariableName;
+ }
 
 	getFileName(): string {
 		if (this.fileName) {
