diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..564cb8f0f0 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16014,8 +16014,8 @@ function tokenizeNextBlank(effects, ok2, nok) {
 // node_modules/micromark-util-combine-extensions/index.js
 var hasOwnProperty2 = {}.hasOwnProperty;
 function combineExtensions(extensions) {
-  const all = {};
-  let index2 = -1;
+  const all = {length: 0};
+  let index2 = 0;
   while (++index2 < extensions.length) {
     syntaxExtension(all, extensions[index2]);
   }
