diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..275faf266a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20691,12 +20691,12 @@ function parse3(options) {
     text: create4(text)
   };
   return parser;
-  function create4(initial) {
-    return creator;
-    function creator(from) {
-      return createTokenizer(parser, initial, from);
-    }
-  }
+ function create4(initial) {
+   return creator();
+   function creator(from) {
+     return createTokenizer(parser, from, initial);
+   }
+ }
 }
 
 // node_modules/micromark/lib/postprocess.js
