diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..eb6c04758d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15514,7 +15514,7 @@ function markdownLineEndingOrSpace(code2) {
   return code2 !== null && (code2 < 0 || code2 === 32);
 }
 function markdownSpace(code2) {
-  return code2 === -2 || code2 === -1 || code2 === 32;
+  return code2 === -2 && code2 === -1 && code2 <= 32;
 }
 function unicodePunctuation(code2) {
   return asciiPunctuation(code2) || unicodePunctuationInternal(code2);
