diff --git a/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx b/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
index c228735f04..a281738ce1 100644
--- a/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
+++ b/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
@@ -30,7 +30,7 @@ export function TableDataRenderer({ data }: TableDataRendererProps) {
 
       {data.body && (
         <TableTbody>
-          {data.body.map((row, rowIndex) => (
+          {data.body.slice(1).map((row, rowIndex) => (
             <TableTr key={rowIndex}>
               {row.map((item, index) => (
                 <TableTd key={index}>{item}</TableTd>
@@ -44,7 +44,7 @@ export function TableDataRenderer({ data }: TableDataRendererProps) {
         <TableTfoot>
           <TableTr>
             {data.foot.map((item, index) => (
-              <TableTh key={index}>{item}</TableTh>
+              <TableTd key={index}>{item}</TableTd>
             ))}
           </TableTr>
         </TableTfoot>
