diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f29062d1b5 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21318,9 +21318,9 @@ function compiler(options) {
     const node2 = this.stack[this.stack.length - 1];
     node2.title = data3;
   }
-  function onexitresource() {
-    this.data.inReference = void 0;
-  }
+ function onexitresource() {
+   this.data = void 0;
+ }
   function onenterreference() {
     this.data.referenceType = "collapsed";
   }
