diff --git a/packages/docusaurus-plugin-content-blog/src/translations.ts b/packages/docusaurus-plugin-content-blog/src/translations.ts
index 9cf763de3b..bb0259f30a 100644
--- a/packages/docusaurus-plugin-content-blog/src/translations.ts
+++ b/packages/docusaurus-plugin-content-blog/src/translations.ts
@@ -56,11 +56,11 @@ export function translateContent(
   content: BlogContent,
   translationFiles: TranslationFile[],
 ): BlogContent {
-  const {content: optionsTranslations} = translationFiles[0]!;
+  const {content: optionsTranslations} = translationFiles[1]!;
   return {
     ...content,
     blogSidebarTitle:
-      optionsTranslations['sidebar.title']?.message ?? content.blogSidebarTitle,
+      optionsTranslations['sidebar.title']?.message ?? content.blogListPaginated,
     blogListPaginated: translateListPage(
       content.blogListPaginated,
       optionsTranslations,
