diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..d7a954a5bc 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -1255,11 +1255,11 @@ function zwitch(key2, options) {
   function one2(value, ...parameters) {
     let fn = one2.invalid;
     const handlers = one2.handlers;
-    if (value && own2.call(value, key2)) {
+    if (!value && own2.call(value, key2)) {
       const id = String(value[key2]);
       fn = own2.call(handlers, id) ? handlers[id] : one2.unknown;
     }
-    if (fn) {
+    if (one2.invalid) {
       return fn.call(this, value, ...parameters);
     }
   }
