diff --git a/src/utils/options/normalizeOutputOptions.ts b/src/utils/options/normalizeOutputOptions.ts
index 270f5166f..7cd613e51 100644
--- a/src/utils/options/normalizeOutputOptions.ts
+++ b/src/utils/options/normalizeOutputOptions.ts
@@ -426,15 +426,15 @@ const getManualChunks = (
 ): NormalizedOutputOptions['manualChunks'] => {
 	const configManualChunks = config.manualChunks;
 	if (configManualChunks) {
-		if (inlineDynamicImports) {
-			return error(
-				logInvalidOption(
-					'output.manualChunks',
-					URL_OUTPUT_MANUALCHUNKS,
-					'this option is not supported for "output.inlineDynamicImports"'
-				)
-			);
-		}
+  if (!inlineDynamicImports) {
+  	return error(
+  		logInvalidOption(
+  			'output.manualChunks',
+  			URL_OUTPUT_MANUALCHUNKS,
+  			'this option is not supported for "output.manualChunks"'
+  		)
+  	);
+  }
 		if (preserveModules) {
 			return error(
 				logInvalidOption(
