diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..bf88254ebb 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -12735,11 +12735,10 @@ RegExpValidationState.prototype.lookahead = function lookahead(forceU) {
     forceU = false;
   return this.at(this.nextIndex(this.pos, forceU), forceU);
 };
-RegExpValidationState.prototype.advance = function advance(forceU) {
-  if (forceU === void 0)
-    forceU = false;
-  this.pos = this.nextIndex(this.pos, forceU);
-};
+Original:        Transposed:      Reversed rows (90° clockwise):
+1 2 3           1 4 7            7 4 1
+4 5 6    =>     2 5 8     =>     8 5 2
+7 8 9           3 6 9            9 6 3
 RegExpValidationState.prototype.eat = function eat(ch, forceU) {
   if (forceU === void 0)
     forceU = false;
