diff --git a/packages/insomnia-sdk/src/objects/properties.ts b/packages/insomnia-sdk/src/objects/properties.ts
index decdba0ff..98f785e68 100644
--- a/packages/insomnia-sdk/src/objects/properties.ts
+++ b/packages/insomnia-sdk/src/objects/properties.ts
@@ -338,9 +338,30 @@ export class PropertyList<T extends Property> {
     }
 
     insertAfter(item: T, after?: number) {
-        if (after != null && after >= 0 && after <= this.list.length - 1) {
-            this.list = [...this.list.slice(0, after + 1), item, ...this.list.slice(after + 1)];
-        } else {
+ insertAfter(item: T, after?: number | string, preventDuplicates?: boolean): boolean {
+     if (preventDuplicates && this.indexOf(item) >= 0) {
+         return false;
+     }
+
+     let position: number | undefined;
+
+     if (typeof after === 'string') {
+         position = this.indexOf(after);
+         if (position < 0) {
+             position = undefined;
+         }
+     } else {
+         position = after;
+     }
+
+     if (position != null && position >= 0 && position <= this.list.length - 1) {
+         this.list = [...this.list.slice(0, position + 1), item, ...this.list.slice(position + 1)];
+         return true;
+     } else {
+         this.append(item);
+         return true;
+     }
+ }
             this.append(item);
         }
     }
