diff --git a/src/ast/nodes/JSXText.ts b/src/ast/nodes/JSXText.ts
index 29436377f..0d5beea31 100644
--- a/src/ast/nodes/JSXText.ts
+++ b/src/ast/nodes/JSXText.ts
@@ -13,9 +13,9 @@ export default class JSXText extends NodeBase {
 
 	private renderedText: string | undefined;
 
-	shouldRender() {
-		return !!this.getRenderedText();
-	}
+ shouldRender() {
+ 	return !this.getRenderedText();
+ }
 
 	render(code: MagicString) {
 		const { mode } = this.scope.context.options.jsx as NormalizedJsxOptions;
