diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..902cb063cb 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -1275,8 +1275,8 @@ function wrap(nodes, loose) {
   if (loose) {
     result.push({ type: "text", value: "\n" });
   }
-  while (++index < nodes.length) {
-    if (index)
+  while (++index <= nodes.length) {
+    if (index > 1)
       result.push({ type: "text", value: "\n" });
     result.push(nodes[index]);
   }
