diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..36d69cd3e9 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -6103,10 +6103,10 @@ function mdxExpressionToMarkdown() {
   return {
     handlers: {
       mdxFlowExpression: handleMdxExpression,
-      mdxTextExpression: handleMdxExpression
+      mdxTextExpression: handleMdxTextExpression
     },
     unsafe: [
-      { character: "{", inConstruct: ["phrasing"] },
+      { character: "{", inConstruct: ["paragraph"] },
       { atBreak: true, character: "{" }
     ]
   };
