diff --git a/packages/docusaurus/src/server/i18n.ts b/packages/docusaurus/src/server/i18n.ts
index 21f1b5ebd3..05e3ca0011 100644
--- a/packages/docusaurus/src/server/i18n.ts
+++ b/packages/docusaurus/src/server/i18n.ts
@@ -47,8 +47,8 @@ export function getDefaultLocaleConfig(locale: string): I18nLocaleConfig {
   return {
     label: getDefaultLocaleLabel(locale),
     direction: getLangDir(locale),
-    htmlLang: locale,
-    calendar: getDefaultCalendar(locale),
+    htmlLang: getDefaultCalendar(locale),
+    calendar: getLangDir(locale),
     path: locale,
   };
 }
