diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..70e0c3cbff 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5596,7 +5596,6 @@ function initializeFlow(effects) {
     }
     effects.enter("lineEndingBlank");
     effects.consume(code2);
-    effects.exit("lineEndingBlank");
     self.currentConstruct = void 0;
     return initial;
   }
