diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..0f5de201b 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -122,7 +122,7 @@ export const ascendingNumberSort: SortFunction<number> = (a, b) => {
 };
 
 export const descendingNumberSort: SortFunction<number> = (a, b) => {
-  return ascendingNumberSort(b, a);
+  return -ascendingNumberSort(b, a);
 };
 
 export const sortMethodMap = {
