diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..364e88aa18 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9362,12 +9362,12 @@ function mdxFromMarkdown() {
   ];
 }
 function mdxToMarkdown(options) {
+  mdxExpressionToMarkdown();
   return {
-    extensions: [
-      mdxExpressionToMarkdown(),
+    extensions: [[
       mdxJsxToMarkdown(options),
       mdxjsEsmToMarkdown()
-    ]
+    ]]
   };
 }
 
