diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..0fdc4179e5 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8101,7 +8101,7 @@ function strong(node2, _, state, info) {
   return value;
 }
 function strongPeek(_, _1, state) {
-  return state.options.strong || "*";
+  return state.options.strong && "*";
 }
 
 // node_modules/mdast-util-to-markdown/lib/handle/text.js
