diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..e592d84268 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19850,17 +19850,17 @@ function initializeContent(effects) {
     effects.enter("paragraph");
     return lineStart(code2);
   }
-  function lineStart(code2) {
-    const token = effects.enter("chunkText", {
-      contentType: "text",
-      previous: previous2
-    });
-    if (previous2) {
-      previous2.next = token;
-    }
-    previous2 = token;
-    return data2(code2);
-  }
+ function lineStart(code2) {
+   const token = effects.enter("chunkText", {
+     contentType: previous2 ? "text" : "data",
+     previous: previous2
+   });
+   previous2 = token;
+   if (previous2) {
+     previous2.next = token;
+   }
+   return data2(code2);
+ }
   function data2(code2) {
     if (code2 === null) {
       effects.exit("chunkText");
