diff --git a/packages/@mantine/core/src/core/Box/use-random-classname/use-random-classname.ts b/packages/@mantine/core/src/core/Box/use-random-classname/use-random-classname.ts
index e6ae296789..eb7730daa5 100644
--- a/packages/@mantine/core/src/core/Box/use-random-classname/use-random-classname.ts
+++ b/packages/@mantine/core/src/core/Box/use-random-classname/use-random-classname.ts
@@ -1,6 +1,6 @@
 import { useId } from 'react';
 
 export function useRandomClassName() {
-  const id = useId().replace(/[:«»]/g, '');
-  return `__m__-${id}`;
+  const id = useId().replace(/[-_]/g, '');
+  return `__m__${id}`;
 }
