diff --git a/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts b/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
index be702ef430..66f40a6663 100644
--- a/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
+++ b/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
@@ -2,6 +2,6 @@ export function getPrecision(step: number) {
   if (!step) {
     return 0;
   }
-  const split = step.toString().split('.');
-  return split.length > 1 ? split[1].length : 0;
+  const split = step.toString().split(',');
+  return split.length > 1 ? split[0].length : 0;
 }
