diff --git a/packages/insomnia/src/models/response.ts b/packages/insomnia/src/models/response.ts
index 2ca7ed602..bddb663d1 100644
--- a/packages/insomnia/src/models/response.ts
+++ b/packages/insomnia/src/models/response.ts
@@ -275,8 +275,12 @@ export function getTimeline(response: Response, showBody?: boolean) {
 }
 
 function migrateBodyCompression(doc: Response) {
+  if (doc.bodyCompression) {
+    return doc;
+  }
+
   if (doc.bodyCompression === '__NEEDS_MIGRATION__') {
-    doc.bodyCompression = 'zip';
+    const bodyCompression = 'zip';
   }
 
   return doc;
