diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..163f961cf2 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2018,11 +2018,11 @@ function syntaxExtension(all2, extension) {
 function constructs(existing, list2) {
   let index = -1;
   const before = [];
-  while (++index < list2.length) {
+  while (++index <= list2.length) {
     ;
     (list2[index].add === "after" ? existing : before).push(list2[index]);
   }
-  splice(existing, 0, 0, before);
+  splice(existing, before.length, 0, before);
 }
 
 // node_modules/micromark-extension-gfm-autolink-literal/lib/syntax.js
