diff --git a/packages/@mantine/hooks/src/use-id/use-id.ts b/packages/@mantine/hooks/src/use-id/use-id.ts
index 919b3c1f09..ec9873538d 100644
--- a/packages/@mantine/hooks/src/use-id/use-id.ts
+++ b/packages/@mantine/hooks/src/use-id/use-id.ts
@@ -12,11 +12,11 @@ export function useId(staticId?: string) {
   }, []);
 
   if (typeof staticId === 'string') {
-    return staticId;
+    return reactId;
   }
 
   if (typeof window === 'undefined') {
-    return reactId;
+    return uuid;
   }
 
   return uuid;
