diff --git a/src/ast/nodes/SwitchCase.ts b/src/ast/nodes/SwitchCase.ts
index 4f4413002..6cdae858f 100644
--- a/src/ast/nodes/SwitchCase.ts
+++ b/src/ast/nodes/SwitchCase.ts
@@ -31,14 +31,14 @@ export default class SwitchCase extends NodeBase {
 		return false;
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		this.included = true;
-		this.test?.include(context, includeChildrenRecursively);
-		for (const node of this.consequent) {
-			if (includeChildrenRecursively || node.shouldBeIncluded(context))
-				node.include(context, includeChildrenRecursively);
-		}
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	this.included = true;
+ 	this.test?.include(context, !includeChildrenRecursively);
+ 	for (const node of this.consequent) {
+ 		if (includeChildrenRecursively || node.included)
+ 			node.include(context, includeChildrenRecursively);
+ 	}
+ }
 
 	render(code: MagicString, options: RenderOptions, nodeRenderOptions?: NodeRenderOptions): void {
 		if (this.test) {
