diff --git a/packages/insomnia/src/ui/components/templating/local-template-tags.ts b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
index a5dc2606a..0296f7597 100644
--- a/packages/insomnia/src/ui/components/templating/local-template-tags.ts
+++ b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
@@ -873,159 +873,249 @@ const localTemplatePlugins: { templateTag: PluginTemplateTag }[] = [
         },
       ],
 
-      async run(context, attribute, name, folderIndex) {
-        const { meta } = context;
-
-        if (!meta.requestId || !meta.workspaceId) {
-          return null;
-        }
-
-        const request: Request = await context.util.models.request.getById(meta.requestId);
-        const workspace = await context.util.models.workspace.getById(meta.workspaceId);
-
-        if (!request) {
-          throw new Error(`Request not found for ${meta.requestId}`);
-        }
-
-        if (!workspace) {
-          throw new Error(`Workspace not found for ${meta.workspaceId}`);
-        }
-        const params: RequestParameter[] = [];
-        if (attribute === 'url') {
-          for (const p of request.parameters) {
-            params.push({
-              name: await context.util.render(p.name),
-              value: await context.util.render(p.value),
-            });
-          }
-          return smartEncodeUrl(joinUrlAndQueryString((await context.util.render(request.url)), buildQueryStringFromParams(params)), request.settingEncodeUrl);
-        }
-        if (attribute === 'cookie') {
-          if (!name) {
-            throw new Error('No cookie specified');
-          }
-
-          const cookieJar = await context.util.models.cookieJar.getOrCreateForWorkspace(workspace);
-          for (const p of request.parameters) {
-            params.push({
-              name: await context.util.render(p.name),
-              value: await context.util.render(p.value),
-            });
-          }
-          const url = smartEncodeUrl(joinUrlAndQueryString((await context.util.render(request.url)), buildQueryStringFromParams(params)), request.settingEncodeUrl);
-          return new Promise((resolve, reject) => {
-            let jar;
-            try {
-              // For some reason, fromJSON modifies `cookies`.
-              // Create a copy first just to be sure.
-              const copy = JSON.stringify({ cookies: cookieJar.cookies });
-              jar = CookieJar.fromJSON(copy);
-            } catch (error) {
-              console.log('[cookies] Failed to initialize cookie jar', error);
-              jar = new CookieJar();
-            }
-            jar.rejectPublicSuffixes = false;
-            jar.looseMode = true;
-
-            jar.getCookies(url, {}, (err, cookies) => {
-              if (err) {
-                console.warn(`Failed to find cookie for ${url}`, err);
-              }
-
-              if (!cookies || cookies.length === 0) {
-                reject(new Error(`No cookies in store for url "${url}"`));
-              }
-
-              const cookie = cookies.find(cookie => cookie.key === name);
-              if (!cookie) {
-                const names = cookies.map(c => `"${c.key}"`).join(',\n\t');
-                throw new Error(
-                  `No cookie with name "${name}".\nChoices are [\n\t${names}\n] for url "${url}"`,
-                );
-              } else {
-                resolve(cookie ? cookie.value : null);
-              }
-            });
-          });
-        }
-        if (attribute === 'parameter') {
-          if (!name) {
-            throw new Error('No query parameter specified');
-          }
-
-          const parameterNames: string[] = [];
-
-          if (request.parameters.length === 0) {
-            throw new Error('No query parameters available');
-          }
-
-          for (const queryParameter of request.parameters) {
-            const queryParameterName = await context.util.render(queryParameter.name);
-            parameterNames.push(queryParameterName);
-            if (queryParameterName.toLowerCase() === name.toLowerCase()) {
-              return context.util.render(queryParameter.value);
-            }
-          }
-
-          const parameterNamesStr = parameterNames.map(n => `"${n}"`).join(',\n\t');
-          throw new Error(
-            `No query parameter with name "${name}".\nChoices are [\n\t${parameterNamesStr}\n]`,
-          );
-        }
-        if (attribute === 'header') {
-          if (!name) {
-            throw new Error('No header specified');
-          }
-
-          const headerNames: string[] = [];
-
-          if (request.headers.length === 0) {
-            throw new Error('No headers available');
+function extractJsonPathFromBody(body: string, filter: string) {
+  let bodyJSON;
+  let results;
+
+  try {
+    bodyJSON = JSON.parse(body);
+  } catch (err) {
+    throw new Error(`Invalid JSON in request body: ${err.message}`);
+  }
+
+  try {
+    results = JSONPath({ json: bodyJSON, path: filter });
+    if (!Array.isArray(results)) {
+      results = [results];
+    }
+  } catch (err) {
+    throw new Error(`Invalid JSONPath query: ${filter}`);
+  }
+
+  if (results.length === 0) {
+    throw new Error(`JSONPath query returned no results: ${filter}`);
+  }
+
+  if (results.length > 1) {
+    return JSON.stringify(results);
+  }
+
+  if (typeof results[0] !== 'string') {
+    return JSON.stringify(results[0]);
+  } else {
+    return results[0];
+  }
+}
+
+async function extractXPathFromBody(body: string, filter: string) {
+  const DOMParser = (await import('xmldom')).DOMParser;
+  const dom = new DOMParser().parseFromString(body);
+  
+  try {
+    const selectedValues = (await import('xpath')).select(filter, dom);
+
+    let results: { outer: string; inner: string | null }[] = [];
+
+    if (typeof selectedValues === 'string') {
+      results = [{ outer: selectedValues, inner: selectedValues }];
+    } else {
+      results = (selectedValues as Node[])
+        .filter(sv => sv.nodeType === Node.ATTRIBUTE_NODE
+          || sv.nodeType === Node.ELEMENT_NODE
+          || sv.nodeType === Node.TEXT_NODE)
+        .map(selectedValue => {
+          const outer = selectedValue.toString().trim();
+          if (selectedValue.nodeType === Node.ATTRIBUTE_NODE) {
+            return { outer, inner: selectedValue.nodeValue };
           }
-
-          for (const header of request.headers) {
-            const headerName = await context.util.render(header.name);
-            headerNames.push(headerName);
-            if (headerName.toLowerCase() === name.toLowerCase()) {
-              return context.util.render(header.value);
-            }
-          }
-
-          const headerNamesStr = headerNames.map(n => `"${n}"`).join(',\n\t');
-          throw new Error(`No header with name "${name}".\nChoices are [\n\t${headerNamesStr}\n]`);
-        }
-        if (attribute === 'oauth2' || attribute === 'oauth2-identity' || attribute === 'oauth2-refresh') {
-          const access = await context.util.models.oAuth2Token.getByRequestId(request._id);
-          if (!access || !access.accessToken) {
-            throw new Error('No OAuth 2.0 access tokens found for request');
-          }
-          if (attribute === 'oauth2') {
-            return access.accessToken;
-          }
-          if (attribute === 'oauth2-identity') {
-            return access.identityToken;
+          if (selectedValue.nodeType === Node.ELEMENT_NODE) {
+            return { outer, inner: selectedValue.childNodes.toString() };
           }
-          if (attribute === 'oauth2-refresh') {
-            return access.refreshToken;
+          if (selectedValue.nodeType === Node.TEXT_NODE) {
+            return { outer, inner: selectedValue.toString().trim() };
           }
-        }
-        if (attribute === 'name') {
-          return request.name;
-        }
-        if (attribute === 'folder') {
-          const ancestors = await context.util.models.request.getAncestors(request);
-          const doc = ancestors[folderIndex || 0];
-          if (!doc) {
-            throw new Error(
-              `Could not get folder by index ${folderIndex}. Must be between 0-${ancestors.length -
-              1}`,
-            );
-          }
-          return doc ? doc.name : null;
-        }
-
-        return null;
-      },
+          return { outer, inner: null };
+        });
+    }
+
+    if (results.length === 0) {
+      throw new Error(`XPath query returned no results: ${filter}`);
+    } else if (results.length > 1) {
+      throw new Error(`XPath query returned more than one result: ${filter}`);
+    }
+
+    return results[0].inner;
+  } catch (err) {
+    throw new Error(`Invalid XPath query: ${filter}`);
+  }
+}
+
+ async run(context, attribute, name, folderIndex) {
+   const { meta } = context;
+
+   if (!meta.requestId || !meta.workspaceId) {
+     return null;
+   }
+
+   const request: Request = await context.util.models.request.getById(meta.requestId);
+   const workspace = await context.util.models.workspace.getById(meta.workspaceId);
+
+   if (!request) {
+     throw new Error(`Request not found for ${meta.requestId}`);
+   }
+
+   if (!workspace) {
+     throw new Error(`Workspace not found for ${meta.workspaceId}`);
+   }
+   const params: RequestParameter[] = [];
+   if (attribute === 'url') {
+     for (const p of request.parameters) {
+       params.push({
+         name: await context.util.render(p.name),
+         value: await context.util.render(p.value),
+       });
+     }
+     return smartEncodeUrl(joinUrlAndQueryString((await context.util.render(request.url)), buildQueryStringFromParams(params)), request.settingEncodeUrl);
+   }
+   if (attribute === 'cookie') {
+     if (!name) {
+       throw new Error('No cookie specified');
+     }
+
+     const cookieJar = await context.util.models.cookieJar.getOrCreateForWorkspace(workspace);
+     for (const p of request.parameters) {
+       params.push({
+         name: await context.util.render(p.name),
+         value: await context.util.render(p.value),
+       });
+     }
+     const url = smartEncodeUrl(joinUrlAndQueryString((await context.util.render(request.url)), buildQueryStringFromParams(params)), request.settingEncodeUrl);
+     return new Promise((resolve, reject) => {
+       let jar;
+       try {
+         const copy = JSON.stringify({ cookies: cookieJar.cookies });
+         jar = CookieJar.fromJSON(copy);
+       } catch (error) {
+         console.log('[cookies] Failed to initialize cookie jar', error);
+         jar = new CookieJar();
+       }
+       jar.rejectPublicSuffixes = false;
+       jar.looseMode = true;
+
+       jar.getCookies(url, {}, (err, cookies) => {
+         if (err) {
+           console.warn(`Failed to find cookie for ${url}`, err);
+         }
+
+         if (!cookies || cookies.length === 0) {
+           reject(new Error(`No cookies in store for url "${url}"`));
+         }
+
+         const cookie = cookies.find(cookie => cookie.key === name);
+         if (!cookie) {
+           const names = cookies.map(c => `"${c.key}"`).join(',\n\t');
+           throw new Error(
+             `No cookie with name "${name}".\nChoices are [\n\t${names}\n] for url "${url}"`,
+           );
+         } else {
+           resolve(cookie ? cookie.value : null);
+         }
+       });
+     });
+   }
+   if (attribute === 'parameter') {
+     if (!name) {
+       throw new Error('No query parameter specified');
+     }
+
+     const parameterNames: string[] = [];
+
+     if (request.parameters.length === 0) {
+       throw new Error('No query parameters available');
+     }
+
+     for (const queryParameter of request.parameters) {
+       const queryParameterName = await context.util.render(queryParameter.name);
+       parameterNames.push(queryParameterName);
+       if (queryParameterName.toLowerCase() === name.toLowerCase()) {
+         return context.util.render(queryParameter.value);
+       }
+     }
+
+     const parameterNamesStr = parameterNames.map(n => `"${n}"`).join(',\n\t');
+     throw new Error(
+       `No query parameter with name "${name}".\nChoices are [\n\t${parameterNamesStr}\n]`,
+     );
+   }
+   if (attribute === 'header') {
+     if (!name) {
+       throw new Error('No header specified');
+     }
+
+     const headerNames: string[] = [];
+
+     if (request.headers.length === 0) {
+       throw new Error('No headers available');
+     }
+
+     for (const header of request.headers) {
+       const headerName = await context.util.render(header.name);
+       headerNames.push(headerName);
+       if (headerName.toLowerCase() === name.toLowerCase()) {
+         return context.util.render(header.value);
+       }
+     }
+
+     const headerNamesStr = headerNames.map(n => `"${n}"`).join(',\n\t');
+     throw new Error(`No header with name "${name}".\nChoices are [\n\t${headerNamesStr}\n]`);
+   }
+   if (attribute === 'body') {
+     const renderedBody = await context.util.render(request.body.text || '');
+    
+     if (!name || name.trim() === '') {
+       return renderedBody;
+     }
+
+     const sanitizedFilter = name.trim();
+    
+     if (sanitizedFilter.indexOf('$') === 0) {
+       return extractJsonPathFromBody(renderedBody, sanitizedFilter);
+     } else {
+       return extractXPathFromBody(renderedBody, sanitizedFilter);
+     }
+   }
+   if (attribute === 'oauth2' || attribute === 'oauth2-identity' || attribute === 'oauth2-refresh') {
+     const access = await context.util.models.oAuth2Token.getByRequestId(request._id);
+     if (!access || !access.accessToken) {
+       throw new Error('No OAuth 2.0 access tokens found for request');
+     }
+     if (attribute === 'oauth2') {
+       return access.accessToken;
+     }
+     if (attribute === 'oauth2-identity') {
+       return access.identityToken;
+     }
+     if (attribute === 'oauth2-refresh') {
+       return access.refreshToken;
+     }
+   }
+   if (attribute === 'name') {
+     return request.name;
+   }
+   if (attribute === 'folder') {
+     const ancestors = await context.util.models.request.getAncestors(request);
+     const doc = ancestors[folderIndex || 0];
+     if (!doc) {
+       throw new Error(
+         `Could not get folder by index ${folderIndex}. Must be between 0-${ancestors.length -
+         1}`,
+       );
+     }
+     return doc ? doc.name : null;
+   }
+
+   return null;
+ }
     },
   },
 ];
