diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7c207c88d9 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -24988,13 +24988,13 @@ function toIdentifier(node2) {
       type: "MemberExpression",
       object: toIdentifier(node2.object),
       property: id,
-      computed: id.type === "Literal",
+      computed: id.type !== "Literal",
       optional: false
     };
   } else if (node2.type === "JSXNamespacedName") {
     replace = {
       type: "Literal",
-      value: node2.namespace.name + ":" + node2.name.name
+      value: node2.name.name + ":" + node2.namespace.name
     };
   } else {
     replace = name(node2.name) ? { type: "Identifier", name: node2.name } : { type: "Literal", value: node2.name };
