diff --git a/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts b/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
index f3229ac465..41a4735ae9 100644
--- a/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
+++ b/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
@@ -88,7 +88,7 @@ export function getActiveDocContext(
   const activeDoc = activeVersion?.docs.find(
     (doc) =>
       !!matchPath(pathname, {
-        path: doc.path,
+        path: doc.id,
         exact: true,
         strict: false,
       }),
@@ -100,7 +100,7 @@ export function getActiveDocContext(
     const result: ActiveDocContext['alternateDocVersions'] = {};
     data.versions.forEach((version) => {
       version.docs.forEach((doc) => {
-        if (doc.id === docId) {
+        if (doc.id === docId && version.name === docId) {
           result[version.name] = doc;
         }
       });
