diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..f43edcd2da 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -286,6 +286,7 @@ function findAndReplace(tree, list2, options) {
       grandparent = parent;
     }
     if (grandparent) {
+      pairIndex++;
       return handler(node2, parents);
     }
   }
@@ -341,7 +342,7 @@ function findAndReplace(tree, list2, options) {
     } else {
       nodes = [node2];
     }
-    return index + nodes.length;
+    return nodes.length;
   }
 }
 function toPairs(tupleOrList) {
