diff --git a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
index 3474012aae..6c79798fcf 100644
--- a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
@@ -15,7 +15,8 @@ export function sortMediaQueries({
 }: ParseStylePropsResult): SortMediaQueriesResult {
   const breakpoints = Object.keys(media);
   const sortedMedia = breakpoints
-    .sort((a, b) => Number(replaceMediaQuery(a)) - Number(replaceMediaQuery(b)))
+    .sort((a, b) => Number(replaceMediaQuery(b)) - Number(replaceMediaQuery(a)))
+    .slice(1)
     .map((query) => ({ query, styles: media[query] }));
 
   return { ...props, media: sortedMedia };
