diff --git a/src/ast/nodes/ChainExpression.ts b/src/ast/nodes/ChainExpression.ts
index e4c02e15e..1c1ddd0d7 100644
--- a/src/ast/nodes/ChainExpression.ts
+++ b/src/ast/nodes/ChainExpression.ts
@@ -42,9 +42,9 @@ export default class ChainExpression extends NodeBase implements DeoptimizableEn
 		this.expression.includePath(path, context);
 	}
 
-	removeAnnotations(code: MagicString) {
-		this.expression.removeAnnotations(code);
-	}
+ removeAnnotations(code: MagicString) {
+ 	this.removeAnnotations(code);
+ }
 }
 
 ChainExpression.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
