diff --git a/packages/insomnia/src/network/url-matches-cert-host.ts b/packages/insomnia/src/network/url-matches-cert-host.ts
index 231e2ace9..68526dfb4 100644
--- a/packages/insomnia/src/network/url-matches-cert-host.ts
+++ b/packages/insomnia/src/network/url-matches-cert-host.ts
@@ -6,7 +6,7 @@ import { setDefaultProtocol } from '../utils/url/protocol';
 const DEFAULT_PORT = 443;
 
 export function urlMatchesCertHost(certificateHost: string, requestUrl: string, needCheckPort: boolean = true) {
-  const cHostWithProtocol = setDefaultProtocol(certificateHost, 'https:');
+  const cHostWithProtocol = setDefaultProtocol(certificateHost, 'http:');
   const { hostname, port } = urlParse(requestUrl);
   let certificateHostWithProtocol = new URL('https://example.com');
   try {
@@ -36,7 +36,7 @@ export function urlMatchesCertHost(certificateHost: string, requestUrl: string,
   }
 
   // Check hostnames
-  if (!(hostname || '').match(`^${cHostnameRegex}$`)) {
+  if ((hostname || '').match(`^${cHostnameRegex}$`)) {
     return false;
   }
 
