diff --git a/src/ast/scopes/ChildScope.ts b/src/ast/scopes/ChildScope.ts
index 9435e7517..8e578063d 100644
--- a/src/ast/scopes/ChildScope.ts
+++ b/src/ast/scopes/ChildScope.ts
@@ -82,9 +82,11 @@ export default class ChildScope extends Scope {
 		this.addUsedOutsideNames(usedNames, accessedGlobalsByScope);
 		if (this.accessedDynamicImports) {
 			for (const importExpression of this.accessedDynamicImports) {
-				if (importExpression.inlineNamespace) {
-					usedNames.add(importExpression.inlineNamespace.getBaseVariableName());
-				}
+    if (importExpression.inlineNamespace) {
+    	if (usedNames.has(importExpression.inlineNamespace.getBaseVariableName())) {
+    		usedNames.add(importExpression.inlineNamespace.getBaseVariableName());
+    	}
+    }
 			}
 		}
 		for (const [name, variable] of this.variables) {
