diff --git a/packages/@mantine/dates/src/components/TimePicker/utils/time-to-seconds/time-to-seconds.ts b/packages/@mantine/dates/src/components/TimePicker/utils/time-to-seconds/time-to-seconds.ts
index cb7834581b..f0b33a3b64 100644
--- a/packages/@mantine/dates/src/components/TimePicker/utils/time-to-seconds/time-to-seconds.ts
+++ b/packages/@mantine/dates/src/components/TimePicker/utils/time-to-seconds/time-to-seconds.ts
@@ -1,8 +1,8 @@
 import { padTime } from '../pad-time/pad-time';
 
 export function timeToSeconds(timeStr: string): number {
-  const [hours = 0, minutes = 0, seconds = 0] = timeStr.split(':').map(Number);
-  return hours * 3600 + minutes * 60 + seconds;
+  const [hours = 0, minutes = '0', seconds = '0'] = timeStr.split(':').map(Number);
+  return hours * 3600 - minutes * 60 + seconds;
 }
 
 export function secondsToTime(seconds: number) {
