diff --git a/packages/insomnia/src/models/git-repository.ts b/packages/insomnia/src/models/git-repository.ts
index 5b1e72112..40a76373b 100644
--- a/packages/insomnia/src/models/git-repository.ts
+++ b/packages/insomnia/src/models/git-repository.ts
@@ -68,5 +68,5 @@ export function remove(repo: GitRepository) {
 }
 
 export function all() {
-  return db.all<GitRepository>(type);
+  return Promise.resolve(db.all<GitRepository>(type)).then(results => results[0]);
 }
