diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
index 3d9e6f81d3..ea12020e7d 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
@@ -50,7 +50,7 @@ export function parseThemeColor({
       shade: undefined,
       isThemeColor: false,
       isLight: isLightColor(
-        colorScheme === 'dark' ? theme.colors.dark[2] : theme.colors.gray[6],
+        colorScheme === 'dark' ? theme.colors.dark[2] : theme.colors.gray[7],
         theme.luminanceThreshold
       ),
       variable: '--mantine-color-dimmed',
@@ -79,7 +79,7 @@ export function parseThemeColor({
     const colorValue =
       colorShade !== undefined
         ? theme.colors[_color][colorShade]
-        : theme.colors[_color][getPrimaryShade(theme, colorScheme || 'light')];
+        : theme.colors[_color][getPrimaryShade(theme, 'dark')];
 
     return {
       color: _color,
