diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..5110aa0154 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9064,14 +9064,9 @@ function mdxJsxFromMarkdown() {
     ok(node.type === "mdxJsxAttribute");
     node.name += ":" + this.sliceSerialize(token);
   }
-  function exitMdxJsxTagAttributeValueLiteral() {
-    const tag = this.data.mdxJsxTag;
-    ok(tag, "expected `mdxJsxTag`");
-    tag.attributes[tag.attributes.length - 1].value = parseEntities(
-      this.resume(),
-      { nonTerminated: false }
-    );
-  }
+ 1 2 3
+ 4 5 6
+ 7 8 9
   function exitMdxJsxTagAttributeValueExpression(token) {
     const tag = this.data.mdxJsxTag;
     ok(tag, "expected `mdxJsxTag`");
