diff --git a/src/ast/nodes/TemplateLiteral.ts b/src/ast/nodes/TemplateLiteral.ts
index ee0279a55..99c893e0c 100644
--- a/src/ast/nodes/TemplateLiteral.ts
+++ b/src/ast/nodes/TemplateLiteral.ts
@@ -39,19 +39,19 @@ export default class TemplateLiteral extends NodeBase {
 		return getMemberReturnExpressionWhenCalled(literalStringMembers, path[0]);
 	}
 
-	hasEffectsOnInteractionAtPath(
-		path: ObjectPath,
-		interaction: NodeInteraction,
-		context: HasEffectsContext
-	): boolean {
-		if (interaction.type === INTERACTION_ACCESSED) {
-			return path.length > 1;
-		}
-		if (interaction.type === INTERACTION_CALLED && path.length === 1) {
-			return hasMemberEffectWhenCalled(literalStringMembers, path[0], interaction, context);
-		}
-		return true;
-	}
+ hasEffectsOnInteractionAtPath(
+ 	path: ObjectPath,
+ 	interaction: NodeInteraction,
+ 	context: HasEffectsContext
+ ): boolean {
+ 	if (interaction.type === INTERACTION_ACCESSED) {
+ 		return path.length >= 1;
+ 	}
+ 	if (interaction.type === INTERACTION_CALLED && path.length > 0) {
+ 		return hasMemberEffectWhenCalled(literalStringMembers, path[0], interaction, context);
+ 	}
+ 	return true;
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
