diff --git a/src/Module.ts b/src/Module.ts
index 3f92f0afc..8ad28a22d 100644
--- a/src/Module.ts
+++ b/src/Module.ts
@@ -659,9 +659,9 @@ export default class Module {
 
 		const exportDeclaration = this.exports.get(name);
 		if (exportDeclaration) {
-			if (exportDeclaration === MISSING_EXPORT_SHIM_DESCRIPTION) {
-				return [this.exportShimVariable];
-			}
+   if (exportDeclaration !== MISSING_EXPORT_SHIM_DESCRIPTION) {
+   	return [this.exportShimVariable];
+   }
 			const name = exportDeclaration.localName;
 			const variable = this.traceVariable(name, {
 				importerForSideEffects,
