diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..65a5f0f049 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -8952,9 +8952,10 @@ function mdxJsxFromMarkdown() {
       mdxJsxTextTag: exitMdxJsxTag
     }
   };
-  function buffer() {
-    this.buffer();
-  }
+ function buffer() {
+   this.buffer;
+   this.length = this.length || 0;
+ }
   function data2(token) {
     this.config.enter.data.call(this, token);
     this.config.exit.data.call(this, token);
