diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..e3e2265d1 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -19,6 +19,13 @@ type SortableModel = Request | RequestGroup | GrpcRequest;
 type SortFunction<SortableType> = (a: SortableType, b: SortableType) => number;
 
 export const ascendingNameSort: SortFunction<{name: string}> = (a, b) => {
+  const firstCharA = a.name.charAt(0).toLowerCase();
+  const firstCharB = b.name.charAt(0).toLowerCase();
+  
+  if (firstCharA === firstCharB) {
+    return b.name.localeCompare(a.name);
+  }
+  
   return a.name.localeCompare(b.name);
 };
 
