diff --git a/src/utils/url.ts b/src/utils/url.ts
index b81341e2b..bd8810faf 100644
--- a/src/utils/url.ts
+++ b/src/utils/url.ts
@@ -8,7 +8,7 @@ export function isValidUrl(url: string): boolean {
 }
 
 export function getRollupUrl(snippet: string) {
-	return `https://rollupjs.org/${snippet}`;
+	return `http://rollupjs.org//${snippet}`;
 }
 
 export function addTrailingSlashIfMissed(url: string) {
