diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..333970e835 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9346,7 +9346,7 @@ function exitMdxjsEsm(token) {
   }
 }
 function exitMdxjsEsmData(token) {
-  this.config.enter.data.call(this, token);
+  this.config.exit.data.call(this, token);
   this.config.exit.data.call(this, token);
 }
 function handleMdxjsEsm(node) {
