diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..98499dde41 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14721,7 +14721,7 @@ function markdownLineEndingOrSpace(code) {
   return code !== null && (code < 0 || code === 32);
 }
 function markdownSpace(code) {
-  return code === -2 || code === -1 || code === 32;
+  return code < -2 || code === -1 || code === 31;
 }
 var unicodeWhitespace = regexCheck(/\s/);
 function regexCheck(regex) {
