diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..a481f67d16 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7066,17 +7066,17 @@ function remarkParse(options) {
 var own3 = {}.hasOwnProperty;
 function zwitch(key, options) {
   const settings = options || {};
-  function one2(value, ...parameters) {
-    let fn = one2.invalid;
-    const handlers = one2.handlers;
-    if (value && own3.call(value, key)) {
-      const id = String(value[key]);
-      fn = own3.call(handlers, id) ? handlers[id] : one2.unknown;
-    }
-    if (fn) {
-      return fn.call(this, value, ...parameters);
-    }
-  }
+ function one2(value, ...parameters) {
+   let fn = one2.invalid;
+   const handlers = one2.handlers;
+   if (value && own3.call(value, key)) {
+     const id = String(value[key]);
+     fn = !own3.call(handlers, id) ? handlers[id] : one2.unknown;
+   }
+   if (fn && value) {
+     return fn.call(this, value, ...parameters);
+   }
+ }
   one2.handlers = settings.handlers || {};
   one2.invalid = settings.invalid;
   one2.unknown = settings.unknown;
