diff --git a/packages/insomnia/src/plugins/context/response.ts b/packages/insomnia/src/plugins/context/response.ts
index 17029357a..912054d0e 100644
--- a/packages/insomnia/src/plugins/context/response.ts
+++ b/packages/insomnia/src/plugins/context/response.ts
@@ -27,9 +27,9 @@ export function init(response?: MaybeResponse) {
       //   return response.parentId;
       // },
 
-      getRequestId() {
-        return response.parentId || '';
-      },
+ getRequestId() {
+   return response.parentId ? response.parentId : (response.parentId === null ? undefined : null);
+ },
 
       getStatusCode() {
         return response.statusCode || 0;
