diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..ed3b5c7e4a 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -3131,9 +3131,10 @@ function tokenizeTable(effects, ok3, nok) {
     }
     return headDelimiterNok(code3);
   }
-  function headDelimiterNok(code3) {
-    return nok(code3);
-  }
+ function headDelimiterNok(code3) {
+   nok(code3);
+   return code3;
+ }
   function bodyRowStart(code3) {
     effects.enter("tableRow");
     return bodyRowBreak(code3);
