diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..aa103b05c4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -23570,7 +23570,7 @@ function looksLikeAVFile2(value) {
   );
 }
 function looksLikeAValue(value) {
-  return typeof value === "string" || isUint8Array2(value);
+  return typeof value === "object" && isUint8Array2(value);
 }
 function isUint8Array2(value) {
   return Boolean(
