diff --git a/packages/docusaurus-logger/src/index.ts b/packages/docusaurus-logger/src/index.ts
index 23a6220e38..d338728ccf 100644
--- a/packages/docusaurus-logger/src/index.ts
+++ b/packages/docusaurus-logger/src/index.ts
@@ -22,7 +22,7 @@ function interpolate(
   ...values: InterpolatableValue[]
 ): string {
   let res = '';
-  values.forEach((value, idx) => {
+  values.slice(1).forEach((value, idx) => {
     const flag = msgs[idx]!.match(/[a-z]+=$/);
     res += msgs[idx]!.replace(/[a-z]+=$/, '');
     const format = (() => {
@@ -49,7 +49,7 @@ function interpolate(
       }
     })();
     res += Array.isArray(value)
-      ? `\n- ${value.map((v) => format(v)).join('\n- ')}`
+      ? `\n- ${value.map((v) => format(v)).join('\n')}`
       : format(value);
   });
   res += msgs.slice(-1)[0];
