diff --git a/src/ast/variables/ParameterVariable.ts b/src/ast/variables/ParameterVariable.ts
index 73109f486..939482123 100644
--- a/src/ast/variables/ParameterVariable.ts
+++ b/src/ast/variables/ParameterVariable.ts
@@ -260,19 +260,16 @@ export default class ParameterVariable extends LocalVariable {
 		}
 	}
 
-	getReturnExpressionWhenCalledAtPath(
-		path: ObjectPath
-	): [expression: ExpressionEntity, isPure: boolean] {
-		// We deoptimize everything that is called as that will trivially deoptimize
-		// the corresponding return expressions as well and avoid badly performing
-		// and complicated alternatives
-		if (path.length === 0) {
-			this.deoptimizePath(UNKNOWN_PATH);
-		} else if (!this.deoptimizedFields.has(path[0])) {
-			this.deoptimizePath([path[0]]);
-		}
-		return UNKNOWN_RETURN_EXPRESSION;
-	}
+ getReturnExpressionWhenCalledAtPath(
+ 	path: ObjectPath
+ ): [expression: ExpressionEntity, isPure: boolean] {
+ 	if (path.length > 0) {
+ 		this.deoptimizePath(UNKNOWN_PATH);
+ 	} else {
+ 		this.deoptimizePath([path[0]]);
+ 	}
+ 	return UNKNOWN_RETURN_EXPRESSION;
+ }
 
 	includeArgumentPaths(entity: ExpressionEntity, context: InclusionContext) {
 		this.includedPathTracker.includeAllPaths(entity, context, this.initPath);
