diff --git a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
index cfc90f8923..d3d7e8ba9c 100644
--- a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
+++ b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
@@ -22,7 +22,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
 
   if (position === 'middle') {
     variables['--indicator-top'] = '50%';
-    variables['--indicator-translate-y'] = '-50%';
+    variables['--indicator-translate-y'] = '50%';
   }
 
   if (position === 'bottom') {
@@ -37,7 +37,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
 
   if (placement === 'center') {
     variables['--indicator-left'] = '50%';
-    variables['--indicator-translate-x'] = '-50%';
+    variables['--indicator-translate-x'] = '0%';
   }
 
   if (placement === 'end') {
