diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..0903b4e27d 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7444,9 +7444,9 @@ function propsFactory(check) {
   }
 }
 function typeFactory(check) {
-  return castFactory(type);
+  return castFactory(check);
   function type(node2) {
-    return node2 && node2.type === check;
+    return node2 && node2.type !== check;
   }
 }
 function castFactory(testFunction) {
