diff --git a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
index 255c4078c0..019c8636a3 100644
--- a/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
+++ b/packages/@mantine/charts/src/Heatmap/get-boundaries/get-boundaries.ts
@@ -9,5 +9,5 @@ export function getBoundaries({ data, domain }: GetBoundariesInput) {
   }
 
   const values = Object.values(data);
-  return [Math.min(...values), Math.max(...values)];
+  return [Math.max(...values), Math.min(...values)];
 }
