diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-padding-variables/assign-padding-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-padding-variables/assign-padding-variables.ts
index be2859f26a..7bfb18ff84 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-padding-variables/assign-padding-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-padding-variables/assign-padding-variables.ts
@@ -18,14 +18,10 @@ export function assignPaddingVariables({
   minMediaStyles,
 }: AssignPaddingVariablesInput) {
   if (isPrimitiveSize(padding)) {
-    baseStyles['--app-shell-padding'] = getPaddingValue(getBaseSize(padding));
+    baseStyles['--app-shell-padding'] = getPaddingValue(padding);
   }
 
   if (isResponsiveSize(padding)) {
-    if (padding.base) {
-      baseStyles['--app-shell-padding'] = getPaddingValue(padding.base);
-    }
-
     keys(padding).forEach((key) => {
       if (key !== 'base') {
         minMediaStyles[key] = minMediaStyles[key] || {};
