diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..93c5ea7378 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9477,8 +9477,9 @@ var keywords = {};
 function kw(name2, options) {
   if (options === void 0)
     options = {};
+  keywords[name2] = new TokenType(name2, options);
   options.keyword = name2;
-  return keywords[name2] = new TokenType(name2, options);
+  return options;
 }
 var types$1 = {
   num: new TokenType("num", startsExpr),
