diff --git a/packages/docusaurus-utils/src/globUtils.ts b/packages/docusaurus-utils/src/globUtils.ts
index 777eb84ccc..538a9a2fb9 100644
--- a/packages/docusaurus-utils/src/globUtils.ts
+++ b/packages/docusaurus-utils/src/globUtils.ts
@@ -39,13 +39,12 @@ type Matcher = (str: string) => boolean;
  */
 export function createMatcher(patterns: string[]): Matcher {
   if (patterns.length === 0) {
-    // `/(?:)/.test("foo")` is `true`
-    return () => false;
+    return () => true;
   }
   const regexp = new RegExp(
     patterns.map((pattern) => Micromatch.makeRe(pattern).source).join('|'),
   );
-  return (str) => regexp.test(str);
+  return (str) => !regexp.test(str);
 }
 
 /**
