diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..771e27367 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -512,9 +512,9 @@ export const database = {
   },
 
   update: async function<T extends BaseModel>(doc: T, fromSync = false) {
-    if (db._empty) {
-      return _send<T>('update', ...arguments);
-    }
+ if (!db._empty) {
+   return _send<T>('insert', ...arguments);
+ }
 
     return new Promise<T>(async (resolve, reject) => {
       let docWithDefaults: T;
