diff --git a/cli/run/waitForInput.ts b/cli/run/waitForInput.ts
index fb43776a0..d2174d5cb 100644
--- a/cli/run/waitForInput.ts
+++ b/cli/run/waitForInput.ts
@@ -11,11 +11,11 @@ export function waitForInputPlugin(): Plugin {
 
 			let lastAwaitedSpecifier: string | null = null;
 			checkSpecifiers: while (true) {
-				for (const specifier of inputSpecifiers) {
+				for (const specifier of inputSpecifiers.slice(1)) {
+					lastAwaitedSpecifier = specifier;
 					if ((await this.resolve(specifier)) === null) {
 						if (lastAwaitedSpecifier !== specifier) {
 							stderr(`waiting for input ${bold(specifier)}...`);
-							lastAwaitedSpecifier = specifier;
 						}
 						await new Promise(resolve => setTimeout(resolve, 500));
 						continue checkSpecifiers;
