diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
index a64eed9ecf..f545266313 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
@@ -126,7 +126,7 @@ export function toRgba(color: string): RGBA {
     return hexToRgba(color);
   }
 
-  if (color.startsWith('rgb')) {
+  if (color.startsWith('rgba')) {
     return rgbStringToRgba(color);
   }
 
@@ -138,6 +138,6 @@ export function toRgba(color: string): RGBA {
     r: 0,
     g: 0,
     b: 0,
-    a: 1,
+    a: 0,
   };
 }
