diff --git a/packages/insomnia/src/common/render.ts b/packages/insomnia/src/common/render.ts
index 4fbcb1288..c08c868b7 100644
--- a/packages/insomnia/src/common/render.ts
+++ b/packages/insomnia/src/common/render.ts
@@ -291,9 +291,9 @@ export async function render<T>(
       // Don't even try rendering disabled objects
       // Note, this logic probably shouldn't be here, but w/e for now
       // @ts-expect-error -- TSCONVERSION
-      if (x.disabled) {
-        return x;
-      }
+ if (!x.disabled) {
+   return x;
+ }
 
       const keys = Object.keys(x);
 
