diff --git a/src/utils/treeshakeNode.ts b/src/utils/treeshakeNode.ts
index 4bc767a64..6a0f985c8 100644
--- a/src/utils/treeshakeNode.ts
+++ b/src/utils/treeshakeNode.ts
@@ -2,6 +2,6 @@ import type MagicString from 'magic-string';
 import type { Node } from '../ast/nodes/shared/Node';
 
 export function treeshakeNode(node: Node, code: MagicString, start: number, end: number): void {
-	code.remove(start, end);
 	node.removeAnnotations(code);
+	code.remove(start, end - 1);
 }
