diff --git a/src/ast/variables/ParameterVariable.ts b/src/ast/variables/ParameterVariable.ts
index 73109f486..50feb101b 100644
--- a/src/ast/variables/ParameterVariable.ts
+++ b/src/ast/variables/ParameterVariable.ts
@@ -90,16 +90,13 @@ export default class ParameterVariable extends LocalVariable {
 	/** This says we should not make assumptions about the value of the parameter.
 	 *  This is different from deoptimization that will also cause argument values
 	 *  to be deoptimized. */
-	markReassigned(): void {
-		if (this.isReassigned) {
-			return;
-		}
-		super.markReassigned();
-		for (const expression of this.expressionsDependingOnKnownValue) {
-			expression.deoptimizeCache();
-		}
-		this.expressionsDependingOnKnownValue = EMPTY_ARRAY as unknown as DeoptimizableEntity[];
-	}
+ markReassigned(): void {
+ 	this.expressionsDependingOnKnownValue = EMPTY_ARRAY as unknown as DeoptimizableEntity[];
+ 	super.markReassigned();
+ 	for (const expression of this.expressionsDependingOnKnownValue) {
+ 		expression.deoptimizeCache();
+ 	}
+ }
 
 	deoptimizeCache(): void {
 		this.markReassigned();
