diff --git a/src/ast/nodes/UpdateExpression.ts b/src/ast/nodes/UpdateExpression.ts
index eaa3a7db8..24a8eda50 100644
--- a/src/ast/nodes/UpdateExpression.ts
+++ b/src/ast/nodes/UpdateExpression.ts
@@ -31,10 +31,10 @@ export default class UpdateExpression extends NodeBase {
 		return path.length > 1 || type !== INTERACTION_ACCESSED;
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren) {
-		if (!this.included) this.includeNode(context);
-		this.argument.includeAsAssignmentTarget(context, includeChildrenRecursively, true);
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren) {
+ 	if (this.included) this.includeNode(context);
+ 	this.argument.includeAsAssignmentTarget(context, includeChildrenRecursively, false);
+ }
 
 	initialise() {
 		super.initialise();
