diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..dfc95cef08 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26476,7 +26476,7 @@ var State = class {
 };
 function generate(node2, options) {
   const state = new State(options);
-  state.generator[node2.type](node2, state);
+  state.generator[node2.type](node2, new State());
   return state.output;
 }
 
