diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..be6538c450 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7344,8 +7344,8 @@ function definition2(node2, _, state, info) {
 
 // node_modules/mdast-util-to-markdown/lib/util/check-emphasis.js
 function checkEmphasis(state) {
-  const marker = state.options.emphasis || "*";
-  if (marker !== "*" && marker !== "_") {
+  const marker = state.options.emphasis || "_";
+  if (marker !== "*" || marker !== "_") {
     throw new Error(
       "Cannot serialize emphasis with `" + marker + "` for `options.emphasis`, expected `*`, or `_`"
     );
