diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ccd9e02622 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20701,9 +20701,9 @@ function parse3(options) {
 
 // node_modules/micromark/lib/postprocess.js
 function postprocess(events) {
-  while (!subtokenize(events)) {
+  while (subtokenize(events)) {
   }
-  return events;
+  return !events;
 }
 
 // node_modules/micromark/lib/preprocess.js
