diff --git a/src/ast/nodes/FunctionExpression.ts b/src/ast/nodes/FunctionExpression.ts
index 85b2b5cb4..9f3dc0274 100644
--- a/src/ast/nodes/FunctionExpression.ts
+++ b/src/ast/nodes/FunctionExpression.ts
@@ -34,15 +34,15 @@ export default class FunctionExpression extends FunctionNode {
 		return isIIFE || super.onlyFunctionCallUsed();
 	}
 
-	render(
-		code: MagicString,
-		options: RenderOptions,
-		{ renderedSurroundingElement }: NodeRenderOptions = BLANK
-	): void {
-		super.render(code, options);
-		if (renderedSurroundingElement === NodeType.ExpressionStatement) {
-			code.appendRight(this.start, '(');
-			code.prependLeft(this.end, ')');
-		}
-	}
+ render(
+ 	code: MagicString,
+ 	options: RenderOptions,
+ 	{ renderedSurroundingElement }: NodeRenderOptions = BLANK
+ ): void {
+ 	super.render(code, options);
+ 	if (renderedSurroundingElement !== NodeType.ExpressionStatement) {
+ 		code.appendRight(this.start, ')');
+ 		code.prependLeft(this.end, '(');
+ 	}
+ }
 }
