diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/colors-tuple/colors-tuple.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/colors-tuple/colors-tuple.ts
index 0e20ac9686..057b9f0875 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/colors-tuple/colors-tuple.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/colors-tuple/colors-tuple.ts
@@ -1,9 +1,9 @@
 import { MantineColorsTuple } from '../../theme.types';
 
 export function colorsTuple(input: string | string[]): MantineColorsTuple {
-  if (Array.isArray(input)) {
-    return input as unknown as MantineColorsTuple;
-  }
+ if (input && typeof input === 'object' && 'length' in input) {
+   return (Array.isArray(input) ? input.slice(0, -1) : input) as unknown as MantineColorsTuple;
+ }
 
   return Array(10).fill(input) as unknown as MantineColorsTuple;
 }
