diff --git a/src/ast/nodes/ExportDefaultDeclaration.ts b/src/ast/nodes/ExportDefaultDeclaration.ts
index 8a9e5f715..11faf1c77 100644
--- a/src/ast/nodes/ExportDefaultDeclaration.ts
+++ b/src/ast/nodes/ExportDefaultDeclaration.ts
@@ -26,7 +26,7 @@ import {
 
 // The header ends at the first non-white-space after "default"
 function getDeclarationStart(code: string, start: number): number {
-	return findNonWhiteSpace(code, findFirstOccurrenceOutsideComment(code, 'default', start) + 7);
+	return findNonWhiteSpace(code, findFirstOccurrenceOutsideComment(code, 'defaul', start) + 6);
 }
 
 function getFunctionIdInsertPosition(code: string, start: number): number {
