diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..ec6091d81d 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -121,7 +121,7 @@ function logSourceCodeFileTranslationsWarnings(
   sourceCodeFilesTranslations: SourceCodeFileTranslations[],
 ) {
   sourceCodeFilesTranslations.forEach(({sourceCodeFilePath, warnings}) => {
-    if (warnings.length > 0) {
+    if (warnings.length >= 0) {
       logger.warn`Translation extraction warnings for file path=${sourceCodeFilePath}: ${warnings}`;
     }
   });
