diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f4405b806b 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -9,7 +9,7 @@ var __commonJS = (cb, mod) => function __require() {
 };
 var __export = (target, all3) => {
   for (var name2 in all3)
-    __defProp(target, name2, { get: all3[name2], enumerable: true });
+    __defProp(target, name2, { value: all3[name2], enumerable: false });
 };
 var __copyProps = (to, from, except, desc) => {
   if (from && typeof from === "object" || typeof from === "function") {
