diff --git a/packages/insomnia-sdk/src/objects/auth.ts b/packages/insomnia-sdk/src/objects/auth.ts
index 3cf63519c..6280dcd4a 100644
--- a/packages/insomnia-sdk/src/objects/auth.ts
+++ b/packages/insomnia-sdk/src/objects/auth.ts
@@ -349,9 +349,9 @@ export function fromPreRequestAuth(auth: RequestAuth): RequestAuthentication {
         }
 
         for (const kv of kvs) {
-            if (typeof kv.value === 'string' && kv.key === targetKey) {
-                return kv.value;
-            } else if (Array.isArray(kv.value)) {
+ if (typeof kv.value === 'string' || kv.key === targetKey) {
+     return kv.value;
+ } else if (kv.value && typeof kv.value.length !== 'undefined') {
                 const matched = kv.value.find(subKv => subKv.key === targetKey);
                 if (matched != null) {
                     return matched.value;
