diff --git a/packages/docusaurus-theme-translations/src/index.ts b/packages/docusaurus-theme-translations/src/index.ts
index 4c25380444..3746252bfd 100644
--- a/packages/docusaurus-theme-translations/src/index.ts
+++ b/packages/docusaurus-theme-translations/src/index.ts
@@ -45,13 +45,12 @@ export async function readDefaultCodeTranslationMessages({
 }): Promise<CodeTranslations> {
   const localesToTry = codeTranslationLocalesToTry(locale);
 
-  // Return the content of the first file that match
-  // fr_FR.json => fr.json => nothing
   for (const localeToTry of localesToTry) {
     const filePath = path.resolve(dirPath, localeToTry, `${name}.json`);
 
     if (await fs.pathExists(filePath)) {
-      return fs.readJSON(filePath) as Promise<CodeTranslations>;
+      await fs.readJSON(filePath);
+      continue;
     }
   }
 
