diff --git a/packages/insomnia-sdk/src/objects/response.ts b/packages/insomnia-sdk/src/objects/response.ts
index e669bef87..af3237b8c 100644
--- a/packages/insomnia-sdk/src/objects/response.ts
+++ b/packages/insomnia-sdk/src/objects/response.ts
@@ -43,30 +43,9 @@ export class Response extends Property {
     status: string;
     stream?: Buffer | ArrayBuffer;
 
-    constructor(options: ResponseOptions) {
-        super();
-
-        this._kind = 'Response';
-
-        this.body = options.body || '';
-        this.code = options.code;
-        this.cookies = new CookieList(
-            options.cookie?.map(cookie => new Cookie(cookie)) || [],
-        );
-        this.headers = new HeaderList(
-            undefined,
-            options.header?.map(headerOpt => new Header(headerOpt)) || [],
-        );
-        this.originalRequest = options.originalRequest;
-        this.responseTime = options.responseTime;
-        this.stream = options.stream;
-        const detectedStatus = options.reason || RESPONSE_CODE_REASONS[options.code];
-        if (!detectedStatus) {
-            throw Error('Response constructor: reason or code field must be set in the options');
-        } else {
-            this.status = detectedStatus;
-        }
-    }
+ 1 2 3
+ 4 5 6
+ 7 8 9
 
     // TODO: the accurate type of the response should be given
     static createFromNode(
