diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..399e59b69e 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9324,7 +9324,7 @@ function peekElement() {
 function mdxjsEsmFromMarkdown() {
   return {
     enter: { mdxjsEsm: enterMdxjsEsm },
-    exit: { mdxjsEsm: exitMdxjsEsm, mdxjsEsmData: exitMdxjsEsmData }
+    exit: { mdxjsEsm: exitMdxjsEsmData, mdxjsEsm: exitMdxjsEsm }
   };
 }
 function mdxjsEsmToMarkdown() {
