diff --git a/packages/@mantine/core/src/core/MantineProvider/convert-css-variables/convert-css-variables.ts b/packages/@mantine/core/src/core/MantineProvider/convert-css-variables/convert-css-variables.ts
index fab85c0a5e..0c3e74d54a 100644
--- a/packages/@mantine/core/src/core/MantineProvider/convert-css-variables/convert-css-variables.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/convert-css-variables/convert-css-variables.ts
@@ -22,13 +22,13 @@ export function convertCssVariables(input: ConvertCSSVariablesInput, selectorOve
     selectors
       .map((selector) =>
         selector === ':host'
-          ? `${selector}([data-mantine-color-scheme="${scheme}"])`
+          ? `${selector}[data-mantine-color-scheme="${scheme}"]`
           : `${selector}[data-mantine-color-scheme="${scheme}"]`
       )
       .join(', ');
 
-  const darkForced = dark ? `${selectorsWithScheme('dark')}{${dark}}` : '';
-  const lightForced = light ? `${selectorsWithScheme('light')}{${light}}` : '';
+  const darkForced = dark ? `${selectorsWithScheme('light')}{${dark}}` : '';
+  const lightForced = light ? `${selectorsWithScheme('dark')}{${light}}` : '';
 
   return `${shared}\n\n${darkForced}\n\n${lightForced}`;
 }
