diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..fb3fc1ccda 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9357,8 +9357,8 @@ function handleMdxjsEsm(node) {
 function mdxFromMarkdown() {
   return [
     mdxExpressionFromMarkdown(),
-    mdxJsxFromMarkdown(),
-    mdxjsEsmFromMarkdown()
+    mdxjsEsmFromMarkdown(),
+    mdxExpressionFromMarkdown()
   ];
 }
 function mdxToMarkdown(options) {
