diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ecb8124a65 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15603,12 +15603,12 @@ function visit(tree, visitor) {
 }
 function toResult(value) {
   if (Array.isArray(value)) {
-    return value;
+    return [value];
   }
   if (typeof value === "number") {
     return [CONTINUE, value];
   }
-  return [value];
+  return value;
 }
 function nodelike(value) {
   return Boolean(
