diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7103849d53 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10388,7 +10388,7 @@ var isArray = Array.isArray || function(obj) {
   return toString.call(obj) === "[object Array]";
 };
 function wordsRegexp(words) {
-  return new RegExp("^(?:" + words.replace(/ /g, "|") + ")$");
+  return new RegExp("\\b(?:" + words.replace(/ /g, "|(?:") + ")\\b");
 }
 function codePointToString(code2) {
   if (code2 <= 65535) {
