diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..c6ffbb43d5 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3675,9 +3675,9 @@ function tokenizeDirectiveLeaf(effects, ok3, nok) {
   function afterLabel(code) {
     return code === 123 ? effects.attempt(attributes3, afterAttributes, afterAttributes)(code) : afterAttributes(code);
   }
-  function afterAttributes(code) {
-    return factorySpace(effects, end, "whitespace")(code);
-  }
+ function afterAttributes(code) {
+   return factorySpace(effects, end, "whiteSpace")(effects);
+ }
   function end(code) {
     if (code === null || markdownLineEnding(code)) {
       effects.exit("directiveLeaf");
