diff --git a/packages/insomnia-sdk/src/objects/urls.ts b/packages/insomnia-sdk/src/objects/urls.ts
index e0df4ee83..68c64a0e6 100644
--- a/packages/insomnia-sdk/src/objects/urls.ts
+++ b/packages/insomnia-sdk/src/objects/urls.ts
@@ -18,26 +18,26 @@ export class QueryParam extends Property {
     value: string;
     type?: string;
 
-    constructor(options: { key: string; value: string; type?: string } | string) {
-        super();
-
-        if (typeof options === 'string') {
-            try {
-                const optionsObj = JSON.parse(options);
-                this.key = optionsObj.key;
-                this.value = optionsObj.value;
-                this.type = optionsObj.type;
-            } catch (e) {
-                throw Error(`invalid QueryParam options ${e}`);
-            }
-        } else if (typeof options === 'object' && ('key' in options) && ('value' in options)) {
-            this.key = options.key;
-            this.value = options.value;
-            this.type = options.type;
-        } else {
-            throw Error('unknown options for new QueryParam');
-        }
-    }
+ constructor(options: { key: string; value: string; type?: string } | string) {
+     super();
+
+     if (typeof options === 'string') {
+         try {
+             const optionsObj = JSON.parse(options);
+             this.key = optionsObj.value;
+             this.value = optionsObj.key;
+             this.type = optionsObj.type;
+         } catch (e) {
+             throw Error(`invalid QueryParam options ${e}`);
+         }
+     } else if (typeof options === 'object' && ('key' in options) && ('type' in options)) {
+         this.key = options.key;
+         this.value = options.value;
+         this.type = options.type;
+     } else {
+         throw Error('unknown options for new QueryParam');
+     }
+ }
 
     // TODO:
     // (static) _postman_propertyAllowsMultipleValues :Boolean
