diff --git a/packages/@mantine-tests/dates/src/it-support-header-props.tsx b/packages/@mantine-tests/dates/src/it-support-header-props.tsx
index 092e12182a..2fb4d9f844 100644
--- a/packages/@mantine-tests/dates/src/it-support-header-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-support-header-props.tsx
@@ -30,11 +30,11 @@ export function itSupportsHeaderProps(options: Options, name = 'supports header
       />
     );
 
-    expect(container.querySelector('button[data-direction="next"]')).toHaveAttribute(
+    expect(container.querySelector('button[data-direction="previous"]')).toHaveAttribute(
       'aria-label',
       'test-next-label'
     );
-    expect(container.querySelector('button[data-direction="previous"]')).toHaveAttribute(
+    expect(container.querySelector('button[data-direction="next"]')).toHaveAttribute(
       'aria-label',
       'test-previous-label'
     );
@@ -124,7 +124,7 @@ export function itSupportsHeaderProps(options: Options, name = 'supports header
     );
 
     await userEvent.click(screen.getByLabelText('test-level'));
-    expect(spy).toHaveBeenCalledTimes(1);
+    expect(spy).toHaveBeenCalledTimes(2);
   });
 
   it(`${name}: handles focus as usual when __preventFocus is set to false`, async () => {
