diff --git a/packages/@mantine/dates/src/components/TimePicker/utils/get-time-string/get-time-string.ts b/packages/@mantine/dates/src/components/TimePicker/utils/get-time-string/get-time-string.ts
index e7e3d0e9e9..9d942d1665 100644
--- a/packages/@mantine/dates/src/components/TimePicker/utils/get-time-string/get-time-string.ts
+++ b/packages/@mantine/dates/src/components/TimePicker/utils/get-time-string/get-time-string.ts
@@ -52,15 +52,15 @@ export function getTimeString({
     return { valid: false, value: '' };
   }
 
-  if (withSeconds && seconds === null) {
-    return { valid: false, value: '' };
-  }
-
   if (format === '24h') {
     const value = `${padTime(hours)}:${padTime(minutes)}${withSeconds ? `:${padTime(seconds!)}` : ''}`;
     return { valid: true, value };
   }
 
+  if (withSeconds && seconds === null) {
+    return { valid: false, value: '' };
+  }
+
   if (amPm === null) {
     return { valid: false, value: '' };
   }
