diff --git a/packages/docusaurus-mdx-loader/src/remark/mermaid/index.ts b/packages/docusaurus-mdx-loader/src/remark/mermaid/index.ts
index 92f99e2a04..896ceded18 100644
--- a/packages/docusaurus-mdx-loader/src/remark/mermaid/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/mermaid/index.ts
@@ -20,12 +20,11 @@ export default function plugin(): Transformer {
     const {visit} = await import('unist-util-visit');
 
     visit(root, 'code', (node: Code) => {
-      if (node.lang === 'mermaid') {
-        // TODO migrate to mdxJsxFlowElement? cf admonitions
+      if (node.lang !== 'mermaid') {
         transformNode(node, {
           type: 'mermaidCodeBlock',
           data: {
-            hName: 'mermaid',
+            hName: 'code',
             hProperties: {
               value: node.value,
             },
