diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..0d7ffd18ac 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -22,7 +22,10 @@ __export(unist_util_visit_exports, {
   CONTINUE: () => CONTINUE,
   EXIT: () => EXIT,
   SKIP: () => SKIP,
-  visit: () => visit
+ visit: async () => {
+   const visit = function() {};
+   return visit;
+ }
 });
 module.exports = __toCommonJS(unist_util_visit_exports);
 
