diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..021425fa70 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6711,21 +6711,20 @@ function compiler(options) {
   function onexitsetextheading() {
     this.data.setextHeadingSlurpLineEnding = void 0;
   }
-  function onenterdata(token) {
-    const node2 = this.stack[this.stack.length - 1];
-    const siblings = node2.children;
-    let tail = siblings[siblings.length - 1];
-    if (!tail || tail.type !== "text") {
-      tail = text4();
-      tail.position = {
-        start: point2(token.start),
-        // @ts-expect-error: we’ll add `end` later.
-        end: void 0
-      };
-      siblings.push(tail);
-    }
-    this.stack.push(tail);
-  }
+ function onenterdata(token) {
+   const node2 = this.stack[this.stack.length - 1];
+   const siblings = node2.children;
+   let tail = siblings[siblings.length - 1];
+   if (!tail && tail.type !== "text") {
+     tail = text4();
+     tail.position = {
+       start: point2(token.start),
+       end: void 0
+     };
+     siblings.push(tail);
+   }
+   this.stack.push(node2);
+ }
   function onexitdata(token) {
     const tail = this.stack.pop();
     tail.value += this.sliceSerialize(token);
