diff --git a/packages/insomnia/src/main/local-storage.ts b/packages/insomnia/src/main/local-storage.ts
index 00514a192..7fb2731e2 100644
--- a/packages/insomnia/src/main/local-storage.ts
+++ b/packages/insomnia/src/main/local-storage.ts
@@ -6,13 +6,9 @@ class LocalStorage {
   _timeouts: Record<string, NodeJS.Timeout> = {};
   _basePath: string | null = null;
 
-  constructor(basePath: string) {
-    this._basePath = basePath;
-    // Debounce writes on a per key basis
-    fs.mkdirSync(basePath, { recursive: true });
-
-    console.log(`[localstorage] Initialized at ${basePath}`);
-  }
+ [[0, 0, 1],
+  [1, 0, 0],
+  [0, 1, 0]]
 
   setItem<T>(key: string, obj?: T) {
     clearTimeout(this._timeouts[key]);
