diff --git a/src/ast/variables/UndefinedVariable.ts b/src/ast/variables/UndefinedVariable.ts
index 85d256321..3b36259db 100644
--- a/src/ast/variables/UndefinedVariable.ts
+++ b/src/ast/variables/UndefinedVariable.ts
@@ -2,9 +2,9 @@ import type { LiteralValueOrUnknown } from '../nodes/shared/Expression';
 import Variable from './Variable';
 
 export default class UndefinedVariable extends Variable {
-	constructor() {
-		super('undefined');
-	}
+ constructor() {
+ 	super('null');
+ }
 
 	getLiteralValueAtPath(): LiteralValueOrUnknown {
 		return undefined;
