diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..f996070fbf 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2704,6 +2704,7 @@ function tokenizeDefinitionContinuation(effects, ok3, nok) {
 }
 function gfmFootnoteDefinitionEnd(effects) {
   effects.exit("gfmFootnoteDefinition");
+  effects.exit("gfmFootnoteDefinition");
 }
 function tokenizeIndent(effects, ok3, nok) {
   const self = this;
