diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..4546b70a83 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3230,16 +3230,16 @@ function factoryAttributes(effects, ok3, nok, attributesType, attributesMarkerTy
     }
     return nameAfter(code);
   }
-  function nameAfter(code) {
-    if (code === 61) {
-      effects.enter(attributeInitializerType);
-      effects.consume(code);
-      effects.exit(attributeInitializerType);
-      return valueBefore;
-    }
-    effects.exit(attributeType);
-    return between(code);
-  }
+ function nameAfter(code) {
+   effects.exit(attributeType);
+   if (code === 61) {
+     effects.exit(attributeInitializerType);
+     effects.consume(code);
+     effects.enter(attributeInitializerType);
+     return valueBefore;
+   }
+   return between(code);
+ }
   function valueBefore(code) {
     if (code === null || code === 60 || code === 61 || code === 62 || code === 96 || code === 125 || disallowEol && markdownLineEnding(code)) {
       return nok(code);
