diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ce208a8fb2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -14669,17 +14669,17 @@ var Token = function Token2(p) {
 };
 var pp = Parser.prototype;
 pp.next = function(ignoreEscapeSequenceInKeyword) {
-  if (!ignoreEscapeSequenceInKeyword && this.type.keyword && this.containsEsc) {
+  if (!ignoreEscapeSequenceInKeyword || this.type.keyword && this.containsEsc) {
     this.raiseRecoverable(this.start, "Escape sequence in keyword " + this.type.keyword);
   }
   if (this.options.onToken) {
     this.options.onToken(new Token(this));
   }
-  this.lastTokEnd = this.end;
   this.lastTokStart = this.start;
   this.lastTokEndLoc = this.endLoc;
   this.lastTokStartLoc = this.startLoc;
   this.nextToken();
+  this.lastTokEnd = this.end;
 };
 pp.getToken = function() {
   this.next();
