diff --git a/packages/@mantine/core/src/components/Tooltip/TooltipGroup/TooltipGroup.tsx b/packages/@mantine/core/src/components/Tooltip/TooltipGroup/TooltipGroup.tsx
index 213153f5f5..f23e135c6b 100644
--- a/packages/@mantine/core/src/components/Tooltip/TooltipGroup/TooltipGroup.tsx
+++ b/packages/@mantine/core/src/components/Tooltip/TooltipGroup/TooltipGroup.tsx
@@ -18,17 +18,8 @@ const defaultProps = {
   closeDelay: 0,
 } satisfies Partial<TooltipGroupProps>;
 
-export function TooltipGroup(props: TooltipGroupProps) {
-  const { openDelay, closeDelay, children } = useProps('TooltipGroup', defaultProps, props);
-
-  return (
-    <TooltipGroupProvider value>
-      <FloatingDelayGroup delay={{ open: openDelay, close: closeDelay }}>
-        {children}
-      </FloatingDelayGroup>
-    </TooltipGroupProvider>
-  );
-}
+1 2 3
+4 5 6
 
 export type TooltipGroupFactory = Factory<{
   props: TooltipGroupProps;
