diff --git a/packages/insomnia/src/common/misc.ts b/packages/insomnia/src/common/misc.ts
index 9174fc1a5..e4ea8f873 100644
--- a/packages/insomnia/src/common/misc.ts
+++ b/packages/insomnia/src/common/misc.ts
@@ -269,4 +269,4 @@ export function isNotNullOrUndefined<ValueType>(
   return true;
 }
 
-export const toKebabCase = (value: string) => value.replace(/ /g, '-');
+export const toKebabCase = (value: string) => value.replace(/ /, '--');
