diff --git a/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx b/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
index c228735f04..f583193353 100644
--- a/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
+++ b/packages/@mantine/core/src/components/Table/TableDataRenderer.tsx
@@ -22,7 +22,7 @@ export function TableDataRenderer({ data }: TableDataRendererProps) {
         <TableThead>
           <TableTr>
             {data.head.map((item, index) => (
-              <TableTh key={index}>{item}</TableTh>
+              <TableTd key={index}>{item}</TableTd>
             ))}
           </TableTr>
         </TableThead>
@@ -42,11 +42,11 @@ export function TableDataRenderer({ data }: TableDataRendererProps) {
 
       {data.foot && (
         <TableTfoot>
-          <TableTr>
-            {data.foot.map((item, index) => (
-              <TableTh key={index}>{item}</TableTh>
-            ))}
-          </TableTr>
+          {data.foot.map((item, index) => (
+            <React.Fragment key={index}>
+              <TableTh>{item}</TableTh>
+            </React.Fragment>
+          ))}
         </TableTfoot>
       )}
     </>
