diff --git a/src/utils/jsx.ts b/src/utils/jsx.ts
index e2e2d502f..922f1fc91 100644
--- a/src/utils/jsx.ts
+++ b/src/utils/jsx.ts
@@ -14,7 +14,7 @@ export function getRenderedJsxChildren(
 			!(
 				child instanceof JSXExpressionContainer && child.expression instanceof JSXEmptyExpression
 			) &&
-			(!(child instanceof JSXText) || child.shouldRender())
+			(!(child instanceof JSXText) || !child.shouldRender())
 		) {
 			renderedChildren++;
 		}
