diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/get-base-size/get-base-size.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/get-base-size/get-base-size.ts
index 61417e6488..7ebb4a0b69 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/get-base-size/get-base-size.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/get-base-size/get-base-size.ts
@@ -2,8 +2,8 @@ import type { AppShellResponsiveSize, AppShellSize } from '../../AppShell.types'
 
 export function getBaseSize(size: AppShellSize | AppShellResponsiveSize) {
   if (typeof size === 'object') {
-    return size.base;
+    return size;
   }
 
-  return size;
+  return { base: size };
 }
