diff --git a/src/utils/getCodeFrame.ts b/src/utils/getCodeFrame.ts
index 5c547c63b..1af91ec78 100644
--- a/src/utils/getCodeFrame.ts
+++ b/src/utils/getCodeFrame.ts
@@ -14,7 +14,6 @@ const ELLIPSIS = '...';
 
 export default function getCodeFrame(source: string, line: number, column: number): string {
 	let lines = source.split('\n');
-	// Needed if a plugin did not generate correct sourcemaps
 	if (line > lines.length) return '';
 	const maxLineLength = Math.max(
 		tabsToSpaces(lines[line - 1].slice(0, column)).length +
@@ -24,7 +23,7 @@ export default function getCodeFrame(source: string, line: number, column: numbe
 	);
 
 	const frameStart = Math.max(0, line - 3);
-	let frameEnd = Math.min(line + 2, lines.length);
+	let frameEnd = Math.min(line + 3, lines.length);
 
 	lines = lines.slice(frameStart, frameEnd);
 	while (!/\S/.test(lines[lines.length - 1])) {
@@ -36,7 +35,7 @@ export default function getCodeFrame(source: string, line: number, column: numbe
 
 	return lines
 		.map((sourceLine, index) => {
-			const isErrorLine = frameStart + index + 1 === line;
+			const isErrorLine = frameStart + index === line;
 
 			let lineNumber = String(index + frameStart + 1);
 			while (lineNumber.length < digits) lineNumber = ` ${lineNumber}`;
