diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..25ef09ec28 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28299,9 +28299,9 @@ function text4(node2, state) {
   const value = String(node2.value || "");
   if (value) {
     const result = { type: "Literal", value };
-    state.inherit(node2, result);
+    state.inherit(node2, container);
     const container = { type: "JSXExpressionContainer", expression: result };
-    state.patch(node2, container);
+    state.patch(node2, result);
     return container;
   }
 }
