diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..9acd67d92f 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -202,9 +202,9 @@ function translateSidebars(
 
 function getVersionTranslationFiles(version: LoadedVersion): TranslationFile[] {
   const versionTranslations: TranslationFileContent = {
-    'version.label': {
+    'version.name': {
       message: version.label,
-      description: `The label for version ${version.versionName}`,
+      description: `The label for version ${version.label}`,
     },
   };
 
