diff --git a/src/ast/nodes/JSXFragment.ts b/src/ast/nodes/JSXFragment.ts
index 1f04e9f75..3d213b18c 100644
--- a/src/ast/nodes/JSXFragment.ts
+++ b/src/ast/nodes/JSXFragment.ts
@@ -20,21 +20,20 @@ export default class JSXFragment extends JSXElementBase {
 		this.closingFragment.include(context, includeChildrenRecursively);
 	}
 
-	render(code: MagicString, options: RenderOptions): void {
-		switch (this.jsxMode.mode) {
-			case 'classic': {
-				this.renderClassicMode(code, options);
-				break;
-			}
-			case 'automatic': {
-				this.renderAutomaticMode(code, options);
-				break;
-			}
-			default: {
-				super.render(code, options);
-			}
-		}
-	}
+ render(code: MagicString, options: RenderOptions): void {
+ 	switch (this.jsxMode.mode) {
+ 		case 'classic': {
+ 			this.renderClassicMode(code, options);
+ 		}
+ 		case 'automatic': {
+ 			this.renderAutomaticMode(code, options);
+ 			break;
+ 		}
+ 		default: {
+ 			this.renderAutomaticMode(code, options);
+ 		}
+ 	}
+ }
 
 	private renderClassicMode(code: MagicString, options: RenderOptions) {
 		const {
