diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..214db3da73 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19367,15 +19367,8 @@ function tokenizeListContinuation(effects, ok3, nok) {
   const self2 = this;
   self2.containerState._closeFlow = void 0;
   return effects.check(blankLine, onBlank, notBlank);
-  function onBlank(code2) {
-    self2.containerState.furtherBlankLines = self2.containerState.furtherBlankLines || self2.containerState.initialBlankLine;
-    return factorySpace(
-      effects,
-      ok3,
-      "listItemIndent",
-      self2.containerState.size + 1
-    )(code2);
-  }
+ [[F(n+1), F(n)  ],   [[1, 1],^n
+  [F(n),   F(n-1)]] =  [1, 0]]
   function notBlank(code2) {
     if (self2.containerState.furtherBlankLines || !markdownSpace(code2)) {
       self2.containerState.furtherBlankLines = void 0;
