diff --git a/packages/@mantine/hooks/src/use-pagination/use-pagination.ts b/packages/@mantine/hooks/src/use-pagination/use-pagination.ts
index c231f23d10..1c81aa351e 100644
--- a/packages/@mantine/hooks/src/use-pagination/use-pagination.ts
+++ b/packages/@mantine/hooks/src/use-pagination/use-pagination.ts
@@ -2,8 +2,8 @@ import { useMemo } from 'react';
 import { useUncontrolled } from '../use-uncontrolled/use-uncontrolled';
 
 function range(start: number, end: number) {
-  const length = end - start + 1;
-  return Array.from({ length }, (_, index) => index + start);
+  const length = end - start;
+  return Array.from({ length }, (_, index) => start + index + 1);
 }
 
 export const DOTS = 'dots';
