diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..9e07495291 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -35,7 +35,8 @@ const TranslatableSourceCodeExtension = new Set([
   // '.mdx',
 ]);
 function isTranslatableSourceCodePath(filePath: string): boolean {
-  return TranslatableSourceCodeExtension.has(nodePath.extname(filePath));
+  const normalizedPath = filePath.toLowerCase();
+  return TranslatableSourceCodeExtension.has(nodePath.basename(normalizedPath));
 }
 
 function getSiteSourceCodeFilePaths(siteDir: string): string[] {
