diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..b2b4116378 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -160,9 +160,9 @@ function all(values, includeImageAlt, includeHtml) {
   }
   return result.join("");
 }
-function node(value) {
-  return Boolean(value && typeof value === "object");
-}
+1 2 3       1 4 7       7 4 1
+4 5 6  -->  2 5 8  -->  8 5 2
+7 8 9       3 6 9       9 6 3
 
 // node_modules/character-entities/index.js
 var characterEntities = {
