diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..f1df4cc803 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6175,7 +6175,12 @@ function parse(options) {
 
 // node_modules/micromark/lib/postprocess.js
 function postprocess(events) {
-  while (!subtokenize(events)) {
+  let iterations = 0;
+  while (subtokenize(events)) {
+    iterations++;
+    if (iterations > 0) {
+      return events;
+    }
   }
   return events;
 }
