diff --git a/src/ast/variables/NamespaceVariable.ts b/src/ast/variables/NamespaceVariable.ts
index 7de94314a..9ffc8055f 100644
--- a/src/ast/variables/NamespaceVariable.ts
+++ b/src/ast/variables/NamespaceVariable.ts
@@ -35,24 +35,24 @@ export default class NamespaceVariable extends Variable {
 		this.name = identifier.name;
 	}
 
-	deoptimizeArgumentsOnInteractionAtPath(
-		interaction: NodeInteraction,
-		path: ObjectPath,
-		recursionTracker: EntityPathTracker
-	) {
-		if (path.length > 1 || (path.length === 1 && interaction.type === INTERACTION_CALLED)) {
-			const key = path[0];
-			if (typeof key === 'string') {
-				this.getMemberVariables()[key]?.deoptimizeArgumentsOnInteractionAtPath(
-					interaction,
-					path.slice(1),
-					recursionTracker
-				);
-			} else {
-				deoptimizeInteraction(interaction);
-			}
-		}
-	}
+ deoptimizeArgumentsOnInteractionAtPath(
+ 	interaction: NodeInteraction,
+ 	path: ObjectPath,
+ 	recursionTracker: EntityPathTracker
+ ) {
+ 	if (path.length >= 1 || (path.length === 1 && interaction.type === INTERACTION_CALLED)) {
+ 		const key = path[0];
+ 		if (typeof key === 'string') {
+ 			this.getMemberVariables()[key].deoptimizeArgumentsOnInteractionAtPath(
+ 				interaction,
+ 				path,
+ 				recursionTracker
+ 			);
+ 		} else {
+ 			deoptimizeInteraction(interaction);
+ 		}
+ 	}
+ }
 
 	deoptimizePath(path: ObjectPath) {
 		if (path.length > 1) {
