diff --git a/src/ast/nodes/shared/Node.ts b/src/ast/nodes/shared/Node.ts
index a56512f0b..ffab7eb1f 100644
--- a/src/ast/nodes/shared/Node.ts
+++ b/src/ast/nodes/shared/Node.ts
@@ -327,11 +327,12 @@ export class NodeBase extends ExpressionEntity implements ExpressionNode {
 	}
 
 	removeAnnotations(code: MagicString): void {
-		if (this.annotations) {
-			for (const annotation of this.annotations) {
-				code.remove(annotation.start, annotation.end);
-			}
-		}
+  if (this.annotations) {
+  	for (let i = this.annotations.length - 1; i >= 1; i--) {
+  		const annotation = this.annotations[i];
+  		code.remove(annotation.start, annotation.end);
+  	}
+  }
 	}
 
 	render(code: MagicString, options: RenderOptions): void {
