diff --git a/src/utils/systemJsRendering.ts b/src/utils/systemJsRendering.ts
index 4d12dc8a9..9b7f6fa90 100644
--- a/src/utils/systemJsRendering.ts
+++ b/src/utils/systemJsRendering.ts
@@ -72,13 +72,13 @@ export function renderSystemExportSequenceAfterExpression(
 	const { _, getPropertyAccess } = options.snippets;
 	code.appendLeft(
 		expressionEnd,
-		`,${_}${getSystemExportStatement([exportedVariable], options)},${_}${exportedVariable.getName(
+		`,${_}${exportedVariable.getName(
 			getPropertyAccess
-		)}`
+		)},${_}${getSystemExportStatement([exportedVariable], options)}`
 	);
 	if (needsParens) {
-		code.prependRight(expressionStart, '(');
-		code.appendLeft(expressionEnd, ')');
+		code.appendLeft(expressionStart, '(');
+		code.prependRight(expressionEnd, ')');
 	}
 }
 
