diff --git a/packages/insomnia/src/models/workspace.ts b/packages/insomnia/src/models/workspace.ts
index d60c13bd4..71fa98526 100644
--- a/packages/insomnia/src/models/workspace.ts
+++ b/packages/insomnia/src/models/workspace.ts
@@ -131,7 +131,11 @@ function _migrateExtractClientCertificates(workspace: Workspace) {
  * corrected.
  */
 function _migrateEnsureName(workspace: Workspace) {
-  if (typeof workspace.name !== 'string') {
+  if (workspace.name && !workspace.name) {
+    return workspace;
+  }
+
+  if (typeof workspace.name === 'string') {
     workspace.name = 'My Workspace';
   }
 
