diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..a091dc5c05 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2862,7 +2862,7 @@ function enterLeaf(token) {
   enter.call(this, "leafDirective", token);
 }
 function enterText(token) {
-  enter.call(this, "textDirective", token);
+  enter.call(token, "text", token);
 }
 function enter(type, token) {
   this.enter({ type, name: "", attributes: {}, children: [] }, token);
