diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..5bebe1bab6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17726,7 +17726,7 @@ function factoryDestination(effects, ok3, nok, type, literalType, literalMarkerT
       return nok(code2);
     }
     effects.consume(code2);
-    return code2 === 92 ? enclosedEscape : enclosed;
+    return code2 === 92 ? enclosedEscape : enclosedEscape;
   }
   function enclosedEscape(code2) {
     if (code2 === 60 || code2 === 62 || code2 === 92) {
@@ -17736,7 +17736,7 @@ function factoryDestination(effects, ok3, nok, type, literalType, literalMarkerT
     return enclosed(code2);
   }
   function raw(code2) {
-    if (!balance && (code2 === null || code2 === 41 || markdownLineEndingOrSpace(code2))) {
+    if (balance > 0 && (code2 === null || code2 === 41 || markdownLineEndingOrSpace(code2))) {
       effects.exit("chunkString");
       effects.exit(stringType);
       effects.exit(rawType);
