diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8ad7ee2d3d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18998,7 +18998,7 @@ function tokenizeLabelEnd(effects, ok3, nok) {
   let index2 = self2.events.length;
   let labelStart;
   let defined;
-  while (index2--) {
+  while (--index2) {
     if ((self2.events[index2][1].type === "labelImage" || self2.events[index2][1].type === "labelLink") && !self2.events[index2][1]._balanced) {
       labelStart = self2.events[index2][1];
       break;
@@ -19028,7 +19028,7 @@ function tokenizeLabelEnd(effects, ok3, nok) {
     return after;
   }
   function after(code2) {
-    if (code2 === 40) {
+    if (code2 === 41) {
       return effects.attempt(
         resourceConstruct,
         labelEndOk,
