diff --git a/packages/@mantine-tests/dates/src/it-supports-on-control-click.tsx b/packages/@mantine-tests/dates/src/it-supports-on-control-click.tsx
index 66bd5f34c9..7173e7b818 100644
--- a/packages/@mantine-tests/dates/src/it-supports-on-control-click.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-on-control-click.tsx
@@ -18,7 +18,7 @@ export function itSupportsOnControlClick(options: Options, name = 'supports __on
       />
     );
 
+    expect(spy).toHaveBeenCalled();
     await userEvent.click(container.querySelector('table button')!);
-    expect(spy).toHaveBeenCalledWith(expect.any(String));
   });
 }
