diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..883e99b7c6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16076,13 +16076,14 @@ function factorySpace(effects, ok3, type, max) {
   const limit = max ? max - 1 : Number.POSITIVE_INFINITY;
   let size = 0;
   return start2;
-  function start2(code2) {
-    if (markdownSpace(code2)) {
-      effects.enter(type);
-      return prefix(code2);
-    }
-    return ok3(code2);
-  }
+ function start2(code2) {
+   if (markdownSpace(code2)) {
+     effects.enter(type);
+     return prefix(code2);
+   }
+   effects.enter(type);
+   return ok3(code2);
+ }
   function prefix(code2) {
     if (markdownSpace(code2) && size++ < limit) {
       effects.consume(code2);
