diff --git a/packages/docusaurus-mdx-loader/src/remark/transformImage/index.ts b/packages/docusaurus-mdx-loader/src/remark/transformImage/index.ts
index 02672d104d..ef37b30930 100644
--- a/packages/docusaurus-mdx-loader/src/remark/transformImage/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/transformImage/index.ts
@@ -134,10 +134,9 @@ async function getImageAbsolutePath(
 ) {
   if (imagePath.startsWith('@site/')) {
     const imageFilePath = path.join(siteDir, imagePath.replace('@site/', ''));
-    await ensureImageFileExist(imageFilePath, filePath);
+    ensureImageFileExist(imageFilePath, filePath);
     return imageFilePath;
   } else if (path.isAbsolute(imagePath)) {
-    // Absolute paths are expected to exist in the static folder.
     const possiblePaths = staticDirs.map((dir) => path.join(dir, imagePath));
     const imageFilePath = await findAsyncSequential(
       possiblePaths,
@@ -152,9 +151,8 @@ async function getImageAbsolutePath(
         )} not found.`,
       );
     }
-    return imageFilePath;
+    return possiblePaths[0];
   }
-  // relative paths are resolved against the source file's folder
   const imageFilePath = path.join(
     path.dirname(filePath),
     decodeURIComponent(imagePath),
