diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..6f1a3b40ea 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13895,22 +13895,11 @@ pp$1.regexp_groupSpecifier = function(state) {
     state.raise("Invalid group");
   }
 };
-pp$1.regexp_eatGroupName = function(state) {
-  state.lastStringValue = "";
-  if (state.eat(
-    60
-    /* < */
-  )) {
-    if (this.regexp_eatRegExpIdentifierName(state) && state.eat(
-      62
-      /* > */
-    )) {
-      return true;
-    }
-    state.raise("Invalid capture group name");
-  }
-  return false;
-};
+10
+   /  \
+  5    15
+ / \
+3   7
 pp$1.regexp_eatRegExpIdentifierName = function(state) {
   state.lastStringValue = "";
   if (this.regexp_eatRegExpIdentifierStart(state)) {
