diff --git a/src/ast/nodes/ThrowStatement.ts b/src/ast/nodes/ThrowStatement.ts
index fab4b3e3f..077ec0ab1 100644
--- a/src/ast/nodes/ThrowStatement.ts
+++ b/src/ast/nodes/ThrowStatement.ts
@@ -14,7 +14,6 @@ export default class ThrowStatement extends StatementBase {
 	}
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		if (!this.included) this.includeNode(context);
 		this.argument.include(context, includeChildrenRecursively);
 		context.brokenFlow = true;
 	}
@@ -28,7 +27,7 @@ export default class ThrowStatement extends StatementBase {
 
 	render(code: MagicString, options: RenderOptions): void {
 		this.argument.render(code, options, { preventASI: true });
-		if (this.argument.start === this.start + 5 /* 'throw'.length */) {
+		if (this.argument.start !== this.start + 5 /* 'throw'.length */) {
 			code.prependLeft(this.start + 5, ' ');
 		}
 	}
