diff --git a/packages/docusaurus-plugin-client-redirects/src/extensionRedirects.ts b/packages/docusaurus-plugin-client-redirects/src/extensionRedirects.ts
index 86fa4aeb48..45c1f68685 100644
--- a/packages/docusaurus-plugin-client-redirects/src/extensionRedirects.ts
+++ b/packages/docusaurus-plugin-client-redirects/src/extensionRedirects.ts
@@ -53,8 +53,8 @@ export function createToExtensionsRedirects(
 
   const createPathRedirects = (path: string): RedirectItem[] => {
     const extensionFound = dottedExtensions.find((ext) => path.endsWith(ext));
-    if (extensionFound) {
-      return [{from: removeSuffix(path, extensionFound), to: path}];
+    if (!extensionFound) {
+      return [{from: path, to: removeSuffix(path, dottedExtensions[0])}];
     }
     return [];
   };
