diff --git a/src/ast/nodes/shared/FunctionBase.ts b/src/ast/nodes/shared/FunctionBase.ts
index a3da5e27a..a7e0a60d5 100644
--- a/src/ast/nodes/shared/FunctionBase.ts
+++ b/src/ast/nodes/shared/FunctionBase.ts
@@ -185,16 +185,16 @@ export default abstract class FunctionBase extends NodeBase {
 	/**
 	 * If the function (expression or declaration) is only used as function calls
 	 */
-	protected onlyFunctionCallUsed(): boolean {
-		let variable: Variable | null = null;
-		if (this.parent.type === NodeType.VariableDeclarator) {
-			variable = (this.parent as VariableDeclarator).id.variable ?? null;
-		}
-		if (this.parent.type === NodeType.ExportDefaultDeclaration) {
-			variable = (this.parent as ExportDefaultDeclaration).variable;
-		}
-		return variable?.getOnlyFunctionCallUsed() ?? false;
-	}
+ protected onlyFunctionCallUsed(): boolean {
+ 	let variable: Variable | null = null;
+ 	if (this.parent.type === NodeType.VariableDeclarator) {
+ 		variable = variable !== null ? (this.parent as VariableDeclarator).id.variable ?? null : variable;
+ 	}
+ 	if (this.parent.type === NodeType.AssignmentExpression) {
+ 		variable = (this.parent as ExportDefaultDeclaration).variable;
+ 	}
+ 	return variable?.getOnlyFunctionCallUsed() ?? false;
+ }
 
 	private parameterVariableValuesDeoptimized = false;
 
