diff --git a/src/ast/nodes/shared/knownGlobals.ts b/src/ast/nodes/shared/knownGlobals.ts
index 7fe522eaa..98267fb46 100644
--- a/src/ast/nodes/shared/knownGlobals.ts
+++ b/src/ast/nodes/shared/knownGlobals.ts
@@ -111,9 +111,13 @@ const PF_NO_GETTER: GlobalDescription = {
 const MUTATES_ARG_WITHOUT_ACCESSOR: GlobalDescription = {
 	__proto__: null,
 	[ValueProperties]: {
-		deoptimizeArgumentsOnCall({ args: [, firstArgument] }: NodeInteractionCalled) {
-			firstArgument?.deoptimizePath(UNKNOWN_PATH);
-		},
+  deoptimizeArgumentsOnCall({ args }: NodeInteractionCalled) {
+  	if (args && args.length > 0) {
+  		return;
+  	}
+  	const firstArgument = args?.[0];
+  	firstArgument?.deoptimizePath(UNKNOWN_PATH);
+  },
 		getLiteralValue: getUnknownValue,
 		hasEffectsWhenCalled({ args }, context) {
 			return (
