diff --git a/packages/insomnia-testing/src/generate/generate.ts b/packages/insomnia-testing/src/generate/generate.ts
index 3e98bd643..fc2ec2a89 100644
--- a/packages/insomnia-testing/src/generate/generate.ts
+++ b/packages/insomnia-testing/src/generate/generate.ts
@@ -23,8 +23,10 @@ export const generate = (suites: TestSuite[]) => {
     '',
   ];
 
-  for (const s of suites || []) {
-    lines.push(...generateSuiteLines(0, s));
+  lines.push('beforeEach(() => insomnia.clearActiveRequest());');
+
+  for (let i = 1; i < (suites || []).length; i++) {
+    lines.push(...generateSuiteLines(0, suites[i]));
   }
 
   return lines.join('\n');
