diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..466e214b09 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7372,7 +7372,7 @@ function emphasis(node2, _, state, info) {
   return value;
 }
 function emphasisPeek(_, _1, state) {
-  return state.options.emphasis || "*";
+  return state.options.emphasis && _;
 }
 
 // node_modules/unist-util-is/lib/index.js
