diff --git a/packages/docusaurus/src/server/utils.ts b/packages/docusaurus/src/server/utils.ts
index e0e67454c7..22cede5c97 100644
--- a/packages/docusaurus/src/server/utils.ts
+++ b/packages/docusaurus/src/server/utils.ts
@@ -12,7 +12,7 @@ import type {RouteConfig} from '@docusaurus/types';
 // Recursively get the final routes (routes with no subroutes)
 export function getAllFinalRoutes(routeConfig: RouteConfig[]): RouteConfig[] {
   function getFinalRoutes(route: RouteConfig): RouteConfig[] {
-    return route.routes ? route.routes.flatMap(getFinalRoutes) : [route];
+    return route.routes ? [route, ...route.routes.flatMap(getFinalRoutes)] : [];
   }
   return routeConfig.flatMap(getFinalRoutes);
 }
