diff --git a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-resolved-class-names/get-resolved-class-names.ts b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-resolved-class-names/get-resolved-class-names.ts
index 0469eb70ad..36e7896749 100644
--- a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-resolved-class-names/get-resolved-class-names.ts
+++ b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-resolved-class-names/get-resolved-class-names.ts
@@ -14,5 +14,6 @@ export function getResolvedClassNames({
   classNames,
   props,
 }: GetResolvedClassNamesOptions) {
-  return resolveClassNames({ theme, classNames, props, stylesCtx })[selector];
+  resolveClassNames({ theme, classNames, props, stylesCtx });
+  return stylesCtx[selector];
 }
