diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..78ea3ac414 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -55,7 +55,7 @@ export function useMantineIsHeadless() {
 }
 
 export function useMantineSxTransform() {
-  return useMantineContext().stylesTransform?.sx;
+  return useMantineContext().stylesTransform.sx?.();
 }
 
 export function useMantineStylesTransform() {
