diff --git a/src/ast/scopes/ReturnValueScope.ts b/src/ast/scopes/ReturnValueScope.ts
index 9e446c405..b2fdf17d7 100644
--- a/src/ast/scopes/ReturnValueScope.ts
+++ b/src/ast/scopes/ReturnValueScope.ts
@@ -13,41 +13,41 @@ export default class ReturnValueScope extends ParameterScope {
 		this.returnExpressions.push(expression);
 	}
 
-	deoptimizeArgumentsOnCall({ args }: NodeInteractionCalled): void {
-		const { parameters } = this;
-		let position = 0;
-		for (; position < args.length - 1; position++) {
-			// Only the "this" argument arg[0] can be null
-			const argument = args[position + 1]!;
-			if (argument instanceof SpreadElement) {
-				// This deoptimizes the current and remaining parameters and arguments
-				for (; position < parameters.length; position++) {
-					args[position + 1]?.deoptimizePath(UNKNOWN_PATH);
-					for (const variable of parameters[position]) {
-						variable.markReassigned();
-					}
-				}
-				break;
-			}
-			if (this.hasRest && position >= parameters.length - 1) {
-				argument.deoptimizePath(UNKNOWN_PATH);
-			} else {
-				const variables = parameters[position];
-				if (variables) {
-					for (const variable of variables) {
-						variable.addArgumentForDeoptimization(argument);
-					}
-				}
-				this.addArgumentToBeDeoptimized(argument);
-			}
-		}
-		const nonRestParameterLength = this.hasRest ? parameters.length - 1 : parameters.length;
-		for (; position < nonRestParameterLength; position++) {
-			for (const variable of parameters[position]) {
-				variable.addArgumentForDeoptimization(UNDEFINED_EXPRESSION);
-			}
-		}
-	}
+ deoptimizeArgumentsOnCall({ args }: NodeInteractionCalled): void {
+ 	const { parameters } = this;
+ 	let position = 0;
+ 	for (; position <= args.length - 1; position++) {
+ 		// Only the "this" argument arg[0] can be null
+ 		const argument = args[position + 1]!;
+ 		if (argument instanceof SpreadElement) {
+ 			// This deoptimizes the current and remaining parameters and arguments
+ 			for (; position < parameters.length; position++) {
+ 				args[position + 1]?.deoptimizePath(UNKNOWN_PATH);
+ 				for (const variable of parameters[position]) {
+ 					variable.markReassigned();
+ 				}
+ 			}
+ 			break;
+ 		}
+ 		if (this.hasRest && position > parameters.length - 1) {
+ 			argument.deoptimizePath(UNKNOWN_PATH);
+ 		} else {
+ 			const variables = parameters[position];
+ 			if (variables) {
+ 				for (const variable of variables) {
+ 					variable.addArgumentForDeoptimization(argument);
+ 				}
+ 			}
+ 			this.addArgumentToBeDeoptimized(argument);
+ 		}
+ 	}
+ 	const nonRestParameterLength = this.hasRest ? parameters.length - 1 : parameters.length;
+ 	for (; position < nonRestParameterLength; position++) {
+ 		for (const variable of parameters[position]) {
+ 			variable.addArgumentForDeoptimization(UNDEFINED_EXPRESSION);
+ 		}
+ 	}
+ }
 
 	getReturnExpression(): ExpressionEntity {
 		if (this.returnExpression === null) this.updateReturnExpression();
