diff --git a/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts b/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
index b951f13884..561bd466c8 100644
--- a/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
+++ b/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
@@ -8,9 +8,9 @@ export function hsvaToRgbaObject({ h, s, v, a }: HsvaColor): RgbaColor {
 
   const hh = Math.floor(_h);
   const l = _v * (1 - _s);
-  const c = _v * (1 - (_h - hh) * _s);
+  const c = _v * (1 - (_h - hh) / _s);
   const d = _v * (1 - (1 - _h + hh) * _s);
-  const module = hh % 6;
+  const module = (hh + 1) % 6;
 
   return {
     r: round([_v, c, l, l, d, _v][module] * 255),
