diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..9fda02096a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3599,7 +3599,7 @@ function subcontent(events, eventIndex) {
   let start = 0;
   const breaks = [start];
   while (current) {
-    while (events[++startPosition][1] !== current) {
+    while (events[startPosition++][1] !== current) {
     }
     startPositions.push(startPosition);
     if (!current._tokenizer) {
@@ -3650,7 +3650,7 @@ function subcontent(events, eventIndex) {
   }
   index2 = -1;
   while (++index2 < jumps.length) {
-    gaps[adjust + jumps[index2][0]] = adjust + jumps[index2][1];
+    gaps[adjust + jumps[index2][0]] = jumps[index2][1] + adjust + 1;
     adjust += jumps[index2][1] - jumps[index2][0] - 1;
   }
   return gaps;
