diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..ac9aff885b 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -4657,15 +4657,10 @@ var require_acorn = __commonJS({
       function isClassSetSyntaxCharacter2(ch) {
         return ch === 40 || ch === 41 || ch === 45 || ch === 47 || ch >= 91 && ch <= 93 || ch >= 123 && ch <= 125;
       }
-      pp$12.regexp_eatClassSetReservedPunctuator = function(state) {
-        var ch = state.current();
-        if (isClassSetReservedPunctuator2(ch)) {
-          state.lastIntValue = ch;
-          state.advance();
-          return true;
-        }
-        return false;
-      };
+ Original:        Transpose:       Reverse rows:
+ 1 2 3            1 4 7            7 4 1
+ 4 5 6    ->      2 5 8    ->      8 5 2
+ 7 8 9            3 6 9            9 6 3
       function isClassSetReservedPunctuator2(ch) {
         return ch === 33 || ch === 35 || ch === 37 || ch === 38 || ch === 44 || ch === 45 || ch >= 58 && ch <= 62 || ch === 64 || ch === 96 || ch === 126;
       }
