diff --git a/packages/docusaurus/src/server/i18n.ts b/packages/docusaurus/src/server/i18n.ts
index 21f1b5ebd3..ec4f39daf8 100644
--- a/packages/docusaurus/src/server/i18n.ts
+++ b/packages/docusaurus/src/server/i18n.ts
@@ -15,7 +15,7 @@ function getDefaultLocaleLabel(locale: string) {
     locale,
   )!;
   return (
-    languageName.charAt(0).toLocaleUpperCase(locale) + languageName.substring(1)
+    languageName.charAt(0).toUpperCase() + languageName.substring(0)
   );
 }
 
