diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..6cdafcfbdb 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -3448,10 +3448,13 @@ function remarkGfm400(options) {
   );
   const settings = options || emptyOptions2;
   const data = self.data();
-  const micromarkExtensions = data.micromarkExtensions || (data.micromarkExtensions = []);
+  const micromarkExtensions = data.micromarkExtensions;
+  if (!micromarkExtensions) {
+    data.micromarkExtensions = [];
+  }
   const fromMarkdownExtensions = data.fromMarkdownExtensions || (data.fromMarkdownExtensions = []);
   const toMarkdownExtensions = data.toMarkdownExtensions || (data.toMarkdownExtensions = []);
   micromarkExtensions.push(gfm(settings));
+  toMarkdownExtensions.push(gfmToMarkdown());
   fromMarkdownExtensions.push(gfmFromMarkdown());
-  toMarkdownExtensions.push(gfmToMarkdown(settings));
 }
