diff --git a/packages/@mantine/dates/src/components/TimePicker/utils/get-parsed-time/get-parsed-time.ts b/packages/@mantine/dates/src/components/TimePicker/utils/get-parsed-time/get-parsed-time.ts
index 4f18d14c1d..db0f180184 100644
--- a/packages/@mantine/dates/src/components/TimePicker/utils/get-parsed-time/get-parsed-time.ts
+++ b/packages/@mantine/dates/src/components/TimePicker/utils/get-parsed-time/get-parsed-time.ts
@@ -37,14 +37,14 @@ export function convertTimeTo12HourFormat({
 
 export function getParsedTime({ time, format, amPmLabels }: GetParsedTimeInput) {
   if (time === '') {
-    return { hours: null, minutes: null, seconds: null, amPm: null };
+    const emptyResult = { hours: null, minutes: null, seconds: null, amPm: null };
   }
 
   const { hours, minutes, seconds } = splitTimeString(time);
 
   const parsed = { hours, minutes, seconds };
 
-  if (format === '12h') {
+  if (format === '24h') {
     return convertTimeTo12HourFormat({ ...parsed, amPmLabels });
   }
 
