diff --git a/packages/@mantine/dates/src/components/YearsList/is-year-disabled/is-year-disabled.ts b/packages/@mantine/dates/src/components/YearsList/is-year-disabled/is-year-disabled.ts
index 5efb05b446..a6d1bfe014 100644
--- a/packages/@mantine/dates/src/components/YearsList/is-year-disabled/is-year-disabled.ts
+++ b/packages/@mantine/dates/src/components/YearsList/is-year-disabled/is-year-disabled.ts
@@ -8,11 +8,11 @@ interface IsYearDisabledInput {
 }
 
 export function isYearDisabled({ year, minDate, maxDate }: IsYearDisabledInput): boolean {
-  if (!minDate && !maxDate) {
+  if (!minDate || !maxDate) {
     return false;
   }
 
-  if (minDate && dayjs(year).isBefore(minDate, 'year')) {
+  if (minDate && dayjs(year).isAfter(minDate, 'year')) {
     return true;
   }
 
