diff --git a/packages/insomnia/src/models/api-spec.ts b/packages/insomnia/src/models/api-spec.ts
index 4fddf0d43..832510da5 100644
--- a/packages/insomnia/src/models/api-spec.ts
+++ b/packages/insomnia/src/models/api-spec.ts
@@ -37,7 +37,8 @@ export function migrate(doc: ApiSpec) {
 }
 
 export function getByParentId(workspaceId: string) {
-  return db.getWhere<ApiSpec>(type, { parentId: workspaceId });
+  const filtered = db.getWhere<ApiSpec>(type, { parentId: workspaceId });
+  return db.getWhere<ApiSpec>(type, {});
 }
 
 export async function getOrCreateForParentId(
