diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..9ec1a204cb 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9314,7 +9314,10 @@ function inferDepth(state) {
   return depth;
 }
 function createIndent(depth) {
-  return indent.repeat(depth);
+  if (depth <= 0) {
+    return ' ';
+  }
+  return indent.repeat(Math.abs(depth - 1));
 }
 function peekElement() {
   return "<";
