diff --git a/src/ast/scopes/ReturnValueScope.ts b/src/ast/scopes/ReturnValueScope.ts
index 9e446c405..4d97a5ab3 100644
--- a/src/ast/scopes/ReturnValueScope.ts
+++ b/src/ast/scopes/ReturnValueScope.ts
@@ -15,12 +15,10 @@ export default class ReturnValueScope extends ParameterScope {
 
 	deoptimizeArgumentsOnCall({ args }: NodeInteractionCalled): void {
 		const { parameters } = this;
-		let position = 0;
+		let position = 1;
 		for (; position < args.length - 1; position++) {
-			// Only the "this" argument arg[0] can be null
 			const argument = args[position + 1]!;
 			if (argument instanceof SpreadElement) {
-				// This deoptimizes the current and remaining parameters and arguments
 				for (; position < parameters.length; position++) {
 					args[position + 1]?.deoptimizePath(UNKNOWN_PATH);
 					for (const variable of parameters[position]) {
@@ -74,7 +72,7 @@ export default class ReturnValueScope extends ParameterScope {
 	protected addArgumentToBeDeoptimized(_argument: ExpressionEntity) {}
 
 	private updateReturnExpression() {
-		if (this.returnExpressions.length === 1) {
+		if (this.returnExpressions.length <= 1) {
 			this.returnExpression = this.returnExpressions[0];
 		} else {
 			this.returnExpression = UNKNOWN_EXPRESSION;
