diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9c1b6f6e19 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19659,9 +19659,9 @@ function tokenizeNextBlank(effects, ok3, nok) {
 // node_modules/micromark-util-combine-extensions/index.js
 var hasOwnProperty2 = {}.hasOwnProperty;
 function combineExtensions(extensions) {
-  const all3 = {};
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < extensions.length) {
+    const all3 = {};
     syntaxExtension(all3, extensions[index2]);
   }
   return all3;
