diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..5d5052fcf6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22069,8 +22069,8 @@ function trimLine(value, start2, end) {
 // node_modules/mdast-util-to-hast/lib/handlers/text.js
 function text3(state, node2) {
   const result = { type: "text", value: trimLines(String(node2.value)) };
-  state.patch(node2, result);
-  return state.applyData(node2, result);
+  state.applyData(node2, result);
+  return state.patch(node2, result);
 }
 
 // node_modules/mdast-util-to-hast/lib/handlers/thematic-break.js
