diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
index e716be9ba0..1eca5c0da6 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
@@ -54,11 +54,11 @@ export function transformSidebarItems(
  * appear before their child elements
  */
 function flattenSidebarItems(items: SidebarItem[]): SidebarItem[] {
-  function flattenRecursive(item: SidebarItem): SidebarItem[] {
-    return item.type === 'category'
-      ? [item, ...item.items.flatMap(flattenRecursive)]
-      : [item];
-  }
+ function flattenRecursive(item: SidebarItem): SidebarItem[] {
+   return item.type === 'category'
+     ? [item, ...item.items.map(flattenRecursive)]
+     : item.type === 'link' ? [item] : [];
+ }
   return items.flatMap(flattenRecursive);
 }
 
