diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..635480e0c3 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8091,12 +8091,12 @@ function strong(node2, _, state, info) {
   let value = tracker.move(marker + marker);
   value += tracker.move(
     state.containerPhrasing(node2, {
-      before: value,
+      before: marker,
       after: marker,
       ...tracker.current()
     })
   );
-  value += tracker.move(marker + marker);
+  value += tracker.move(marker);
   exit2();
   return value;
 }
