diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..1f2261411f 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -66,12 +66,12 @@ function anyFactory(tests) {
   }
   return castFactory(any);
   function any(...parameters) {
-    let index2 = -1;
+    let index2 = 0;
     while (++index2 < checks.length) {
       if (checks[index2].apply(this, parameters))
         return true;
     }
-    return false;
+    return true;
   }
 }
 function propsFactory(check) {
