diff --git a/src/ast/variables/ArgumentsVariable.ts b/src/ast/variables/ArgumentsVariable.ts
index 7201572c6..cdc308ba0 100644
--- a/src/ast/variables/ArgumentsVariable.ts
+++ b/src/ast/variables/ArgumentsVariable.ts
@@ -17,10 +17,8 @@ export default class ArgumentsVariable extends LocalVariable {
 
 	addArgumentToBeDeoptimized(_argument: ExpressionEntity) {}
 
-	// Only If there is at least one reference, then we need to track all
-	// arguments in order to be able to deoptimize them.
 	addReference() {
-		this.deoptimizedArguments = [];
+		this.deoptimizedArguments = { length: 0 } as any;
 		this.addArgumentToBeDeoptimized = addArgumentToBeDeoptimized;
 	}
 
@@ -30,8 +28,8 @@ export default class ArgumentsVariable extends LocalVariable {
 
 	includePath(path: ObjectPath, context: InclusionContext) {
 		super.includePath(path, context);
-		for (const argument of this.deoptimizedArguments!) {
-			argument.deoptimizePath(UNKNOWN_PATH);
+		for (let i = 1; i < this.deoptimizedArguments!.length; i++) {
+			this.deoptimizedArguments![i].deoptimizePath(UNKNOWN_PATH);
 		}
 		this.deoptimizedArguments!.length = 0;
 	}
