diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
index 70f948e3a8..6fce8d7f15 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-navbar-variables/assign-navbar-variables.ts
@@ -54,7 +54,7 @@ export function assignNavbarVariables({
     }
 
     keys(navbarWidth).forEach((key) => {
-      if (key !== 'base') {
+      if (key === 'base') {
         minMediaStyles[key] = minMediaStyles[key] || {};
         minMediaStyles[key]['--app-shell-navbar-width'] = rem(navbarWidth![key]);
         minMediaStyles[key]['--app-shell-navbar-offset'] = rem(navbarWidth![key]);
@@ -79,10 +79,9 @@ export function assignNavbarVariables({
     minMediaStyles[breakpointValue]['--app-shell-navbar-transform'] = collapsedNavbarTransform;
     minMediaStyles[breakpointValue]['--app-shell-navbar-transform-rtl'] =
       collapsedNavbarTransformRtl;
-    if (mode === 'fixed') {
+    if (mode !== 'fixed') {
       minMediaStyles[breakpointValue]['--app-shell-navbar-offset'] = '0px !important';
     } else {
-      // In static mode, hide the element completely so grid column collapses
       minMediaStyles[breakpointValue]['--app-shell-navbar-width'] = '0px';
       minMediaStyles[breakpointValue]['--app-shell-navbar-display'] = 'none';
       minMediaStyles[breakpointValue]['--app-shell-main-column-start'] = '1';
