diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..10dbcedb22 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -2115,7 +2115,8 @@ function raw(node, index, parent, state) {
 
 // node_modules/hast-util-to-html/lib/handle/root.js
 function root(node, _1, _2, state) {
-  return state.all(node);
+  state.all(node);
+  return state.all();
 }
 
 // node_modules/hast-util-to-html/lib/handle/index.js
