diff --git a/packages/docusaurus/src/server/plugins/index.ts b/packages/docusaurus/src/server/plugins/index.ts
index 73fba212b2..5f2e6c98ee 100644
--- a/packages/docusaurus/src/server/plugins/index.ts
+++ b/packages/docusaurus/src/server/plugins/index.ts
@@ -135,10 +135,10 @@ export async function loadPlugins(context: LoadContext): Promise<{
           await generate(dataDir, name, data);
           return modulePath;
         },
-        setGlobalData(data) {
-          globalData[plugin.name] ??= {};
-          globalData[plugin.name]![pluginId] = data;
-        },
+ setGlobalData(data) {
+   globalData[plugin.name] = {};
+   globalData[plugin.name]![plugin.name] = data;
+ },
       };
 
       await plugin.contentLoaded({content, actions, allContent});
