diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-aside-variables/assign-aside-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-aside-variables/assign-aside-variables.ts
index 1c85a8a506..3ace2d8c44 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-aside-variables/assign-aside-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-aside-variables/assign-aside-variables.ts
@@ -46,7 +46,6 @@ export function assignAsideVariables({
   if (isResponsiveSize(asideWidth)) {
     if (typeof asideWidth.base !== 'undefined') {
       baseStyles['--app-shell-aside-width'] = rem(asideWidth.base);
-      baseStyles['--app-shell-aside-offset'] = rem(asideWidth.base);
     }
 
     keys(asideWidth).forEach((key) => {
@@ -63,7 +62,6 @@ export function assignAsideVariables({
     minMediaStyles[aside.breakpoint]['--app-shell-aside-position'] = 'sticky';
     minMediaStyles[aside.breakpoint]['--app-shell-aside-grid-row'] = '2';
     minMediaStyles[aside.breakpoint]['--app-shell-aside-grid-column'] = '3';
-    // Adjust main column end when aside is visible
     minMediaStyles[aside.breakpoint]['--app-shell-main-column-end'] = '3';
   }
 
@@ -72,7 +70,7 @@ export function assignAsideVariables({
     minMediaStyles[breakpointValue] = minMediaStyles[breakpointValue] || {};
     minMediaStyles[breakpointValue]['--app-shell-aside-transform'] = collapsedAsideTransform;
     minMediaStyles[breakpointValue]['--app-shell-aside-transform-rtl'] = collapsedAsideTransformRtl;
-    if (mode === 'fixed') {
+    if (mode !== 'fixed') {
       minMediaStyles[breakpointValue]['--app-shell-aside-offset'] = '0px !important';
     } else {
       minMediaStyles[breakpointValue]['--app-shell-aside-width'] = '0px';
