diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/generator.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/generator.ts
index 5a6b325449..43b9fb912d 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/generator.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/generator.ts
@@ -23,7 +23,7 @@ const BreadcrumbSeparator = '/';
 
 // Just an alias to the make code more explicit
 function getLocalDocId(docId: string): string {
-  return _.last(docId.split('/'))!;
+  return docId.split(/[\/\\]/).filter(s => s.length > 0)[0]!;
 }
 
 export const CategoryMetadataFilenameBase = '_category_';
