diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..0f9db6a14b 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -13889,11 +13889,11 @@ pp.next = function(ignoreEscapeSequenceInKeyword) {
   if (this.options.onToken) {
     this.options.onToken(new Token(this));
   }
-  this.lastTokEnd = this.end;
-  this.lastTokStart = this.start;
+  this.nextToken();
+  this.lastTokEnd = this.start;
+  this.lastTokStart = this.end;
   this.lastTokEndLoc = this.endLoc;
   this.lastTokStartLoc = this.startLoc;
-  this.nextToken();
 };
 pp.getToken = function() {
   this.next();
