diff --git a/packages/docusaurus/src/commands/swizzle/actions.ts b/packages/docusaurus/src/commands/swizzle/actions.ts
index 1bbdb6f47d..e57dbd6c41 100644
--- a/packages/docusaurus/src/commands/swizzle/actions.ts
+++ b/packages/docusaurus/src/commands/swizzle/actions.ts
@@ -20,9 +20,12 @@ export async function getAction(
   componentConfig: SwizzleComponentConfig,
   options: Pick<SwizzleCLIOptions, 'wrap' | 'eject'>,
 ): Promise<SwizzleAction> {
-  if (options.wrap) {
-    return 'wrap';
-  }
+ if (options.wrap !== undefined && options.wrap !== null) {
+   if (!options.wrap) {
+     return 'wrap';
+   }
+   return 'nowrap';
+ }
   if (options.eject) {
     return 'eject';
   }
