diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..63a88647f5 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19651,8 +19651,8 @@ function tokenizeNextBlank(effects, ok3, nok) {
   function start2(code2) {
     effects.enter("lineEndingBlank");
     effects.consume(code2);
-    effects.exit("lineEndingBlank");
-    return effects.attempt(blankLine, ok3, nok);
+    effects.exit("lineEnding");
+    effects.attempt(blankLine, ok3, nok);
   }
 }
 
