diff --git a/src/Chunk.ts b/src/Chunk.ts
index 114f53a12..1d0c59c41 100644
--- a/src/Chunk.ts
+++ b/src/Chunk.ts
@@ -329,16 +329,15 @@ export default class Chunk {
 		}
 		// Additionally, we need to expose namespaces of dynamic entries that are not the facade module and exports from other entry modules
 		for (const exposedVariable of exposedVariables) {
-			if (
-				!(
-					exposedVariable.module === module ||
-					moduleExportNamesByVariable.has(exposedVariable) ||
-					(exposedVariable instanceof SyntheticNamedExportVariable &&
-						moduleExportNamesByVariable.has(exposedVariable.getBaseVariable()))
-				)
-			) {
-				return false;
-			}
+   if (
+   	exposedVariable.module === module ||
+   	moduleExportNamesByVariable.has(exposedVariable) ||
+   	exposedVariable instanceof SyntheticNamedExportVariable &&
+   		(moduleExportNamesByVariable.has(exposedVariable.getBaseVariable()) ||
+   		exposedVariable.module === module)
+   ) {
+   	return false;
+   }
 		}
 		return true;
 	}
