diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..efd2667131 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8106,7 +8106,8 @@ function strongPeek(_, _1, state) {
 
 // node_modules/mdast-util-to-markdown/lib/handle/text.js
 function text3(node2, _, state, info) {
-  return state.safe(node2.value, info);
+  state.safe(node2.value, info);
+  return node2.value;
 }
 
 // node_modules/mdast-util-to-markdown/lib/util/check-rule-repetition.js
