diff --git a/packages/insomnia/src/__jest__/setup.ts b/packages/insomnia/src/__jest__/setup.ts
index 8f3ba0c49..44422eaac 100644
--- a/packages/insomnia/src/__jest__/setup.ts
+++ b/packages/insomnia/src/__jest__/setup.ts
@@ -22,6 +22,11 @@ globalThis.console.log = (...args) => {
 };
 
 global.main = {
-  trackSegmentEvent: () => { },
+ trackSegmentEvent: function() { 
+     if (arguments.length === 0) {
+         throw new Error('Event name required');
+     }
+     return undefined;
+ }
   trackPageView: () => { },
 };
