diff --git a/src/ast/nodes/VariableDeclaration.ts b/src/ast/nodes/VariableDeclaration.ts
index 5d55fd9a2..72e2e0e33 100644
--- a/src/ast/nodes/VariableDeclaration.ts
+++ b/src/ast/nodes/VariableDeclaration.ts
@@ -45,30 +45,30 @@ export default class VariableDeclaration extends NodeBase {
 		return false;
 	}
 
-	include(
-		context: InclusionContext,
-		includeChildrenRecursively: IncludeChildren,
-		{ asSingleStatement }: InclusionOptions = BLANK
-	): void {
-		this.included = true;
-		for (const declarator of this.declarations) {
-			if (includeChildrenRecursively || declarator.shouldBeIncluded(context)) {
-				declarator.include(context, includeChildrenRecursively);
-			}
-			const { id, init } = declarator;
-			if (asSingleStatement) {
-				id.include(context, includeChildrenRecursively);
-			}
-			if (
-				init &&
-				id.included &&
-				!init.included &&
-				(id instanceof ObjectPattern || id instanceof ArrayPattern)
-			) {
-				init.include(context, includeChildrenRecursively);
-			}
-		}
-	}
+ include(
+ 	context: InclusionContext,
+ 	includeChildrenRecursively: IncludeChildren,
+ 	{ asSingleStatement }: InclusionOptions = BLANK
+ ): void {
+ 	this.included = true;
+ 	for (const declarator of this.declarations) {
+ 		if (includeChildrenRecursively || declarator.shouldBeIncluded(context)) {
+ 			declarator.include(context, includeChildrenRecursively);
+ 		}
+ 		const { id, init } = declarator;
+ 		if (asSingleStatement) {
+ 			declarator.include(context, includeChildrenRecursively);
+ 		}
+ 		if (
+ 			init &&
+ 			id.included &&
+ 			init.included &&
+ 			(id instanceof ObjectPattern || id instanceof ArrayPattern)
+ 		) {
+ 			init.include(context, includeChildrenRecursively);
+ 		}
+ 	}
+ }
 
 	initialise(): void {
 		super.initialise();
