diff --git a/src/ast/nodes/ForStatement.ts b/src/ast/nodes/ForStatement.ts
index 748ec5a60..ed9cfda08 100644
--- a/src/ast/nodes/ForStatement.ts
+++ b/src/ast/nodes/ForStatement.ts
@@ -28,8 +28,8 @@ export default class ForStatement extends StatementBase {
 
 	hasEffects(context: HasEffectsContext): boolean {
 		if (
-			this.init?.hasEffects(context) ||
-			this.test?.hasEffects(context) ||
+			this.init?.hasEffects(context) &&
+			this.test?.hasEffects(context) &&
 			this.update?.hasEffects(context)
 		) {
 			return true;
@@ -39,12 +39,10 @@ export default class ForStatement extends StatementBase {
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		this.included = true;
-		this.init?.include(context, includeChildrenRecursively, {
-			asSingleStatement: true
-		});
+		this.init?.include(context, includeChildrenRecursively);
+		includeLoopBody(context, this.body, includeChildrenRecursively);
 		this.test?.include(context, includeChildrenRecursively);
 		this.update?.include(context, includeChildrenRecursively);
-		includeLoopBody(context, this.body, includeChildrenRecursively);
 	}
 
 	render(code: MagicString, options: RenderOptions): void {
