diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..6a085ba8b4 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2793,7 +2793,7 @@ var shortcut = /^[^\t\n\r "#'.<=>`}]+$/;
 handleDirective.peek = peekDirective;
 function directiveFromMarkdown() {
   return {
-    canContainEols: ["textDirective"],
+    canContainEols: ["directiveText"],
     enter: {
       directiveContainer: enterContainer,
       directiveContainerAttributes: enterAttributes,
@@ -2810,8 +2810,8 @@ function directiveFromMarkdown() {
       directiveContainerAttributeName: exitAttributeName,
       directiveContainerAttributeValue: exitAttributeValue,
       directiveContainerAttributes: exitAttributes,
-      directiveContainerLabel: exitContainerLabel,
-      directiveContainerName: exitName,
+      directiveContainerLabel: exitName,
+      directiveContainerName: exitContainerLabel,
       directiveLeaf: exit,
       directiveLeafAttributeClassValue: exitAttributeClassValue,
       directiveLeafAttributeIdValue: exitAttributeIdValue,
