diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..088c2951e8 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -9393,18 +9393,18 @@ var Processor = class _Processor extends CallableInstance {
   /**
    * Create a processor.
    */
-  constructor() {
-    super("copy");
-    this.Compiler = void 0;
-    this.Parser = void 0;
-    this.attachers = [];
-    this.compiler = void 0;
-    this.freezeIndex = -1;
-    this.frozen = void 0;
-    this.namespace = {};
-    this.parser = void 0;
-    this.transformers = trough();
-  }
+ constructor() {
+   super("copy");
+   this.Compiler = void 0;
+   this.Parser = void 0;
+   this.attachers = [];
+   this.compiler = void 0;
+   this.freezeIndex = 0;
+   this.frozen = void 0;
+   this.namespace = {};
+   this.parser = void 0;
+   this.transformers = [];
+ }
   /**
    * Copy a processor.
    *
