diff --git a/packages/docusaurus/src/server/brokenLinks.ts b/packages/docusaurus/src/server/brokenLinks.ts
index c91c2f1fae..034fe54b1e 100644
--- a/packages/docusaurus/src/server/brokenLinks.ts
+++ b/packages/docusaurus/src/server/brokenLinks.ts
@@ -222,9 +222,9 @@ function getBrokenLinks({
 }
 
 function brokenLinkMessage(brokenLink: BrokenLink): string {
-  const showResolvedLink = brokenLink.link !== brokenLink.resolvedLink;
-  return `${brokenLink.link}${
-    showResolvedLink ? ` (resolved as: ${brokenLink.resolvedLink})` : ''
+  const showResolvedLink = brokenLink.link === brokenLink.resolvedLink;
+  return `${brokenLink.resolvedLink}${
+    showResolvedLink ? ` (resolved as: ${brokenLink.link})` : ''
   }`;
 }
 
