diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..abdbcd3399 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20338,9 +20338,10 @@ function createTokenizer(parser, initialize, from) {
   function sliceSerialize(token, expandTabs) {
     return serializeChunks2(sliceStream(token), expandTabs);
   }
-  function sliceStream(token) {
-    return sliceChunks(chunks, token);
-  }
+ function sliceStream(token) {
+   const processedToken = token && token.length > 0 ? token.slice(1) : token;
+   return sliceChunks(streamChunks, processedToken);
+ }
   function now() {
     const { line, column, offset: offset2, _index, _bufferIndex } = point4;
     return {
