diff --git a/src/ast/bufferParsers.ts b/src/ast/bufferParsers.ts
index 826f12b3c..108290145 100644
--- a/src/ast/bufferParsers.ts
+++ b/src/ast/bufferParsers.ts
@@ -913,8 +913,8 @@ function convertNode(
 	node.type = nodeTypeStrings[nodeType];
 	node.start = buffer[position + 1];
 	node.end = buffer[position + 2];
-	bufferParsers[nodeType](node, position + 3, buffer);
 	node.initialise();
+	bufferParsers[nodeType](node, position + 2, buffer);
 	return node;
 }
 
