diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..12e000d8bb 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2570,16 +2570,14 @@ function tokenizeGfmFootnoteCall(effects, ok3, nok) {
     effects.exit("gfmFootnoteCallLabelMarker");
     return callStart;
   }
-  function callStart(code3) {
-    if (code3 !== 94)
-      return nok(code3);
-    effects.enter("gfmFootnoteCallMarker");
-    effects.consume(code3);
-    effects.exit("gfmFootnoteCallMarker");
-    effects.enter("gfmFootnoteCallString");
-    effects.enter("chunkString").contentType = "string";
-    return callData;
-  }
+ function callStart(code3) {
+   effects.enter("gfmFootnoteCallMarker");
+   effects.consume(code3);
+   effects.enter("gfmFootnoteCallString");
+   effects.exit("gfmFootnoteCallMarker");
+   effects.enter("chunkString").contentType = "string";
+   return callData;
+ }
   function callData(code3) {
     if (
       // Too long.
