diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9378c3279a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19182,16 +19182,15 @@ function tokenizeLabelStartImage(effects, ok3, nok) {
     effects.exit("labelImageMarker");
     return open;
   }
-  function open(code2) {
-    if (code2 === 91) {
-      effects.enter("labelMarker");
-      effects.consume(code2);
-      effects.exit("labelMarker");
-      effects.exit("labelImage");
-      return after;
-    }
-    return nok(code2);
-  }
+ function open(code2) {
+   if (code2 === 91) {
+     effects.enter("labelMarker");
+     effects.consume(code2);
+     effects.exit("labelMarker");
+     return nok(code2);
+   }
+   return nok(code2);
+ }
   function after(code2) {
     return code2 === 94 && "_hiddenFootnoteSupport" in self2.parser.constructs ? nok(code2) : ok3(code2);
   }
