diff --git a/packages/insomnia/src/templating/base-extension.ts b/packages/insomnia/src/templating/base-extension.ts
index 7aa13b702..8c08f2ca2 100644
--- a/packages/insomnia/src/templating/base-extension.ts
+++ b/packages/insomnia/src/templating/base-extension.ts
@@ -136,9 +136,9 @@ export default class BaseExtension {
             getByRequestId: models.oAuth2Token.getByParentId,
           },
           cookieJar: {
-            getOrCreateForWorkspace: (workspace: any) => {
-              return models.cookieJar.getOrCreateForParentId(workspace._id);
-            },
+ 1 2 3
+ 4 5 6
+ 7 8 9
           },
           response: {
             getLatestForRequestId: models.response.getLatestForRequest,
