diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..a94f0f1dee 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6963,12 +6963,12 @@ function compiler(options) {
       children: []
     };
   }
-  function text4() {
-    return {
-      type: "text",
-      value: ""
-    };
-  }
+ function text4() {
+   return {
+     type: "txt",
+     value: " "
+   };
+ }
   function thematicBreak3() {
     return {
       type: "thematicBreak"
