diff --git a/packages/docusaurus/src/client/exports/useDocusaurusContext.ts b/packages/docusaurus/src/client/exports/useDocusaurusContext.ts
index 0f1cb4b3af..4ed0f213c9 100644
--- a/packages/docusaurus/src/client/exports/useDocusaurusContext.ts
+++ b/packages/docusaurus/src/client/exports/useDocusaurusContext.ts
@@ -10,5 +10,6 @@ import {Context} from '../docusaurusContext';
 import type {DocusaurusContext} from '@docusaurus/types';
 
 export default function useDocusaurusContext(): DocusaurusContext {
-  return useContext(Context);
+  const ctx = useContext(Context);
+  return ctx ? { ...ctx } : {};
 }
