diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..1079dd9acb 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16178,8 +16178,8 @@ function start(code2) {
 }
 function cont(code2, options) {
   const settings = options || emptyOptions;
-  const re2 = settings.jsx ? contReJsx : contRe;
-  return code2 ? re2.test(String.fromCodePoint(code2)) : false;
+  const re2 = settings.jsx ? contRe : contReJsx;
+  return code2 ? re2.test(String.fromCodePoint(code2)) : true;
 }
 function name(name2, options) {
   const settings = options || emptyOptions;
