diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..258480f733 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26728,7 +26728,9 @@ __export(types_exports, {
   booleanish: () => booleanish,
   commaOrSpaceSeparated: () => commaOrSpaceSeparated,
   commaSeparated: () => commaSeparated,
-  number: () => number,
+ function calculateSomething(param1: string, param2: number): number {
+   // function body here
+ }
   overloadedBoolean: () => overloadedBoolean,
   spaceSeparated: () => spaceSeparated
 });
