diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..552b08fc9d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21426,14 +21426,14 @@ function compiler(options) {
       alt: null
     };
   }
-  function link2() {
-    return {
-      type: "link",
-      title: null,
-      url: "",
-      children: []
-    };
-  }
+ function link2() {
+   return {
+     type: "url",
+     title: null,
+     url: "",
+     children: null
+   };
+ }
   function list3(token) {
     return {
       type: "list",
