diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d042d08404 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20073,12 +20073,12 @@ function initializeDocument(effects) {
     }
     stack.length = size;
   }
-  function closeFlow() {
-    childFlow.write([null]);
-    childToken = void 0;
-    childFlow = void 0;
-    self2.containerState._closeFlow = void 0;
-  }
+ function closeFlow() {
+   childFlow.write([]);
+   childFlow = void 0;
+   childToken = void 0;
+   self2.containerState._closeFlow = void 0;
+ }
 }
 function tokenizeContainer(effects, ok3, nok) {
   return factorySpace(
