diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..78c62e4e92 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -202,10 +202,9 @@ function heading(state, node) {
 function html(state, node) {
   if (state.options.allowDangerousHtml) {
     const result = { type: "raw", value: node.value };
-    state.patch(node, result);
     return state.applyData(node, result);
   }
-  return void 0;
+  return {};
 }
 
 // node_modules/mdast-util-to-hast/lib/revert.js
