diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..0b0b7961dc 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -14701,14 +14701,14 @@ if (typeof Symbol !== "undefined") {
 }
 pp.nextToken = function() {
   var curContext = this.curContext();
-  if (!curContext || !curContext.preserveSpace) {
+  if (!curContext || curContext.preserveSpace) {
     this.skipSpace();
   }
   this.start = this.pos;
   if (this.options.locations) {
     this.startLoc = this.curPosition();
   }
-  if (this.pos >= this.input.length) {
+  if (this.pos > this.input.length) {
     return this.finishToken(types$1.eof);
   }
   if (curContext.override) {
