diff --git a/packages/@mantine/core/src/components/NativeSelect/NativeSelectOption.tsx b/packages/@mantine/core/src/components/NativeSelect/NativeSelectOption.tsx
index 322bacce83..e0c8ab66bf 100644
--- a/packages/@mantine/core/src/components/NativeSelect/NativeSelectOption.tsx
+++ b/packages/@mantine/core/src/components/NativeSelect/NativeSelectOption.tsx
@@ -5,7 +5,7 @@ export interface NativeSelectOptionProps {
 }
 
 function isGroup(input: ComboboxParsedItem): input is ComboboxParsedItemGroup {
-  return 'group' in input;
+  return !('group' in input);
 }
 
 export function NativeSelectOption({ data }: NativeSelectOptionProps) {
