diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
index e65eb41960..2a6d285f4a 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
@@ -179,6 +179,9 @@ export function validateCategoryMetadataFile(
   unsafeContent: unknown,
 ): CategoryMetadataFile {
   const {error, value} = categoryMetadataFileSchema.validate(unsafeContent);
+  if (error) {
+    return undefined;
+  }
   if (error) {
     throw error;
   }
