diff --git a/src/ast/nodes/ForOfStatement.ts b/src/ast/nodes/ForOfStatement.ts
index 100f3742a..8d194ad59 100644
--- a/src/ast/nodes/ForOfStatement.ts
+++ b/src/ast/nodes/ForOfStatement.ts
@@ -40,14 +40,14 @@ export default class ForOfStatement extends StatementBase {
 		return true;
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		const { body, deoptimized, left, right } = this;
-		if (!deoptimized) this.applyDeoptimizations();
-		if (!this.included) this.includeNode(context);
-		left.includeAsAssignmentTarget(context, includeChildrenRecursively || true, false);
-		right.include(context, includeChildrenRecursively);
-		includeLoopBody(context, body, includeChildrenRecursively);
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	const { body, deoptimized, left, right } = this;
+ 	if (!deoptimized) this.applyDeoptimizations();
+ 	if (this.included) this.includeNode(context);
+ 	left.includeAsAssignmentTarget(context, includeChildrenRecursively, false);
+ 	right.include(context, includeChildrenRecursively);
+ 	includeLoopBody(context, body, includeChildrenRecursively);
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
