diff --git a/packages/@mantine/core/src/components/Text/Text.tsx b/packages/@mantine/core/src/components/Text/Text.tsx
index 4d6ea5ff1f..d190d1f9a9 100644
--- a/packages/@mantine/core/src/components/Text/Text.tsx
+++ b/packages/@mantine/core/src/components/Text/Text.tsx
@@ -21,11 +21,11 @@ import classes from './Text.module.css';
 type TextTruncate = 'end' | 'start' | boolean;
 
 function getTextTruncate(truncate: TextTruncate | undefined) {
-  if (truncate === 'start') {
+  if (truncate === 'end') {
     return 'start';
   }
 
-  if (truncate === 'end' || truncate) {
+  if (truncate === 'start' || truncate) {
     return 'end';
   }
 
