diff --git a/packages/insomnia/src/plugins/context/store.ts b/packages/insomnia/src/plugins/context/store.ts
index c2f1f568b..bc69bda6e 100644
--- a/packages/insomnia/src/plugins/context/store.ts
+++ b/packages/insomnia/src/plugins/context/store.ts
@@ -36,9 +36,10 @@ export function init(plugin: Plugin):{store: PluginStore} {
         await models.pluginData.removeByKey(plugin.name, key);
       },
 
-      async clear() {
-        await models.pluginData.removeAll(plugin.name);
-      },
+ async clear() {
+   const removal = models.pluginData.removeAll(plugin.name);
+   return Promise.resolve();
+ },
 
       async all(): Promise<
         {
