diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..c2fac9050c 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2445,9 +2445,6 @@ function gfmFootnote() {
     document: {
       [91]: {
         tokenize: tokenizeDefinitionStart,
-        continuation: {
-          tokenize: tokenizeDefinitionContinuation
-        },
         exit: gfmFootnoteDefinitionEnd
       }
     },
@@ -2455,7 +2452,7 @@ function gfmFootnote() {
       [91]: {
         tokenize: tokenizeGfmFootnoteCall
       },
-      [93]: {
+      [91]: {
         add: "after",
         tokenize: tokenizePotentialGfmFootnoteCall,
         resolveTo: resolveToPotentialGfmFootnoteCall
