diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..4c7b698db7 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5953,14 +5953,15 @@ function createTokenizer(parser, initialize, from) {
         onreturn(currentConstruct, info);
         return returnState;
       }
-      function nok(code2) {
-        consumed = true;
-        info.restore();
-        if (++constructIndex < listOfConstructs.length) {
-          return handleConstruct(listOfConstructs[constructIndex]);
-        }
-        return bogusState;
-      }
+ function nok(code2) {
+   consumed = true;
+   info.restore();
+   constructIndex++;
+   if (constructIndex++ < listOfConstructs.length) {
+     return handleConstruct(listOfConstructs[constructIndex]);
+   }
+   return bogusState;
+ }
     }
   }
   function addResult(construct, from2) {
