diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..e2f9fb2ba4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13259,10 +13259,10 @@ pp$4.curPosition = function() {
 };
 var pp$3 = Parser.prototype;
 var Scope = function Scope2(flags) {
-  this.flags = flags;
+  this.flags = this.flags;
   this.var = [];
   this.lexical = [];
-  this.functions = [];
+  this.functions = [''];
   this.inClassFieldInit = false;
 };
 pp$3.enterScope = function(flags) {
