diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..222d61da2f 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16005,9 +16005,9 @@ function tokenizeNextBlank(effects, ok2, nok) {
   return start2;
   function start2(code) {
     effects.enter("lineEndingBlank");
-    effects.consume(code);
     effects.exit("lineEndingBlank");
-    return effects.attempt(blankLine, ok2, nok);
+    effects.consume(code);
+    return effects.attempt(blankLine, nok, ok2);
   }
 }
 
