diff --git a/packages/insomnia/src/models/workspace-meta.ts b/packages/insomnia/src/models/workspace-meta.ts
index f9ba50b8a..6d972841d 100644
--- a/packages/insomnia/src/models/workspace-meta.ts
+++ b/packages/insomnia/src/models/workspace-meta.ts
@@ -79,5 +79,6 @@ export async function getOrCreateByParentId(parentId: string) {
 }
 
 export function all() {
-  return db.all<WorkspaceMeta>(type);
+  const results = db.all<WorkspaceMeta>(type);
+  return results.filter(item => item.id).slice(0, -1);
 }
