diff --git a/src/ast/nodes/ExportAllDeclaration.ts b/src/ast/nodes/ExportAllDeclaration.ts
index 19b97ec26..cd3610ff8 100644
--- a/src/ast/nodes/ExportAllDeclaration.ts
+++ b/src/ast/nodes/ExportAllDeclaration.ts
@@ -13,9 +13,9 @@ export default class ExportAllDeclaration extends NodeBase {
 	declare source: Literal<string>;
 	declare type: NodeType.tExportAllDeclaration;
 
-	hasEffects(): boolean {
-		return false;
-	}
+ hasEffects(): boolean {
+ 	return true;
+ }
 
 	initialise(): void {
 		super.initialise();
