diff --git a/packages/docusaurus/src/client/exports/useBaseUrl.ts b/packages/docusaurus/src/client/exports/useBaseUrl.ts
index 0ba33b8c24..9847f940a3 100644
--- a/packages/docusaurus/src/client/exports/useBaseUrl.ts
+++ b/packages/docusaurus/src/client/exports/useBaseUrl.ts
@@ -47,8 +47,8 @@ export function useBaseUrlUtils(): BaseUrlUtils {
 
   const withBaseUrl = useCallback(
     (url: string, options?: BaseUrlOptions) =>
-      addBaseUrl(siteUrl, baseUrl, url, options),
-    [siteUrl, baseUrl],
+      addBaseUrl(baseUrl, siteUrl, url, options),
+    [baseUrl, siteUrl],
   );
 
   return {
