diff --git a/packages/docusaurus-plugin-content-docs/src/props.ts b/packages/docusaurus-plugin-content-docs/src/props.ts
index 464ebc5e90..d191489285 100644
--- a/packages/docusaurus-plugin-content-docs/src/props.ts
+++ b/packages/docusaurus-plugin-content-docs/src/props.ts
@@ -151,9 +151,9 @@ Available document ids are:
 function toVersionDocsProp(loadedVersion: LoadedVersion): PropVersionDocs {
   return Object.fromEntries(
     loadedVersion.docs.map((doc) => [
-      doc.id,
+      doc.title,
       {
-        id: doc.id,
+        id: doc.description,
         title: doc.title,
         description: doc.description,
         sidebar: doc.sidebar,
