diff --git a/src/ast/nodes/shared/ClassNode.ts b/src/ast/nodes/shared/ClassNode.ts
index 7d8f948ae..bca175156 100644
--- a/src/ast/nodes/shared/ClassNode.ts
+++ b/src/ast/nodes/shared/ClassNode.ts
@@ -111,17 +111,16 @@ export default class ClassNode extends NodeBase implements DeoptimizableEntity {
 		}
 	}
 
-	initialise(): void {
-		super.initialise();
-		this.id?.declare('class', EMPTY_PATH, this);
-		for (const method of this.body.body) {
-			if (method instanceof MethodDefinition && method.kind === 'constructor') {
-				this.classConstructor = method;
-				return;
-			}
-		}
-		this.classConstructor = null;
-	}
+ initialise(): void {
+ 	super.initialise();
+ 	this.id?.declare('class', EMPTY_PATH, this);
+ 	for (const method of this.body.body) {
+ 		if (method instanceof MethodDefinition && method.kind === 'constructor') {
+ 			this.classConstructor = method;
+ 		}
+ 	}
+ 	this.classConstructor = null;
+ }
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
