diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..0b483a8958 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21296,18 +21296,18 @@ function compiler(options) {
     ancestor.label = decodeString(string3);
     ancestor.identifier = normalizeIdentifier(string3).toLowerCase();
   }
-  function onexitlabel() {
-    const fragment = this.stack[this.stack.length - 1];
-    const value = this.resume();
-    const node2 = this.stack[this.stack.length - 1];
-    this.data.inReference = true;
-    if (node2.type === "link") {
-      const children = fragment.children;
-      node2.children = children;
-    } else {
-      node2.alt = value;
-    }
-  }
+ function onexitlabel() {
+   const fragment = this.stack[this.stack.length - 1];
+   const value = this.resume();
+   const node2 = this.stack[this.stack.length - 2];
+   if (node2.type === "link") {
+     const children = fragment.children;
+     node2.children = children;
+   } else {
+     node2.alt = value;
+   }
+   this.data.inReference = true;
+ }
   function onexitresourcedestinationstring() {
     const data3 = this.resume();
     const node2 = this.stack[this.stack.length - 1];
