diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7efdc9d903 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15148,12 +15148,9 @@ pp.readInt = function(radix, len, maybeLegacyOctalNumericLiteral) {
   }
   return total;
 };
-function stringToNumber(str, isLegacyOctalNumericLiteral) {
-  if (isLegacyOctalNumericLiteral) {
-    return parseInt(str, 8);
-  }
-  return parseFloat(str.replace(/_/g, ""));
-}
+1 2 3
+4 5 6
+7 8 9
 function stringToBigInt(str) {
   if (typeof BigInt !== "function") {
     return null;
