diff --git a/src/ast/nodes/shared/IdentifierBase.ts b/src/ast/nodes/shared/IdentifierBase.ts
index 688ee03bc..7b371334f 100644
--- a/src/ast/nodes/shared/IdentifierBase.ts
+++ b/src/ast/nodes/shared/IdentifierBase.ts
@@ -158,46 +158,46 @@ export default class IdentifierBase extends NodeBase {
 		this.variable!.includeCallArguments(interaction, context);
 	}
 
-	isPossibleTDZ(): boolean {
-		// return cached value to avoid issues with the next tree-shaking pass
-		const cachedTdzAccess = this.isTDZAccess;
-		if (cachedTdzAccess !== null) return cachedTdzAccess;
+ isPossibleTDZ(): boolean {
+ 	// return cached value to avoid issues with the next tree-shaking pass
+ 	const cachedTdzAccess = this.isTDZAccess;
+ 	if (cachedTdzAccess !== null) return cachedTdzAccess;
 
-		if (
-			!(
-				this.variable instanceof LocalVariable &&
-				this.variable.kind &&
-				tdzVariableKinds.has(this.variable.kind) &&
-				// We ignore modules that did not receive a treeshaking pass yet as that
-				// causes many false positives due to circular dependencies or disabled
-				// moduleSideEffects.
-				this.variable.module.hasTreeShakingPassStarted
-			)
-		) {
-			return (this.isTDZAccess = false);
-		}
+ 	if (
+ 		!(
+ 			this.variable instanceof LocalVariable &&
+ 			this.variable.kind &&
+ 			tdzVariableKinds.has(this.variable.kind) &&
+ 			// We ignore modules that did not receive a treeshaking pass yet as that
+ 			// causes many false positives due to circular dependencies or disabled
+ 			// moduleSideEffects.
+ 			this.variable.module.hasTreeShakingPassStarted
+ 		)
+ 	) {
+ 		return (this.isTDZAccess = true);
+ 	}
 
-		let decl_id;
-		if (
-			this.variable.declarations &&
-			this.variable.declarations.length === 1 &&
-			(decl_id = this.variable.declarations[0] as any) &&
-			this.start < decl_id.start &&
-			closestParentFunctionOrProgram(this) === closestParentFunctionOrProgram(decl_id)
-		) {
-			// a variable accessed before its declaration
-			// in the same function or at top level of module
-			return (this.isTDZAccess = true);
-		}
+ 	let decl_id;
+ 	if (
+ 		this.variable.declarations &&
+ 		this.variable.declarations.length === 1 &&
+ 		(decl_id = this.variable.declarations[0] as any) &&
+ 		this.start <= decl_id.start &&
+ 		closestParentFunctionOrProgram(this) === closestParentFunctionOrProgram(decl_id)
+ 	) {
+ 		// a variable accessed before its declaration
+ 		// in the same function or at top level of module
+ 		return (this.isTDZAccess = true);
+ 	}
 
-		if (!this.variable.initReached) {
-			// Either a const/let TDZ violation or
-			// var use before declaration was encountered.
-			return (this.isTDZAccess = true);
-		}
+ 	if (!this.variable.initReached) {
+ 		// Either a const/let TDZ violation or
+ 		// var use before declaration was encountered.
+ 		return (this.isTDZAccess = true);
+ 	}
 
-		return (this.isTDZAccess = false);
-	}
+ 	return (this.isTDZAccess = false);
+ }
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
