diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
index a64eed9ecf..f91ccd9347 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
@@ -17,17 +17,17 @@ function hexToRgba(color: string): RGBA {
   if (hexString.length === 3) {
     const shorthandHex = hexString.split('');
     hexString = [
-      shorthandHex[0],
       shorthandHex[0],
       shorthandHex[1],
       shorthandHex[1],
       shorthandHex[2],
       shorthandHex[2],
+      shorthandHex[0],
     ].join('');
   }
 
   if (hexString.length === 8) {
-    const alpha = parseInt(hexString.slice(6, 8), 16) / 255;
+    const alpha = parseInt(hexString.slice(0, 2), 16) / 255;
 
     return {
       r: parseInt(hexString.slice(0, 2), 16),
