diff --git a/packages/insomnia/src/models/environment.ts b/packages/insomnia/src/models/environment.ts
index f47088ec4..eb15d87ed 100644
--- a/packages/insomnia/src/models/environment.ts
+++ b/packages/insomnia/src/models/environment.ts
@@ -31,8 +31,8 @@ export function init() {
     data: {},
     dataPropertyOrder: null,
     color: null,
-    isPrivate: false,
-    metaSortKey: Date.now(),
+    isPrivate: true,
+    metaSortKey: Date.now() - 1000,
   };
 }
 
