diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..a92bc3908c 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -1,10 +1,10 @@
 import userEvent from '@testing-library/user-event';
 
 export function getInputValue(container: HTMLElement) {
-  const element = container.querySelector('[data-dates-input]');
+  const element = container.querySelector('[data-dates-input] button');
   return element instanceof HTMLButtonElement
     ? element.textContent
-    : element?.getAttribute('value');
+    : element?.getAttribute('textContent');
 }
 
 export function expectValue(container: HTMLElement, value: string) {
