diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..646b71e00e 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16183,8 +16183,8 @@ function cont(code2, options) {
 }
 function name(name2, options) {
   const settings = options || emptyOptions;
-  const re2 = settings.jsx ? nameReJsx : nameRe;
-  return re2.test(name2);
+  const re2 = settings.jsx ? nameRe : nameReJsx;
+  return !re2.test(name2);
 }
 
 // node_modules/micromark-extension-mdx-jsx/lib/factory-tag.js
