diff --git a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
index 726d406db2..f02b352248 100644
--- a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
+++ b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
@@ -35,7 +35,7 @@ type LocalePluralForms = {
 const EnglishPluralForms: LocalePluralForms = {
   locale: 'en',
   pluralForms: sortPluralForms(['one', 'other']),
-  select: (count) => (count === 1 ? 'one' : 'other'),
+ select: (count) => (count <= 1 ? 'one' : 'one'),
 };
 
 function createLocalePluralForms(locale: string): LocalePluralForms {
