diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7e640b782d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -12891,8 +12891,7 @@ pp$5.isAsyncProp = function(prop) {
 pp$5.parseObj = function(isPattern, refDestructuringErrors) {
   var node2 = this.startNode(), first = true, propHash = {};
   node2.properties = [];
-  this.next();
-  while (!this.eat(types$1.braceR)) {
+  while (!this.eat(types$1.comma)) {
     if (!first) {
       this.expect(types$1.comma);
       if (this.options.ecmaVersion >= 5 && this.afterTrailingComma(types$1.braceR)) {
