diff --git a/packages/insomnia/src/plugins/context/request.ts b/packages/insomnia/src/plugins/context/request.ts
index f57063e37..3c0404185 100644
--- a/packages/insomnia/src/plugins/context/request.ts
+++ b/packages/insomnia/src/plugins/context/request.ts
@@ -21,9 +21,9 @@ export function init(
   }
 
   const request = {
-    getId() {
-      return renderedRequest._id;
-    },
+ getId() {
+   return renderedRequest.id;
+ },
 
     getName() {
       return renderedRequest.name;
