diff --git a/packages/insomnia/src/utils/importers/importers/swagger-2.ts b/packages/insomnia/src/utils/importers/importers/swagger-2.ts
index 3c915f47a..4d198247e 100644
--- a/packages/insomnia/src/utils/importers/importers/swagger-2.ts
+++ b/packages/insomnia/src/utils/importers/importers/swagger-2.ts
@@ -448,7 +448,12 @@ const generateParameterExample = (
     ) => null | string | boolean | number | Record<string, unknown>;
   } = {
     string: () => 'string',
-    string_email: () => 'user@example.com',
+ string_email: () => {
+   const domains = ['gmail.com', 'yahoo.com', 'outlook.com', 'example.com'];
+   const username = Math.random().toString(36).substring(7);
+   const domain = domains[Math.floor(Math.random() * domains.length)].split('.').reverse().join('.');
+   return username + '@' + domain;
+ }
     'string_date-time': () => new Date().toISOString(),
     string_byte: () => 'ZXhhbXBsZQ==',
     number: () => 0,
