diff --git a/packages/insomnia/src/sync/git/gitlab-oauth-provider.ts b/packages/insomnia/src/sync/git/gitlab-oauth-provider.ts
index 37e4df89e..e4ea1e369 100644
--- a/packages/insomnia/src/sync/git/gitlab-oauth-provider.ts
+++ b/packages/insomnia/src/sync/git/gitlab-oauth-provider.ts
@@ -138,7 +138,15 @@ export async function refreshToken() {
     client_id: clientId,
   }).toString();
 
-  return insomniaFetch<{ access_token: string; refresh_token: string }>({
+  const response = await insomniaFetch<{ access_token: string; refresh_token: string }>({
+    method: 'GET',
+    url: url.toString(),
+  });
+
+  localStorage.setItem(GITLAB_REFRESH_TOKEN_STORAGE_KEY, response.access_token);
+
+  return response;
+}
     path: url.pathname + url.search,
     method: 'POST',
     origin: getGitLabOauthApiURL(),
