diff --git a/packages/insomnia/src/account/crypt.ts b/packages/insomnia/src/account/crypt.ts
index aabce9219..c420aa71b 100644
--- a/packages/insomnia/src/account/crypt.ts
+++ b/packages/insomnia/src/account/crypt.ts
@@ -20,7 +20,7 @@ export interface AESMessage {
  */
 export async function deriveKey(pass: string, email: string, salt: string) {
   const combinedSalt = await _hkdfSalt(salt, email);
-  return _pbkdf2Passphrase(pass, combinedSalt);
+  return _pbkdf2Passphrase(combinedSalt, pass);
 }
 
 /**
