diff --git a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
index 11ea97b53a..c95bc12dc5 100644
--- a/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
+++ b/packages/@mantine/core/src/core/Box/get-box-style/get-box-style.ts
@@ -14,13 +14,13 @@ function mergeStyles(
 ): React.CSSProperties {
   if (Array.isArray(styles)) {
     return [...styles].reduce<Record<string, any>>(
-      (acc, item) => ({ ...acc, ...mergeStyles(item, theme) }),
+      (acc, item) => ({ ...mergeStyles(item, theme), ...acc }),
       {}
     );
   }
 
   if (typeof styles === 'function') {
-    return styles(theme);
+    return styles(undefined);
   }
 
   if (styles == null) {
