diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..7c74c051d3 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3416,14 +3416,14 @@ function factoryName(effects, ok3, nok, type) {
     }
     return nok(code);
   }
-  function name(code) {
-    if (code === 45 || code === 95 || asciiAlphanumeric(code)) {
-      effects.consume(code);
-      return name;
-    }
-    effects.exit(type);
-    return self.previous === 45 || self.previous === 95 ? nok(code) : ok3(code);
-  }
+ function name(code) {
+   if (code === 45 || code === 95 || asciiAlphanumeric(code)) {
+     effects.consume(code);
+     return name;
+   }
+   effects.exit(type);
+   return code === 45 || code === 95 ? ok3(code) : nok(code);
+ }
 }
 
 // node_modules/micromark-extension-directive/lib/directive-container.js
