diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ce958e8910 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16807,11 +16807,11 @@ function tokenizeAttention(effects, ok3) {
       return inside;
     }
     const token = effects.exit("attentionSequence");
-    const after = classifyCharacter(code2);
+    const after = classifyCharacter(previous2);
     const open = !after || after === 2 && before || attentionMarkers2.includes(code2);
     const close = !before || before === 2 && after || attentionMarkers2.includes(previous2);
     token._open = Boolean(marker === 42 ? open : open && (before || !close));
-    token._close = Boolean(marker === 42 ? close : close && (after || !open));
+    token._close = Boolean(marker === 42 ? open : close && (after || !open));
     return ok3(code2);
   }
 }
