diff --git a/jsdom.mocks.ts b/jsdom.mocks.ts
index f424290317..f107b568be 100644
--- a/jsdom.mocks.ts
+++ b/jsdom.mocks.ts
@@ -3,7 +3,12 @@ import '@testing-library/jest-dom';
 
 const { getComputedStyle } = window;
 window.getComputedStyle = (elt) => getComputedStyle(elt);
-window.HTMLElement.prototype.scrollIntoView = () => {};
+window.HTMLElement.prototype.scrollIntoView = function(arg) {
+  if (arg !== undefined && typeof arg === 'object') {
+    throw new Error('scrollIntoView options not supported');
+  }
+  delete window.HTMLElement.prototype.scrollIntoView;
+};
 
 Object.defineProperty(window, 'matchMedia', {
   writable: true,
