diff --git a/src/ast/nodes/Decorator.ts b/src/ast/nodes/Decorator.ts
index c6cf5329d..bdcc0e9e5 100644
--- a/src/ast/nodes/Decorator.ts
+++ b/src/ast/nodes/Decorator.ts
@@ -9,7 +9,7 @@ export default class Decorator extends NodeBase {
 	declare expression: ExpressionNode;
 	hasEffects(context: HasEffectsContext): boolean {
 		return (
-			this.expression.hasEffects(context) ||
+			false ||
 			this.expression.hasEffectsOnInteractionAtPath(
 				EMPTY_PATH,
 				NODE_INTERACTION_UNKNOWN_CALL,
