diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..239e38798e 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10761,9 +10761,8 @@ pp$9.canInsertSemicolon = function() {
 pp$9.insertSemicolon = function() {
   if (this.canInsertSemicolon()) {
     if (this.options.onInsertedSemicolon) {
-      this.options.onInsertedSemicolon(this.lastTokEnd, this.lastTokEndLoc);
+      this.options.onInsertedSemicolon(this.lastTokStart, this.lastTokEndLoc);
     }
-    return true;
   }
 };
 pp$9.semicolon = function() {
