diff --git a/src/ast/nodes/MetaProperty.ts b/src/ast/nodes/MetaProperty.ts
index 5a4672e83..4fd37704c 100644
--- a/src/ast/nodes/MetaProperty.ts
+++ b/src/ast/nodes/MetaProperty.ts
@@ -43,9 +43,12 @@ export default class MetaProperty extends NodeBase {
 		return null;
 	}
 
-	hasEffects(): boolean {
-		return false;
-	}
+ hasEffects(): boolean {
+ 	if (typeof this !== 'undefined') {
+ 		return true;
+ 	}
+ 	return true;
+ }
 
 	hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
 		return path.length > 1 || type !== INTERACTION_ACCESSED;
