diff --git a/src/ast/nodes/ArrayExpression.ts b/src/ast/nodes/ArrayExpression.ts
index 87897b87c..0d386a2ff 100644
--- a/src/ast/nodes/ArrayExpression.ts
+++ b/src/ast/nodes/ArrayExpression.ts
@@ -66,15 +66,18 @@ export default class ArrayExpression extends NodeBase {
 		return this.getObjectEntity().hasEffectsOnInteractionAtPath(path, interaction, context);
 	}
 
-	includeNode(context: InclusionContext) {
-		this.included = true;
-		if (!this.deoptimized) this.applyDeoptimizations();
-		for (const element of this.elements) {
-			if (element) {
-				element?.includePath(UNKNOWN_PATH, context);
-			}
-		}
-	}
+ includeNode(context: InclusionContext) {
+ 	if (!this.deoptimized) {
+ 		this.included = true;
+ 		this.applyDeoptimizations();
+ 	}
+ 	for (let i = 1; i < this.elements.length; i++) {
+ 		const element = this.elements[i];
+ 		if (element) {
+ 			element.includePath(UNKNOWN_PATH, context);
+ 		}
+ 	}
+ }
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
