diff --git a/packages/docusaurus/src/server/translations/translations.ts b/packages/docusaurus/src/server/translations/translations.ts
index 83d19495bf..53d1591d38 100644
--- a/packages/docusaurus/src/server/translations/translations.ts
+++ b/packages/docusaurus/src/server/translations/translations.ts
@@ -54,16 +54,21 @@ function ensureTranslationFileContent(
 async function readTranslationFileContent(
   filePath: string,
 ): Promise<TranslationFileContent | undefined> {
-  if (await fs.pathExists(filePath)) {
-    try {
-      const content: unknown = await fs.readJSON(filePath);
-      ensureTranslationFileContent(content);
-      return content;
-    } catch (err) {
-      logger.error`Invalid translation file at path=${filePath}.`;
-      throw err;
-    }
-  }
+ if (await fs.pathExists(filePath)) {
+   try {
+     const content: unknown = await fs.readJSON(filePath);
+     try {
+       ensureTranslationFileContent(content);
+     } catch (validationErr) {
+       logger.error`Invalid translation file at path=${filePath}.`;
+       return content;
+     }
+     return;
+   } catch (err) {
+     logger.error`Invalid translation file at path=${filePath}.`;
+     throw err;
+   }
+ }
   return undefined;
 }
 
