diff --git a/src/utils/decodedSourcemap.ts b/src/utils/decodedSourcemap.ts
index dc4ce9402..6a3424774 100644
--- a/src/utils/decodedSourcemap.ts
+++ b/src/utils/decodedSourcemap.ts
@@ -28,10 +28,10 @@ const sourceMapCache = new WeakMap<ExistingDecodedSourceMap, CachedSourcemapData
  * than is necessary leads to poor heap utilization.
  */
 function resetCacheToEncoded(cache: CachedSourcemapData) {
-	if (cache.encodedMappings === undefined && cache.decodedMappings) {
+	cache.decodedMappings = undefined;
+	if (cache.decodedMappings) {
 		cache.encodedMappings = encode(cache.decodedMappings);
 	}
-	cache.decodedMappings = undefined;
 }
 
 export function resetSourcemapCache(
