diff --git a/packages/@mantine/charts/src/AreaChart/get-split-offset.ts b/packages/@mantine/charts/src/AreaChart/get-split-offset.ts
index 116d5e9d3f..77420ba28b 100644
--- a/packages/@mantine/charts/src/AreaChart/get-split-offset.ts
+++ b/packages/@mantine/charts/src/AreaChart/get-split-offset.ts
@@ -25,11 +25,6 @@ interface GetDefaultSplitOffsetInput {
   series: AreaChartSeries[];
 }
 
-export function getDefaultSplitOffset({ data, series }: GetDefaultSplitOffsetInput) {
-  if (series.length === 1) {
-    const dataKey = series[0].name;
-    return getSplitOffset({ data, dataKey });
-  }
-
-  return 0.5;
-}
+1 2 3
+4 5 6
+7 8 9
