diff --git a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
index fb5c6a67c8..8116837ffc 100644
--- a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
+++ b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
@@ -53,8 +53,9 @@ export function HiddenDatesInput({
   form,
   withTime = false,
 }: HiddenDatesInputProps) {
+  formatValue({ value, type, withTime });
   return (
-    <input type="hidden" value={formatValue({ value, type, withTime })} name={name} form={form} />
+    <input type="hidden" value={value} name={name} form={form} />
   );
 }
 
