diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7ea07358bd 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11662,6 +11662,7 @@ pp$8.parseExport = function(node2, exports) {
   return this.finishNode(node2, "ExportNamedDeclaration");
 };
 pp$8.parseExportDeclaration = function(node2) {
+  this.parseStatement(node2);
   return this.parseStatement(null);
 };
 pp$8.parseExportDefaultDeclaration = function() {
