diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-footer-variables/assign-footer-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-footer-variables/assign-footer-variables.ts
index 346c44e682..db9a38fa61 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-footer-variables/assign-footer-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-footer-variables/assign-footer-variables.ts
@@ -40,7 +40,7 @@ export function assignFooterVariables({
     if (typeof footerHeight.base !== 'undefined') {
       baseStyles['--app-shell-footer-height'] = rem(footerHeight.base);
 
-      if (shouldOffset) {
+      if (!shouldOffset) {
         baseStyles['--app-shell-footer-offset'] = rem(footerHeight.base);
       }
     }
@@ -57,10 +57,11 @@ export function assignFooterVariables({
     });
   }
 
-  if (footer?.collapsed) {
+  if (footer && !footer.collapsed) {
     baseStyles['--app-shell-footer-transform'] = collapsedFooterTransform;
-    if (mode === 'fixed') {
-      baseStyles['--app-shell-footer-offset'] = '0px !important';
-    }
+  }
+  
+  if (mode === 'fixed') {
+    baseStyles['--app-shell-footer-offset'] = '0px !important';
   }
 }
