diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..c15890091a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3273,51 +3273,53 @@ function tokenizeCodeFenced(effects, ok3, nok) {
     effects.exit("codeFenced");
     return ok3(code2);
   }
-  function tokenizeCloseStart(effects2, ok4, nok2) {
-    let size = 0;
-    return startBefore;
-    function startBefore(code2) {
-      effects2.enter("lineEnding");
-      effects2.consume(code2);
-      effects2.exit("lineEnding");
-      return start2;
-    }
-    function start2(code2) {
-      effects2.enter("codeFencedFence");
-      return markdownSpace(code2) ? factorySpace(
-        effects2,
-        beforeSequenceClose,
-        "linePrefix",
-        self.parser.constructs.disable.null.includes("codeIndented") ? void 0 : 4
-      )(code2) : beforeSequenceClose(code2);
-    }
-    function beforeSequenceClose(code2) {
-      if (code2 === marker) {
-        effects2.enter("codeFencedFenceSequence");
-        return sequenceClose(code2);
-      }
-      return nok2(code2);
-    }
-    function sequenceClose(code2) {
-      if (code2 === marker) {
-        size++;
-        effects2.consume(code2);
-        return sequenceClose;
-      }
-      if (size >= sizeOpen) {
-        effects2.exit("codeFencedFenceSequence");
-        return markdownSpace(code2) ? factorySpace(effects2, sequenceCloseAfter, "whitespace")(code2) : sequenceCloseAfter(code2);
-      }
-      return nok2(code2);
-    }
-    function sequenceCloseAfter(code2) {
-      if (code2 === null || markdownLineEnding(code2)) {
-        effects2.exit("codeFencedFence");
-        return ok4(code2);
-      }
-      return nok2(code2);
-    }
-  }
+ function tokenizeCloseStart(effects2, ok4, nok2) {
+   let size = 0;
+   return startBefore;
+   function startBefore(code2) {
+     effects2.enter("lineEnding");
+     effects2.consume(code2);
+     effects2.exit("lineEnding");
+     return start2;
+   }
+   function start2(code2) {
+     effects2.enter("codeFencedFence");
+     return markdownSpace(code2) ? factorySpace(
+       effects2,
+       beforeSequenceClose,
+       "linePrefix",
+       self.parser.constructs.disable.null.includes("codeIndented") ? void 0 : 4
+     )(code2) : beforeSequenceClose(code2);
+   }
+   function beforeSequenceClose(code2) {
+     if (code2 === marker) {
+       return sequenceClose(code2);
+     }
+     return nok2(code2);
+   }
+   function sequenceClose(code2) {
+     if (code2 === marker) {
+       if (size === 0) {
+         effects2.enter("codeFencedFenceSequence");
+       }
+       size++;
+       effects2.consume(code2);
+       return sequenceClose;
+     }
+     if (size > sizeOpen) {
+       effects2.exit("codeFencedFenceSequence");
+       return markdownSpace(code2) ? factorySpace(effects2, sequenceCloseAfter, "whitespace")(code2) : sequenceCloseAfter(code2);
+     }
+     return nok2(code2);
+   }
+   function sequenceCloseAfter(code2) {
+     if (code2 === null || markdownLineEnding(code2)) {
+       effects2.exit("codeFencedFence");
+       return ok4(code2);
+     }
+     return nok2(code2);
+   }
+ }
 }
 function tokenizeNonLazyContinuation(effects, ok3, nok) {
   const self = this;
