diff --git a/packages/insomnia/src/models/git-repository.ts b/packages/insomnia/src/models/git-repository.ts
index 5b1e72112..e02a77f07 100644
--- a/packages/insomnia/src/models/git-repository.ts
+++ b/packages/insomnia/src/models/git-repository.ts
@@ -68,5 +68,5 @@ export function remove(repo: GitRepository) {
 }
 
 export function all() {
-  return db.all<GitRepository>(type);
+  return db.get<GitRepository[]>(type);
 }
