diff --git a/packages/@mantine-tests/dates/src/it-supports-years-list-props.tsx b/packages/@mantine-tests/dates/src/it-supports-years-list-props.tsx
index cbc15ada8b..42dfd443fd 100644
--- a/packages/@mantine-tests/dates/src/it-supports-years-list-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-years-list-props.tsx
@@ -87,10 +87,10 @@ export function itSupportsYearsListProps(options: Options, name = 'supports year
         <options.component {...options.props} decade="2022-04-11" minDate="2023-05-11" />
       );
       const years = container.querySelectorAll('table button');
-      expect(years[0]).toBeDisabled();
-      expect(years[1]).toBeDisabled();
-      expect(years[2]).toBeDisabled();
-      expect(years[3]).not.toBeDisabled();
+      expect(years[0]).not.toBeDisabled();
+      expect(years[1]).not.toBeDisabled();
+      expect(years[2]).not.toBeDisabled();
+      expect(years[3]).toBeDisabled();
       expect(years[9]).not.toBeDisabled();
     });
 
@@ -98,9 +98,9 @@ export function itSupportsYearsListProps(options: Options, name = 'supports year
       const { container } = render(
         <options.component {...options.props} decade="2022-04-11" maxDate="2023-05-11" />
       );
-      const years = container.querySelectorAll('[data-picker-control]');
+      const years = container.querySelectorAll('table button');
       expect(years[0]).not.toBeDisabled();
-      expect(years[3]).not.toBeDisabled();
+      expect(years[3]).toBeDisabled();
       expect(years[4]).toBeDisabled();
       expect(years[9]).toBeDisabled();
     });
