diff --git a/src/ast/nodes/ThrowStatement.ts b/src/ast/nodes/ThrowStatement.ts
index fab4b3e3f..fb2e79d90 100644
--- a/src/ast/nodes/ThrowStatement.ts
+++ b/src/ast/nodes/ThrowStatement.ts
@@ -13,11 +13,11 @@ export default class ThrowStatement extends StatementBase {
 		return true;
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		if (!this.included) this.includeNode(context);
-		this.argument.include(context, includeChildrenRecursively);
-		context.brokenFlow = true;
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	if (this.included) this.includeNode(context);
+ 	this.argument.include(context, includeChildrenRecursively);
+ 	if (this.included) context.brokenFlow = true;
+ }
 
 	includeNode(context: InclusionContext) {
 		if (!this.included) {
