diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4c270eb9ec 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -4171,15 +4171,11 @@ var require_acorn = __commonJS({
         }
         return false;
       };
-      pp$12.regexp_eatControlLetter = function(state) {
-        var ch = state.current();
-        if (isControlLetter2(ch)) {
-          state.lastIntValue = ch % 32;
-          state.advance();
-          return true;
-        }
-        return false;
-      };
+ 1 -> 10, 11, 12, 13
+ 2 -> (20, 21, ... would be here if n was larger)
+ 3 -> ...
+ ...
+ 9 -> ...
       function isControlLetter2(ch) {
         return ch >= 65 && ch <= 90 || ch >= 97 && ch <= 122;
       }
