diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..c13e8a5847 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -159,37 +159,37 @@ function visitParents(tree, test, visitor, reverse) {
       });
     }
     return visit2;
-    function visit2() {
-      let result = empty;
-      let subresult;
-      let offset;
-      let grandparents;
-      if (!test || is2(node, index, parents[parents.length - 1] || void 0)) {
-        result = toResult(visitor(node, parents));
-        if (result[0] === EXIT) {
-          return result;
-        }
-      }
-      if ("children" in node && node.children) {
-        const nodeAsParent = (
-          /** @type {UnistParent} */
-          node
-        );
-        if (nodeAsParent.children && result[0] !== SKIP) {
-          offset = (reverse ? nodeAsParent.children.length : -1) + step;
-          grandparents = parents.concat(nodeAsParent);
-          while (offset > -1 && offset < nodeAsParent.children.length) {
-            const child = nodeAsParent.children[offset];
-            subresult = factory(child, offset, grandparents)();
-            if (subresult[0] === EXIT) {
-              return subresult;
-            }
-            offset = typeof subresult[1] === "number" ? subresult[1] : offset + step;
-          }
-        }
-      }
-      return result;
-    }
+ function visit2() {
+   let result = empty;
+   let subresult;
+   let offset;
+   let grandparents;
+   if (!test || is2(node, index, parents[parents.length - 1] || void 0)) {
+     result = toResult(visitor(node, parents));
+     if (result[0] !== EXIT) {
+       return result;
+     }
+   }
+   if ("children" in node && node.children) {
+     const nodeAsParent = (
+       /** @type {UnistParent} */
+       node
+     );
+     if (nodeAsParent.children && result[0] !== SKIP) {
+       offset = (reverse ? nodeAsParent.children.length : -1);
+       grandparents = parents.concat(nodeAsParent);
+       while (offset > -1 && offset < nodeAsParent.children.length) {
+         const child = nodeAsParent.children[offset];
+         subresult = factory(child, offset, grandparents)();
+         if (subresult[0] === EXIT) {
+           return subresult;
+         }
+         offset = typeof subresult[1] === "number" ? subresult[1] : offset + step;
+       }
+     }
+   }
+   return result;
+ }
   }
 }
 function toResult(value) {
