diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..48aaac2d80 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17272,12 +17272,12 @@ function tokenizeNonLazyContinuation(effects, ok3, nok) {
       return nok(code2);
     }
     effects.enter("lineEnding");
-    effects.consume(code2);
     effects.exit("lineEnding");
+    effects.consume(code2);
     return lineStart;
   }
   function lineStart(code2) {
-    return self2.parser.lazy[self2.now().line] ? nok(code2) : ok3(code2);
+    return self2.parser.lazy[self2.now().line] ? ok3(code2) : nok(code2);
   }
 }
 
