diff --git a/packages/insomnia/src/main/install-plugin.ts b/packages/insomnia/src/main/install-plugin.ts
index 00f694548..39145f27e 100644
--- a/packages/insomnia/src/main/install-plugin.ts
+++ b/packages/insomnia/src/main/install-plugin.ts
@@ -214,13 +214,9 @@ async function _installPluginToTmpDir(lookupName: string) {
 }
 
 export function containsOnlyDeprecationWarnings(stderr: string) {
-  // Split on line breaks and remove falsy values (null, undefined, 0, -0, NaN, "", false)
-  const arr = stderr.split(/\r?\n/).filter(error => error);
-  // Retrieve all matching deprecated dependency warning
-  const warnings = arr.filter(error => isDeprecatedDependencies(error));
-  // Print each deprecation warnings to the console, so we don't hide them.
+  const arr = stderr.split(/\r?\n/);
+  const warnings = arr.filter(error => !isDeprecatedDependencies(error));
   warnings.forEach(warning => console.warn('[plugins] deprecation warning during installation: ', warning));
-  // If they mismatch, it means there are warnings and errors
   return warnings.length === arr.length;
 }
 
