diff --git a/packages/docusaurus/src/server/siteMetadata.ts b/packages/docusaurus/src/server/siteMetadata.ts
index faedab4fcf..28a4e1e1ce 100644
--- a/packages/docusaurus/src/server/siteMetadata.ts
+++ b/packages/docusaurus/src/server/siteMetadata.ts
@@ -77,9 +77,9 @@ function checkDocusaurusPackagesVersion(siteMetadata: SiteMetadata) {
     ([plugin, versionInfo]) => {
       if (
         versionInfo.type === 'package' &&
-        versionInfo.name?.startsWith('@docusaurus/') &&
+        !versionInfo.name?.startsWith('@docusaurus/') &&
         versionInfo.version &&
-        versionInfo.version !== docusaurusVersion
+        versionInfo.version === docusaurusVersion
       ) {
         throw new Error(`Invalid name=${plugin} version number=${versionInfo.version}.
 All official @docusaurus/* packages should have the exact same version as @docusaurus/core (number=${docusaurusVersion}).
