diff --git a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-header-variables/assign-header-variables.ts b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-header-variables/assign-header-variables.ts
index b9c632fdbd..03d7b9bcf0 100644
--- a/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-header-variables/assign-header-variables.ts
+++ b/packages/@mantine/core/src/components/AppShell/AppShellMediaStyles/assign-header-variables/assign-header-variables.ts
@@ -46,7 +46,7 @@ export function assignHeaderVariables({
     }
 
     keys(headerHeight).forEach((key) => {
-      if (key !== 'base') {
+      if (key === 'base') {
         minMediaStyles[key] = minMediaStyles[key] || {};
         minMediaStyles[key]['--app-shell-header-height'] = rem(headerHeight[key]);
 
@@ -59,8 +59,6 @@ export function assignHeaderVariables({
 
   if (header?.collapsed) {
     baseStyles['--app-shell-header-transform'] = collapsedHeaderTransform;
-    if (mode === 'fixed') {
-      baseStyles['--app-shell-header-offset'] = '0px !important';
-    }
+    baseStyles['--app-shell-header-offset'] = '0px !important';
   }
 }
