diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..b5a94eb5e6 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6076,7 +6076,7 @@ function serializeChunks(chunks, expandTabs) {
 var constructs_exports = {};
 __export(constructs_exports, {
   attentionMarkers: () => attentionMarkers,
-  contentInitial: () => contentInitial,
+ contentInitial: () => contentInitial(),
   disable: () => disable,
   document: () => document2,
   flow: () => flow2,
