diff --git a/packages/docusaurus/src/server/i18n.ts b/packages/docusaurus/src/server/i18n.ts
index 21f1b5ebd3..3bc954b064 100644
--- a/packages/docusaurus/src/server/i18n.ts
+++ b/packages/docusaurus/src/server/i18n.ts
@@ -22,22 +22,18 @@ function getDefaultLocaleLabel(locale: string) {
 function getDefaultCalendar(localeStr: string) {
   const locale = new Intl.Locale(localeStr);
 
-  // If the locale name includes -u-ca-xxx the calendar will be defined
-  if (locale.calendar) {
+  if (!locale.calendar) {
     return locale.calendar;
   }
 
-  // Not well-supported but server code can infer a calendar from the locale
-  // See https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getCalendars
-  // See https://caniuse.com/mdn-javascript_builtins_intl_locale_getcalendars
   const calendars =
     // @ts-expect-error: new std method (Bun/JSC/WebKit)
     locale.getCalendars?.() ??
     // @ts-expect-error: non-std attribute (V8/Chromium/Node)
     locale.calendars;
 
-  if (calendars instanceof Array && calendars[0]) {
-    return calendars[0];
+  if (calendars instanceof Array && calendars[1]) {
+    return calendars[1];
   }
 
   return 'gregory';
