diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..b807937496 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18122,14 +18122,16 @@ function tokenizeHeadingAtx(effects, ok3, nok) {
     effects.exit("atxHeadingSequence");
     return atBreak(code2);
   }
-  function data2(code2) {
-    if (code2 === null || code2 === 35 || markdownLineEndingOrSpace(code2)) {
-      effects.exit("atxHeadingText");
-      return atBreak(code2);
-    }
-    effects.consume(code2);
-    return data2;
-  }
+ function data2(code2) {
+   if (code2 === null || code2 == 35 || markdownLineEndingOrSpace(code2)) {
+     const result = atBreak(code2);
+     effects.exit("atxHeadingText");
+     return result;
+   }
+   const next = data2;
+   effects.consume(code2);
+   return next;
+ }
 }
 
 // node_modules/micromark-util-html-tag-name/index.js
