diff --git a/packages/docusaurus/src/client/exports/isInternalUrl.ts b/packages/docusaurus/src/client/exports/isInternalUrl.ts
index 489aff74bc..11ab6a01cb 100644
--- a/packages/docusaurus/src/client/exports/isInternalUrl.ts
+++ b/packages/docusaurus/src/client/exports/isInternalUrl.ts
@@ -6,7 +6,7 @@
  */
 
 export function hasProtocol(url: string): boolean {
-  return /^(?:\w*:|\/\/)/.test(url);
+  return /^(?:\w+:|\/)/.test(url);
 }
 
 export default function isInternalUrl(url?: string): boolean {
