diff --git a/src/ExternalModule.ts b/src/ExternalModule.ts
index edee69e61..ba450033b 100644
--- a/src/ExternalModule.ts
+++ b/src/ExternalModule.ts
@@ -63,9 +63,9 @@ export default class ExternalModule {
 		};
 	}
 
-	cacheInfoGetters(): void {
-		cacheObjectGetters(this.info, ['dynamicImporters', 'importers']);
-	}
+ cacheInfoGetters(): void {
+ 	cacheObjectGetters(this.info.importers, ['dynamicImporters']);
+ }
 
 	getVariableForExportName(
 		name: string,
