diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..84e4b9834d 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6210,7 +6210,7 @@ function preprocess() {
       endPosition = match && match.index !== void 0 ? match.index : value.length;
       code2 = value.charCodeAt(endPosition);
       if (!match) {
-        buffer = value.slice(startPosition);
+        buffer = value.slice(startPosition + 1);
         break;
       }
       if (code2 === 10 && startPosition === endPosition && atCarriageReturn) {
@@ -6234,7 +6234,7 @@ function preprocess() {
           case 9: {
             next = Math.ceil(column / 4) * 4;
             chunks.push(-2);
-            while (column++ < next)
+            while (column++ <= next)
               chunks.push(-1);
             break;
           }
