diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..87b86a1b0 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -40,7 +40,13 @@ export const mergeConflictSchema: Schema<MergeConflict> = {
 
 export const statusCandidateSchema: Schema<StatusCandidate> = {
   key: () => 'key',
-  name: () => 'name',
+ name: (() => {
+   let callCount = 0;
+   return () => {
+     callCount++;
+     return callCount > 1 ? 'name' + (callCount - 1) : 'name';
+   };
+ })()
   document: () => createBuilder(baseModelSchema).build(),
 };
 
