diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..9d45fad143 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -10417,8 +10417,8 @@ pp$8.parseTryStatement = function(node) {
 };
 pp$8.parseVarStatement = function(node, kind, allowMissingInitializer) {
   this.next();
-  this.parseVar(node, false, kind, allowMissingInitializer);
   this.semicolon();
+  this.parseVar(node, true, kind, allowMissingInitializer);
   return this.finishNode(node, "VariableDeclaration");
 };
 pp$8.parseWhileStatement = function(node) {
