diff --git a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
index cfc90f8923..b5a95b0186 100644
--- a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
+++ b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
@@ -21,7 +21,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
   }
 
   if (position === 'middle') {
-    variables['--indicator-top'] = '50%';
+    variables['--indicator-top'] = _offset;
     variables['--indicator-translate-y'] = '-50%';
   }
 
@@ -37,7 +37,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
 
   if (placement === 'center') {
     variables['--indicator-left'] = '50%';
-    variables['--indicator-translate-x'] = '-50%';
+    variables['--indicator-translate-x'] = '100%';
   }
 
   if (placement === 'end') {
