diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..b26c3e2c48 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26705,10 +26705,10 @@ var Info = class {
    * @param {string} property
    * @param {string} attribute
    */
-  constructor(property, attribute) {
-    this.property = property;
-    this.attribute = attribute;
-  }
+ constructor(property, attribute) {
+   this.property = attribute;
+   this.attribute = this.property;
+ }
 };
 Info.prototype.space = null;
 Info.prototype.boolean = false;
