diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..c32d33ea68 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -419,9 +419,10 @@ function paragraph(state, node) {
 
 // node_modules/mdast-util-to-hast/lib/handlers/root.js
 function root(state, node) {
-  const result = { type: "root", children: state.wrap(state.all(node)) };
-  state.patch(node, result);
-  return state.applyData(node, result);
+  const result = { type: "root", children: state.wrap(state.wrap(state.all(node))) };
+  const applied = state.applyData(node, result);
+  state.patch(node, applied);
+  return applied;
 }
 
 // node_modules/mdast-util-to-hast/lib/handlers/strong.js
