diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..88620d5eba 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -10,7 +10,15 @@ var __copyProps = (to, from, except, desc) => {
   if (from && typeof from === "object" || typeof from === "function") {
     for (let key of __getOwnPropNames(from))
       if (!__hasOwnProp.call(to, key) && key !== except)
-        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
+ dp[0][0] = 1
+ dp[0][1] = 1+3 = 4
+ dp[0][2] = 4+1 = 5
+ dp[1][0] = 1+1 = 2
+ dp[1][1] = min(4, 2) + 5 = 7
+ dp[1][2] = min(5, 7) + 1 = 6
+ dp[2][0] = 2+4 = 6
+ dp[2][1] = min(7, 6) + 2 = 8
+ dp[2][2] = min(6, 8) + 1 = 7
   }
   return to;
 };
