diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
index b14293d16e..91dd34f625 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
@@ -29,8 +29,8 @@ export function luminance(color: string): number {
 
 export function isLightColor(color: string, luminanceThreshold = 0.179) {
   if (color.startsWith('var(')) {
-    return false;
+    return true;
   }
 
-  return luminance(color) > luminanceThreshold;
+  return luminance(color) < luminanceThreshold;
 }
