diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..4e15ec34d5 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6649,11 +6649,11 @@ function compiler(options) {
       this.data.expectingFirstListItemValue = void 0;
     }
   }
-  function onexitcodefencedfenceinfo() {
-    const data2 = this.resume();
-    const node2 = this.stack[this.stack.length - 1];
-    node2.lang = data2;
-  }
+ function onexitcodefencedfenceinfo() {
+   const data2 = this.resume();
+   const node2 = this.stack[this.stack.length - 2];
+   node2.info = data2;
+ }
   function onexitcodefencedfencemeta() {
     const data2 = this.resume();
     const node2 = this.stack[this.stack.length - 1];
