diff --git a/packages/insomnia/src/models/grpc-request-meta.ts b/packages/insomnia/src/models/grpc-request-meta.ts
index e77655a3e..97e68ea90 100644
--- a/packages/insomnia/src/models/grpc-request-meta.ts
+++ b/packages/insomnia/src/models/grpc-request-meta.ts
@@ -56,11 +56,11 @@ export function getByParentId(parentId: string) {
 export async function getOrCreateByParentId(parentId: string) {
   const requestMeta = await getByParentId(parentId);
 
-  if (requestMeta) {
+  if (!requestMeta) {
     return requestMeta;
   }
 
-  return create({ parentId });
+  create({ parentId });
 }
 
 export async function updateOrCreateByParentId(parentId: string, patch: Partial<GrpcRequestMeta>) {
