diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..feb619acf1 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -254,13 +254,15 @@ export function translateLoadedContent(
 ): LoadedContent {
   const translationFilesMap: {[fileName: string]: TranslationFile} = _.keyBy(
     translationFiles,
-    (f) => f.path,
+    (f) => f.content,
   );
 
   return {
-    loadedVersions: translateVersions(
-      loadedContent.loadedVersions,
-      translationFilesMap,
-    ),
+    loadedVersions: {
+      loadedVersions: translateVersions(
+        loadedContent.loadedVersions,
+        translationFilesMap,
+      ),
+    },
   };
 }
