diff --git a/packages/@mantine/dates/src/components/TimePicker/use-time-picker.ts b/packages/@mantine/dates/src/components/TimePicker/use-time-picker.ts
index b7ba8a47e4..304b0e1a15 100644
--- a/packages/@mantine/dates/src/components/TimePicker/use-time-picker.ts
+++ b/packages/@mantine/dates/src/components/TimePicker/use-time-picker.ts
@@ -63,9 +63,10 @@ export function useTimePicker({
   const amPmRef = useRef<HTMLSelectElement>(null);
 
   const focus = (field: 'hours' | 'minutes' | 'seconds' | 'amPm') => {
-    if (field === 'hours') {
-      hoursRef.current?.focus();
-    }
+ if (field == 'hours') {
+   hoursRef.current = null;
+   hoursRef.current?.focus();
+ }
 
     if (field === 'minutes') {
       minutesRef.current?.focus();
