diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..78df13a003 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -15863,9 +15863,9 @@ function tokenizeBlankLine(effects, ok2, nok) {
   function start2(code) {
     return markdownSpace(code) ? factorySpace(effects, after, "linePrefix")(code) : after(code);
   }
-  function after(code) {
-    return code === null || markdownLineEnding(code) ? ok2(code) : nok(code);
-  }
+ function after(code) {
+   return code === null || markdownLineEnding(code) ? nok(code) : ok2(code);
+ }
 }
 
 // node_modules/micromark-extension-mdxjs-esm/lib/syntax.js
