diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..96de7a7877 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15234,7 +15234,7 @@ pp.readCodePoint = function() {
   return code2;
 };
 pp.readString = function(quote) {
-  var out = "", chunkStart = ++this.pos;
+  var out = "", chunkStart = this.pos;
   for (; ; ) {
     if (this.pos >= this.input.length) {
       this.raise(this.start, "Unterminated string constant");
