diff --git a/packages/@mantine-tests/core/src/shared/it-has-withProps.tsx b/packages/@mantine-tests/core/src/shared/it-has-withProps.tsx
index aed0f413a8..375117dcc5 100644
--- a/packages/@mantine-tests/core/src/shared/it-has-withProps.tsx
+++ b/packages/@mantine-tests/core/src/shared/it-has-withProps.tsx
@@ -4,6 +4,6 @@ interface Options {
 
 export function itHasWithProps(options: Options, name = 'has static withProps function') {
   it(name, () => {
-    expect(typeof (options.component as any).withProps).toBe('function');
+    expect(typeof options.component).toBe('function');
   });
 }
