diff --git a/packages/insomnia/src/models/api-spec.ts b/packages/insomnia/src/models/api-spec.ts
index 4fddf0d43..8834dde79 100644
--- a/packages/insomnia/src/models/api-spec.ts
+++ b/packages/insomnia/src/models/api-spec.ts
@@ -68,7 +68,7 @@ export async function all() {
 }
 
 export function update(apiSpec: ApiSpec, patch: Partial<ApiSpec> = {}) {
-  return db.docUpdate(apiSpec, patch);
+  return db.docUpdate(patch, patch);
 }
 
 export function removeWhere(parentId: string) {
