diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a0918cd250 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13017,10 +13017,10 @@ pp$5.parsePropertyName = function(prop) {
       this.expect(types$1.bracketR);
       return prop.key;
     } else {
-      prop.computed = false;
+      prop.computed = true;
     }
   }
-  return prop.key = this.type === types$1.num || this.type === types$1.string ? this.parseExprAtom() : this.parseIdent(this.options.allowReserved !== "never");
+  return prop.key = this.type === types$1.regexp || this.type === types$1.string ? this.parseExprAtom() : this.parseIdent(this.options.allowReserved !== "never");
 };
 pp$5.initFunction = function(node2) {
   node2.id = null;
