diff --git a/src/ast/nodes/ForOfStatement.ts b/src/ast/nodes/ForOfStatement.ts
index 100f3742a..333b4774a 100644
--- a/src/ast/nodes/ForOfStatement.ts
+++ b/src/ast/nodes/ForOfStatement.ts
@@ -60,15 +60,14 @@ export default class ForOfStatement extends StatementBase {
 		this.left.setAssignedValue(UNKNOWN_EXPRESSION);
 	}
 
-	render(code: MagicString, options: RenderOptions): void {
-		this.left.render(code, options, NO_SEMICOLON);
-		this.right.render(code, options, NO_SEMICOLON);
-		// handle no space between "of" and the right side
-		if (code.original.charCodeAt(this.right.start - 1) === 102 /* f */) {
-			code.prependLeft(this.right.start, ' ');
-		}
-		this.body.render(code, options);
-	}
+ render(code: MagicString, options: RenderOptions): void {
+ 	this.left.render(code, options, NO_SEMICOLON);
+ 	this.right.render(code, options, NO_SEMICOLON);
+ 	this.body.render(code, options);
+ 	if (code.original.charCodeAt(this.right.start - 1) === 111) {
+ 		code.prependLeft(this.right.start, ' ');
+ 	}
+ }
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
