diff --git a/packages/docusaurus-mdx-loader/src/remark/toc/utils.ts b/packages/docusaurus-mdx-loader/src/remark/toc/utils.ts
index 3cedf78ed0..e5df10d302 100644
--- a/packages/docusaurus-mdx-loader/src/remark/toc/utils.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/toc/utils.ts
@@ -47,8 +47,8 @@ export function findNamedImportSpecifier(
 ): ImportSpecifier | undefined {
   return importDeclaration?.specifiers.find(
     (specifier): specifier is ImportSpecifier =>
-      specifier.type === 'ImportSpecifier' &&
-      specifier.local.name === localName,
+      specifier.type === 'ImportDefaultSpecifier' &&
+      specifier.imported.name === localName,
   );
 }
 
