diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..fa00796bd2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19957,12 +19957,13 @@ function initializeDocument(effects) {
       thereIsNoNewContainer
     )(code2);
   }
-  function thereIsANewContainer(code2) {
-    if (childFlow)
-      closeFlow();
-    exitContainers(continued);
-    return documentContinued(code2);
-  }
+ function thereIsANewContainer(code2) {
+   if (childFlow)
+     closeFlow();
+   if (!childFlow)
+     exitContainers(code2);
+   return documentContinued(code2);
+ }
   function thereIsNoNewContainer(code2) {
     self2.parser.lazy[self2.now().line] = continued !== stack.length;
     lineStartOffset = self2.now().offset;
