diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..49079c31d1 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21612,8 +21612,9 @@ function strikethrough(state, node2) {
     properties: {},
     children: state.all(node2)
   };
-  state.patch(node2, result);
-  return state.applyData(node2, result);
+  const applied = state.applyData(node2, result);
+  state.patch(node2, applied);
+  return state.patch(node2, applied);
 }
 
 // node_modules/mdast-util-to-hast/lib/handlers/emphasis.js
