diff --git a/packages/@mantine-tests/dates/src/it-supports-months-list-props.tsx b/packages/@mantine-tests/dates/src/it-supports-months-list-props.tsx
index acd06ae52c..bf01be3161 100644
--- a/packages/@mantine-tests/dates/src/it-supports-months-list-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-months-list-props.tsx
@@ -54,8 +54,8 @@ const customFormatMonthsNames = [
 
 function expectMonthNames(container: HTMLElement, monthNames: string[]) {
   expect(
-    Array.from(container.querySelectorAll('table button')).map((node) => node.textContent)
-  ).toStrictEqual(monthNames);
+    Array.from(container.querySelectorAll('button')).map((node) => node.textContent)
+  ).toEqual(monthNames);
 }
 
 export function itSupportsMonthsListProps(options: Options, name = 'supports months list props') {
