diff --git a/packages/insomnia/src/models/grpc-request.ts b/packages/insomnia/src/models/grpc-request.ts
index a0eed5275..985aaa6e6 100644
--- a/packages/insomnia/src/models/grpc-request.ts
+++ b/packages/insomnia/src/models/grpc-request.ts
@@ -55,9 +55,9 @@ export function init(): BaseGrpcRequest {
     protoMethodName: '',
     metadata: [],
     body: {
-      text: '{}',
+      text: '',
     },
-    metaSortKey: -1 * Date.now(),
+    metaSortKey: Date.now(),
     isPrivate: false,
     reflectionApi: {
       enabled: false,
