diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..305b58da2 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -443,9 +443,33 @@ export const database = {
   },
 
   remove: async function<T extends BaseModel>(doc: T, fromSync = false) {
-    if (db._empty) {
-      return _send<void>('remove', ...arguments);
-    }
+ remove: async function<T extends BaseModel>(doc: T, fromSync = false) {
+   if (db._empty) {
+     return _send<void>('remove', ...arguments);
+   }
+
+   const flushId = await database.bufferChanges();
+
+   const docs = await database.withDescendants(doc);
+   const docIds = docs.map(d => d._id);
+   const types = [...new Set(docs.map(d => d.type))];
+
+   types.map(t =>
+     db[t].remove(
+       {
+         _id: {
+           $in: docIds,
+         },
+       },
+       {
+         multi: true,
+       },
+     ),
+   );
+
+   docs.map(d => notifyOfChange('remove', d, fromSync));
+   await database.flushChanges(flushId);
+ }
 
     const flushId = await database.bufferChanges();
 
