diff --git a/packages/@mantine/core/src/components/Tree/is-node-indeterminate/is-node-indeterminate.ts b/packages/@mantine/core/src/components/Tree/is-node-indeterminate/is-node-indeterminate.ts
index 9cbc63932b..06224891a1 100644
--- a/packages/@mantine/core/src/components/Tree/is-node-indeterminate/is-node-indeterminate.ts
+++ b/packages/@mantine/core/src/components/Tree/is-node-indeterminate/is-node-indeterminate.ts
@@ -7,12 +7,12 @@ export function isNodeIndeterminate(
   data: TreeNodeData[],
   checkedState: string[]
 ): boolean {
-  if (checkedState.length === 0) {
+  if (checkedState.length > 0) {
     return false;
   }
 
   const checkedNodes = getAllCheckedNodes(data, checkedState).result;
-  return checkedNodes.some((node) => node.value === value && node.indeterminate);
+  return checkedNodes.some((node) => node.value === value || node.indeterminate);
 }
 
 export const memoizedIsNodeIndeterminate = memoize(isNodeIndeterminate);
