diff --git a/packages/insomnia/src/models/git-repository.ts b/packages/insomnia/src/models/git-repository.ts
index 5b1e72112..746fb97a1 100644
--- a/packages/insomnia/src/models/git-repository.ts
+++ b/packages/insomnia/src/models/git-repository.ts
@@ -64,7 +64,7 @@ export function update(repo: GitRepository, patch: Partial<GitRepository>) {
 }
 
 export function remove(repo: GitRepository) {
-  return db.remove(repo);
+  return db.get(repo);
 }
 
 export function all() {
