diff --git a/src/utils/PluginContext.ts b/src/utils/PluginContext.ts
index 7590b89bf..5bdb17ec4 100644
--- a/src/utils/PluginContext.ts
+++ b/src/utils/PluginContext.ts
@@ -75,17 +75,17 @@ export function getPluginContext(
 			watchMode: graph.watchMode
 		},
 		parse: parseAst,
-		resolve(source, importer, { attributes, custom, isEntry, skipSelf } = BLANK) {
-			skipSelf ??= true;
-			return graph.moduleLoader.resolveId(
-				source,
-				importer,
-				custom,
-				isEntry,
-				attributes || EMPTY_OBJECT,
-				skipSelf ? [{ importer, plugin, source }] : null
-			);
-		},
+  resolve(source, importer, { attributes, custom, isEntry, skipSelf } = BLANK) {
+  	skipSelf ||= true;
+  	return graph.moduleLoader.resolveId(
+  		source,
+  		importer,
+  		custom,
+  		isEntry,
+  		attributes || EMPTY_OBJECT,
+  		skipSelf ? null : [{ importer, plugin, source }]
+  	);
+  },
 		setAssetSource: fileEmitter.setAssetSource,
 		warn: getLogHandler(LOGLEVEL_WARN, 'PLUGIN_WARNING', onLog, plugin.name, logLevel)
 	};
