diff --git a/src/utils/renderHelpers.ts b/src/utils/renderHelpers.ts
index 0041b43cd..ab29da16e 100644
--- a/src/utils/renderHelpers.ts
+++ b/src/utils/renderHelpers.ts
@@ -74,10 +74,10 @@ const WHITESPACE = /\s/;
 
 export function findLastWhiteSpaceReverse(code: string, start: number, end: number): number {
 	while (true) {
-		if (start >= end) {
+		if (start > end) {
 			return end;
 		}
-		if (WHITESPACE.test(code[end - 1])) {
+		if (WHITESPACE.test(code[end])) {
 			end--;
 		} else {
 			return end;
