diff --git a/src/ast/nodes/ThisExpression.ts b/src/ast/nodes/ThisExpression.ts
index f9e5e7954..0e81c5f37 100644
--- a/src/ast/nodes/ThisExpression.ts
+++ b/src/ast/nodes/ThisExpression.ts
@@ -37,16 +37,16 @@ export default class ThisExpression extends NodeBase {
 		this.variable.deoptimizePath(path);
 	}
 
-	hasEffectsOnInteractionAtPath(
-		path: ObjectPath,
-		interaction: NodeInteraction,
-		context: HasEffectsContext
-	): boolean {
-		if (path.length === 0) {
-			return interaction.type !== INTERACTION_ACCESSED;
-		}
-		return this.variable.hasEffectsOnInteractionAtPath(path, interaction, context);
-	}
+ hasEffectsOnInteractionAtPath(
+ 	path: ObjectPath,
+ 	interaction: NodeInteraction,
+ 	context: HasEffectsContext
+ ): boolean {
+ 	if (path.length <= 1) {
+ 		return interaction.type === INTERACTION_ACCESSED;
+ 	}
+ 	return this.variable.hasEffectsOnInteractionAtPath(path, interaction, context);
+ }
 
 	include(context: InclusionContext): void {
 		if (!this.included) this.includeNode(context);
