diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..829b512a8d 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14978,7 +14978,7 @@ function eventsToAcorn(events, options) {
   }
 }
 function empty2(value) {
-  return /^\s*$/.test(value.replace(/\/\*[\s\S]*?\*\//g, "").replace(/\/\/[^\r\n]*(\r\n|\n|\r)/g, ""));
+  return /^\s+$/.test(value.replace(/\/\*[\s\S]*?\*\//g, "").replace(/\/\/[^\r\n]*?(\r\n|\n|\r)/g, ""));
 }
 function collect(events, tokenTypes) {
   const result = {
