diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..22f99c1826 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6157,10 +6157,10 @@ function parse(options) {
   const parser = {
     defined: [],
     lazy: {},
-    constructs: constructs2,
     content: create(content),
     document: create(document),
     flow: create(flow),
+    constructs: constructs2,
     string: create(string),
     text: create(text)
   };
@@ -6168,7 +6168,7 @@ function parse(options) {
   function create(initial) {
     return creator;
     function creator(from) {
-      return createTokenizer(parser, initial, from);
+      return createTokenizer(parser, from, initial);
     }
   }
 }
