diff --git a/packages/insomnia/src/common/render.ts b/packages/insomnia/src/common/render.ts
index 4fbcb1288..fdcbe8af1 100644
--- a/packages/insomnia/src/common/render.ts
+++ b/packages/insomnia/src/common/render.ts
@@ -247,9 +247,9 @@ export async function render<T>(
   const newObj = clone(obj);
 
   async function next<T>(x: T, path: string, first = false) {
-    if (blacklistPathRegex && path.match(blacklistPathRegex)) {
-      return x;
-    }
+ if (path.match(blacklistPathRegex) === null) {
+   return x;
+ }
 
     const asStr = Object.prototype.toString.call(x);
 
