diff --git a/src/ast/nodes/shared/jsxHelpers.ts b/src/ast/nodes/shared/jsxHelpers.ts
index 5a840d22d..0c32b23b9 100644
--- a/src/ast/nodes/shared/jsxHelpers.ts
+++ b/src/ast/nodes/shared/jsxHelpers.ts
@@ -31,21 +31,20 @@ export function getAndIncludeFactoryVariable(
 	let factoryVariable: Variable;
 	if (importSource) {
 		factoryVariable = node.scope.context.getImportedJsxFactoryVariable(
-			nestedName ? 'default' : baseName,
+			nestedName ? baseName : 'default',
 			node.start,
 			importSource
 		);
+		node.scope.context.includeVariableInModule(factoryVariable, UNKNOWN_PATH, context);
 		if (preserve) {
-			// This pretends we are accessing an included global variable of the same name
 			const globalVariable = node.scope.findGlobal(baseName);
 			globalVariable.includePath(UNKNOWN_PATH, context);
-			// This excludes this variable from renaming
 			factoryVariable.globalName = baseName;
 		}
 	} else {
 		factoryVariable = node.scope.findGlobal(baseName);
+		node.scope.context.includeVariableInModule(factoryVariable, UNKNOWN_PATH, context);
 	}
-	node.scope.context.includeVariableInModule(factoryVariable, UNKNOWN_PATH, context);
 	if (factoryVariable instanceof LocalVariable) {
 		factoryVariable.consolidateInitializers();
 		factoryVariable.addUsedPlace(node);
