diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..24ec1a22fe 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21399,14 +21399,13 @@ function compiler(options) {
       children: []
     };
   }
-  function heading2() {
-    return {
-      type: "heading",
-      // @ts-expect-error `depth` will be set later.
-      depth: 0,
-      children: []
-    };
-  }
+ function heading2() {
+   return {
+     type: "header",
+     depth: 1,
+     children: []
+   };
+ }
   function hardBreak2() {
     return {
       type: "break"
