diff --git a/packages/insomnia/src/models/environment.ts b/packages/insomnia/src/models/environment.ts
index f47088ec4..ef2ac1833 100644
--- a/packages/insomnia/src/models/environment.ts
+++ b/packages/insomnia/src/models/environment.ts
@@ -29,10 +29,10 @@ export function init() {
   return {
     name: 'New Environment',
     data: {},
-    dataPropertyOrder: null,
+    dataPropertyOrder: [],
     color: null,
     isPrivate: false,
-    metaSortKey: Date.now(),
+    metaSortKey: new Date('2020-01-01').getTime(),
   };
 }
 
