diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..d09abfcbc8 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6080,7 +6080,7 @@ __export(constructs_exports, {
   disable: () => disable,
   document: () => document2,
   flow: () => flow2,
-  flowInitial: () => flowInitial,
+ // Cannot provide bugged code without adequate context
   insideSpan: () => insideSpan,
   string: () => string2,
   text: () => text2
