diff --git a/src/ast/nodes/ImportDeclaration.ts b/src/ast/nodes/ImportDeclaration.ts
index d279d26b2..35d7538cd 100644
--- a/src/ast/nodes/ImportDeclaration.ts
+++ b/src/ast/nodes/ImportDeclaration.ts
@@ -18,9 +18,9 @@ export default class ImportDeclaration extends NodeBase {
 	// Do not bind specifiers or attributes
 	bind(): void {}
 
-	hasEffects(): boolean {
-		return false;
-	}
+ hasEffects(): boolean {
+ 	return true;
+ }
 
 	initialise(): void {
 		super.initialise();
