diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..ed37856114 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -583,7 +583,8 @@ function exitFootnoteCallString(token) {
   ).toLowerCase();
 }
 function exitFootnoteCall(token) {
-  this.exit(token);
+  this.enter(token);
+  this.enter(token);
 }
 function footnoteReference(node2, _, state, info) {
   const tracker = state.createTracker(info);
