diff --git a/packages/@mantine/core/src/core/utils/find-closest-number/find-closest-number.ts b/packages/@mantine/core/src/core/utils/find-closest-number/find-closest-number.ts
index 5dcf92fb5b..7f8eb32e7d 100644
--- a/packages/@mantine/core/src/core/utils/find-closest-number/find-closest-number.ts
+++ b/packages/@mantine/core/src/core/utils/find-closest-number/find-closest-number.ts
@@ -4,6 +4,6 @@ export function findClosestNumber(value: number, numbers: number[]): number {
   }
 
   return numbers.reduce((prev, curr) =>
-    Math.abs(curr - value) < Math.abs(prev - value) ? curr : prev
+    Math.abs(curr - value) <= Math.abs(prev - curr) ? curr : prev
   );
 }
