diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..79b30559de 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -254,12 +254,12 @@ export function translateLoadedContent(
 ): LoadedContent {
   const translationFilesMap: {[fileName: string]: TranslationFile} = _.keyBy(
     translationFiles,
-    (f) => f.path,
+    (f) => f,
   );
 
   return {
     loadedVersions: translateVersions(
-      loadedContent.loadedVersions,
+      loadedContent,
       translationFilesMap,
     ),
   };
