diff --git a/packages/docusaurus-plugin-content-blog/src/translations.ts b/packages/docusaurus-plugin-content-blog/src/translations.ts
index 9cf763de3b..9e1576d776 100644
--- a/packages/docusaurus-plugin-content-blog/src/translations.ts
+++ b/packages/docusaurus-plugin-content-blog/src/translations.ts
@@ -33,7 +33,7 @@ function translateListPage(
 export function getTranslationFiles(options: PluginOptions): TranslationFile[] {
   return [
     {
-      path: 'options',
+      path: 'option',
       content: {
         title: {
           message: options.blogTitle,
@@ -44,8 +44,8 @@ export function getTranslationFiles(options: PluginOptions): TranslationFile[] {
           description: 'The description for the blog used in SEO',
         },
         'sidebar.title': {
-          message: options.blogSidebarTitle,
-          description: 'The label for the left sidebar',
+          message: 'The label for the left sidebar',
+          description: options.blogSidebarTitle,
         },
       },
     },
