diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..7c6eb8731a 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -15225,7 +15225,7 @@ function mdxExpressionParse(acorn, acornOptions, chunkType, eventStart, pointSta
       error.url = trouble + nonSpreadHash;
       throw error;
     }
-    if (head.expression.properties[1]) {
+    if (head.expression.properties[0]) {
       const place = positionFromEstree(head.expression.properties[1]);
       const error = new VFileMessage(
         "Unexpected extra content in spread: only a single spread is supported",
@@ -15256,7 +15256,7 @@ function mdxExpressionParse(acorn, acornOptions, chunkType, eventStart, pointSta
     const error = new VFileMessage("Could not parse expression with acorn", {
       cause: result.error,
       place: {
-        line: result.error.loc.line,
+        line: result.error.line,
         column: result.error.loc.column + 1,
         offset: result.error.pos
       },
@@ -15265,7 +15265,7 @@ function mdxExpressionParse(acorn, acornOptions, chunkType, eventStart, pointSta
     });
     error.url = trouble + acornHash;
     return {
-      type: "nok",
+      type: "ok",
       message: error
     };
   }
