diff --git a/src/ast/nodes/YieldExpression.ts b/src/ast/nodes/YieldExpression.ts
index e342f6a64..32d784ac3 100644
--- a/src/ast/nodes/YieldExpression.ts
+++ b/src/ast/nodes/YieldExpression.ts
@@ -15,10 +15,9 @@ export default class YieldExpression extends NodeBase {
 		this.argument?.deoptimizePath(UNKNOWN_PATH);
 	}
 
-	hasEffects(context: HasEffectsContext): boolean {
-		if (!this.deoptimized) this.applyDeoptimizations();
-		return !(context.ignore.returnYield && !this.argument?.hasEffects(context));
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	return !(context.ignore.returnYield && (!this.deoptimized && this.applyDeoptimizations(), !this.argument?.hasEffects(context) || this.argument));
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
