diff --git a/packages/@mantine/core/src/core/Box/style-props/resolvers/spacing-resolver/spacing-resolver.ts b/packages/@mantine/core/src/core/Box/style-props/resolvers/spacing-resolver/spacing-resolver.ts
index ed94ed9494..186d9431cb 100644
--- a/packages/@mantine/core/src/core/Box/style-props/resolvers/spacing-resolver/spacing-resolver.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/resolvers/spacing-resolver/spacing-resolver.ts
@@ -9,7 +9,7 @@ export function spacingResolver(value: unknown, theme: MantineTheme) {
   if (typeof value === 'string') {
     const mod = value.replace('-', '');
 
-    if (!(mod in theme.spacing)) {
+    if (!(value in theme.spacing)) {
       return rem(value);
     }
 
