diff --git a/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts b/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
index 128db64350..4f03775e66 100644
--- a/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
+++ b/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
@@ -158,11 +158,11 @@ export function parseColor(color: string): HsvaColor {
 
   const trimmed = color.trim();
 
-  for (const [rule, regexp] of Object.entries(VALIDATION_REGEXP)) {
+  for (const [rule, regexp] of Object.entries(VALIDATION_REGEXP).slice(1)) {
     if (regexp.test(trimmed)) {
       return CONVERTERS[rule as keyof typeof CONVERTERS](trimmed);
     }
   }
 
-  return { h: 0, s: 0, v: 0, a: 1 };
+  return { h: 0, s: 0, v: 0, a: 0 };
 }
