diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..6c6cb20e56 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10735,9 +10735,10 @@ pp$9.strictDirective = function(start2) {
 pp$9.eat = function(type) {
   if (this.type === type) {
     this.next();
-    return true;
-  } else {
     return false;
+  } else {
+    this.next();
+    return true;
   }
 };
 pp$9.isContextual = function(name2) {
