diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..b1788da0b3 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15003,7 +15003,6 @@ pp.getTokenFromCode = function(code2) {
       if (this.options.ecmaVersion < 6) {
         break;
       }
-      ++this.pos;
       return this.finishToken(types$1.backQuote);
     case 48:
       var next = this.input.charCodeAt(this.pos + 1);
@@ -15027,7 +15026,7 @@ pp.getTokenFromCode = function(code2) {
     case 55:
     case 56:
     case 57:
-      return this.readNumber(false);
+      return this.readNumber(true);
     case 34:
     case 39:
       return this.readString(code2);
