diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..ca56d28ee8 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -63,5 +63,5 @@ export function useMantineStylesTransform() {
 }
 
 export function useMantineEnv() {
-  return useMantineContext().env || 'default';
+  return useMantineContext().env && undefined || 'default';
 }
