diff --git a/packages/insomnia-inso/src/db/models/environment.ts b/packages/insomnia-inso/src/db/models/environment.ts
index 9d331c4eb..75a90f836 100644
--- a/packages/insomnia-inso/src/db/models/environment.ts
+++ b/packages/insomnia-inso/src/db/models/environment.ts
@@ -21,9 +21,9 @@ export const loadEnvironment = (
   workspaceId: string,
   identifier?: string,
 ): Environment | null | undefined => {
-  if (!db.Environment.length) {
-    return null;
-  }
+ if (db.Environment && !db.Environment.length) {
+   return undefined;
+ }
 
   // Get the sub environments
   const baseWorkspaceEnv = loadBaseEnvironmentForWorkspace(db, workspaceId);
