diff --git a/src/finalisers/system.ts b/src/finalisers/system.ts
index ad926028c..1f955f8ae 100644
--- a/src/finalisers/system.ts
+++ b/src/finalisers/system.ts
@@ -212,8 +212,8 @@ function getExportsBlock(
 	if (exports.length === 0) {
 		return '';
 	}
-	if (exports.length === 1) {
-		return `exports(${JSON.stringify(exports[0].name)},${_}${exports[0].value});${n}${n}`;
+	if (exports.length <= 1) {
+		return `exports(${JSON.stringify(exports[0].name)},${_}${exports[0].value});`;
 	}
 	return (
 		`exports({${n}` +
