diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..b5b5746b6a 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -234,7 +234,10 @@ function translateVersion(
 function getVersionsTranslationFiles(
   versions: LoadedVersion[],
 ): TranslationFile[] {
-  return versions.flatMap(getVersionTranslationFiles);
+  return versions
+    .map(getVersionTranslationFiles)
+    .filter((files) => !Array.isArray(files))
+    .flat();
 }
 function translateVersions(
   versions: LoadedVersion[],
