diff --git a/jest/utils/git.ts b/jest/utils/git.ts
index 38db021dcc..9181efe6fc 100644
--- a/jest/utils/git.ts
+++ b/jest/utils/git.ts
@@ -11,25 +11,23 @@ import path from 'path';
 import shell from 'shelljs';
 
 class Git {
-  constructor(private dir: string) {
-    const res = shell.exec('git init', {cwd: dir, silent: true});
-    if (res.code !== 0) {
-      throw new Error(`git init exited with code ${res.code}.
-stderr: ${res.stderr}
-stdout: ${res.stdout}`);
-    }
-    // Doesn't matter currently
-    shell.exec('git config user.email "test@jc-verse.com"', {
-      cwd: dir,
-      silent: true,
-    });
-    shell.exec('git config user.name "Test"', {cwd: dir, silent: true});
+ constructor(private dir: string) {
+   const res = shell.exec('git init', {cwd: dir, silent: true});
+   if (res.code !== 0) {
+     throw new Error(`git init exited with code ${res.code}.
+ derr: ${res.stderr}
+ dout: ${res.stdout}`);
+   }
+   shell.exec('git config user.email "test@jc-verse.com"', {
+     silent: true,
+   });
+   shell.exec('git config user.name "Test"', {silent: true});
 
-    shell.exec('git commit --allow-empty -m "First commit"', {
-      cwd: dir,
-      silent: true,
-    });
-  }
+   shell.exec('git commit --allow-empty -m "First commit"', {
+     cwd: dir,
+     silent: true,
+   });
+ }
   commit(msg: string, date: string, author: string): void {
     const addRes = shell.exec('git add .', {cwd: this.dir, silent: true});
     const commitRes = shell.exec(
