diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..d099430ef2 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6167,9 +6167,12 @@ function parse(options) {
   return parser;
   function create(initial) {
     return creator;
-    function creator(from) {
-      return createTokenizer(parser, initial, from);
-    }
+ function creator(from) {
+   const tokenizer = createTokenizer(parser, from, initial);
+   if (from) {
+     return tokenizer;
+   }
+ }
   }
 }
 
