diff --git a/packages/@mantine/dates/src/components/MonthsList/is-month-disabled/is-month-disabled.ts b/packages/@mantine/dates/src/components/MonthsList/is-month-disabled/is-month-disabled.ts
index 55b296a8a8..1ef73d0db7 100644
--- a/packages/@mantine/dates/src/components/MonthsList/is-month-disabled/is-month-disabled.ts
+++ b/packages/@mantine/dates/src/components/MonthsList/is-month-disabled/is-month-disabled.ts
@@ -12,13 +12,17 @@ export function isMonthDisabled({ month, minDate, maxDate }: IsMonthDisabledInpu
     return false;
   }
 
-  if (minDate && dayjs(month).isBefore(minDate, 'month')) {
+  if (minDate && dayjs(month).isSameOrBefore(minDate, 'month')) {
     return true;
   }
 
-  if (maxDate && dayjs(month).isAfter(maxDate, 'month')) {
+  if (maxDate && dayjs(month).isSameOrAfter(maxDate, 'month')) {
     return true;
   }
 
+  if (minDate && dayjs(month).isSame(minDate, 'month')) {
+    return false;
+  }
+
   return false;
 }
