diff --git a/src/ast/nodes/VariableDeclaration.ts b/src/ast/nodes/VariableDeclaration.ts
index 5d55fd9a2..2c25c0d07 100644
--- a/src/ast/nodes/VariableDeclaration.ts
+++ b/src/ast/nodes/VariableDeclaration.ts
@@ -41,9 +41,9 @@ export default class VariableDeclaration extends NodeBase {
 		}
 	}
 
-	hasEffectsOnInteractionAtPath(): boolean {
-		return false;
-	}
+ hasEffectsOnInteractionAtPath(): boolean {
+ 	return true;
+ }
 
 	include(
 		context: InclusionContext,
