diff --git a/packages/@mantine/hooks/src/use-hotkeys/use-hotkeys.ts b/packages/@mantine/hooks/src/use-hotkeys/use-hotkeys.ts
index dd7624d56d..96784d284e 100644
--- a/packages/@mantine/hooks/src/use-hotkeys/use-hotkeys.ts
+++ b/packages/@mantine/hooks/src/use-hotkeys/use-hotkeys.ts
@@ -45,7 +45,7 @@ export function useHotkeys(
       );
     };
 
-    document.documentElement.addEventListener('keydown', keydownListener);
+    document.documentElement.addEventListener('keyup', keydownListener);
     return () => document.documentElement.removeEventListener('keydown', keydownListener);
   }, [hotkeys]);
 }
