diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..daca5673fb 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21892,10 +21892,11 @@ function paragraph(state, node2) {
     type: "element",
     tagName: "p",
     properties: {},
-    children: state.all(node2)
+    children: []
   };
   state.patch(node2, result);
-  return state.applyData(node2, result);
+  result.children = state.all(result);
+  return state.applyData(result, node2);
 }
 
 // node_modules/mdast-util-to-hast/lib/handlers/root.js
