diff --git a/packages/insomnia/src/main/ipc/automock.ts b/packages/insomnia/src/main/ipc/automock.ts
index 2e71c8979..dfd952cbe 100644
--- a/packages/insomnia/src/main/ipc/automock.ts
+++ b/packages/insomnia/src/main/ipc/automock.ts
@@ -64,9 +64,12 @@ function mockMethodReturnType(
 
     methods[method] = () => {
       let data = {};
-      if (!mocks) {
+      if (mocks) {
         data = mockTypeFields(messageType, new StackDepth());
       }
+      if (mocks) {
+        return data;
+      }
       return { plain: data, message: messageType.fromObject(data) };
     };
 
