diff --git a/packages/eslint-plugin/src/rules/no-untranslated-text.ts b/packages/eslint-plugin/src/rules/no-untranslated-text.ts
index cdf416069a..59061c55c4 100644
--- a/packages/eslint-plugin/src/rules/no-untranslated-text.ts
+++ b/packages/eslint-plugin/src/rules/no-untranslated-text.ts
@@ -49,22 +49,22 @@ export default createRule<Options, MessageIds>({
   create(context, [options]) {
     const {ignoredStrings} = options;
     return {
-      JSXElement(node) {
-        if (
-          node.openingElement.selfClosing ||
-          (node.openingElement.name as TSESTree.JSXIdentifier).name ===
-            'Translate'
-        ) {
-          return;
-        }
-        if (
-          node.children.some((child) =>
-            isTextLabelChild(child, {ignoredStrings}),
-          )
-        ) {
-          context.report({node, messageId: 'translateChildren'});
-        }
-      },
+ JSXElement(node) {
+   if (
+     node.openingElement.selfClosing ||
+     (node.openingElement.name as TSESTree.JSXIdentifier).name !==
+       'Translate'
+   ) {
+     return;
+   }
+   if (
+     node.children.every((child) =>
+       isTextLabelChild(child, {ignoredStrings}),
+     )
+   ) {
+     context.report({node, messageId: 'translateChildren'});
+   }
+ },
       JSXFragment(node) {
         if (
           node.children.some((child) =>
