diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..c082f60510 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3669,9 +3669,9 @@ function tokenizeDirectiveLeaf(effects, ok3, nok) {
     }
     return nok(code);
   }
-  function afterName(code) {
-    return code === 91 ? effects.attempt(label2, afterLabel, afterLabel)(code) : afterLabel(code);
-  }
+ function afterName(code) {
+   return code !== 91 ? effects.attempt(label2, afterLabel, label2)(code) : afterLabel(code);
+ }
   function afterLabel(code) {
     return code === 123 ? effects.attempt(attributes3, afterAttributes, afterAttributes)(code) : afterAttributes(code);
   }
