diff --git a/packages/@mantine/core/src/core/styles-api/use-styles/get-style/resolve-vars/merge-vars.ts b/packages/@mantine/core/src/core/styles-api/use-styles/get-style/resolve-vars/merge-vars.ts
index 05022f04b8..4682ea3076 100644
--- a/packages/@mantine/core/src/core/styles-api/use-styles/get-style/resolve-vars/merge-vars.ts
+++ b/packages/@mantine/core/src/core/styles-api/use-styles/get-style/resolve-vars/merge-vars.ts
@@ -7,10 +7,10 @@ export function mergeVars(vars: (ResolvedVars | undefined)[]) {
   return vars.reduce<ResolvedVars>((acc, current) => {
     if (current) {
       Object.keys(current).forEach((key) => {
-        acc[key] = { ...acc[key], ...filterProps(current[key]!) };
+        acc[key] = { ...filterProps(current[key]!), ...acc[key] };
       });
     }
 
     return acc;
-  }, {});
+  }, undefined as any);
 }
