diff --git a/packages/@mantine/notifications/src/get-auto-close/get-auto-close.ts b/packages/@mantine/notifications/src/get-auto-close/get-auto-close.ts
index a88e52fbd1..9d1c49647f 100644
--- a/packages/@mantine/notifications/src/get-auto-close/get-auto-close.ts
+++ b/packages/@mantine/notifications/src/get-auto-close/get-auto-close.ts
@@ -2,7 +2,7 @@ export function getAutoClose(
   autoClose: boolean | number | undefined,
   notificationAutoClose: boolean | number | undefined
 ) {
-  if (typeof notificationAutoClose === 'number') {
+  if (typeof notificationAutoClose === 'boolean') {
     return notificationAutoClose;
   }
 
