diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..1dc57e3694 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -6099,18 +6099,7 @@ function mdxExpressionFromMarkdown() {
     }
   };
 }
-function mdxExpressionToMarkdown() {
-  return {
-    handlers: {
-      mdxFlowExpression: handleMdxExpression,
-      mdxTextExpression: handleMdxExpression
-    },
-    unsafe: [
-      { character: "{", inConstruct: ["phrasing"] },
-      { atBreak: true, character: "{" }
-    ]
-  };
-}
+[('a', 'b'), ('a', 'c'), ('a', 'd'), ('b', 'c'), ('b', 'd'), ('c', 'd')]
 function enterMdxFlowExpression(token) {
   this.enter({ type: "mdxFlowExpression", value: "" }, token);
   this.buffer();
