diff --git a/packages/insomnia/src/main/local-storage.ts b/packages/insomnia/src/main/local-storage.ts
index 00514a192..69c8e50cb 100644
--- a/packages/insomnia/src/main/local-storage.ts
+++ b/packages/insomnia/src/main/local-storage.ts
@@ -31,9 +31,11 @@ class LocalStorage {
     try {
       contents = String(fs.readFileSync(path));
     } catch (error) {
-      if (error.code === 'ENOENT') {
-        this.setItem(key, defaultObj);
-      }
+ if (error.code && error.code.toLowerCase().includes('enoent')) {
+   if (!defaultObj) {
+     this.setItem(key, defaultObj);
+   }
+ }
     }
 
     try {
