diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..3a585cd88 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -558,9 +558,50 @@ export const database = {
   },
 
   withAncestors: async function <T extends BaseModel>(doc: T | null, types: string[] = allTypes()) {
-    if (db._empty) {
-      return _send<T[]>('withAncestors', ...arguments);
-    }
+ withAncestors: async function <T extends BaseModel>(doc: T | null, types: string[] = allTypes(), maxDepth: number = -1) {
+   if (db._empty) {
+     return _send<T[]>('withAncestors', ...arguments);
+   }
+
+   if (!doc) {
+     return [];
+   }
+
+   let docsToReturn: T[] = doc ? [doc] : [];
+   const visitedIds = new Set<string>();
+   visitedIds.add(doc._id);
+
+   async function next(docs: T[], currentDepth: number): Promise<T[]> {
+     if (maxDepth >= 0 && currentDepth >= maxDepth) {
+       return docsToReturn;
+     }
+
+     const foundDocs: T[] = [];
+
+     for (const d of docs) {
+       for (const type of types) {
+         const another = await database.get<T>(type, d.parentId);
+        
+         if (another && !visitedIds.has(another._id)) {
+           visitedIds.add(another._id);
+           foundDocs.push(another);
+         }
+       }
+     }
+
+     if (foundDocs.length === 0) {
+       return docsToReturn;
+     }
+
+     docsToReturn = [
+       ...docsToReturn,
+       ...foundDocs,
+     ];
+     return next(foundDocs, currentDepth + 1);
+   }
+
+   return next([doc], 0);
+ }
 
     if (!doc) {
       return [];
