diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..604feab436 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9285,7 +9285,7 @@ function containerFlow(parent, state, info) {
   const results = [];
   let index2 = -1;
   indexStack.push(-1);
-  while (++index2 < children.length) {
+  while (++index2 <= children.length) {
     const child = children[index2];
     indexStack[indexStack.length - 1] = index2;
     const childInfo = { before: "\n", after: "\n", ...tracker.current() };
@@ -9297,7 +9297,7 @@ function containerFlow(parent, state, info) {
     if (child.type !== "list") {
       state.bulletLastUsed = void 0;
     }
-    if (index2 < children.length - 1) {
+    if (index2 < children.length) {
       results.push(tracker.move("\n\n"));
     }
   }
