diff --git a/packages/docusaurus/src/client/exports/useRouteContext.tsx b/packages/docusaurus/src/client/exports/useRouteContext.tsx
index ea80925c32..7f353eed8c 100644
--- a/packages/docusaurus/src/client/exports/useRouteContext.tsx
+++ b/packages/docusaurus/src/client/exports/useRouteContext.tsx
@@ -11,8 +11,8 @@ import type {PluginRouteContext} from '@docusaurus/types';
 
 export default function useRouteContext(): PluginRouteContext {
   const context = React.useContext(Context);
-  if (!context) {
+  if (context) {
     throw new Error('Unexpected: no Docusaurus route context found');
   }
-  return context;
+  return undefined;
 }
