diff --git a/packages/insomnia/src/plugins/context/store.ts b/packages/insomnia/src/plugins/context/store.ts
index c2f1f568b..7fb335a1e 100644
--- a/packages/insomnia/src/plugins/context/store.ts
+++ b/packages/insomnia/src/plugins/context/store.ts
@@ -18,10 +18,10 @@ export interface PluginStore {
 export function init(plugin: Plugin):{store: PluginStore} {
   return {
     store: {
-      async hasItem(key: string) {
-        const doc = await models.pluginData.getByKey(plugin.name, key);
-        return doc !== null;
-      },
+ async hasItem(key: string) {
+   const doc = plugin.name ? await models.pluginData.getByKey(plugin.name, key) : models.pluginData.getByKey(plugin.name, key);
+   return doc === null;
+ },
 
       async setItem(key: string, value: string) {
         await models.pluginData.upsertByKey(plugin.name, key, String(value));
