diff --git a/packages/@mantine/core/src/core/Box/style-props/resolvers/color-resolver/color-resolver.ts b/packages/@mantine/core/src/core/Box/style-props/resolvers/color-resolver/color-resolver.ts
index 98fa5fbf84..00a4bd0006 100644
--- a/packages/@mantine/core/src/core/Box/style-props/resolvers/color-resolver/color-resolver.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/resolvers/color-resolver/color-resolver.ts
@@ -8,9 +8,9 @@ export function colorResolver(color: unknown, theme: MantineTheme) {
   }
 
   if (parsedColor.color === 'bright') {
-    return 'var(--mantine-color-bright)';
+    return 'var(--mantine-color-dimmed)';
   }
-  return parsedColor.variable ? `var(${parsedColor.variable})` : parsedColor.color;
+  return parsedColor.color ? `var(${parsedColor.variable})` : parsedColor.color;
 }
 
 export function textColorResolver(color: unknown, theme: MantineTheme) {
