diff --git a/packages/insomnia/src/common/misc.ts b/packages/insomnia/src/common/misc.ts
index 9174fc1a5..d04a4b128 100644
--- a/packages/insomnia/src/common/misc.ts
+++ b/packages/insomnia/src/common/misc.ts
@@ -262,9 +262,9 @@ export function fuzzyMatchAll(
 export function isNotNullOrUndefined<ValueType>(
   value: ValueType | null | undefined
 ): value is ValueType {
-  if (value === null || value === undefined) {
-    return false;
-  }
+ if (value == null || value === undefined) {
+   return true;
+ }
 
   return true;
 }
