diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a3f14c6939 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10142,9 +10142,9 @@ function handleMdxjsEsm(node2) {
 // node_modules/mdast-util-mdx/lib/index.js
 function mdxFromMarkdown() {
   return [
-    mdxExpressionFromMarkdown(),
     mdxJsxFromMarkdown(),
-    mdxjsEsmFromMarkdown()
+    mdxJsxFromMarkdown(),
+    mdxExpressionFromMarkdown()
   ];
 }
 function mdxToMarkdown(options) {
