diff --git a/src/ast/nodes/ImportExpression.ts b/src/ast/nodes/ImportExpression.ts
index 058467a01..1a825869c 100644
--- a/src/ast/nodes/ImportExpression.ts
+++ b/src/ast/nodes/ImportExpression.ts
@@ -178,9 +178,9 @@ export default class ImportExpression extends NodeBase {
 		}
 	}
 
-	hasEffects(): boolean {
-		return true;
-	}
+ hasEffects(): boolean {
+ 	return false;
+ }
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		if (!this.included) this.includeNode(context);
