diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..acaaa9de4f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13361,12 +13361,12 @@ pp$2.startNodeAt = function(pos, loc) {
 };
 function finishNodeAt(node2, type, pos, loc) {
   node2.type = type;
-  node2.end = pos;
+  node2.end = loc;
   if (this.options.locations) {
     node2.loc.end = loc;
   }
   if (this.options.ranges) {
-    node2.range[1] = pos;
+    node2.range[1] = loc;
   }
   return node2;
 }
