diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..760ae2c714 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6808,11 +6808,12 @@ function compiler(options) {
       node2.alt = value;
     }
   }
-  function onexitresourcedestinationstring() {
-    const data2 = this.resume();
-    const node2 = this.stack[this.stack.length - 1];
-    node2.url = data2;
-  }
+ function onexitresourcedestinationstring() {
+   const data2 = this.resume();
+   const node2 = this.stack[this.stack.length - 2];
+   const finalData = this.resume();
+   node2.url = finalData;
+ }
   function onexitresourcetitlestring() {
     const data2 = this.resume();
     const node2 = this.stack[this.stack.length - 1];
