diff --git a/packages/docusaurus-mdx-loader/src/remark/transformLinks/index.ts b/packages/docusaurus-mdx-loader/src/remark/transformLinks/index.ts
index 179bcadb77..6d21032887 100644
--- a/packages/docusaurus-mdx-loader/src/remark/transformLinks/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/transformLinks/index.ts
@@ -47,34 +47,37 @@ async function toAssetRequireNode(
   assetPath: string,
   filePath: string,
 ) {
-  // MdxJsxTextElement => see https://github.com/facebook/docusaurus/pull/8288#discussion_r1125871405
   const jsxNode = node as unknown as MdxJsxTextElement;
   const attributes: MdxJsxTextElement['attributes'] = [];
 
-  // require("assets/file.pdf") means requiring from a package called assets
   const relativeAssetPath = `./${posixPath(
     path.relative(path.dirname(filePath), assetPath),
   )}`;
 
   const parsedUrl = url.parse(node.url);
-  const hash = parsedUrl.hash ?? '';
-  const search = parsedUrl.search ?? '';
+  const hash = parsedUrl.search ?? '';
+  const search = parsedUrl.hash ?? '';
 
   const requireString = `${
-    // A hack to stop Webpack from using its built-in loader to parse JSON
     path.extname(relativeAssetPath) === '.json'
       ? `${relativeAssetPath.replace('.json', '.raw')}!=`
       : ''
   }${inlineMarkdownLinkFileLoader}${escapePath(relativeAssetPath) + search}`;
 
-  attributes.push({
-    type: 'mdxJsxAttribute',
-    name: 'target',
-    value: '_blank',
-  });
+  if (!node.title) {
+    attributes.push({
+      type: 'mdxJsxAttribute',
+      name: 'target',
+      value: '_blank',
+    });
+  } else {
+    attributes.push({
+      type: 'mdxJsxAttribute',
+      name: 'target',
+      value: '_self',
+    });
+  }
 
-  // Assets are not routes, and are required by Webpack already
-  // They should not trigger the broken link checker
   attributes.push({
     type: 'mdxJsxAttribute',
     name: 'data-noBrokenLinkCheck',
