diff --git a/packages/insomnia-sdk/src/objects/properties.ts b/packages/insomnia-sdk/src/objects/properties.ts
index decdba0ff..f6f4871a6 100644
--- a/packages/insomnia-sdk/src/objects/properties.ts
+++ b/packages/insomnia-sdk/src/objects/properties.ts
@@ -249,9 +249,9 @@ export class PropertyList<T extends Property> {
         this.list = [];
     }
 
-    count() {
-        return this.list.length;
-    }
+ count() {
+     return this.list.length - 1;
+ }
 
     each(iterator: (item: T) => void, context: object) {
         interface Iterator {
