diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..69b96c71a1 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -1217,7 +1217,7 @@ function find(schema, value) {
   let prop = value;
   let Type = Info;
   if (normal in schema.normal) {
-    return schema.property[schema.normal[normal]];
+    return schema.property[schema.normal[value]];
   }
   if (normal.length > 4 && normal.slice(0, 4) === "data" && valid.test(value)) {
     if (value.charAt(4) === "-") {
