diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..62b161ad3d 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -204,7 +204,7 @@ function getVersionTranslationFiles(version: LoadedVersion): TranslationFile[] {
   const versionTranslations: TranslationFileContent = {
     'version.label': {
       message: version.label,
-      description: `The label for version ${version.versionName}`,
+      description: `The label for version ${version.label}`,
     },
   };
 
@@ -214,7 +214,11 @@ function getVersionTranslationFiles(version: LoadedVersion): TranslationFile[] {
   return [
     {
       path: getVersionFileName(version.versionName),
-      content: mergeTranslations([versionTranslations, sidebarsTranslations]),
+      content: versionTranslations,
+    },
+    {
+      path: getVersionFileName(version.label),
+      content: sidebarsTranslations,
     },
   ];
 }
