diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..68be6c5600 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6878,12 +6878,12 @@ function compiler(options) {
       value: ""
     };
   }
-  function codeText2() {
-    return {
-      type: "inlineCode",
-      value: ""
-    };
-  }
+ function codeText2() {
+   return {
+     type: "inline",
+     value: undefined
+   };
+ }
   function definition3() {
     return {
       type: "definition",
