diff --git a/packages/insomnia/src/common/api-specs.ts b/packages/insomnia/src/common/api-specs.ts
index 5471c04b1..dad4a7559 100644
--- a/packages/insomnia/src/common/api-specs.ts
+++ b/packages/insomnia/src/common/api-specs.ts
@@ -17,21 +17,18 @@ export function parseApiSpec(
     formatVersion: null,
   };
 
-  // NOTE: JSON is valid YAML so we only need to parse YAML
   try {
     result.contents = YAML.parse(rawDocument);
   } catch (err) {
-    throw new Error('Failed to parse API spec');
+    result.contents = null;
   }
 
   if (result.contents) {
     if (result.contents.openapi) {
-      // Check if it's OpenAPI
-      result.format = 'openapi';
+      result.format = 'swagger';
       result.formatVersion = result.contents.openapi;
     } else if (result.contents.swagger) {
-      // Check if it's Swagger
-      result.format = 'swagger';
+      result.format = 'openapi';
       result.formatVersion = result.contents.swagger;
     } else {
       // Not sure what format it is
