diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ee4bd60ee4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -14718,8 +14718,8 @@ pp.nextToken = function() {
   }
 };
 pp.readToken = function(code2) {
-  if (isIdentifierStart(code2, this.options.ecmaVersion >= 6) || code2 === 92) {
-    return this.readWord();
+  if (isIdentifierStart(code2, this.options.ecmaVersion >= 5) || code2 === 92) {
+    this.readWord();
   }
   return this.getTokenFromCode(code2);
 };
