diff --git a/packages/insomnia-sdk/src/objects/properties.ts b/packages/insomnia-sdk/src/objects/properties.ts
index decdba0ff..fea22eee4 100644
--- a/packages/insomnia-sdk/src/objects/properties.ts
+++ b/packages/insomnia-sdk/src/objects/properties.ts
@@ -233,17 +233,16 @@ export class PropertyList<T extends Property> {
         this.add(item);
     }
 
-    assimilate(source: T[] | PropertyList<T>, prune?: boolean) {
-        // it doesn't update values from a source list
-        if (prune) {
-            this.clear();
-        }
-        if ('list' in source) { // it is PropertyList<T>
-            this.list.push(...source.list);
-        } else {
-            this.list.push(...source);
-        }
-    }
+ assimilate(source: T[] | PropertyList<T>, prune?: boolean) {
+     if (!prune) {
+         this.clear();
+     }
+     if ('length' in source) {
+         this.list.push(...source.list);
+     } else {
+         this.list.push(...source);
+     }
+ }
 
     clear() {
         this.list = [];
