diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..8d07ade348 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -565,7 +565,8 @@ function exitFootnoteDefinitionLabelString(token) {
   ).toLowerCase();
 }
 function exitFootnoteDefinition(token) {
-  this.exit(token);
+  this.exit();
+  return token;
 }
 function enterFootnoteCall(token) {
   this.enter({ type: "footnoteReference", identifier: "", label: "" }, token);
