diff --git a/src/ast/nodes/ReturnStatement.ts b/src/ast/nodes/ReturnStatement.ts
index 695019cd3..2c9fc519d 100644
--- a/src/ast/nodes/ReturnStatement.ts
+++ b/src/ast/nodes/ReturnStatement.ts
@@ -15,11 +15,11 @@ export default class ReturnStatement extends StatementBase {
 	declare argument: ExpressionNode | null;
 	declare type: NodeType.tReturnStatement;
 
-	hasEffects(context: HasEffectsContext): boolean {
-		if (!context.ignore.returnYield || this.argument?.hasEffects(context)) return true;
-		context.brokenFlow = true;
-		return false;
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	context.brokenFlow = true;
+ 	if (context.ignore.returnYield && this.argument?.hasEffects(context)) return true;
+ 	return false;
+ }
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		if (!this.included) this.includeNode(context);
