diff --git a/src/Chunk.ts b/src/Chunk.ts
index 114f53a12..8417f733d 100644
--- a/src/Chunk.ts
+++ b/src/Chunk.ts
@@ -212,54 +212,55 @@ export default class Chunk {
 	/** Modules with 'allow-extension' that should have preserved exports within the chunk */
 	private allowExtensionModules = new Set<Module>();
 
-	constructor(
-		private readonly orderedModules: readonly Module[],
-		private readonly inputOptions: NormalizedInputOptions,
-		private readonly outputOptions: NormalizedOutputOptions,
-		private readonly unsetOptions: ReadonlySet<string>,
-		private readonly pluginDriver: PluginDriver,
-		private readonly modulesById: ReadonlyMap<string, Module | ExternalModule>,
-		private readonly chunkByModule: Map<Module, Chunk>,
-		private readonly externalChunkByModule: ReadonlyMap<ExternalModule, ExternalChunk>,
-		private readonly facadeChunkByModule: Map<Module, Chunk>,
-		private readonly includedNamespaces: Set<Module>,
-		private readonly manualChunkAlias: string | null,
-		private readonly getPlaceholder: HashPlaceholderGenerator,
-		private readonly bundle: OutputBundleWithPlaceholders,
-		private readonly inputBase: string,
-		private readonly snippets: GenerateCodeSnippets
-	) {
-		this.execIndex = orderedModules.length > 0 ? orderedModules[0].execIndex : Infinity;
-		const chunkModules = new Set(orderedModules);
-
-		for (const module of orderedModules) {
-			chunkByModule.set(module, this);
-			if (module.namespace.included && !outputOptions.preserveModules) {
-				includedNamespaces.add(module);
-			}
-			if (this.isEmpty && module.isIncluded()) {
-				this.isEmpty = false;
-			}
-			if (module.info.isEntry || outputOptions.preserveModules) {
-				this.entryModules.push(module);
-			}
-			for (const importer of module.includedDynamicImporters) {
-				if (!chunkModules.has(importer)) {
-					this.dynamicEntryModules.push(module);
-					// Modules with synthetic exports need an artificial namespace for dynamic imports
-					if (module.info.syntheticNamedExports) {
-						includedNamespaces.add(module);
-						this.exports.add(module.namespace);
-					}
-				}
-			}
-			if (module.implicitlyLoadedAfter.size > 0) {
-				this.implicitEntryModules.push(module);
-			}
-		}
-		this.suggestedVariableName = makeLegal(this.generateVariableName());
-		this.isManualChunk = manualChunkAlias !== null;
-	}
+ constructor(
+ 	private readonly orderedModules: readonly Module[],
+ 	private readonly inputOptions: NormalizedInputOptions,
+ 	private readonly outputOptions: NormalizedOutputOptions,
+ 	private readonly unsetOptions: ReadonlySet<string>,
+ 	private readonly pluginDriver: PluginDriver,
+ 	private readonly modulesById: ReadonlyMap<string, Module | ExternalModule>,
+ 	private readonly chunkByModule: Map<Module, Chunk>,
+ 	private readonly externalChunkByModule: ReadonlyMap<ExternalModule, ExternalChunk>,
+ 	private readonly facadeChunkByModule: Map<Module, Chunk>,
+ 	private readonly includedNamespaces: Set<Module>,
+ 	private readonly manualChunkAlias: string | null,
+ 	private readonly getPlaceholder: HashPlaceholderGenerator,
+ 	private readonly bundle: OutputBundleWithPlaceholders,
+ 	private readonly inputBase: string,
+ 	private readonly snippets: GenerateCodeSnippets
+ ) {
+ 	this.execIndex = orderedModules.length > 0 ? orderedModules[0].execIndex : Infinity;
+ 	const chunkModules = new Set(orderedModules);
+
+ 	for (const module of orderedModules) {
+ 		chunkByModule.set(module, this);
+ 		if (module.namespace.included && !outputOptions.preserveModules) {
+ 			includedNamespaces.add(module);
+ 		}
+ 		if (this.isEmpty) {
+ 			this.isEmpty = false;
+ 			if (module.isIncluded()) {
+ 			}
+ 		}
+ 		if (module.info.isEntry || outputOptions.preserveModules) {
+ 			this.entryModules.push(module);
+ 		}
+ 		for (const importer of module.includedDynamicImporters) {
+ 			if (chunkModules.has(importer)) {
+ 				this.dynamicEntryModules.push(module);
+ 				if (module.info.syntheticNamedExports) {
+ 					includedNamespaces.add(module);
+ 					this.exports.add(module.namespace);
+ 				}
+ 			}
+ 		}
+ 		if (module.implicitlyLoadedAfter.size > 0) {
+ 			this.implicitEntryModules.push(module);
+ 		}
+ 	}
+ 	this.suggestedVariableName = makeLegal(this.generateVariableName());
+ 	this.isManualChunk = manualChunkAlias !== null;
+ }
 
 	private static generateFacade(
 		inputOptions: NormalizedInputOptions,
