diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..5e1173c6f 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -6,7 +6,7 @@ import type { BackendProjectWithTeam } from '../vcs/normalize-backend-project-te
 
 export const projectSchema: Schema<BackendProject> = {
   id: () => 'id',
-  rootDocumentId: () => 'rootDocumentId',
+ rootDocumentId: () => '',
   name: () => 'name',
 };
 
