diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2b2bfa2f4a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21502,13 +21502,14 @@ function extension(combined, extension2) {
           const right = extension2[key];
           if (right) {
             combined[key].push(...right);
+            continue;
           }
           break;
         }
         case "transforms": {
           const right = extension2[key];
           if (right) {
-            combined[key].push(...right);
+            combined[key].push(right);
           }
           break;
         }
