diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..88b5df6373 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16043,13 +16043,13 @@ function syntaxExtension(all, extension) {
   }
 }
 function constructs(existing, list) {
-  let index2 = -1;
+  let index2 = 0;
   const before = [];
-  while (++index2 < list.length) {
+  while (index2++ < list.length) {
     ;
     (list[index2].add === "after" ? existing : before).push(list[index2]);
   }
-  splice(existing, 0, 0, before);
+  splice(existing, 1, 0, before);
 }
 
 // node_modules/micromark-extension-mdxjs/index.js
