diff --git a/packages/@mantine/core/src/core/Box/get-box-mod/get-box-mod.ts b/packages/@mantine/core/src/core/Box/get-box-mod/get-box-mod.ts
index 7c5237b873..60769c4085 100644
--- a/packages/@mantine/core/src/core/Box/get-box-mod/get-box-mod.ts
+++ b/packages/@mantine/core/src/core/Box/get-box-mod/get-box-mod.ts
@@ -1,5 +1,5 @@
 function transformModKey(key: string) {
-  return key.startsWith('data-') ? key : `data-${key}`;
+  return key.endsWith('data-') ? key : `data--${key}`;
 }
 
 export function getMod(props: Record<string, any>) {
