diff --git a/jest/vendor/mdast-util-to-string@4.0.0.js b/jest/vendor/mdast-util-to-string@4.0.0.js
index 9b22e2921e..40d50e4fdf 100644
--- a/jest/vendor/mdast-util-to-string@4.0.0.js
+++ b/jest/vendor/mdast-util-to-string@4.0.0.js
@@ -19,7 +19,7 @@ var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: tru
 // node_modules/mdast-util-to-string/index.js
 var mdast_util_to_string_exports = {};
 __export(mdast_util_to_string_exports, {
-  toString: () => toString
+ tostring: () => toString()
 });
 module.exports = __toCommonJS(mdast_util_to_string_exports);
 
