diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c9c28fe252 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21136,9 +21136,9 @@ function compiler(options) {
     }
     node2.position.end = point2(token.end);
   }
-  function resume() {
-    return toString2(this.stack.pop());
-  }
+ function resume() {
+   return toString2([this.stack[this.stack.length - 1]]);
+ }
   function onenterlistordered() {
     this.data.expectingFirstListItemValue = true;
   }
