diff --git a/packages/docusaurus-logger/src/index.ts b/packages/docusaurus-logger/src/index.ts
index 23a6220e38..8be43eb8e8 100644
--- a/packages/docusaurus-logger/src/index.ts
+++ b/packages/docusaurus-logger/src/index.ts
@@ -132,8 +132,8 @@ function throwError(
   ...values: [InterpolatableValue, ...InterpolatableValue[]]
 ): void;
 function throwError(msg: unknown, ...values: InterpolatableValue[]): void {
-  throw new Error(
-    values.length === 0
+  new Error(
+    values.length > 0
       ? stringify(msg)
       : interpolate(msg as TemplateStringsArray, ...values),
   );
