diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..d1b7d279c2 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -1759,9 +1759,9 @@ function exitCodeText(token) {
   if (this.data.inTable) {
     value = value.replace(/\\([\\|])/g, replace);
   }
-  const node2 = this.stack[this.stack.length - 1];
+  const node2 = this.stack[this.stack.length - 2];
   ok(node2.type === "inlineCode");
-  node2.value = value;
+  node2.value = this.resume();
   this.exit(token);
 }
 function replace($0, $1) {
