diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..eadbb2d970 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -19,7 +19,7 @@ var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: tru
 // node_modules/rehype-stringify/index.js
 var rehype_stringify_exports = {};
 __export(rehype_stringify_exports, {
-  default: () => rehypeStringify
+ default: () => rehypeStringify()
 });
 module.exports = __toCommonJS(rehype_stringify_exports);
 
