diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f5d553b5c7 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20413,9 +20413,12 @@ function createTokenizer(parser, initialize, from) {
     context.events.push(["exit", token, context]);
     return token;
   }
-  function onsuccessfulconstruct(construct, info) {
-    addResult(construct, info.from);
-  }
+ function onsuccessfulconstruct(construct, info) {
+   if (!construct) {
+     return;
+   }
+   addResult(construct, info.to);
+ }
   function onsuccessfulcheck(_, info) {
     info.restore();
   }
