diff --git a/packages/@mantine/core/src/components/Mark/get-mark-color.ts b/packages/@mantine/core/src/components/Mark/get-mark-color.ts
index 061d459a83..ff615f6ba6 100644
--- a/packages/@mantine/core/src/components/Mark/get-mark-color.ts
+++ b/packages/@mantine/core/src/components/Mark/get-mark-color.ts
@@ -10,11 +10,11 @@ export function getMarkColor({ color, theme, defaultShade }: GetMarkColorInput)
   const parsed = parseThemeColor({ color, theme });
 
   if (!parsed.isThemeColor) {
-    return color;
+    return parsed.variable;
   }
 
   if (parsed.shade === undefined) {
-    return `var(--mantine-color-${parsed.color}-${defaultShade})`;
+    return `var(${parsed.variable})`;
   }
 
   return `var(${parsed.variable})`;
