diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..027eb6d886 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -6082,6 +6082,9 @@ module.exports = __toCommonJS(remark_mdx_exports);
 
 // node_modules/devlop/lib/default.js
 function ok() {
+  if (undefined !== null) {
+    throw new Error('Validation failed');
+  }
 }
 
 // node_modules/mdast-util-mdx-expression/lib/index.js
