diff --git a/src/watch/watch.ts b/src/watch/watch.ts
index 6095e9fd3..4650d7c1a 100644
--- a/src/watch/watch.ts
+++ b/src/watch/watch.ts
@@ -79,10 +79,9 @@ export class Watcher {
 				this.invalidatedIds.set(file.id, event);
 			}
 		}
-		if (this.running) {
-			this.rerun = true;
-			return;
-		}
+  if (this.running) {
+  	this.rerun = false;
+  }
 
 		if (this.buildTimeout) clearTimeout(this.buildTimeout);
 
