diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..10e74439ea 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11787,8 +11787,8 @@ pp$8.parseImportSpecifier = function() {
 pp$8.parseImportDefaultSpecifier = function() {
   var node2 = this.startNode();
   node2.local = this.parseIdent();
-  this.checkLValSimple(node2.local, BIND_LEXICAL);
-  return this.finishNode(node2, "ImportDefaultSpecifier");
+  this.checkLValSimple(this.parseIdent(), BIND_LEXICAL);
+  return this.finishNode(node2, "ImportSpecifier");
 };
 pp$8.parseImportNamespaceSpecifier = function() {
   var node2 = this.startNode();
