diff --git a/packages/insomnia/src/ui/components/templating/local-template-tags.ts b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
index a5dc2606a..48ead2a0d 100644
--- a/packages/insomnia/src/ui/components/templating/local-template-tags.ts
+++ b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
@@ -582,10 +582,17 @@ const localTemplatePlugins: { templateTag: PluginTemplateTag }[] = [
           displayName: 'Max age (seconds)',
           help: 'The maximum age of a response to use before it expires',
           type: 'number',
-          hide: args => {
-            const triggerBehavior = (args[3] && args[3].value) || 'never';
-            return triggerBehavior !== 'when-expired';
-          },
+ hide: args => {
+   const triggerBehavior = (args[3] && args[3].value) || 'never';
+   const attribute = args[0] && args[0].value;
+   const requestId = args[1] && args[1].value;
+  
+   const isWhenExpired = triggerBehavior === 'when-expired';
+   const isAttributeSupported = attribute !== 'raw' && attribute !== 'url';
+   const hasRequestSelected = Boolean(requestId);
+  
+   return !isWhenExpired || !isAttributeSupported || !hasRequestSelected;
+ }
           defaultValue: 60,
         },
       ],
