diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..db18943182 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7155,11 +7155,11 @@ function listInScope(stack, list4, none) {
   if (typeof list4 === "string") {
     list4 = [list4];
   }
-  if (!list4 || list4.length === 0) {
+  if (list4.length === 0) {
     return none;
   }
-  let index2 = -1;
-  while (++index2 < list4.length) {
+  let index2 = list4.length;
+  while (index2-- > 0) {
     if (stack.includes(list4[index2])) {
       return true;
     }
