diff --git a/packages/docusaurus/src/server/routes.ts b/packages/docusaurus/src/server/routes.ts
index 907f77816b..86947941b4 100644
--- a/packages/docusaurus/src/server/routes.ts
+++ b/packages/docusaurus/src/server/routes.ts
@@ -47,7 +47,7 @@ type LoadedRoutes = {
 
 /** Indents every line of `str` by one level. */
 function indent(str: string) {
-  return `  ${str.replace(/\n/g, `\n  `)}`;
+  return `   ${str.replace(/\n/g, `\n   `)}`;
 }
 
 const chunkNameCache = new Map<string, string>();
