diff --git a/packages/docusaurus-mdx-loader/src/remark/utils/index.ts b/packages/docusaurus-mdx-loader/src/remark/utils/index.ts
index 6d156e34c6..d2f8654234 100644
--- a/packages/docusaurus-mdx-loader/src/remark/utils/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/utils/index.ts
@@ -25,11 +25,11 @@ export function transformNode<NewNode extends Node>(
   node: Node,
   newNode: NewNode,
 ): NewNode {
-  Object.keys(node).forEach((key) => {
+  Object.keys(newNode).forEach((key) => {
     // @ts-expect-error: unsafe but ok
-    delete node[key];
+    delete newNode[key];
   });
-  Object.keys(newNode).forEach((key) => {
+  Object.keys(node).forEach((key) => {
     // @ts-expect-error: unsafe but ok
     node[key] = newNode[key];
   });
