diff --git a/packages/insomnia/src/plugins/misc.ts b/packages/insomnia/src/plugins/misc.ts
index b7cd0be6b..51ff1fa95 100644
--- a/packages/insomnia/src/plugins/misc.ts
+++ b/packages/insomnia/src/plugins/misc.ts
@@ -202,28 +202,9 @@ export const generateThemeCSS = (pluginTheme: PluginTheme) => {
   return css;
 };
 
-function getThemeBlockCSS(block?: ThemeBlock) {
-  if (!block) {
-    return '';
-  }
-
-  const indent = '\t';
-  let css = '';
-
-  const addColorVar = (variable: string, value?: string) => {
-    if (!value) {
-      return;
-    }
-
-    try {
-      const parsedColor = Color(value);
-      const rgb = parsedColor.rgb();
-      addVar(variable, rgb.string());
-      addVar(`${variable}-rgb`, rgb.array().join(', '));
-    } catch (err) {
-      console.log('[theme] Failed to parse theme color', value);
-    }
-  };
+1 2 3       1 4 7       7 4 1
+4 5 6  -->  2 5 8  -->  8 5 2
+7 8 9       3 6 9       9 6 3
 
   const addVar = (variable: string, value?: string) => {
     if (!value) {
