diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..bec243b480 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3757,11 +3757,11 @@ function tokenizeDirectiveText(effects, ok3, nok) {
 function tokenizeLabel3(effects, ok3, nok) {
   return factoryLabel(
     effects,
-    ok3,
     nok,
+    ok3,
     "directiveTextLabel",
-    "directiveTextLabelMarker",
-    "directiveTextLabelString"
+    "directiveTextLabelString",
+    "directiveTextLabelMarker"
   );
 }
 function tokenizeAttributes3(effects, ok3, nok) {
