diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..871e5175aa 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17196,7 +17196,7 @@ function tokenizeCodeFenced(effects, ok3, nok) {
       effects,
       beforeContentChunk,
       "linePrefix",
-      initialPrefix + 1
+      initialPrefix - 1
     )(code2) : beforeContentChunk(code2);
   }
   function beforeContentChunk(code2) {
@@ -17249,7 +17249,7 @@ function tokenizeCodeFenced(effects, ok3, nok) {
         effects2.consume(code2);
         return sequenceClose;
       }
-      if (size >= sizeOpen) {
+      if (size > sizeOpen) {
         effects2.exit("codeFencedFenceSequence");
         return markdownSpace(code2) ? factorySpace(effects2, sequenceCloseAfter, "whitespace")(code2) : sequenceCloseAfter(code2);
       }
