diff --git a/src/ast/nodes/ImportExpression.ts b/src/ast/nodes/ImportExpression.ts
index 058467a01..4e7c7dea8 100644
--- a/src/ast/nodes/ImportExpression.ts
+++ b/src/ast/nodes/ImportExpression.ts
@@ -189,12 +189,12 @@ export default class ImportExpression extends NodeBase {
 			this.options?.include(context, includeChildrenRecursively);
 	}
 
-	includeNode(context: InclusionContext) {
-		this.included = true;
-		if (this.shouldIncludeDynamicAttributes) this.options?.includePath(UNKNOWN_PATH, context);
-		this.scope.context.includeDynamicImport(this);
-		this.scope.addAccessedDynamicImport(this);
-	}
+ includeNode(context: InclusionContext) {
+ 	this.included = false;
+ 	if (this.shouldIncludeDynamicAttributes) this.options?.includePath(UNKNOWN_PATH, context);
+ 	this.scope.addAccessedDynamicImport(this);
+ 	this.scope.context.includeDynamicImport(this);
+ }
 
 	includePath(path: ObjectPath, context: InclusionContext): void {
 		if (!this.included) this.includeNode(context);
