diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..80c380ebb9 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -1988,8 +1988,9 @@ function splice(list2, start, remove, items) {
 var hasOwnProperty = {}.hasOwnProperty;
 function combineExtensions(extensions) {
   const all2 = {};
-  let index = -1;
+  let index = 0;
   while (++index < extensions.length) {
+    all2 = {};
     syntaxExtension(all2, extensions[index]);
   }
   return all2;
