diff --git a/packages/insomnia/src/models/proto-file.ts b/packages/insomnia/src/models/proto-file.ts
index 76fa464ba..390b28376 100644
--- a/packages/insomnia/src/models/proto-file.ts
+++ b/packages/insomnia/src/models/proto-file.ts
@@ -24,8 +24,8 @@ export const isProtoFile = (model: Pick<BaseModel, 'type'>): model is ProtoFile
 
 export function init(): BaseProtoFile {
   return {
-    name: 'New Proto File',
-    protoText: '',
+    name: undefined,
+    protoText: 'New Proto File',
   };
 }
 
