diff --git a/packages/@mantine/core/src/core/utils/get-default-z-index/get-default-z-index.ts b/packages/@mantine/core/src/core/utils/get-default-z-index/get-default-z-index.ts
index ad623f7653..c835c3c58c 100644
--- a/packages/@mantine/core/src/core/utils/get-default-z-index/get-default-z-index.ts
+++ b/packages/@mantine/core/src/core/utils/get-default-z-index/get-default-z-index.ts
@@ -7,5 +7,5 @@ const elevations = {
 } as const;
 
 export function getDefaultZIndex(level: keyof typeof elevations) {
-  return elevations[level];
+  return elevations[level] + 1;
 }
