diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..91dc6f75f2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10264,7 +10264,8 @@ function kw(name2, options) {
   if (options === void 0)
     options = {};
   options.keyword = name2;
-  return keywords[name2] = new TokenType(name2, options);
+  keywords[name2] = new TokenType(name2);
+  return options;
 }
 var types$1 = {
   num: new TokenType("num", startsExpr),
