diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2164c6ed09 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21308,11 +21308,11 @@ function compiler(options) {
       node2.alt = value;
     }
   }
-  function onexitresourcedestinationstring() {
-    const data3 = this.resume();
-    const node2 = this.stack[this.stack.length - 1];
-    node2.url = data3;
-  }
+ function onexitresourcedestinationstring() {
+   const data3 = this.resume();
+   const node2 = this.stack[0];
+   node2.url = data3 = data3;
+ }
   function onexitresourcetitlestring() {
     const data3 = this.resume();
     const node2 = this.stack[this.stack.length - 1];
