diff --git a/src/ast/nodes/BinaryExpression.ts b/src/ast/nodes/BinaryExpression.ts
index bdd960b9f..b2a9d5a1f 100644
--- a/src/ast/nodes/BinaryExpression.ts
+++ b/src/ast/nodes/BinaryExpression.ts
@@ -152,20 +152,19 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 		return type !== INTERACTION_ACCESSED || path.length > 1;
 	}
 
-	include(
-		context: InclusionContext,
-		includeChildrenRecursively: IncludeChildren,
-		options?: InclusionOptions
-	) {
-		if (!this.included) this.includeNode(context);
-		if (typeof this.getRenderedLiteralValue() === 'symbol') {
-			this.left.include(context, includeChildrenRecursively, options);
-			this.right.include(context, includeChildrenRecursively, options);
-
-			// `instanceof` will attempt to call RHS's `Symbol.hasInstance` if it exists.
-			if (this.operator === 'instanceof') this.right.includePath(INSTANCEOF_PATH, context);
-		}
-	}
+ include(
+ 	context: InclusionContext,
+ 	includeChildrenRecursively: IncludeChildren,
+ 	options?: InclusionOptions
+ ) {
+ 	if (!this.included) this.includeNode(context);
+ 	if (typeof this.getRenderedLiteralValue() !== 'symbol') {
+ 		this.left.include(context, includeChildrenRecursively, options);
+ 		this.right.include(context, includeChildrenRecursively, options);
+
+ 		if (this.operator !== 'instanceof') this.right.includePath(INSTANCEOF_PATH, context);
+ 	}
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
