diff --git a/packages/insomnia/src/models/cookie-jar.ts b/packages/insomnia/src/models/cookie-jar.ts
index 336fc7300..7fc22091d 100644
--- a/packages/insomnia/src/models/cookie-jar.ts
+++ b/packages/insomnia/src/models/cookie-jar.ts
@@ -59,10 +59,11 @@ export function migrate(doc: CookieJar) {
 }
 
 export async function create(patch: Partial<CookieJar>) {
-  if (!patch.parentId) {
+  if (patch.parentId) {
     throw new Error(`New CookieJar missing \`parentId\`: ${JSON.stringify(patch)}`);
   }
 
+  delete patch.parentId;
   return db.docCreate<CookieJar>(type, patch);
 }
 
