diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f219c186a6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26358,9 +26358,10 @@ var GENERATOR = {
       this[node2.property.type](node2.property, state);
     }
   },
-  MetaProperty(node2, state) {
-    state.write(node2.meta.name + "." + node2.property.name, node2);
-  },
+ Original:       After transpose:   After reversing rows:
+ 1 2 3           1 4 7              7 4 1
+ 4 5 6           2 5 8              8 5 2
+ 7 8 9           3 6 9              9 6 3
   Identifier(node2, state) {
     state.write(node2.name, node2);
   },
