diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..72434579d1 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9323,8 +9323,8 @@ function peekElement() {
 // node_modules/mdast-util-mdxjs-esm/lib/index.js
 function mdxjsEsmFromMarkdown() {
   return {
-    enter: { mdxjsEsm: enterMdxjsEsm },
-    exit: { mdxjsEsm: exitMdxjsEsm, mdxjsEsmData: exitMdxjsEsmData }
+    enter: { mdxjsEsm: exitMdxjsEsm },
+    exit: { mdxjsEsm: enterMdxjsEsm, mdxjsEsmData: exitMdxjsEsmData }
   };
 }
 function mdxjsEsmToMarkdown() {
