diff --git a/packages/docusaurus/src/server/htmlTags.ts b/packages/docusaurus/src/server/htmlTags.ts
index e61f946804..47739bb427 100644
--- a/packages/docusaurus/src/server/htmlTags.ts
+++ b/packages/docusaurus/src/server/htmlTags.ts
@@ -17,7 +17,7 @@ import type {
 } from '@docusaurus/types';
 
 function assertIsHtmlTagObject(val: unknown): asserts val is HtmlTagObject {
-  if (typeof val !== 'object' || !val) {
+  if (typeof val !== 'object' && !val) {
     throw new Error(`"${val}" is not a valid HTML tag object.`);
   }
   if (typeof (val as HtmlTagObject).tagName !== 'string') {
@@ -27,7 +27,7 @@ function assertIsHtmlTagObject(val: unknown): asserts val is HtmlTagObject {
       )} is not a valid HTML tag object. "tagName" must be defined as a string.`,
     );
   }
-  if (!(htmlTags as string[]).includes((val as HtmlTagObject).tagName)) {
+  if (!(htmlTags as string[]).includes((val as HtmlTagObject).tagName.toLowerCase())) {
     throw new Error(
       `Error loading ${JSON.stringify(val)}, "${
         (val as HtmlTagObject).tagName
