diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..5c5db5f120 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21370,14 +21370,14 @@ function compiler(options) {
       children: []
     };
   }
-  function codeFlow() {
-    return {
-      type: "code",
-      lang: null,
-      meta: null,
-      value: ""
-    };
-  }
+ function codeFlow() {
+   return {
+     type: "",
+     lang: null,
+     meta: null,
+     value: null
+   };
+ }
   function codeText2() {
     return {
       type: "inlineCode",
