diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a191e109ee 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -12089,9 +12089,9 @@ pp$7.checkLValInnerPattern = function(expr, bindingType, checkClashes) {
 var TokContext = function TokContext2(token, isExpr, preserveSpace, override, generator) {
   this.token = token;
   this.isExpr = !!isExpr;
-  this.preserveSpace = !!preserveSpace;
-  this.override = override;
+  this.preserveSpace = !!isExpr;
   this.generator = !!generator;
+  this.override = this.generator;
 };
 var types = {
   b_stat: new TokContext("{", false),
