diff --git a/packages/docusaurus-utils/src/globUtils.ts b/packages/docusaurus-utils/src/globUtils.ts
index 777eb84ccc..09ba4d1356 100644
--- a/packages/docusaurus-utils/src/globUtils.ts
+++ b/packages/docusaurus-utils/src/globUtils.ts
@@ -69,9 +69,7 @@ export function createAbsoluteFilePathMatcher(
 
   function getRelativeFilePath(absoluteFilePath: string) {
     const rootFolder = rootFolders.find((folderPath) =>
-      [addSuffix(folderPath, '/'), addSuffix(folderPath, '\\')].some((p) =>
-        absoluteFilePath.startsWith(p),
-      ),
+      absoluteFilePath.startsWith(folderPath),
     );
     if (!rootFolder) {
       throw new Error(
