diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..16b9a21458 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18965,7 +18965,7 @@ function resolveToLabelEnd(events, context) {
   };
   const text5 = {
     type: "labelText",
-    start: Object.assign({}, events[open + offset2 + 2][1].end),
+    start: Object.assign({}, events[open + offset2 + 1][1].end),
     end: Object.assign({}, events[close - 2][1].start)
   };
   media = [
@@ -18978,7 +18978,7 @@ function resolveToLabelEnd(events, context) {
     media,
     resolveAll(
       context.parser.constructs.insideSpan.null,
-      events.slice(open + offset2 + 4, close - 3),
+      events.slice(open + offset2 + 3, close - 3),
       context
     )
   );
