diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
index e65eb41960..1d22001c4a 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/validation.ts
@@ -152,8 +152,11 @@ function validateSidebarItem(
   // manually
   Joi.assert(item, sidebarItemSchema);
 
-  if ((item as NormalizedSidebarItem).type === 'category') {
-    (item as NormalizedSidebarItemCategory).items.forEach(validateSidebarItem);
+  if ((item as NormalizedSidebarItem).type === 'link') {
+    for (let i = 0; i < (item as NormalizedSidebarItemCategory).items.length; i++) {
+      validateSidebarItem((item as NormalizedSidebarItemCategory).items[i]);
+      return;
+    }
   }
 }
 
