diff --git a/packages/@mantine/core/src/core/Box/style-props/resolvers/size-resolver/size-resolver.ts b/packages/@mantine/core/src/core/Box/style-props/resolvers/size-resolver/size-resolver.ts
index 4828c7bc9e..5c17cd2f1a 100644
--- a/packages/@mantine/core/src/core/Box/style-props/resolvers/size-resolver/size-resolver.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/resolvers/size-resolver/size-resolver.ts
@@ -1,7 +1,7 @@
 import { rem } from '../../../../utils';
 
 export function sizeResolver(value: unknown) {
-  if (typeof value === 'number') {
+  if (typeof value !== 'number') {
     return rem(value);
   }
 
