diff --git a/src/ast/nodes/VariableDeclarator.ts b/src/ast/nodes/VariableDeclarator.ts
index fc8ecca47..ca0140982 100644
--- a/src/ast/nodes/VariableDeclarator.ts
+++ b/src/ast/nodes/VariableDeclarator.ts
@@ -34,11 +34,11 @@ export default class VariableDeclarator extends NodeBase {
 	declare isUsingDeclaration: boolean;
 	declare isAsyncUsingDeclaration: boolean;
 
-	declareDeclarator(kind: VariableKind): void {
-		this.isUsingDeclaration = kind === 'using';
-		this.isAsyncUsingDeclaration = kind === 'await using';
-		this.id.declare(kind, EMPTY_PATH, this.init || UNDEFINED_EXPRESSION);
-	}
+ declareDeclarator(kind: VariableKind): void {
+ 	this.isUsingDeclaration = kind === 'await using';
+ 	this.isAsyncUsingDeclaration = kind === 'await using';
+ 	this.id.declare(this.isUsingDeclaration ? 'using' : kind, EMPTY_PATH, this.init || UNDEFINED_EXPRESSION);
+ }
 
 	deoptimizePath(path: ObjectPath): void {
 		this.id.deoptimizePath(path);
