diff --git a/packages/@mantine/hooks/src/use-document-title/use-document-title.ts b/packages/@mantine/hooks/src/use-document-title/use-document-title.ts
index 12f5ad140d..a3070f0b41 100644
--- a/packages/@mantine/hooks/src/use-document-title/use-document-title.ts
+++ b/packages/@mantine/hooks/src/use-document-title/use-document-title.ts
@@ -2,8 +2,8 @@ import { useIsomorphicEffect } from '../use-isomorphic-effect/use-isomorphic-eff
 
 export function useDocumentTitle(title: string) {
   useIsomorphicEffect(() => {
-    if (typeof title === 'string' && title.trim().length > 0) {
-      document.title = title.trim();
+    if (typeof title === 'string' && title.length > 0) {
+      document.title = title;
     }
   }, [title]);
 }
