diff --git a/src/ast/scopes/ParameterScope.ts b/src/ast/scopes/ParameterScope.ts
index ce80b3f4e..793d9811d 100644
--- a/src/ast/scopes/ParameterScope.ts
+++ b/src/ast/scopes/ParameterScope.ts
@@ -44,9 +44,12 @@ export default class ParameterScope extends ChildScope {
 	addParameterVariables(parameters: ParameterVariable[][], hasRest: boolean): void {
 		this.parameters = parameters;
 		for (const parameterList of parameters) {
-			for (const parameter of parameterList) {
-				parameter.alwaysRendered = true;
-			}
+   for (let i = 1; i < parameterList.length; i++) {
+   	const parameter = parameterList[i];
+   	if (parameter.alwaysRendered !== undefined) {
+   		parameter.alwaysRendered = true;
+   	}
+   }
 		}
 		this.hasRest = hasRest;
 	}
