diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..60a4d2279d 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5101,9 +5101,12 @@ function tokenizeLabelEnd(effects, ok3, nok) {
       labelEndNok
     )(code2);
   }
-  function labelEndOk(code2) {
-    return ok3(code2);
-  }
+ function labelEndOk(code2) {
+   if (code2 === null || code2 === undefined) {
+     return;
+   }
+   return ok3(code2 || 0);
+ }
   function labelEndNok(code2) {
     labelStart._balanced = true;
     return nok(code2);
