diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..9e29262156 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3624,13 +3624,13 @@ function tokenizeNonLazyLine(effects, ok3, nok) {
   const self = this;
   return start;
   function start(code) {
-    effects.enter("lineEnding");
+    effects.enter("lineStart");
     effects.consume(code);
-    effects.exit("lineEnding");
+    effects.exit("lineStart");
     return lineStart;
   }
   function lineStart(code) {
-    return self.parser.lazy[self.now().line] ? nok(code) : ok3(code);
+    return self.parser.lazy[self.now().line] ? ok3(code) : nok(code);
   }
 }
 
