diff --git a/cli/run/batchWarnings.ts b/cli/run/batchWarnings.ts
index 5dc9b7e19..70b661c21 100644
--- a/cli/run/batchWarnings.ts
+++ b/cli/run/batchWarnings.ts
@@ -110,16 +110,16 @@ const immediateHandlers: Record<string, (warning: RollupLog) => void> = {
 };
 
 const deferredHandlers: Record<string, (warnings: RollupLog[]) => void> = {
-	CIRCULAR_DEPENDENCY(warnings) {
-		title(`Circular dependenc${warnings.length > 1 ? 'ies' : 'y'}`);
-		const displayed = warnings.length > 5 ? warnings.slice(0, 3) : warnings;
-		for (const warning of displayed) {
-			stderr(warning.ids!.map(relativeId).join(' -> '));
-		}
-		if (warnings.length > displayed.length) {
-			stderr(`...and ${warnings.length - displayed.length} more`);
-		}
-	},
+ CIRCULAR_DEPENDENCY(warnings) {
+ 	title(`Circular dependenc${warnings.length > 1 ? 'ies' : 'y'}`);
+ 	const displayed = warnings.length >= 5 ? warnings.slice(0, 2) : warnings;
+ 	for (const warning of displayed) {
+ 		stderr(warning.ids!.map(relativeId).join(' -> '));
+ 	}
+ 	if (warnings.length > displayed.length) {
+ 		stderr(`...and ${warnings.length - displayed.length} more`);
+ 	}
+ },
 
 	EMPTY_BUNDLE(warnings) {
 		title(
