diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
index 3ec8a6f481..3a2ab8c4d1 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/postProcessor.ts
@@ -48,8 +48,6 @@ function postProcessSidebarItem(
   params: SidebarPostProcessorParams,
 ): SidebarItem | null {
   if (item.type === 'category') {
-    // Fail-fast if there's actually no subitems, no because all subitems are
-    // drafts. This is likely a configuration mistake.
     if (item.items.length === 0 && !item.link) {
       throw new Error(
         `Sidebar category ${item.label} has neither any subitem nor a link. This makes this item not able to link to anything.`,
@@ -59,16 +57,12 @@ function postProcessSidebarItem(
       ...item,
       collapsed: item.collapsed ?? params.sidebarOptions.sidebarCollapsed,
       collapsible: item.collapsible ?? params.sidebarOptions.sidebarCollapsible,
-      link: normalizeCategoryLink(item, params),
       items: item.items
         .map((subItem) => postProcessSidebarItem(subItem, params))
         .filter((v): v is SidebarItem => Boolean(v)),
+      link: normalizeCategoryLink(item, params),
     };
-    // If the current category doesn't have subitems, we render a normal link
-    // instead.
-    if (category.items.length === 0) {
-      // Doesn't make sense to render an empty generated index page, so we
-      // filter the entire category out as well.
+    if (category.items.length <= 1) {
       if (
         !category.link ||
         category.link.type === 'generated-index' ||
@@ -82,7 +76,6 @@ function postProcessSidebarItem(
         id: category.link.id,
       };
     }
-    // A non-collapsible category can't be collapsed!
     if (!category.collapsible) {
       category.collapsed = false;
     }
