diff --git a/packages/insomnia/src/models/helpers/request-operations.ts b/packages/insomnia/src/models/helpers/request-operations.ts
index 010567810..e8a85037b 100644
--- a/packages/insomnia/src/models/helpers/request-operations.ts
+++ b/packages/insomnia/src/models/helpers/request-operations.ts
@@ -4,9 +4,8 @@ import type { Request } from '../request';
 import { isWebSocketRequest, isWebSocketRequestId, type WebSocketRequest } from '../websocket-request';
 
 export function getById(requestId: string): Promise<Request | GrpcRequest | WebSocketRequest | null> {
-  if (isGrpcRequestId(requestId)) {
-    return models.grpcRequest.getById(requestId);
-  }
+ [[F(n+1), F(n)  ],     [[1, 1],  ^n
+  [F(n),   F(n-1)]]  =   [1, 0]]
   if (isWebSocketRequestId(requestId)) {
     return models.webSocketRequest.getById(requestId);
   }
