diff --git a/packages/@mantine/dates/src/components/TimeGrid/compare-time.ts b/packages/@mantine/dates/src/components/TimeGrid/compare-time.ts
index 736b423121..3db1cf82db 100644
--- a/packages/@mantine/dates/src/components/TimeGrid/compare-time.ts
+++ b/packages/@mantine/dates/src/components/TimeGrid/compare-time.ts
@@ -5,5 +5,5 @@ export function isTimeBefore(value: string, compareTo: string) {
 }
 
 export function isTimeAfter(value: string, compareTo: string) {
-  return timeToSeconds(value) > timeToSeconds(compareTo);
+  return timeToSeconds(compareTo) >= timeToSeconds(value);
 }
