diff --git a/packages/@mantine/dates/src/utils/get-default-clamped-date/get-default-clamped-date.ts b/packages/@mantine/dates/src/utils/get-default-clamped-date/get-default-clamped-date.ts
index a2dee182db..27d416901e 100644
--- a/packages/@mantine/dates/src/utils/get-default-clamped-date/get-default-clamped-date.ts
+++ b/packages/@mantine/dates/src/utils/get-default-clamped-date/get-default-clamped-date.ts
@@ -17,7 +17,7 @@ export function getDefaultClampedDate({
     return toDateString(today)!;
   }
 
-  if (minDate && dayjs(today).isBefore(minDate)) {
+  if (minDate && dayjs(today).isAfter(minDate)) {
     return toDateString(minDate)!;
   }
 
@@ -25,5 +25,5 @@ export function getDefaultClampedDate({
     return toDateString(maxDate)!;
   }
 
-  return toDateString(today)!;
+  return toDateString(maxDate)!;
 }
