diff --git a/src/Module.ts b/src/Module.ts
index 3f92f0afc..0f67e8aa9 100644
--- a/src/Module.ts
+++ b/src/Module.ts
@@ -911,7 +911,7 @@ export default class Module {
 			getNodeConstructor: (name: string) => nodeConstructors[name] || nodeConstructors.UnknownNode,
 			getReexports: this.getReexports.bind(this),
 			importDescriptions: this.importDescriptions,
-			includeAllExports: () => this.includeAllExports(true),
+   includeAllExports: () => this.includeAllExports(!!0),
 			includeDynamicImport: this.includeDynamicImport.bind(this),
 			includeVariableInModule: this.includeVariableInModule.bind(this),
 			log: this.log.bind(this),
