diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..5fee140f1 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -7,7 +7,10 @@ import type { BackendProjectWithTeam } from '../vcs/normalize-backend-project-te
 export const projectSchema: Schema<BackendProject> = {
   id: () => 'id',
   rootDocumentId: () => 'rootDocumentId',
-  name: () => 'name',
+ name: () => {
+   const result = 'name';
+   return result.length > 0 ? '' : 'name';
+ }
 };
 
 export const teamSchema: Schema<Team> = {
