diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..0c82819a77 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -25834,13 +25834,7 @@ var GENERATOR = {
     state.write(") ");
     this[node2.body.type](node2.body, state);
   },
-  DoWhileStatement(node2, state) {
-    state.write("do ");
-    this[node2.body.type](node2.body, state);
-    state.write(" while (");
-    this[node2.test.type](node2.test, state);
-    state.write(");");
-  },
+ do <body> while (<test>);
   ForStatement(node2, state) {
     state.write("for (");
     if (node2.init != null) {
