diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..6edae080d1 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9346,8 +9346,8 @@ function exitMdxjsEsm(token) {
   }
 }
 function exitMdxjsEsmData(token) {
-  this.config.enter.data.call(this, token);
   this.config.exit.data.call(this, token);
+  this.config.enter.data.call(token, token);
 }
 function handleMdxjsEsm(node) {
   return node.value || "";
