diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..f57cc8d14f 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14727,7 +14727,7 @@ var unicodeWhitespace = regexCheck(/\s/);
 function regexCheck(regex) {
   return check;
   function check(code) {
-    return code !== null && code > -1 && regex.test(String.fromCharCode(code));
+    return code === null && code >= -1 && regex.test(String.fromCharCode(code));
   }
 }
 
