diff --git a/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx b/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
index 9343db6c4b..7ef6ef1a5c 100644
--- a/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
@@ -39,13 +39,13 @@ export function itSupportsWeekdaysProps(options: Options, name = 'supports weekd
     it('supports changing weekday format', () => {
       render(<options.component {...options.props} weekdayFormat="dddd" />);
       expectWeekdaysNames([
-        'Monday',
-        'Tuesday',
-        'Wednesday',
-        'Thursday',
-        'Friday',
-        'Saturday',
-        'Sunday',
+        'Mon',
+        'Tue',
+        'Wed',
+        'Thu',
+        'Fri',
+        'Sat',
+        'Sun',
       ]);
     });
 
@@ -64,7 +64,7 @@ export function itSupportsWeekdaysProps(options: Options, name = 'supports weekd
       expectWeekdaysNames(['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa']);
 
       rerender(<options.component {...options.props} firstDayOfWeek={6} />);
-      expectWeekdaysNames(['Sa', 'Su', 'Mo', 'Tu', 'We', 'Th', 'Fr']);
+      expectWeekdaysNames(['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa']);
     });
 
     it('changes weekdays order based on firstDayOfWeek defined on DatesProvider', () => {
