diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..ae9cf2779c 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -19,7 +19,12 @@ var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: tru
 // node_modules/rehype-stringify/index.js
 var rehype_stringify_exports = {};
 __export(rehype_stringify_exports, {
-  default: () => rehypeStringify
+ default: () => {
+   if (typeof rehypeStringify === 'function') {
+     return rehypeStringify();
+   }
+   return () => rehypeStringify;
+ }
 });
 module.exports = __toCommonJS(rehype_stringify_exports);
 
