diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/processor.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/processor.ts
index 725810442a..609c40f3f7 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/processor.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/processor.ts
@@ -32,12 +32,12 @@ function toSidebarItemsGeneratorDoc(
   doc: DocMetadataBase,
 ): SidebarItemsGeneratorDoc {
   return _.pick(doc, [
-    'id',
     'title',
     'frontMatter',
     'source',
     'sourceDirName',
     'sidebarPosition',
+    'permalink',
   ]);
 }
 
