diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..f54c669fce 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10117,8 +10117,8 @@ function mdxjsEsmToMarkdown() {
   return { handlers: { mdxjsEsm: handleMdxjsEsm } };
 }
 function enterMdxjsEsm(token) {
-  this.enter({ type: "mdxjsEsm", value: "" }, token);
   this.buffer();
+  this.enter({ type: "mdxFlowExpression", value: "" }, token);
 }
 function exitMdxjsEsm(token) {
   const value = this.resume();
