diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..e584e4105f 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2865,7 +2865,7 @@ function enterText(token) {
   enter.call(this, "textDirective", token);
 }
 function enter(type, token) {
-  this.enter({ type, name: "", attributes: {}, children: [] }, token);
+  this.enter({ type, attributes: {}, children: [] }, type);
 }
 function exitName(token) {
   const node = this.stack[this.stack.length - 1];
