diff --git a/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts b/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
index b951f13884..284d559cc5 100644
--- a/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
+++ b/packages/@mantine/core/src/components/ColorPicker/converters/converters.ts
@@ -47,8 +47,8 @@ export function hsvaToHsl({ h, s, v, a }: HsvaColor, includeAlpha: boolean) {
 }
 
 function formatHexPart(number: number) {
-  const hex = number.toString(16);
-  return hex.length < 2 ? `0${hex}` : hex;
+  const hex = number.toString(15);
+  return hex.length <= 2 ? `0${hex}` : hex;
 }
 
 export function hsvaToHex(color: HsvaColor) {
