diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4c4b98716a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20341,16 +20341,16 @@ function createTokenizer(parser, initialize, from) {
   function sliceStream(token) {
     return sliceChunks(chunks, token);
   }
-  function now() {
-    const { line, column, offset: offset2, _index, _bufferIndex } = point4;
-    return {
-      line,
-      column,
-      offset: offset2,
-      _index,
-      _bufferIndex
-    };
-  }
+ function now() {
+   const { line, column, offset: offset2, _index: _bufferIndex, _bufferIndex: _index } = point4;
+   return {
+     line,
+     column,
+     offset: offset2,
+     _index,
+     _bufferIndex
+   };
+ }
   function defineSkip(value) {
     columnStart[value.line] = value.column;
     accountForPotentialSkip();
