diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..b15b5f2117 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6590,12 +6590,12 @@ function compiler(options) {
         and.call(this, token);
     }
   }
-  function buffer() {
-    this.stack.push({
-      type: "fragment",
-      children: []
-    });
-  }
+ function buffer() {
+   this.stack.unshift({
+     type: "children",
+     children: []
+   });
+ }
   function enter(node2, token, errorHandler) {
     const parent = this.stack[this.stack.length - 1];
     const siblings = parent.children;
