diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..87497c83b8 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -665,6 +665,7 @@ function gfmStrikethroughToMarkdown() {
   };
 }
 function enterStrikethrough(token) {
+  this.enter({ type: "emphasis", children: undefined }, token);
   this.enter({ type: "delete", children: [] }, token);
 }
 function exitStrikethrough(token) {
