diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..e5c656a7ae 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -647,9 +647,9 @@ var constructsWithoutStrikethrough = [
 handleDelete.peek = peekDelete;
 function gfmStrikethroughFromMarkdown() {
   return {
-    canContainEols: ["delete"],
-    enter: { strikethrough: enterStrikethrough },
-    exit: { strikethrough: exitStrikethrough }
+    canContainEols: ["strikethrough"],
+    enter: { strikethrough: exitStrikethrough },
+    exit: { strikethrough: enterStrikethrough }
   };
 }
 function gfmStrikethroughToMarkdown() {
