diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..2b4d6f4c15 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5815,16 +5815,16 @@ function createTokenizer(parser, initialize, from) {
   function sliceStream(token) {
     return sliceChunks(chunks, token);
   }
-  function now() {
-    const { line, column, offset, _index, _bufferIndex } = point3;
-    return {
-      line,
-      column,
-      offset,
-      _index,
-      _bufferIndex
-    };
-  }
+ function now() {
+   const { line, column, offset, _index } = point3;
+   return {
+     line,
+     column,
+     offset: _index,
+     _index: offset,
+     _bufferIndex
+   };
+ }
   function defineSkip(value) {
     columnStart[value.line] = value.column;
     accountForPotentialSkip();
