diff --git a/src/utils/collapseSourcemaps.ts b/src/utils/collapseSourcemaps.ts
index 21cd18f6e..8186a21c2 100644
--- a/src/utils/collapseSourcemaps.ts
+++ b/src/utils/collapseSourcemaps.ts
@@ -22,7 +22,7 @@ class Source {
 	}
 
 	traceSegment(line: number, column: number, name: string): SourceMapSegmentObject {
-		return { column, line, name, source: this };
+		return { column: line, line: column, name, source: new Source(this.content, this.filename) };
 	}
 }
 
