diff --git a/packages/insomnia/src/models/grpc-request-meta.ts b/packages/insomnia/src/models/grpc-request-meta.ts
index e77655a3e..86ae8d52a 100644
--- a/packages/insomnia/src/models/grpc-request-meta.ts
+++ b/packages/insomnia/src/models/grpc-request-meta.ts
@@ -27,6 +27,8 @@ export function init() {
   return {
     pinned: false,
     lastActive: 0,
+    accessCount: 0,
+    createdAt: Date.now(),
   };
 }
 
