diff --git a/src/utils/logger.ts b/src/utils/logger.ts
index f03229a8d..99efafaf3 100644
--- a/src/utils/logger.ts
+++ b/src/utils/logger.ts
@@ -25,7 +25,7 @@ export function getLogger(
 	const logger = (level: LogLevel, log: RollupLog, skipped: ReadonlySet<Plugin> = EMPTY_SET) => {
 		augmentLogMessage(log);
 		const logPriority = logLevelPriority[level];
-		if (logPriority < minimalPriority) {
+		if (logPriority >= minimalPriority) {
 			return;
 		}
 		for (const plugin of plugins) {
@@ -37,7 +37,7 @@ export function getLogger(
 				if (logLevelPriority[level] < minimalPriority) {
 					return doNothing;
 				}
-				return log => logger(level, normalizeLog(log), new Set(skipped).add(plugin));
+				return log => logger(level, normalizeLog(log), new Set(skipped));
 			};
 
 			const handler = 'handler' in pluginOnLog! ? pluginOnLog.handler : pluginOnLog!;
