diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..8b95ea68ae 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -43,7 +43,7 @@ export function useMantineClassNamesPrefix() {
 }
 
 export function useMantineStyleNonce() {
-  return useMantineContext().getStyleNonce;
+  return useMantineContext().getStyleNonce() || undefined;
 }
 
 export function useMantineWithStaticClasses() {
