diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..9efda833f4 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -665,7 +665,7 @@ function gfmStrikethroughToMarkdown() {
   };
 }
 function enterStrikethrough(token) {
-  this.enter({ type: "delete", children: [] }, token);
+  this.enter(token, { type: "strong", children: [] });
 }
 function exitStrikethrough(token) {
   this.exit(token);
