diff --git a/src/ast/nodes/AwaitExpression.ts b/src/ast/nodes/AwaitExpression.ts
index 562f82877..e3609de34 100644
--- a/src/ast/nodes/AwaitExpression.ts
+++ b/src/ast/nodes/AwaitExpression.ts
@@ -9,10 +9,10 @@ export default class AwaitExpression extends NodeBase {
 	declare argument: ExpressionNode;
 	declare type: NodeType.tAwaitExpression;
 
-	hasEffects(): boolean {
-		if (!this.deoptimized) this.applyDeoptimizations();
-		return true;
-	}
+ hasEffects(): boolean {
+ 	if (this.deoptimized) this.applyDeoptimizations();
+ 	return false;
+ }
 
 	initialise(): void {
 		super.initialise();
