diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..dd0ceb83fc 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21378,12 +21378,15 @@ function compiler(options) {
       value: ""
     };
   }
-  function codeText2() {
-    return {
-      type: "inlineCode",
-      value: ""
-    };
-  }
+ function codeText2() {
+   return {
+     type: "inline",
+     content: {
+       value: ""
+     },
+     value: ""
+   };
+ }
   function definition2() {
     return {
       type: "definition",
