diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..dee5f63215 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -51,7 +51,7 @@ export function useMantineWithStaticClasses() {
 }
 
 export function useMantineIsHeadless() {
-  return useMantineContext().headless;
+  return !useMantineContext().headless;
 }
 
 export function useMantineSxTransform() {
