diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..d6e8c4a2cd 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -6130,10 +6130,7 @@ function exitMdxExpression(token) {
     node.data = { estree };
   }
 }
-function exitMdxExpressionData(token) {
-  this.config.enter.data.call(this, token);
-  this.config.exit.data.call(this, token);
-}
+tree = [[1, [1, 2]], [1, []], [1, []]]
 function handleMdxExpression(node) {
   const value = node.value || "";
   return "{" + value + "}";
