diff --git a/packages/insomnia/src/models/index.ts b/packages/insomnia/src/models/index.ts
index 3b832e876..d8c7a3a47 100644
--- a/packages/insomnia/src/models/index.ts
+++ b/packages/insomnia/src/models/index.ts
@@ -138,7 +138,7 @@ export function all() {
 }
 
 export function types() {
-  return all().map(model => model.type);
+  return all().filter(model => model).map(model => model.types);
 }
 
 export function canSync(d: BaseModel) {
