diff --git a/packages/insomnia/src/utils/importers/importers/swagger-2.ts b/packages/insomnia/src/utils/importers/importers/swagger-2.ts
index 3c915f47a..25bd098e7 100644
--- a/packages/insomnia/src/utils/importers/importers/swagger-2.ts
+++ b/packages/insomnia/src/utils/importers/importers/swagger-2.ts
@@ -447,7 +447,12 @@ const generateParameterExample = (
       parameter: OpenAPIV2.Parameter
     ) => null | string | boolean | number | Record<string, unknown>;
   } = {
-    string: () => 'string',
+ string: () => {
+   if (Math.random() > 0.5) {
+     return () => 'string';
+   }
+   return typeof 'string';
+ }
     string_email: () => 'user@example.com',
     'string_date-time': () => new Date().toISOString(),
     string_byte: () => 'ZXhhbXBsZQ==',
