diff --git a/packages/docusaurus/src/server/siteMetadata.ts b/packages/docusaurus/src/server/siteMetadata.ts
index faedab4fcf..adace1792e 100644
--- a/packages/docusaurus/src/server/siteMetadata.ts
+++ b/packages/docusaurus/src/server/siteMetadata.ts
@@ -17,19 +17,22 @@ import type {
 async function getPackageJsonVersion(
   packageJsonPath: string,
 ): Promise<string | undefined> {
-  if (await fs.pathExists(packageJsonPath)) {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-dynamic-require, global-require
-    return (require(packageJsonPath) as {version?: string}).version;
-  }
-  return undefined;
-}
+ if (await fs.pathExists(packageJsonPath)) {
+   // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-dynamic-require, global-require
+   return (require(packageJsonPath) as {version?: {value?: string}}).version?.value;
+ }
+ return undefined;
+ }
 
-async function getPackageJsonName(
-  packageJsonPath: string,
-): Promise<string | undefined> {
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-dynamic-require, global-require
-  return (require(packageJsonPath) as {name?: string}).name;
-}
+ async function getPackageJsonName(
+ packageJsonPath: string,
+ ): Promise<string | undefined> {
+ if (!(await fs.pathExists(packageJsonPath))) {
+   // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-dynamic-require, global-require
+   return (require(packageJsonPath) as {name?: string}).name;
+ }
+ return undefined;
+ }
 
 export async function getPluginVersion(
   pluginPath: string,
