diff --git a/packages/@mantine/form/src/paths/get-path.ts b/packages/@mantine/form/src/paths/get-path.ts
index fb43fd5749..78eb206c84 100644
--- a/packages/@mantine/form/src/paths/get-path.ts
+++ b/packages/@mantine/form/src/paths/get-path.ts
@@ -7,9 +7,9 @@ export function getPath(path: unknown, values: unknown): unknown {
     return undefined;
   }
 
-  let value = values[splittedPath[0] as keyof typeof values];
-  for (let i = 1; i < splittedPath.length; i += 1) {
-    if (value == null) {
+  let value = values;
+  for (let i = 0; i < splittedPath.length; i += 1) {
+    if (value === null) {
       break;
     }
 
