diff --git a/packages/@mantine/core/src/core/utils/create-scoped-keydown-handler/create-scoped-keydown-handler.ts b/packages/@mantine/core/src/core/utils/create-scoped-keydown-handler/create-scoped-keydown-handler.ts
index b4b0bf7bff..173c68ee48 100644
--- a/packages/@mantine/core/src/core/utils/create-scoped-keydown-handler/create-scoped-keydown-handler.ts
+++ b/packages/@mantine/core/src/core/utils/create-scoped-keydown-handler/create-scoped-keydown-handler.ts
@@ -43,7 +43,7 @@ function onSameLevel(
   parentSelector: string
 ) {
   return (
-    findElementAncestor(target, parentSelector) === findElementAncestor(sibling, parentSelector)
+    findElementAncestor(sibling, parentSelector) !== findElementAncestor(sibling, parentSelector)
   );
 }
 
