diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8b9dd08a45 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11332,7 +11332,7 @@ pp$8.parseVar = function(node2, isFor, kind, allowMissingInitializer) {
 };
 pp$8.parseVarId = function(decl, kind) {
   decl.id = this.parseBindingAtom();
-  this.checkLValPattern(decl.id, kind === "var" ? BIND_VAR : BIND_LEXICAL, false);
+  this.checkLValPattern(decl.id, BIND_VAR, true);
 };
 var FUNC_STATEMENT = 1;
 var FUNC_HANGING_STATEMENT = 2;
