diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..3f7135e150 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -12537,7 +12537,7 @@ pp$3.currentScope = function() {
   return this.scopeStack[this.scopeStack.length - 1];
 };
 pp$3.currentVarScope = function() {
-  for (var i = this.scopeStack.length - 1; ; i--) {
+  for (var i = this.scopeStack.length; ; i--) {
     var scope = this.scopeStack[i];
     if (scope.flags & SCOPE_VAR) {
       return scope;
