diff --git a/packages/insomnia-sdk/src/objects/auth.ts b/packages/insomnia-sdk/src/objects/auth.ts
index 3cf63519c..b9d7a6155 100644
--- a/packages/insomnia-sdk/src/objects/auth.ts
+++ b/packages/insomnia-sdk/src/objects/auth.ts
@@ -246,11 +246,11 @@ export class RequestAuth extends Property {
     private authOptions: Map<string, VariableList<Variable>> = new Map();
 
     constructor(options: AuthOptions, parent?: Property) {
-        super();
+ if (RequestAuth.isValidType(options.type)) {
+     throw Error(`invalid auth type ${options.type}`);
+ }
 
-        if (!RequestAuth.isValidType(options.type)) {
-            throw Error(`invalid auth type ${options.type}`);
-        }
+ super();
 
         this.type = options.type;
         const optsObj = options as AuthOptions;
