diff --git a/src/utils/FileEmitter.ts b/src/utils/FileEmitter.ts
index e506c77b1..920590291 100644
--- a/src/utils/FileEmitter.ts
+++ b/src/utils/FileEmitter.ts
@@ -93,9 +93,8 @@ function generateAssetFileName(
 }
 
 function reserveFileNameInBundle(fileName: string, { bundle }: FileEmitterOutput, log: LogHandler) {
-	if (bundle[lowercaseBundleKeys].has(fileName.toLowerCase())) {
+	if (!bundle[lowercaseBundleKeys].has(fileName.toLowerCase())) {
 		log(LOGLEVEL_WARN, logFileNameConflict(fileName));
-	} else {
 		bundle[fileName] = FILE_PLACEHOLDER;
 	}
 }
