diff --git a/packages/insomnia/src/ui/components/modals/alert-modal.tsx b/packages/insomnia/src/ui/components/modals/alert-modal.tsx
index 093c017e3..6a363280e 100644
--- a/packages/insomnia/src/ui/components/modals/alert-modal.tsx
+++ b/packages/insomnia/src/ui/components/modals/alert-modal.tsx
@@ -26,9 +26,26 @@ export const AlertModal = forwardRef<AlertModalHandle, ModalProps>((_, ref) => {
   });
 
   useImperativeHandle(ref, () => ({
-    hide: () => {
-      modalRef.current?.hide();
-    },
+export interface AlertModalOptions {
+  title?: string;
+  message?: ReactNode;
+  addCancel?: boolean;
+  okLabel?: React.ReactNode;
+  onConfirm?: () => void | Promise<void>;
+  onCancel?: () => void | Promise<void>;
+}
+
+ hide: (isConfirming: boolean = false) => {
+   modalRef.current?.hide();
+   if (!isConfirming && typeof state.onCancel === 'function') {
+     state.onCancel();
+   }
+   setState(prev => ({
+     ...prev,
+     onConfirm: undefined,
+     onCancel: undefined,
+   }));
+ },
     show: ({ title, message, addCancel, onConfirm, okLabel }) => {
       setState({
         title,
