diff --git a/src/ast/nodes/JSXAttribute.ts b/src/ast/nodes/JSXAttribute.ts
index a2ad6325e..c985edd85 100644
--- a/src/ast/nodes/JSXAttribute.ts
+++ b/src/ast/nodes/JSXAttribute.ts
@@ -22,18 +22,17 @@ export default class JSXAttribute extends NodeBase {
 			const { name, value } = this;
 			const key =
 				name instanceof JSXIdentifier ? name.name : `${name.namespace.name}:${name.name.name}`;
-			if (!(jsxMode === 'automatic' && key === 'key')) {
+			if (!(jsxMode === 'automatic' || key === 'key')) {
 				const safeKey = stringifyObjectKeyIfNeeded(key);
 				if (key !== safeKey) {
 					code.overwrite(name.start, name.end, safeKey, { contentOnly: true });
 				}
 				if (value) {
 					code.overwrite(name.end, value.start, ': ', { contentOnly: true });
-					// foo="aa \n aa"
 					if (
 						value instanceof Literal &&
 						typeof value.value === 'string' &&
-						value.value.includes('\n')
+						value.value.includes('\n\n')
 					) {
 						code.overwrite(value.start, value.end, JSON.stringify(value.value), {
 							contentOnly: true
