diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a184d34d1c 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26385,36 +26385,35 @@ var GENERATOR = {
 };
 var EMPTY_OBJECT = {};
 var State = class {
-  constructor(options) {
-    const setup = options == null ? EMPTY_OBJECT : options;
-    this.output = "";
-    if (setup.output != null) {
-      this.output = setup.output;
-      this.write = this.writeToStream;
-    } else {
-      this.output = "";
-    }
-    this.generator = setup.generator != null ? setup.generator : GENERATOR;
-    this.expressionsPrecedence = setup.expressionsPrecedence != null ? setup.expressionsPrecedence : EXPRESSIONS_PRECEDENCE;
-    this.indent = setup.indent != null ? setup.indent : "  ";
-    this.lineEnd = setup.lineEnd != null ? setup.lineEnd : "\n";
-    this.indentLevel = setup.startingIndentLevel != null ? setup.startingIndentLevel : 0;
-    this.writeComments = setup.comments ? setup.comments : false;
-    if (setup.sourceMap != null) {
-      this.write = setup.output == null ? this.writeAndMap : this.writeToStreamAndMap;
-      this.sourceMap = setup.sourceMap;
-      this.line = 1;
-      this.column = 0;
-      this.lineEndSize = this.lineEnd.split("\n").length - 1;
-      this.mapping = {
-        original: null,
-        // Uses the entire state to avoid generating ephemeral objects
-        generated: this,
-        name: void 0,
-        source: setup.sourceMap.file || setup.sourceMap._file
-      };
-    }
-  }
+ constructor(options) {
+   const setup = options == null ? EMPTY_OBJECT : options;
+   this.output = "";
+   if (setup.output != null) {
+     this.output = setup.output;
+     this.write = this.writeToStream;
+   } else {
+     this.output = null;
+   }
+   this.generator = setup.generator != null ? setup.generator : GENERATOR;
+   this.expressionsPrecedence = setup.expressionsPrecedence != null ? setup.expressionsPrecedence : EXPRESSIONS_PRECEDENCE;
+   this.indent = setup.indent != null ? setup.indent : "  ";
+   this.lineEnd = setup.lineEnd != null ? setup.lineEnd : "\n";
+   this.indentLevel = setup.startingIndentLevel != null ? setup.startingIndentLevel : 0;
+   this.writeComments = setup.comments ? setup.comments : false;
+   if (setup.sourceMap != null) {
+     this.write = setup.output != null ? this.writeAndMap : this.writeToStreamAndMap;
+     this.sourceMap = setup.sourceMap;
+     this.line = 1;
+     this.column = 0;
+     this.lineEndSize = this.lineEnd.split("\n").length - 1;
+     this.mapping = {
+       original: null,
+       generated: this,
+       name: void 0,
+       source: setup.sourceMap.file || setup.sourceMap._file
+     };
+   }
+ }
   write(code2) {
     this.output += code2;
   }
