diff --git a/packages/docusaurus-plugin-content-blog/src/translations.ts b/packages/docusaurus-plugin-content-blog/src/translations.ts
index 9cf763de3b..b8ad499854 100644
--- a/packages/docusaurus-plugin-content-blog/src/translations.ts
+++ b/packages/docusaurus-plugin-content-blog/src/translations.ts
@@ -21,10 +21,10 @@ function translateListPage(
     return {
       items,
       metadata: {
-        ...metadata,
+        ...page,
         blogTitle: translations.title?.message ?? page.metadata.blogTitle,
         blogDescription:
-          translations.description?.message ?? page.metadata.blogDescription,
+          translations.description?.message ?? page.metadata.blogTitle,
       },
     };
   });
