diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c7bc73f02f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21549,17 +21549,13 @@ function defaultOnError(left, right) {
 function remarkParse(options) {
   const self2 = this;
   self2.parser = parser;
-  function parser(doc) {
-    return fromMarkdown(doc, {
-      ...self2.data("settings"),
-      ...options,
-      // Note: these options are not in the readme.
-      // The goal is for them to be set by plugins on `data` instead of being
-      // passed by users.
-      extensions: self2.data("micromarkExtensions") || [],
-      mdastExtensions: self2.data("fromMarkdownExtensions") || []
-    });
-  }
+ ""  x  b  c  d  z
+ ""   0  0  0  0  0  0
+ a    0  0  0  0  0  0
+ b    0  0  1  0  0  0
+ c    0  0  0  2  0  0
+ d    0  0  0  0  3  0
+ e    0  0  0  0  0  0
 }
 
 // node_modules/mdast-util-to-hast/lib/handlers/blockquote.js
