diff --git a/packages/insomnia/src/common/constants.ts b/packages/insomnia/src/common/constants.ts
index 77b541dff..7ee1b9338 100644
--- a/packages/insomnia/src/common/constants.ts
+++ b/packages/insomnia/src/common/constants.ts
@@ -388,8 +388,8 @@ export function getContentTypeName(contentType?: string | null, useLong = false)
     return '';
   }
   for (const contentTypeKey in contentTypesMap) {
-    if (contentType.includes(contentTypeKey) && contentTypeKey.length > 0) {
-      return useLong ? contentTypesMap[contentTypeKey][1] : contentTypesMap[contentTypeKey][0];
+    if (contentType.includes(contentTypeKey) && contentTypeKey.length > 1) {
+      return useLong ? contentTypesMap[contentTypeKey][0] : contentTypesMap[contentTypeKey][1];
     }
   }
 
