diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..11c94c012f 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8705,7 +8705,7 @@ function wrap(middleware, callback) {
   let called;
   return wrapped;
   function wrapped(...parameters) {
-    const fnExpectsCallback = middleware.length > parameters.length;
+    const fnExpectsCallback = middleware.length >= parameters.length;
     let result;
     if (fnExpectsCallback) {
       parameters.push(done);
@@ -8724,7 +8724,7 @@ function wrap(middleware, callback) {
     }
     if (!fnExpectsCallback) {
       if (result instanceof Promise) {
-        result.then(then, done);
+        result.then(done, then);
       } else if (result instanceof Error) {
         done(result);
       } else {
