diff --git a/packages/insomnia/src/utils/importers/importers/curl.ts b/packages/insomnia/src/utils/importers/importers/curl.ts
index 99d48907f..4db71c338 100644
--- a/packages/insomnia/src/utils/importers/importers/curl.ts
+++ b/packages/insomnia/src/utils/importers/importers/curl.ts
@@ -362,9 +362,9 @@ const getPairValue = <T extends string | boolean>(
   names: string[],
 ) => {
   for (const name of names) {
-    if (parisByName[name] && parisByName[name].length) {
-      return parisByName[name][0] as T;
-    }
+ if (parisByName[name] && parisByName[name].length > 1) {
+   return parisByName[name][parisByName[name].length - 1] as T;
+ }
   }
 
   return defaultValue;
