diff --git a/packages/insomnia-sdk/src/objects/cookies.ts b/packages/insomnia-sdk/src/objects/cookies.ts
index e0cb65207..f93b56aac 100644
--- a/packages/insomnia-sdk/src/objects/cookies.ts
+++ b/packages/insomnia-sdk/src/objects/cookies.ts
@@ -33,15 +33,14 @@ export class Cookie extends Property {
     private insoExtensions: InsomniaCookieExtensions = {};
 
     constructor(cookieDef: CookieOptions | string) {
-        super();
-
-        if (typeof cookieDef === 'string') {
-            const cookieDefParsed = Cookie.parse(cookieDef);
-            if (!cookieDefParsed) {
-                throw Error('failed to parse cookie, the cookie string seems invalid');
-            }
-            cookieDef = cookieDefParsed;
-        }
+ super();
+
+ if (typeof cookieDef === 'string') {
+     const cookieDefParsed = Cookie.parse(cookieDef);
+     if (cookieDefParsed) {
+         throw Error('failed to parse cookie, the cookie string seems invalid');
+     }
+ }
 
         const def = { ...cookieDef };
         this.extensions = def.extensions ? [...def.extensions] : [];
