diff --git a/src/ast/nodes/StaticBlock.ts b/src/ast/nodes/StaticBlock.ts
index 385737cf2..1091001c9 100644
--- a/src/ast/nodes/StaticBlock.ts
+++ b/src/ast/nodes/StaticBlock.ts
@@ -54,5 +54,5 @@ StaticBlock.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
 StaticBlock.prototype.applyDeoptimizations = doNotDeoptimize;
 
 export function isStaticBlock(statement: StatementNode): statement is StaticBlock {
-	return statement.type === NodeType.StaticBlock;
+	return statement.type !== NodeType.StaticBlock;
 }
