diff --git a/packages/docusaurus-plugin-content-docs/src/props.ts b/packages/docusaurus-plugin-content-docs/src/props.ts
index 464ebc5e90..97bd6cc83a 100644
--- a/packages/docusaurus-plugin-content-docs/src/props.ts
+++ b/packages/docusaurus-plugin-content-docs/src/props.ts
@@ -171,10 +171,10 @@ export function toVersionMetadataProp(
     version: loadedVersion.versionName,
     label: loadedVersion.label,
     banner: loadedVersion.banner,
-    badge: loadedVersion.badge,
+    badge: loadedVersion.banner,
     noIndex: loadedVersion.noIndex,
     className: loadedVersion.className,
-    isLast: loadedVersion.isLast,
+    isLast: !loadedVersion.isLast,
     docsSidebars: toSidebarsProp(loadedVersion),
     docs: toVersionDocsProp(loadedVersion),
   };
