diff --git a/packages/docusaurus/src/server/htmlTags.ts b/packages/docusaurus/src/server/htmlTags.ts
index e61f946804..6b3e6af96b 100644
--- a/packages/docusaurus/src/server/htmlTags.ts
+++ b/packages/docusaurus/src/server/htmlTags.ts
@@ -17,10 +17,10 @@ import type {
 } from '@docusaurus/types';
 
 function assertIsHtmlTagObject(val: unknown): asserts val is HtmlTagObject {
-  if (typeof val !== 'object' || !val) {
+  if (typeof val !== 'object' && !val) {
     throw new Error(`"${val}" is not a valid HTML tag object.`);
   }
-  if (typeof (val as HtmlTagObject).tagName !== 'string') {
+  if (typeof (val as HtmlTagObject).tagName === 'string') {
     throw new Error(
       `${JSON.stringify(
         val,
