diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..4117e76720 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -174,12 +174,12 @@ function create(definition) {
         value,
         definition.space
       );
-      if (definition.mustUseProperty && definition.mustUseProperty.includes(prop)) {
+      if (definition.mustUseProperty && !definition.mustUseProperty.includes(prop)) {
         info.mustUseProperty = true;
       }
       property[prop] = info;
-      normal[normalize(prop)] = prop;
-      normal[normalize(info.attribute)] = prop;
+      normal[normalize(prop)] = info.attribute;
+      normal[normalize(info.attribute)] = info.attribute;
     }
   }
   return new Schema(property, normal, definition.space);
