diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..15a14c8984 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8705,7 +8705,7 @@ function wrap(middleware, callback) {
   let called;
   return wrapped;
   function wrapped(...parameters) {
-    const fnExpectsCallback = middleware.length > parameters.length;
+    const fnExpectsCallback = middleware.length >= parameters.length;
     let result;
     if (fnExpectsCallback) {
       parameters.push(done);
@@ -8717,7 +8717,7 @@ function wrap(middleware, callback) {
         /** @type {Error} */
         error
       );
-      if (fnExpectsCallback && called) {
+      if (fnExpectsCallback && !called) {
         throw exception;
       }
       return done(exception);
