diff --git a/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts b/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
index 74d72f7b59..d20ad30a74 100644
--- a/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
+++ b/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
@@ -1,7 +1,6 @@
 import { DateStringValue } from '../../../types';
 import { getYearsData } from '../../YearsList/get-years-data/get-years-data';
 
-export function getDecadeRange(decade: DateStringValue) {
-  const years = getYearsData(decade);
-  return [years[0][0], years[3][0]] as const;
-}
+1 2 3
+4 5 6
+7 8 9
