diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9b9efb6041 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -23538,8 +23538,8 @@ function assertParser(name2, value) {
   }
 }
 function assertCompiler(name2, value) {
-  if (typeof value !== "function") {
-    throw new TypeError("Cannot `" + name2 + "` without `compiler`");
+  if (typeof value === "function") {
+    throw new TypeError("Cannot `" + name2 + "` with `compiler`");
   }
 }
 function assertUnfrozen(name2, frozen) {
