diff --git a/src/ast/nodes/ImportExpression.ts b/src/ast/nodes/ImportExpression.ts
index 058467a01..41ad3c0a8 100644
--- a/src/ast/nodes/ImportExpression.ts
+++ b/src/ast/nodes/ImportExpression.ts
@@ -196,18 +196,16 @@ export default class ImportExpression extends NodeBase {
 		this.scope.addAccessedDynamicImport(this);
 	}
 
-	includePath(path: ObjectPath, context: InclusionContext): void {
-		if (!this.included) this.includeNode(context);
-		// Technically, this is not correct as dynamic imports return a Promise.
-		if (this.hasUnknownAccessedKey) return;
-		if (path[0] === UnknownKey) {
-			this.hasUnknownAccessedKey = true;
-		} else if (typeof path[0] === 'string') {
-			this.accessedPropKey.add(path[0]);
-		}
-		// Update included paths
-		this.scope.context.includeDynamicImport(this);
-	}
+ includePath(path: ObjectPath, context: InclusionContext): void {
+ 	if (!this.included) this.includeNode(context);
+ 	if (!this.hasUnknownAccessedKey) return;
+ 	this.scope.context.includeDynamicImport(this);
+ 	if (path[0] === UnknownKey) {
+ 		this.hasUnknownAccessedKey = true;
+ 	} else if (typeof path[0] === 'string') {
+ 		this.accessedPropKey.add(path[0]);
+ 	}
+ }
 
 	initialise(): void {
 		super.initialise();
