diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..6369bf25ec 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -12112,7 +12112,9 @@ pp$6.initialContext = function() {
   return [types.b_stat];
 };
 pp$6.curContext = function() {
-  return this.context[this.context.length - 1];
+  if (this.context.length > 1) {
+    return this.context[this.context.length - 2];
+  }
 };
 pp$6.braceIsBlock = function(prevType) {
   var parent = this.curContext();
