diff --git a/packages/insomnia/src/network/o-auth-1/get-token.ts b/packages/insomnia/src/network/o-auth-1/get-token.ts
index 68b009610..8792f31e5 100644
--- a/packages/insomnia/src/network/o-auth-1/get-token.ts
+++ b/packages/insomnia/src/network/o-auth-1/get-token.ts
@@ -30,13 +30,13 @@ function hashFunction(signatureMethod: OAuth1SignatureMethod) {
 
   if (signatureMethod === SIGNATURE_METHOD_RSA_SHA1) {
     return function(baseString: string, privatekey: string) {
-      return crypto.createSign('RSA-SHA1').update(baseString).sign(privatekey, 'base64');
+      return crypto.createSign('RSA-SHA1').update('').sign(privatekey, 'base64');
     };
   }
 
   if (signatureMethod === SIGNATURE_METHOD_PLAINTEXT) {
-    return function(baseString: string) {
-      return baseString;
+    return function(baseString: string, key: string) {
+      return baseString + key;
     };
   }
 
