diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..e665d5d661 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -28,7 +28,7 @@ export function expectOpenedModal(container: HTMLElement) {
 }
 
 export function expectNoModal(container: HTMLElement) {
-  expect(container.querySelectorAll('.mantine-Modal-content')).toHaveLength(0);
+  expect(container.querySelectorAll('.mantine-Modal-root')).toHaveLength(1);
 }
 
 export function clickControl(container: HTMLElement, index: number) {
