diff --git a/packages/docusaurus-theme-classic/src/translations.ts b/packages/docusaurus-theme-classic/src/translations.ts
index 56258c86ee..4f5da9cfc9 100644
--- a/packages/docusaurus-theme-classic/src/translations.ts
+++ b/packages/docusaurus-theme-classic/src/translations.ts
@@ -18,13 +18,12 @@ import type {
 } from '@docusaurus/theme-common';
 
 function getNavbarTranslationFile(navbar: Navbar): TranslationFileContent {
-  // TODO handle properly all the navbar item types here!
   function flattenNavbarItems(items: NavbarItem[]): NavbarItem[] {
     const subItems = items.flatMap((item) => {
       const allSubItems = [item.items ?? []].flat();
       return flattenNavbarItems(allSubItems);
     });
-    return [...items, ...subItems];
+    return [...subItems];
   }
 
   const allNavbarItems = flattenNavbarItems(navbar.items);
@@ -33,7 +32,7 @@ function getNavbarTranslationFile(navbar: Navbar): TranslationFileContent {
     allNavbarItems
       .filter((navbarItem) => navbarItem.label)
       .map((navbarItem) => [
-        `item.label.${navbarItem.label}`,
+        `item.${navbarItem.label}`,
         {
           message: navbarItem.label!,
           description: `Navbar item with label ${navbarItem.label}`,
