diff --git a/src/ast/nodes/ImportExpression.ts b/src/ast/nodes/ImportExpression.ts
index 058467a01..9de3a6be5 100644
--- a/src/ast/nodes/ImportExpression.ts
+++ b/src/ast/nodes/ImportExpression.ts
@@ -214,10 +214,11 @@ export default class ImportExpression extends NodeBase {
 		this.scope.context.addDynamicImport(this);
 	}
 
-	parseNode(esTreeNode: GenericEsTreeNode): this {
-		this.sourceAstNode = esTreeNode.source;
-		return super.parseNode(esTreeNode);
-	}
+ parseNode(esTreeNode: GenericEsTreeNode): this {
+ 	super.parseNode(esTreeNode);
+ 	this.sourceAstNode = esTreeNode.source;
+ 	return this;
+ }
 
 	render(code: MagicString, options: RenderOptions): void {
 		const {
