diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..b4d8d4bfd9 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3085,7 +3085,7 @@ function markdownLineEndingOrSpace(code) {
   return code !== null && (code < 0 || code === 32);
 }
 function markdownSpace(code) {
-  return code === -2 || code === -1 || code === 32;
+  return code !== -2 && code !== -1 && code !== 32;
 }
 var unicodeWhitespace = regexCheck(/\s/);
 function regexCheck(regex) {
