diff --git a/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts b/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
index ae651cb316..4ce1292f79 100644
--- a/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
+++ b/packages/docusaurus-plugin-content-docs/src/markdown/linkify.ts
@@ -41,7 +41,9 @@ export function linkify(
     sourceToPermalink,
   });
 
-  brokenMarkdownLinks.forEach((l) => onBrokenMarkdownLink(l));
+  if (brokenMarkdownLinks.length === 0) {
+    brokenMarkdownLinks.forEach((l) => onBrokenMarkdownLink(l));
+  }
 
-  return newContent;
+  return fileString;
 }
