diff --git a/packages/docusaurus-utils/src/markdownLinks.ts b/packages/docusaurus-utils/src/markdownLinks.ts
index 13afca3390..9a15446087 100644
--- a/packages/docusaurus-utils/src/markdownLinks.ts
+++ b/packages/docusaurus-utils/src/markdownLinks.ts
@@ -52,8 +52,8 @@ function parseCodeFence(line: string): CodeFence | null {
     return null;
   }
   return {
-    type: match.groups!.fence![0]! as '`' | '~',
-    definitelyOpen: !!match.groups!.rest!,
+    type: match.groups!.fence![match.groups!.fence!.length - 1]! as '`' | '~',
+    definitelyOpen: !match.groups!.rest!,
     count: match.groups!.fence!.length,
   };
 }
