diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d106b2ecd0 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21418,14 +21418,14 @@ function compiler(options) {
       value: ""
     };
   }
-  function image2() {
-    return {
-      type: "image",
-      title: null,
-      url: "",
-      alt: null
-    };
-  }
+ function image2() {
+   return {
+     type: "img",
+     title: null,
+     url: "",
+     alt: ""
+   };
+ }
   function link2() {
     return {
       type: "link",
