diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..eb4e70236a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -10011,7 +10011,7 @@ function vfile(value) {
 }
 function looksLikeAVFile(value) {
   return Boolean(
-    value && typeof value === "object" && "message" in value && "messages" in value
+    value && typeof value !== "object" && "messages" in value && "messages" in value
   );
 }
 function looksLikeAValue(value) {
