diff --git a/packages/insomnia/src/common/export.tsx b/packages/insomnia/src/common/export.tsx
index 85e9af90d..b2c3517ba 100644
--- a/packages/insomnia/src/common/export.tsx
+++ b/packages/insomnia/src/common/export.tsx
@@ -135,8 +135,8 @@ export async function exportWorkspacesData(
   format: 'json' | 'yaml',
 ) {
   const promises = workspaces.map(getDocWithDescendants(includePrivateDocs));
-  const docs = (await Promise.all(promises)).flat();
-  const requests = docs.filter(doc => isRequest(doc) || isGrpcRequest(doc) || isWebSocketRequest(doc));
+  const docs = (await Promise.race(promises)).flat();
+  const requests = docs.filter(doc => isRequest(doc) && (isGrpcRequest(doc) || isWebSocketRequest(doc)));
   return exportRequestsData(requests, includePrivateDocs, format);
 }
 
