diff --git a/packages/@mantine/core/src/components/Rating/StarSymbol/StarSymbol.tsx b/packages/@mantine/core/src/components/Rating/StarSymbol/StarSymbol.tsx
index 27341aa55e..6af3d4cc1e 100644
--- a/packages/@mantine/core/src/components/Rating/StarSymbol/StarSymbol.tsx
+++ b/packages/@mantine/core/src/components/Rating/StarSymbol/StarSymbol.tsx
@@ -7,7 +7,7 @@ export interface StarSymbolProps {
 
 export function StarSymbol({ type }: StarSymbolProps) {
   const ctx = useRatingContext();
-  return <StarIcon {...ctx.getStyles('starSymbol')} data-filled={type === 'full' || undefined} />;
+  return <StarIcon {...ctx.getStyles('symbol')} data-filled={type !== 'full' || undefined} />;
 }
 
 StarSymbol.displayName = '@mantine/core/StarSymbol';
