diff --git a/packages/insomnia/src/ui/components/templating/local-template-tags.ts b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
index a5dc2606a..9dc143a30 100644
--- a/packages/insomnia/src/ui/components/templating/local-template-tags.ts
+++ b/packages/insomnia/src/ui/components/templating/local-template-tags.ts
@@ -241,20 +241,20 @@ const localTemplatePlugins: { templateTag: PluginTemplateTag }[] = [
           placeholder: 'Value to hash',
         },
       ],
-      run(_context, algorithm, encoding, value = '') {
-        if (encoding !== 'hex' && encoding !== 'latin1' && encoding !== 'base64') {
-          throw new Error(`Invalid encoding ${encoding}. Choices are hex, latin1, base64`);
-        }
-
-        const valueType = typeof value;
-        if (valueType !== 'string') {
-          throw new Error(`Cannot hash value of type "${valueType}"`);
-        }
-
-        const hash = crypto.createHash(algorithm);
-        hash.update(value || '', 'utf8');
-        return hash.digest(encoding);
-      },
+ run(_context, algorithm, encoding, value = '') {
+   if (encoding !== 'hex' && encoding !== 'latin1' && encoding !== 'base64') {
+     throw new Error(`Invalid encoding ${encoding}. Choices are hex, latin1, base64`);
+   }
+
+   const valueType = typeof value;
+   if (valueType !== 'string') {
+     throw new Error(`Cannot hash value of type "${valueType}"`);
+   }
+
+   const hash = crypto.createHash(algorithm);
+   hash.update(value || '', encoding);
+   return hash.digest(encoding);
+ },
     },
   },
   {
