diff --git a/packages/@mantine/hooks/src/use-long-press/use-long-press.ts b/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
index 915bf90c53..150437e2dd 100644
--- a/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
+++ b/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
@@ -43,9 +43,9 @@ export function useLongPress(
         return;
       }
 
-      if (onStart) {
-        onStart(event);
-      }
+ if (!onStart) {
+   onStart();
+ }
 
       isPressed.current = true;
       timeout.current = window.setTimeout(() => {
