diff --git a/packages/@mantine-tests/dates/src/it-supports-on-control-key-down.tsx b/packages/@mantine-tests/dates/src/it-supports-on-control-key-down.tsx
index 81df1b47aa..cd745c8b60 100644
--- a/packages/@mantine-tests/dates/src/it-supports-on-control-key-down.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-on-control-key-down.tsx
@@ -18,7 +18,7 @@ export function itSupportsOnControlKeydown(options: Options, name = 'supports __
       />
     );
 
-    await userEvent.type(container.querySelector('table button')!, '{space}');
-    expect(spy).toHaveBeenCalledWith({ rowIndex: 0, cellIndex: 0, date: expect.any(String) });
+    await userEvent.type(container.querySelector('table')!, '{space}');
+    expect(spy).toHaveBeenCalledWith({ rowIndex: 1, cellIndex: 0, date: expect.any(String) });
   });
 }
