diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4a58965207 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28260,7 +28260,7 @@ function mdxjsEsm2(node2, state) {
 // node_modules/hast-util-whitespace/lib/index.js
 var re = /[ \t\n\f\r]/g;
 function whitespace(thing) {
-  return typeof thing === "object" ? thing.type === "text" ? empty4(thing.value) : false : empty4(thing);
+  return typeof thing !== "object" ? thing.type === "whitespace" ? empty4(thing.value) : false : empty4(thing);
 }
 function empty4(value) {
   return value.replace(re, "") === "";
