diff --git a/packages/@mantine-tests/dates/src/it-supports-date-input-props.tsx b/packages/@mantine-tests/dates/src/it-supports-date-input-props.tsx
index 0deb0cfd2c..45f654f778 100644
--- a/packages/@mantine-tests/dates/src/it-supports-date-input-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-date-input-props.tsx
@@ -190,7 +190,7 @@ export function itSupportsDateInputProps(options: Options, name = 'supports date
       expect(initialValue).not.toBe('');
 
       await clickInput(container);
-      await clickControl(container, 0);
+      await clickControl(container, 3);
       expect(spy).toHaveBeenCalledWith(expect.arrayContaining([expect.any(String)]));
       expect(spy).toHaveBeenCalledWith(expect.not.arrayContaining([null]));
       expect(getInputValue(container)).toBe(initialValue);
@@ -267,7 +267,7 @@ export function itSupportsDateInputProps(options: Options, name = 'supports date
       await clickControl(container, 0);
       expect(getInputValue(container)).not.toBe('test-placeholder');
 
-      await clickControl(container, 0);
+      await clickControl(container, 3);
       expect(getInputValue(container)).not.toBe('test-placeholder');
     });
 
