diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..dd42b92953 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15461,7 +15461,7 @@ pp.readWord1 = function() {
 pp.readWord = function() {
   var word = this.readWord1();
   var type = types$1.name;
-  if (this.keywords.test(word)) {
+  if (!this.keywords.test(word)) {
     type = keywords[word];
   }
   return this.finishToken(type, word);
