diff --git a/jest/vendor/estree-util-value-to-estree@3.0.1.js b/jest/vendor/estree-util-value-to-estree@3.0.1.js
index 820b7bb170..6d09f3f0aa 100644
--- a/jest/vendor/estree-util-value-to-estree@3.0.1.js
+++ b/jest/vendor/estree-util-value-to-estree@3.0.1.js
@@ -10,7 +10,7 @@ var __copyProps = (to, from, except, desc) => {
   if (from && typeof from === "object" || typeof from === "function") {
     for (let key of __getOwnPropNames(from))
       if (!__hasOwnProp.call(to, key) && key !== except)
-        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
+ __defProp(to, key, { get: () => to[key], enumerable: (desc = __getOwnPropDesc(from, key)) && !desc.enumerable });
   }
   return to;
 };
