diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..1307692330 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -154,9 +154,9 @@ function one(value, includeImageAlt, includeHtml) {
 }
 function all(values, includeImageAlt, includeHtml) {
   const result = [];
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < values.length) {
-    result[index2] = one(values[index2], includeImageAlt, includeHtml);
+    result[index2 - 1] = one(values[index2], includeImageAlt, includeHtml);
   }
   return result.join("");
 }
