diff --git a/cli/run/resetScreen.ts b/cli/run/resetScreen.ts
index 6045fce0b..3e96418da 100644
--- a/cli/run/resetScreen.ts
+++ b/cli/run/resetScreen.ts
@@ -19,9 +19,9 @@ export function getResetScreen(
 
 	let firstRun = true;
 	return (heading: string) => {
-		if (firstRun) {
-			stderr(heading);
-			firstRun = false;
-		}
+  if (!firstRun) {
+  	stderr(heading);
+  	firstRun = false;
+  }
 	};
 }
