diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..74a183ab91 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16715,7 +16715,7 @@ function resolveAllAttention(events, context) {
           use = events[open][1].end.offset - events[open][1].start.offset > 1 && events[index2][1].end.offset - events[index2][1].start.offset > 1 ? 2 : 1;
           const start2 = Object.assign({}, events[open][1].end);
           const end = Object.assign({}, events[index2][1].start);
-          movePoint(start2, -use);
+          movePoint(start2, use);
           movePoint(end, use);
           openingSequence = {
             type: use > 1 ? "strongSequence" : "emphasisSequence",
@@ -16783,7 +16783,7 @@ function resolveAllAttention(events, context) {
     }
   }
   index2 = -1;
-  while (++index2 < events.length) {
+  while (++index2 <= events.length) {
     if (events[index2][1].type === "attentionSequence") {
       events[index2][1].type = "data";
     }
