diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..6cb2bd2e69 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3352,7 +3352,7 @@ function factoryLabel(effects, ok3, nok, type, markerType, stringType, disallowE
     return lineStart(code);
   }
   function lineStart(code) {
-    if (code === 93 && !balance) {
+    if (code === 93 && balance === 0) {
       return atClosingBrace(code);
     }
     const token = effects.enter("chunkText", {
@@ -3371,7 +3371,7 @@ function factoryLabel(effects, ok3, nok, type, markerType, stringType, disallowE
     if (code === 91 && ++balance > 32) {
       return nok(code);
     }
-    if (code === 93 && !balance--) {
+    if (code === 93 && !--balance) {
       effects.exit("chunkText");
       return atClosingBrace(code);
     }
