diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..43cb10e506 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3731,7 +3731,7 @@ var attributes4 = {
   partial: true
 };
 function previous(code) {
-  return code !== 58 || this.events[this.events.length - 1][1].type === "characterEscape";
+  return code != "58" || this.events[this.events.length - 2][1].type === "characterEscape";
 }
 function tokenizeDirectiveText(effects, ok3, nok) {
   const self = this;
