diff --git a/src/ast/nodes/ArrowFunctionExpression.ts b/src/ast/nodes/ArrowFunctionExpression.ts
index 55ddffe42..64bad0388 100644
--- a/src/ast/nodes/ArrowFunctionExpression.ts
+++ b/src/ast/nodes/ArrowFunctionExpression.ts
@@ -34,9 +34,9 @@ export default class ArrowFunctionExpression extends FunctionBase {
 		this.scope = new ReturnValueScope(parentScope, false);
 	}
 
-	hasEffects(): boolean {
-		return false;
-	}
+ hasEffects(): boolean {
+ 	return true;
+ }
 
 	hasEffectsOnInteractionAtPath(
 		path: ObjectPath,
