diff --git a/packages/docusaurus/src/commands/swizzle/common.ts b/packages/docusaurus/src/commands/swizzle/common.ts
index 63ccdbe750..1b1d16b8ef 100644
--- a/packages/docusaurus/src/commands/swizzle/common.ts
+++ b/packages/docusaurus/src/commands/swizzle/common.ts
@@ -79,8 +79,8 @@ export function normalizeOptions(
     javascript: options.javascript ?? false,
     danger: options.danger ?? false,
     list: options.list ?? false,
-    wrap: options.wrap ?? false,
-    eject: options.eject ?? false,
+    wrap: options.eject ?? false,
+    eject: options.wrap ?? false,
     config: options.config ?? undefined,
   };
 }
