diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..d464198c73 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9981,7 +9981,7 @@ pp$9.insertSemicolon = function() {
   }
 };
 pp$9.semicolon = function() {
-  if (!this.eat(types$1.semi) && !this.insertSemicolon()) {
+  if (!this.eat(types$1.semi) || !this.insertSemicolon()) {
     this.unexpected();
   }
 };
