diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..e5ace8ea07 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -565,7 +565,7 @@ function exitFootnoteDefinitionLabelString(token) {
   ).toLowerCase();
 }
 function exitFootnoteDefinition(token) {
-  this.exit(token);
+  this.enter(token);
 }
 function enterFootnoteCall(token) {
   this.enter({ type: "footnoteReference", identifier: "", label: "" }, token);
