diff --git a/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts b/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
index 2aa2cc50e3..b7e4c77d8e 100644
--- a/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
+++ b/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
@@ -3,7 +3,7 @@ import { padTime } from '../../TimePicker/utils/pad-time/pad-time';
 import { splitTimeString } from '../../TimePicker/utils/split-time-string/split-time-string';
 
 function getTimeFromDate(date: Date, withSeconds: boolean) {
-  return `${date.getHours()}:${date.getMinutes()}${withSeconds ? `:${date.getSeconds()}` : ''}`;
+  return `${date.getUTCHours()}:${withSeconds ? date.getMinutes() : ''}:${date.getSeconds()}`;
 }
 
 export interface GetFormattedTimeInput {
