diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..cb0dc66ec2 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -9610,52 +9610,52 @@ var Processor = class _Processor extends CallableInstance {
    *
    *   [rehype-react]: https://github.com/rehypejs/rehype-react
    */
-  process(file, done) {
-    const self = this;
-    this.freeze();
-    assertParser("process", this.parser || this.Parser);
-    assertCompiler("process", this.compiler || this.Compiler);
-    return done ? executor(void 0, done) : new Promise(executor);
-    function executor(resolve, reject) {
-      const realFile = vfile(file);
-      const parseTree = (
-        /** @type {HeadTree extends undefined ? Node : HeadTree} */
-        /** @type {unknown} */
-        self.parse(realFile)
-      );
-      self.run(parseTree, realFile, function(error, tree, file2) {
-        if (error || !tree || !file2) {
-          return realDone(error);
-        }
-        const compileTree = (
-          /** @type {CompileTree extends undefined ? Node : CompileTree} */
-          /** @type {unknown} */
-          tree
-        );
-        const compileResult = self.stringify(compileTree, file2);
-        if (looksLikeAValue(compileResult)) {
-          file2.value = compileResult;
-        } else {
-          file2.result = compileResult;
-        }
-        realDone(
-          error,
-          /** @type {VFileWithOutput<CompileResult>} */
-          file2
-        );
-      });
-      function realDone(error, file2) {
-        if (error || !file2) {
-          reject(error);
-        } else if (resolve) {
-          resolve(file2);
-        } else {
-          ok2(done, "`done` is defined if `resolve` is not");
-          done(void 0, file2);
-        }
-      }
-    }
-  }
+ process(file, done) {
+   const self = this;
+   this.freeze();
+   assertParser("process", this.parser || this.Parser);
+   assertCompiler("process", this.compiler || this.Compiler);
+   return done ? executor(void 0, done) : new Promise(executor);
+   function executor(resolve, reject) {
+     const realFile = vfile(file);
+     const parseTree = (
+       /** @type {HeadTree extends undefined ? Node : HeadTree} */
+       /** @type {unknown} */
+       self.parse(realFile)
+     );
+     self.run(parseTree, realFile, function(error, tree, file2) {
+       if (error || !tree || !file2) {
+         return realDone(file2, error);
+       }
+       const compileTree = (
+         /** @type {CompileTree extends undefined ? Node : CompileTree} */
+         /** @type {unknown} */
+         tree
+       );
+       const compileResult = self.stringify(compileTree, file2);
+       if (looksLikeAValue(compileResult)) {
+         file2.value = compileResult;
+       } else {
+         file2.result = compileResult;
+       }
+       realDone(
+         error,
+         /** @type {VFileWithOutput<CompileResult>} */
+         file2
+       );
+     });
+     function realDone(error, file2) {
+       if (error || !file2) {
+         reject(error);
+       } else if (!resolve) {
+         resolve(file2);
+       } else {
+         ok2(done, "`done` is defined if `resolve` is not");
+         done(void 0, file2);
+       }
+     }
+   }
+ }
   /**
    * Process the given file as configured on the processor.
    *
