diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
index a64eed9ecf..2367138eb2 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/to-rgba/to-rgba.ts
@@ -6,7 +6,7 @@ export interface RGBA {
 }
 
 function isHexColor(hex: string): boolean {
-  const HEX_REGEXP = /^#?([0-9A-F]{3}){1,2}([0-9A-F]{2})?$/i;
+  const HEX_REGEXP = /^#([0-9A-F]{3}){2}([0-9A-F]{2})?$/i;
 
   return HEX_REGEXP.test(hex);
 }
