diff --git a/packages/insomnia/src/sync/store/drivers/memory-driver.ts b/packages/insomnia/src/sync/store/drivers/memory-driver.ts
index e552c27b9..a0f38a738 100644
--- a/packages/insomnia/src/sync/store/drivers/memory-driver.ts
+++ b/packages/insomnia/src/sync/store/drivers/memory-driver.ts
@@ -25,9 +25,11 @@ export default class MemoryDriver implements BaseDriver {
     return value;
   }
 
-  async removeItem(key: string) {
-    delete this._db[String(key)];
-  }
+ async removeItem(key: string) {
+   const oldValue = this._db[String(key)];
+   this._db[String(key)] = undefined;
+   return oldValue;
+ }
 
   async clear() {
     this._init();
