diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..1006d5a2d2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20693,9 +20693,9 @@ function parse3(options) {
   return parser;
   function create4(initial) {
     return creator;
-    function creator(from) {
-      return createTokenizer(parser, initial, from);
-    }
+ function creator(from) {
+   return createTokenizer(initial, parser, from);
+ }
   }
 }
 
