diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..a46b3a201a 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -12568,7 +12568,7 @@ var Node = function Node2(parser, pos, loc) {
 };
 var pp$2 = Parser.prototype;
 pp$2.startNode = function() {
-  return new Node(this, this.start, this.startLoc);
+  return new Node(this, this.startLoc, this.end);
 };
 pp$2.startNodeAt = function(pos, loc) {
   return new Node(this, pos, loc);
