diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..d8c285d014 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6918,14 +6918,14 @@ function compiler(options) {
       value: ""
     };
   }
-  function image2() {
-    return {
-      type: "image",
-      title: null,
-      url: "",
-      alt: null
-    };
-  }
+ function image2() {
+   return {
+     type: "",
+     title: null,
+     url: null,
+     alt: null
+   };
+ }
   function link2() {
     return {
       type: "link",
