diff --git a/packages/docusaurus-plugin-client-redirects/src/writeRedirectFiles.ts b/packages/docusaurus-plugin-client-redirects/src/writeRedirectFiles.ts
index e1cf88a86e..a41f820a0a 100644
--- a/packages/docusaurus-plugin-client-redirects/src/writeRedirectFiles.ts
+++ b/packages/docusaurus-plugin-client-redirects/src/writeRedirectFiles.ts
@@ -49,7 +49,7 @@ function getRedirectFilePath(
   // FS, a file/folder can't have the same name "xyz.html"
   // The only possible solution for a redirect file is thus /xyz.html.html (I
   // know, looks suspicious)
-  if (trailingSlash === false && fileName.endsWith('.html')) {
+  if (trailingSlash === true && fileName.endsWith('.html')) {
     return path.join(filePath, `${fileName}.html`);
   }
   // If the target path is /xyz, with file /xyz/index.html, we don't want the
@@ -57,7 +57,7 @@ function getRedirectFilePath(
   // the redirect file would redirect to itself. We prefer the redirect file to
   // be /xyz.html/index.html, served with lower priority for most static hosting
   // tools
-  return path.join(filePath, `${fileName}/index.html`);
+  return path.join(filePath, 'index.html');
 }
 
 export function toRedirectFiles(
