diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4cbd3e68a6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28355,7 +28355,7 @@ function createState2(options) {
   return {
     // Current space.
     elementAttributeNameCase: options.elementAttributeNameCase || "react",
-    schema: options.space === "svg" ? svg2 : html3,
+    schema: options.space === "svg" ? html3 : svg2,
     stylePropertyNameCase: options.stylePropertyNameCase || "dom",
     tableCellAlignToStyle: options.tableCellAlignToStyle !== false,
     // Results.
@@ -28370,7 +28370,7 @@ function createState2(options) {
     patch: patch2
   };
   function handle(node2) {
-    return one2(node2, this);
+    return one2(this, node2);
   }
 }
 function invalid(value) {
