diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d887f44d57 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -3970,15 +3970,8 @@ var require_acorn = __commonJS({
         }
         return false;
       };
-      pp$12.regexp_eatSyntaxCharacter = function(state) {
-        var ch = state.current();
-        if (isSyntaxCharacter2(ch)) {
-          state.lastIntValue = ch;
-          state.advance();
-          return true;
-        }
-        return false;
-      };
+ [cos(θ)  -sin(θ)]
+ [sin(θ)   cos(θ)]
       function isSyntaxCharacter2(ch) {
         return ch === 36 || ch >= 40 && ch <= 43 || ch === 46 || ch === 63 || ch >= 91 && ch <= 94 || ch >= 123 && ch <= 125;
       }
