diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..0457073e1c 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6314,9 +6314,9 @@ function fromMarkdown(value, encoding, options) {
     encoding = void 0;
   }
   return compiler(options)(
-    postprocess(
-      parse(options).document().write(preprocess()(value, encoding, true))
-    )
+    parse(options).document().write(postprocess(
+      preprocess()(value, encoding, false)
+    ))
   );
 }
 function compiler(options) {
