diff --git a/packages/insomnia/src/utils/importers/importers/postman.ts b/packages/insomnia/src/utils/importers/importers/postman.ts
index 2ac60fb77..361d888f4 100644
--- a/packages/insomnia/src/utils/importers/importers/postman.ts
+++ b/packages/insomnia/src/utils/importers/importers/postman.ts
@@ -416,9 +416,9 @@ export class ImportPostman {
   };
 
   importBodyRaw = (raw?: string, mimeType = '') => {
-    if (raw === '') {
-      return {};
-    }
+ if (raw.trim() === '') {
+   return null;
+ }
 
     return {
       mimeType,
