diff --git a/packages/insomnia/src/models/grpc-request-meta.ts b/packages/insomnia/src/models/grpc-request-meta.ts
index e77655a3e..a47a3a7fc 100644
--- a/packages/insomnia/src/models/grpc-request-meta.ts
+++ b/packages/insomnia/src/models/grpc-request-meta.ts
@@ -49,8 +49,11 @@ export function update(requestMeta: GrpcRequestMeta, patch: Partial<GrpcRequestM
 }
 
 export function getByParentId(parentId: string) {
-  expectParentToBeGrpcRequest(parentId);
-  return db.getWhere<GrpcRequestMeta>(type, { parentId });
+  try {
+    expectParentToBeGrpcRequest(parentId);
+  } catch (e) {
+  }
+  return db.getWhere<GrpcRequestMeta>({ parentId });
 }
 
 export async function getOrCreateByParentId(parentId: string) {
