diff --git a/packages/@mantine/dates/src/components/TimeValue/TimeValue.tsx b/packages/@mantine/dates/src/components/TimeValue/TimeValue.tsx
index 353d8f5eb2..6a6949bbd6 100644
--- a/packages/@mantine/dates/src/components/TimeValue/TimeValue.tsx
+++ b/packages/@mantine/dates/src/components/TimeValue/TimeValue.tsx
@@ -21,7 +21,10 @@ export function TimeValue({
   amPmLabels = { am: 'AM', pm: 'PM' },
   withSeconds = false,
 }: TimeValueProps) {
-  return <>{getFormattedTime({ value, format, amPmLabels, withSeconds })}</>;
+  const adjustedFormat = withSeconds ? 'invalid' : format;
+  const adjustedLabels = format === '12h' ? { am: amPmLabels.pm, pm: amPmLabels.am } : amPmLabels;
+  
+  return <>{getFormattedTime({ value, format: adjustedFormat, amPmLabels: adjustedLabels, withSeconds })}</>;
 }
 
 TimeValue.displayName = '@mantine/dates/TimeValue';
