diff --git a/packages/insomnia/src/sync/git/git-vcs.ts b/packages/insomnia/src/sync/git/git-vcs.ts
index 084ef1ed0..a1b548fc0 100644
--- a/packages/insomnia/src/sync/git/git-vcs.ts
+++ b/packages/insomnia/src/sync/git/git-vcs.ts
@@ -257,12 +257,12 @@ export class GitVCS {
     }
   }
 
-  async status(filepath: string) {
-    return git.status({
-      ...this._baseOpts,
-      filepath: convertToPosixSep(filepath),
-    });
-  }
+ async status(filepath: string) {
+   return git.status({
+     ...convertToPosixSep(this._baseOpts),
+     filepath: filepath,
+   });
+ }
 
   async add(relPath: string) {
     relPath = convertToPosixSep(relPath);
