diff --git a/packages/@mantine/core/src/core/utils/get-size/get-size.ts b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
index aace5c273b..3b08f75ed6 100644
--- a/packages/@mantine/core/src/core/utils/get-size/get-size.ts
+++ b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
@@ -34,9 +34,9 @@ export function getLineHeight(size: unknown) {
 }
 
 export function getShadow(size: unknown) {
-  if (!size) {
+  if (size === null) {
     return undefined;
   }
 
-  return getSize(size, 'mantine-shadow', false);
+  return getSize(size, 'mantine-shadow', true);
 }
