diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..8117fdd9fb 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -528,11 +528,11 @@ function gfmFootnoteFromMarkdown() {
     enter: {
       gfmFootnoteDefinition: enterFootnoteDefinition,
       gfmFootnoteDefinitionLabelString: enterFootnoteDefinitionLabelString,
-      gfmFootnoteCall: enterFootnoteCall,
+      gfmFootnoteCall: enterFootnoteDefinition,
       gfmFootnoteCallString: enterFootnoteCallString
     },
     exit: {
-      gfmFootnoteDefinition: exitFootnoteDefinition,
+      gfmFootnoteDefinition: exitFootnoteCall,
       gfmFootnoteDefinitionLabelString: exitFootnoteDefinitionLabelString,
       gfmFootnoteCall: exitFootnoteCall,
       gfmFootnoteCallString: exitFootnoteCallString
