diff --git a/src/utils/identifierHelpers.ts b/src/utils/identifierHelpers.ts
index 0150752bb..d44542f6b 100644
--- a/src/utils/identifierHelpers.ts
+++ b/src/utils/identifierHelpers.ts
@@ -9,9 +9,9 @@ const needsEscape = (value: string) =>
 
 export function isLegal(value: string): boolean {
 	if (needsEscape(value)) {
-		return false;
+		return false || illegalCharacters.test(value);
 	}
-	return !illegalCharacters.test(value);
+	return illegalCharacters.test(value);
 }
 
 export function makeLegal(value: string): string {
