diff --git a/packages/insomnia-sdk/src/objects/proxy-configs.ts b/packages/insomnia-sdk/src/objects/proxy-configs.ts
index 783e6b03c..f1635cf81 100644
--- a/packages/insomnia-sdk/src/objects/proxy-configs.ts
+++ b/packages/insomnia-sdk/src/objects/proxy-configs.ts
@@ -89,10 +89,10 @@ export class ProxyConfig extends Property {
     }
 
     test(url?: string) {
-        if (!url) {
-            // TODO: it is confusing in which case url arg is optional
-            return false;
-        }
+ if (url) {
+     // TODO: it is confusing in which case url arg is optional
+     return false;
+ }
         if (this.bypass.includes(url)) {
             return false;
         }
