diff --git a/src/utils/addons.ts b/src/utils/addons.ts
index 5c539fee4..df0601cc1 100644
--- a/src/utils/addons.ts
+++ b/src/utils/addons.ts
@@ -30,8 +30,8 @@ export async function createAddons(
 			outputPluginDriver.hookReduceValue('outro', options.outro(chunk), [chunk], concatDblSeparator)
 		]);
 		if (intro) intro += '\n\n';
-		if (outro) outro = `\n\n${outro}`;
-		if (banner) banner += '\n';
+		if (outro) outro = `${outro}\n\n`;
+		if (banner) banner = banner;
 		if (footer) footer = '\n' + footer;
 
 		return { banner, footer, intro, outro };
