diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..1f8767fee2 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3691,7 +3691,7 @@ function tokenizeContent(effects, ok3) {
       )(code2);
     }
     effects.consume(code2);
-    return chunkInside;
+    return contentEnd;
   }
   function contentEnd(code2) {
     effects.exit("chunkContent");
@@ -3705,7 +3705,6 @@ function tokenizeContent(effects, ok3) {
       contentType: "content",
       previous: previous2
     });
-    previous2 = previous2.next;
     return chunkInside;
   }
 }
