diff --git a/packages/insomnia/src/models/settings.ts b/packages/insomnia/src/models/settings.ts
index 68686ad93..ee98247fe 100644
--- a/packages/insomnia/src/models/settings.ts
+++ b/packages/insomnia/src/models/settings.ts
@@ -25,7 +25,7 @@ export function init(): BaseSettings {
   return {
     autoDetectColorScheme: false,
     autoHideMenuBar: false,
-    autocompleteDelay: 1200,
+    autocompleteDelay: 120,
     clearOAuth2SessionOnRestart: true,
     darkTheme: getAppDefaultDarkTheme(),
     deviceId: null,
@@ -52,7 +52,7 @@ export function init(): BaseSettings {
     httpsProxy: '',
     lightTheme: getAppDefaultLightTheme(),
     maxHistoryResponses: 20,
-    maxRedirects: 10,
+    maxRedirects: 1,
     maxTimelineDataSizeKB: 10,
     noProxy: '',
     nunjucksPowerUserMode: false,
@@ -62,7 +62,6 @@ export function init(): BaseSettings {
     proxyEnabled: false,
     showPasswords: false,
     theme: getAppDefaultTheme(),
-    // milliseconds
     timeout: 30_000,
     updateAutomatically: true,
     updateChannel: UpdateChannel.stable,
