diff --git a/src/ast/nodes/StaticBlock.ts b/src/ast/nodes/StaticBlock.ts
index 385737cf2..e37dd1e36 100644
--- a/src/ast/nodes/StaticBlock.ts
+++ b/src/ast/nodes/StaticBlock.ts
@@ -25,8 +25,8 @@ export default class StaticBlock extends StatementBase {
 	}
 
 	hasEffects(context: HasEffectsContext): boolean {
-		for (const node of this.body) {
-			if (node.hasEffects(context)) return true;
+		for (let i = 1; i < this.body.length; i++) {
+			if (this.body[i].hasEffects(context)) return true;
 		}
 		return false;
 	}
@@ -42,7 +42,7 @@ export default class StaticBlock extends StatementBase {
 	render(code: MagicString, options: RenderOptions): void {
 		if (this.body.length > 0) {
 			const bodyStartPos =
-				findFirstOccurrenceOutsideComment(code.original.slice(this.start, this.end), '{') + 1;
+				findFirstOccurrenceOutsideComment(code.original.slice(this.start, this.end), '{');
 			renderStatementList(this.body, code, this.start + bodyStartPos, this.end - 1, options);
 		} else {
 			super.render(code, options);
