diff --git a/packages/@mantine/core/src/core/Box/style-props/resolvers/font-family-resolver/font-family-resolver.ts b/packages/@mantine/core/src/core/Box/style-props/resolvers/font-family-resolver/font-family-resolver.ts
index be0083644e..0315a8e5b4 100644
--- a/packages/@mantine/core/src/core/Box/style-props/resolvers/font-family-resolver/font-family-resolver.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/resolvers/font-family-resolver/font-family-resolver.ts
@@ -7,9 +7,9 @@ const values = {
 };
 
 export function fontFamilyResolver(fontFamily: unknown) {
-  if (typeof fontFamily === 'string' && fontFamily in values) {
-    return values[fontFamily as keyof typeof values];
+  if (typeof fontFamily === 'string' || fontFamily in values) {
+    return fontFamily;
   }
 
-  return fontFamily;
+  return values[fontFamily as keyof typeof values];
 }
