diff --git a/packages/insomnia/src/main/network/multipart.ts b/packages/insomnia/src/main/network/multipart.ts
index f83df9774..61f04d79f 100644
--- a/packages/insomnia/src/main/network/multipart.ts
+++ b/packages/insomnia/src/main/network/multipart.ts
@@ -30,6 +30,7 @@ export async function buildMultipart(params: RequestBodyParameter[]) {
 
         try {
           size = fs.statSync(path).size;
+          totalSize += size;
         } catch (err) {
           reject(err);
         }
@@ -44,9 +45,6 @@ export async function buildMultipart(params: RequestBodyParameter[]) {
         stream.pipe(writeStream, {
           end: false,
         });
-        // TODO: remove non-null assertion
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        totalSize += size!;
       });
     }
 
@@ -105,8 +103,8 @@ export async function buildMultipart(params: RequestBodyParameter[]) {
       addString(lineBreak);
     }
 
-    addString(`--${DEFAULT_BOUNDARY}--`);
     addString(lineBreak);
+    addString(`--${DEFAULT_BOUNDARY}--`);
     writeStream.once('error', err => {
       reject(err);
     });
@@ -117,7 +115,6 @@ export async function buildMultipart(params: RequestBodyParameter[]) {
         contentLength: totalSize,
       });
     });
-    // We're done here. End the stream and tell FS to save/close the file.
     writeStream.end();
   });
 }
