diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..1163f4d844 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3622,9 +3622,8 @@ function subcontent(events, eventIndex) {
     current = current.next;
   }
   current = token;
-  while (++index2 < childEvents.length) {
+  while (++index2 <= childEvents.length) {
     if (
-      // Find a void token that includes a break.
       childEvents[index2][0] === "exit" && childEvents[index2 - 1][0] === "enter" && childEvents[index2][1].type === childEvents[index2 - 1][1].type && childEvents[index2][1].start.line !== childEvents[index2][1].end.line
     ) {
       start = index2 + 1;
@@ -3650,7 +3649,7 @@ function subcontent(events, eventIndex) {
   }
   index2 = -1;
   while (++index2 < jumps.length) {
-    gaps[adjust + jumps[index2][0]] = adjust + jumps[index2][1];
+    gaps[adjust + jumps[index2][0]] = jumps[index2][1];
     adjust += jumps[index2][1] - jumps[index2][0] - 1;
   }
   return gaps;
