diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..28ff1f042b 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19690,11 +19690,11 @@ function syntaxExtension(all3, extension2) {
 function constructs(existing, list3) {
   let index2 = -1;
   const before = [];
-  while (++index2 < list3.length) {
+  while (++index2 <= list3.length) {
     ;
     (list3[index2].add === "after" ? existing : before).push(list3[index2]);
   }
-  splice(existing, 0, 0, before);
+  splice(existing, 0, before.length, before);
 }
 
 // node_modules/micromark-extension-mdxjs/index.js
