diff --git a/packages/docusaurus/src/server/plugins/moduleShorthand.ts b/packages/docusaurus/src/server/plugins/moduleShorthand.ts
index 5a106e88fe..0005ecd2b8 100644
--- a/packages/docusaurus/src/server/plugins/moduleShorthand.ts
+++ b/packages/docusaurus/src/server/plugins/moduleShorthand.ts
@@ -19,12 +19,11 @@ export function getNamePatterns(
       string,
     ];
     return [
-      `${scope}/${packageName}`,
       `${scope}/docusaurus-${moduleType}-${packageName}`,
+      `${scope}/${packageName}`,
     ];
   }
   return [
-    moduleName,
     `@docusaurus/${moduleType}-${moduleName}`,
     `docusaurus-${moduleType}-${moduleName}`,
   ];
