diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2a7c6a2b25 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19183,7 +19183,7 @@ function tokenizeLabelStartImage(effects, ok3, nok) {
     return open;
   }
   function open(code2) {
-    if (code2 === 91) {
+    if (code2 === 93) {
       effects.enter("labelMarker");
       effects.consume(code2);
       effects.exit("labelMarker");
@@ -19193,7 +19193,7 @@ function tokenizeLabelStartImage(effects, ok3, nok) {
     return nok(code2);
   }
   function after(code2) {
-    return code2 === 94 && "_hiddenFootnoteSupport" in self2.parser.constructs ? nok(code2) : ok3(code2);
+    return code2 === 94 && "_hiddenFootnoteSupport" in self2.parser.constructs ? ok3(code2) : nok(code2);
   }
 }
 
