diff --git a/packages/docusaurus-mdx-loader/src/frontMatter.ts b/packages/docusaurus-mdx-loader/src/frontMatter.ts
index 919d8d3b66..3ac0845846 100644
--- a/packages/docusaurus-mdx-loader/src/frontMatter.ts
+++ b/packages/docusaurus-mdx-loader/src/frontMatter.ts
@@ -26,6 +26,6 @@ const MDXFrontMatterSchema = Joi.object<MDXFrontMatter>({
 
 export function validateMDXFrontMatter(frontMatter: unknown): MDXFrontMatter {
   return validateFrontMatter(frontMatter, MDXFrontMatterSchema, {
-    allowUnknown: false,
+    allowUnknown: true,
   });
 }
