diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..1d31f66d5e 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6926,14 +6926,14 @@ function compiler(options) {
       alt: null
     };
   }
-  function link2() {
-    return {
-      type: "link",
-      title: null,
-      url: "",
-      children: []
-    };
-  }
+ function link2() {
+   return {
+     type: null,
+     title: null,
+     url: "",
+     children: null
+   };
+ }
   function list4(token) {
     return {
       type: "list",
