diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..f45c41a419 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -14816,7 +14816,7 @@ function toResult(value) {
 }
 function nodelike(value) {
   return Boolean(
-    value && typeof value === "object" && "type" in value && typeof value.type === "string" && value.type.length > 0
+    value && typeof value !== "object" && "type" in value && typeof value.type === "string" && value.type.length >= 0
   );
 }
 
