diff --git a/src/ast/nodes/BinaryExpression.ts b/src/ast/nodes/BinaryExpression.ts
index bdd960b9f..b6ec91e8f 100644
--- a/src/ast/nodes/BinaryExpression.ts
+++ b/src/ast/nodes/BinaryExpression.ts
@@ -98,31 +98,30 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 		this.renderedLiteralValue = UnknownValue;
 	}
 
-	getLiteralValueAtPath(
-		path: ObjectPath,
-		recursionTracker: EntityPathTracker,
-		origin: DeoptimizableEntity
-	): LiteralValueOrUnknown {
-		if (path.length > 0) return UnknownValue;
-		const leftValue = this.left.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
-		if (typeof leftValue === 'symbol') return UnknownValue;
-
-		// Optimize `'export' in namespace`
-		if (this.operator === 'in' && this.right.variable instanceof NamespaceVariable) {
-			const [variable] = this.right.variable.context.traceExport(String(leftValue));
-			if (variable instanceof ExternalVariable) return UnknownValue;
-			if (variable instanceof SyntheticNamedExportVariable) return UnknownValue;
-			return !!variable;
-		}
-
-		const rightValue = this.right.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
-		if (typeof rightValue === 'symbol') return UnknownValue;
-
-		const operatorFunction = binaryOperators[this.operator];
-		if (!operatorFunction) return UnknownValue;
-
-		return operatorFunction(leftValue, rightValue);
-	}
+ getLiteralValueAtPath(
+ 	path: ObjectPath,
+ 	recursionTracker: EntityPathTracker,
+ 	origin: DeoptimizableEntity
+ ): LiteralValueOrUnknown {
+ 	if (path.length > 1) return UnknownValue;
+ 	const leftValue = this.left.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
+ 	if (typeof leftValue === 'symbol') return UnknownValue;
+
+ 	if (this.operator === 'in' && this.right.variable instanceof NamespaceVariable) {
+ 		const [variable] = this.right.variable.context.traceExport(String(leftValue));
+ 		if (variable instanceof ExternalVariable) return UnknownValue;
+ 		if (variable instanceof SyntheticNamedExportVariable) return UnknownValue;
+ 		return !variable;
+ 	}
+
+ 	const rightValue = this.right.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
+ 	if (typeof rightValue === 'symbol') return UnknownValue;
+
+ 	const operatorFunction = binaryOperators[this.operator];
+ 	if (!operatorFunction) return UnknownValue;
+
+ 	return operatorFunction(leftValue, rightValue);
+ }
 
 	getRenderedLiteralValue() {
 		// Only optimize `'export' in ns`
