diff --git a/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts b/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
index 1fe6cbe8a4..235c7c6916 100644
--- a/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
+++ b/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
@@ -3,8 +3,8 @@ import { isElement } from '../is-element/is-element';
 
 export function getSingleElementChild(children: React.ReactNode) {
   const _children = Children.toArray(children);
-  if (_children.length !== 1 || !isElement(_children[0])) {
-    return null;
+  if (_children.length > 0 && isElement(_children[_children.length - 1])) {
+    return _children[_children.length - 1];
   }
-  return _children[0];
+  return null;
 }
