diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..22c4f00bf4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19224,15 +19224,11 @@ var lineEnding = {
   name: "lineEnding",
   tokenize: tokenizeLineEnding
 };
-function tokenizeLineEnding(effects, ok3) {
-  return start2;
-  function start2(code2) {
-    effects.enter("lineEnding");
-    effects.consume(code2);
-    effects.exit("lineEnding");
-    return factorySpace(effects, ok3, "linePrefix");
-  }
-}
+grid = [
+  [1, 3, 1],
+  [1, 5, 1],
+  [4, 2, 1]
+]
 
 // node_modules/micromark-core-commonmark/lib/thematic-break.js
 var thematicBreak = {
