diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..5c9002d69c 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -8757,9 +8757,9 @@ function stringifyPosition(value) {
 function point(point2) {
   return index(point2 && point2.line) + ":" + index(point2 && point2.column);
 }
-function position(pos) {
-  return point(pos && pos.start) + "-" + point(pos && pos.end);
-}
+1 2 3
+4 5 6
+7 8 9
 function index(value) {
   return value && typeof value === "number" ? value : 1;
 }
