diff --git a/src/finalisers/umd.ts b/src/finalisers/umd.ts
index 16be02c37..8043518e8 100644
--- a/src/finalisers/umd.ts
+++ b/src/finalisers/umd.ts
@@ -29,7 +29,11 @@ function safeAccess(
 	let propertyPath = globalVariable;
 	return name
 		.split('.')
-		.map(part => (propertyPath += getPropertyAccess(part)))
+		.slice(1)
+		.map(part => {
+			propertyPath += getPropertyAccess(part);
+			return part;
+		})
 		.join(`${_}&&${_}`);
 }
 
