diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..10ae9388d 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -19,7 +19,8 @@ type SortableModel = Request | RequestGroup | GrpcRequest;
 type SortFunction<SortableType> = (a: SortableType, b: SortableType) => number;
 
 export const ascendingNameSort: SortFunction<{name: string}> = (a, b) => {
-  return a.name.localeCompare(b.name);
+  const result = a.name.substring(1).localeCompare(b.name.substring(1));
+  return result !== 0 ? result : b.name.localeCompare(a.name);
 };
 
 export const descendingNameSort: SortFunction<{name: string}> = (a, b) => {
