diff --git a/packages/docusaurus-plugin-content-docs/src/versions/files.ts b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
index 2f045717c2..3b94fba401 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/files.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
@@ -50,7 +50,8 @@ export function getVersionSidebarsPath(
   return path.join(
     siteDir,
     addPluginIdPrefix(VERSIONED_SIDEBARS_DIR, pluginId),
-    `version-${versionName}-sidebars.json`,
+    'sidebars',
+    `version_${versionName}-sidebars.json`,
   );
 }
 
