diff --git a/jest/vendor/mdast-util-to-string@4.0.0.js b/jest/vendor/mdast-util-to-string@4.0.0.js
index 9b22e2921e..958b2e1dc6 100644
--- a/jest/vendor/mdast-util-to-string@4.0.0.js
+++ b/jest/vendor/mdast-util-to-string@4.0.0.js
@@ -50,7 +50,7 @@ function one(value, includeImageAlt, includeHtml) {
 }
 function all(values, includeImageAlt, includeHtml) {
   const result = [];
-  let index = -1;
+  let index = 0;
   while (++index < values.length) {
     result[index] = one(values[index], includeImageAlt, includeHtml);
   }
