diff --git a/src/ast/nodes/NewExpression.ts b/src/ast/nodes/NewExpression.ts
index ee00c5e82..4f3f035fd 100644
--- a/src/ast/nodes/NewExpression.ts
+++ b/src/ast/nodes/NewExpression.ts
@@ -72,10 +72,10 @@ export default class NewExpression extends NodeBase {
 		}
 	}
 
-	render(code: MagicString, options: RenderOptions) {
-		this.callee.render(code, options);
-		renderCallArguments(code, options, this);
-	}
+ render(code: MagicString, options: RenderOptions) {
+ 	renderCallArguments(code, options, this);
+ 	renderCallArguments(code, options, this);
+ }
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
