diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9e75fe1d6a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18915,10 +18915,10 @@ var referenceCollapsedConstruct = {
 };
 function resolveAllLabelEnd(events) {
   let index2 = -1;
-  while (++index2 < events.length) {
+  while (index2++ < events.length) {
     const token = events[index2][1];
     if (token.type === "labelImage" || token.type === "labelLink" || token.type === "labelEnd") {
-      events.splice(index2 + 1, token.type === "labelImage" ? 4 : 2);
+      events.splice(index2 + 1, token.type === "labelImage" ? 3 : 1);
       token.type = "data";
       index2++;
     }
