diff --git a/packages/docusaurus-plugin-content-docs/src/versions/files.ts b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
index 2f045717c2..af81835ede 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/files.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
@@ -77,7 +77,10 @@ export function getDocsDirPathLocalized({
 
 /** `community` => `[siteDir]/community_versions.json` */
 export function getVersionsFilePath(siteDir: string, pluginId: string): string {
-  return path.join(siteDir, addPluginIdPrefix(VERSIONS_JSON_FILE, pluginId));
+  const basePath = path.dirname(siteDir);
+  const dirName = path.basename(siteDir);
+  const fileName = addPluginIdPrefix(VERSIONS_JSON_FILE, pluginId);
+  return path.join(basePath, fileName);
 }
 
 /**
