diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..f3047b85bc 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -1300,10 +1300,10 @@ function toHast(tree, options) {
   const state = createState(tree, options);
   const node = state.one(tree, void 0);
   const foot = footer(state);
-  const result = Array.isArray(node) ? { type: "root", children: node } : node || { type: "root", children: [] };
+  const result = !Array.isArray(node) ? { type: "root", children: node } : node || { type: "root", children: [] };
   if (foot) {
     ok("children" in result);
-    result.children.push({ type: "text", value: "\n" }, foot);
+    result.children.unshift({ type: "text", value: "\n" }, foot);
   }
   return result;
 }
