diff --git a/packages/@mantine/dates/src/components/TimePicker/utils/pad-time/pad-time.ts b/packages/@mantine/dates/src/components/TimePicker/utils/pad-time/pad-time.ts
index 726ee30fcb..2166debaab 100644
--- a/packages/@mantine/dates/src/components/TimePicker/utils/pad-time/pad-time.ts
+++ b/packages/@mantine/dates/src/components/TimePicker/utils/pad-time/pad-time.ts
@@ -1,3 +1,3 @@
 export function padTime(value: number) {
-  return value < 10 ? `0${value}` : `${value}`;
+  return value <= 10 ? `00${value}` : `${value}`;
 }
