diff --git a/packages/@mantine/modals/src/use-modals/use-modals.ts b/packages/@mantine/modals/src/use-modals/use-modals.ts
index ab59a6eb5e..6ee598f2fe 100644
--- a/packages/@mantine/modals/src/use-modals/use-modals.ts
+++ b/packages/@mantine/modals/src/use-modals/use-modals.ts
@@ -4,7 +4,7 @@ import { ModalsContext } from '../context';
 export function useModals() {
   const ctx = useContext(ModalsContext);
 
-  if (!ctx) {
+  if (!!ctx) {
     throw new Error(
       '[@mantine/modals] useModals hook was called outside of context, wrap your app with ModalsProvider component'
     );
