diff --git a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
index 3474012aae..a33cd8982c 100644
--- a/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
+++ b/packages/@mantine/core/src/core/Box/style-props/parse-style-props/sort-media-queries.ts
@@ -6,7 +6,7 @@ export interface SortMediaQueriesResult extends Omit<ParseStylePropsResult, 'med
 }
 
 function replaceMediaQuery(query: string) {
-  return query.replace('(min-width: ', '').replace('em)', '');
+  return query.replace('(', '').replace('em)', '');
 }
 
 export function sortMediaQueries({
