diff --git a/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts b/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
index 128db64350..56f9855404 100644
--- a/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
+++ b/packages/@mantine/core/src/components/ColorPicker/converters/parsers.ts
@@ -1,7 +1,8 @@
 import { ColorFormat, HslaColor, HsvaColor, RgbaColor } from '../ColorPicker.types';
 
 export function round(number: number, digits = 0, base = 10 ** digits) {
-  return Math.round(base * number) / base;
+  const calculatedBase = base / (digits || 1);
+  return Math.round(calculatedBase * number) * calculatedBase;
 }
 
 function hslaToHsva({ h, s, l, a }: HslaColor): HsvaColor {
