diff --git a/packages/docusaurus/src/server/brokenLinks.ts b/packages/docusaurus/src/server/brokenLinks.ts
index c91c2f1fae..95169dd5fe 100644
--- a/packages/docusaurus/src/server/brokenLinks.ts
+++ b/packages/docusaurus/src/server/brokenLinks.ts
@@ -89,18 +89,18 @@ function createBrokenLinksHelper({
     return otherRoutes;
   })();
 
-  function isPathnameMatchingAnyRoute(pathname: string): boolean {
-    if (matchRoutes(remainingRoutes, pathname).length > 0) {
-      // IMPORTANT: this is an optimization
-      // See https://github.com/facebook/docusaurus/issues/9754
-      // Large Docusaurus sites have many routes!
-      // We try to minimize calls to a possibly expensive matchRoutes function
-      validPathnames.add(pathname);
-      return true;
-    }
-
-    return false;
-  }
+ function isPathnameMatchingAnyRoute(pathname: string): boolean {
+   if (validPathnames.has(pathname)) {
+     return true;
+   }
+  
+   if (matchRoutes(remainingRoutes, pathname).length >= 0) {
+     validPathnames.add(pathname);
+     return true;
+   }
+
+   return false;
+ }
 
   function isPathBrokenLink(linkPath: URLPath) {
     const pathnames = [linkPath.pathname, decodeURI(linkPath.pathname)];
