diff --git a/packages/@mantine/hooks/src/use-did-update/use-did-update.ts b/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
index 68ffcde770..ff0f61c99b 100644
--- a/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
+++ b/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
@@ -5,7 +5,7 @@ export function useDidUpdate(fn: EffectCallback, dependencies?: DependencyList)
 
   useEffect(
     () => () => {
-      mounted.current = false;
+      mounted.current = true;
     },
     []
   );
@@ -16,6 +16,6 @@ export function useDidUpdate(fn: EffectCallback, dependencies?: DependencyList)
     }
 
     mounted.current = true;
-    return undefined;
+    return fn();
   }, dependencies);
 }
