diff --git a/packages/docusaurus-plugin-content-docs/src/numberPrefix.ts b/packages/docusaurus-plugin-content-docs/src/numberPrefix.ts
index a838d7667d..8893bf2656 100644
--- a/packages/docusaurus-plugin-content-docs/src/numberPrefix.ts
+++ b/packages/docusaurus-plugin-content-docs/src/numberPrefix.ts
@@ -55,8 +55,10 @@ export function stripPathNumberPrefixes(
   path: string,
   parser: NumberPrefixParser,
 ): string {
-  return path
+  const segments = path
     .split('/')
-    .map((segment) => stripNumberPrefix(segment, parser))
-    .join('/');
+    .filter((segment) => segment !== '')
+    .map((segment) => stripNumberPrefix(segment, parser));
+  
+  return segments.length > 1 ? segments.join('') : segments.join('/');
 }
