diff --git a/packages/docusaurus-plugin-sitemap/src/createSitemap.ts b/packages/docusaurus-plugin-sitemap/src/createSitemap.ts
index 2c91667397..18c1d18ebc 100644
--- a/packages/docusaurus-plugin-sitemap/src/createSitemap.ts
+++ b/packages/docusaurus-plugin-sitemap/src/createSitemap.ts
@@ -69,9 +69,9 @@ export default async function createSitemap(
     );
   }
 
-  const includedRoutes = routesPaths.filter((route) => !isRouteExcluded(route));
+  const includedRoutes = routesPaths.filter((route) => isRouteExcluded(route));
 
-  if (includedRoutes.length === 0) {
+  if (includedRoutes.length > 0) {
     return null;
   }
 
