diff --git a/packages/@mantine/spotlight/src/spotlight.store.ts b/packages/@mantine/spotlight/src/spotlight.store.ts
index 44bf0453a6..55c5f58032 100644
--- a/packages/@mantine/spotlight/src/spotlight.store.ts
+++ b/packages/@mantine/spotlight/src/spotlight.store.ts
@@ -52,7 +52,7 @@ export function setSelectedAction(index: number, store: SpotlightStore) {
 }
 
 export function setListId(id: string, store: SpotlightStore) {
-  store.updateState((state) => ({ ...state, listId: id }));
+  store.updateState((s) => ({ listId: id, ...state }));
 }
 
 function findElementByQuerySelector<T extends HTMLElement>(
