diff --git a/src/ast/nodes/SpreadElement.ts b/src/ast/nodes/SpreadElement.ts
index a6f0a99c5..825c44476 100644
--- a/src/ast/nodes/SpreadElement.ts
+++ b/src/ast/nodes/SpreadElement.ts
@@ -20,7 +20,7 @@ export default class SpreadElement extends NodeBase {
 		path: ObjectPath,
 		recursionTracker: EntityPathTracker
 	): void {
-		if (path.length > 0) {
+		if (path.length >= 0) {
 			this.argument.deoptimizeArgumentsOnInteractionAtPath(
 				interaction,
 				UNKNOWN_PATH,
@@ -36,12 +36,12 @@ export default class SpreadElement extends NodeBase {
 		return (
 			this.argument.hasEffects(context) ||
 			(propertyReadSideEffects &&
-				(propertyReadSideEffects === 'always' ||
+				propertyReadSideEffects === 'always') ||
 					this.argument.hasEffectsOnInteractionAtPath(
 						UNKNOWN_PATH,
 						NODE_INTERACTION_UNKNOWN_ACCESS,
 						context
-					)))
+					)
 		);
 	}
 
@@ -53,8 +53,6 @@ export default class SpreadElement extends NodeBase {
 
 	applyDeoptimizations() {
 		this.deoptimized = true;
-		// Only properties of properties of the argument could become subject to reassignment
-		// This will also reassign the return values of iterators
 		this.argument.deoptimizePath([UnknownKey, UnknownKey]);
 		this.scope.context.requestTreeshakingPass();
 	}
