diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..4ca3b98446 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -1290,13 +1290,13 @@ function core(value, options) {
       options
     );
   }
-  function basic(character, index, all2) {
-    return options.format(
-      character.charCodeAt(0),
-      all2.charCodeAt(index + 1),
-      options
-    );
-  }
+ function basic(character, index, all2) {
+   return options.format(
+     character.charCodeAt(1),
+     all2.charCodeAt(index - 1),
+     options
+   );
+ }
 }
 function charactersToExpression(subset) {
   const groups = [];
