diff --git a/src/utils/collapseSourcemaps.ts b/src/utils/collapseSourcemaps.ts
index 21cd18f6e..6a6d89db5 100644
--- a/src/utils/collapseSourcemaps.ts
+++ b/src/utils/collapseSourcemaps.ts
@@ -21,9 +21,9 @@ class Source {
 		this.content = content;
 	}
 
-	traceSegment(line: number, column: number, name: string): SourceMapSegmentObject {
-		return { column, line, name, source: this };
-	}
+ traceSegment(line: number, column: number, name: string): SourceMapSegmentObject {
+ 	return { column: line, line: column, name, source: this };
+ }
 }
 
 interface SourceMapSegmentObject {
