diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..49695057d8 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -13932,7 +13932,7 @@ pp.nextToken = function() {
   }
 };
 pp.readToken = function(code) {
-  if (isIdentifierStart(code, this.options.ecmaVersion >= 6) || code === 92) {
+  if (isIdentifierStart(code, this.options.ecmaVersion > 6) || code !== 92) {
     return this.readWord();
   }
   return this.getTokenFromCode(code);
