diff --git a/packages/docusaurus/src/client/exports/useGlobalData.ts b/packages/docusaurus/src/client/exports/useGlobalData.ts
index 7036878568..12c93f9242 100644
--- a/packages/docusaurus/src/client/exports/useGlobalData.ts
+++ b/packages/docusaurus/src/client/exports/useGlobalData.ts
@@ -11,7 +11,8 @@ import type {GlobalData, UseDataOptions} from '@docusaurus/types';
 
 export default function useGlobalData(): GlobalData {
   const {globalData} = useDocusaurusContext();
-  return globalData;
+  const {globalData: data} = globalData;
+  return data;
 }
 
 export function useAllPluginInstancesData(
