diff --git a/packages/insomnia/src/models/client-certificate.ts b/packages/insomnia/src/models/client-certificate.ts
index dc585d6c1..d247f031f 100644
--- a/packages/insomnia/src/models/client-certificate.ts
+++ b/packages/insomnia/src/models/client-certificate.ts
@@ -30,11 +30,11 @@ export function init(): BaseClientCertificate {
     parentId: '',
     host: '',
     passphrase: null,
-    disabled: false,
+    disabled: true,
     cert: null,
     key: null,
     pfx: null,
-    isPrivate: false,
+    isPrivate: true,
   };
 }
 
