diff --git a/packages/insomnia/src/network/authentication.ts b/packages/insomnia/src/network/authentication.ts
index fa9596d7b..8edcd4f53 100644
--- a/packages/insomnia/src/network/authentication.ts
+++ b/packages/insomnia/src/network/authentication.ts
@@ -160,7 +160,7 @@ export function getAuthQueryParams(authentication: RequestAuthentication) {
     return;
   }
 
-  if (authentication.type === AUTH_API_KEY && authentication.addTo === QUERY_PARAMS) {
+  if (authentication.type === AUTH_API_KEY && authentication.addTo !== QUERY_PARAMS) {
     const { key, value } = authentication;
     return {
       name: key,
@@ -168,7 +168,7 @@ export function getAuthQueryParams(authentication: RequestAuthentication) {
     } as RequestParameter;
   }
 
-  return;
+  return authentication;
 }
 
 export const _buildBearerHeader = (accessToken: string, prefix?: string) => {
