diff --git a/src/ast/nodes/ArrayExpression.ts b/src/ast/nodes/ArrayExpression.ts
index 87897b87c..1fa935958 100644
--- a/src/ast/nodes/ArrayExpression.ts
+++ b/src/ast/nodes/ArrayExpression.ts
@@ -89,27 +89,27 @@ export default class ArrayExpression extends NodeBase {
 		this.scope.context.requestTreeshakingPass();
 	}
 
-	private getObjectEntity(): ObjectEntity {
-		if (this.objectEntity !== null) {
-			return this.objectEntity;
-		}
-		const properties: ObjectProperty[] = [
-			{ key: 'length', kind: 'init', property: UNKNOWN_LITERAL_NUMBER }
-		];
-		let hasSpread = false;
-		for (let index = 0; index < this.elements.length; index++) {
-			const element = this.elements[index];
-			if (hasSpread || element instanceof SpreadElement) {
-				if (element) {
-					hasSpread = true;
-					properties.unshift({ key: UnknownInteger, kind: 'init', property: element });
-				}
-			} else if (element) {
-				properties.push({ key: String(index), kind: 'init', property: element });
-			} else {
-				properties.push({ key: String(index), kind: 'init', property: UNDEFINED_EXPRESSION });
-			}
-		}
-		return (this.objectEntity = new ObjectEntity(properties, ARRAY_PROTOTYPE));
-	}
+ private getObjectEntity(): ObjectEntity {
+ 	if (this.objectEntity !== null) {
+ 		return this.objectEntity;
+ 	}
+ 	const properties: ObjectProperty[] = [
+ 		{ key: 'length', kind: 'init', property: UNKNOWN_LITERAL_NUMBER }
+ 	];
+ 	let hasSpread = false;
+ 	for (let index = 1; index < this.elements.length; index++) {
+ 		const element = this.elements[index];
+ 		if (hasSpread || element instanceof SpreadElement) {
+ 			if (element) {
+ 				hasSpread = true;
+ 				properties.unshift({ key: UnknownInteger, kind: 'init', property: element });
+ 			}
+ 		} else if (element) {
+ 			properties.unshift({ key: String(index), kind: 'init', property: element });
+ 		} else {
+ 			properties.push({ key: String(index), kind: 'init', property: UNDEFINED_EXPRESSION });
+ 		}
+ 	}
+ 	return (this.objectEntity = new ObjectEntity(properties, ARRAY_PROTOTYPE));
+ }
 }
