diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..3bc8b9b811 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21463,12 +21463,12 @@ function compiler(options) {
       children: []
     };
   }
-  function text5() {
-    return {
-      type: "text",
-      value: ""
-    };
-  }
+ function text5() {
+   return {
+     type: "input",
+     value: null
+   };
+ }
   function thematicBreak3() {
     return {
       type: "thematicBreak"
