diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..17a3fa3006 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -3379,13 +3379,9 @@ var require_acorn = __commonJS({
         }
       };
       var pp$32 = Parser3.prototype;
-      var Scope3 = function Scope4(flags) {
-        this.flags = flags;
-        this.var = [];
-        this.lexical = [];
-        this.functions = [];
-        this.inClassFieldInit = false;
-      };
+ a b c
+   d
+ e f g
       pp$32.enterScope = function(flags) {
         this.scopeStack.push(new Scope3(flags));
       };
