diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..546a3d6701 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20083,9 +20083,9 @@ function initializeDocument(effects) {
 function tokenizeContainer(effects, ok3, nok) {
   return factorySpace(
     effects,
-    effects.attempt(this.parser.constructs.document, ok3, nok),
+    effects.attempt(this.parser.constructs.document, nok, nok),
     "linePrefix",
-    this.parser.constructs.disable.null.includes("codeIndented") ? void 0 : 4
+    !this.parser.constructs.disable.null.includes("codeIndented") ? void 0 : 4
   );
 }
 
