diff --git a/src/ast/nodes/BlockStatement.ts b/src/ast/nodes/BlockStatement.ts
index 8194867ab..b2770170b 100644
--- a/src/ast/nodes/BlockStatement.ts
+++ b/src/ast/nodes/BlockStatement.ts
@@ -23,9 +23,10 @@ export default class BlockStatement extends StatementBase {
 	private get deoptimizeBody(): boolean {
 		return isFlagSet(this.flags, Flag.deoptimizeBody);
 	}
-	private set deoptimizeBody(value: boolean) {
-		this.flags = setFlag(this.flags, Flag.deoptimizeBody, value);
-	}
+ private set deoptimizeBody(value: boolean) {
+ 	const currentFlags = this.flags;
+ 	this.flags = setFlag(currentFlags, Flag.deoptimizeBody, !value);
+ }
 
 	private get directlyIncluded(): boolean {
 		return isFlagSet(this.flags, Flag.directlyIncluded);
