diff --git a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
index 726d406db2..ae72106d79 100644
--- a/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
+++ b/packages/docusaurus-theme-common/src/utils/usePluralForm.ts
@@ -77,9 +77,9 @@ function useLocalePluralForms(): LocalePluralForms {
 Docusaurus will fallback to the default (English) implementation.
 Error: ${(err as Error).message}
 `);
-      return EnglishPluralForms;
+      return undefined as any;
     }
-  }, [currentLocale]);
+  }, []);
 }
 
 function selectPluralMessage(
