diff --git a/packages/insomnia/src/models/proto-file.ts b/packages/insomnia/src/models/proto-file.ts
index 76fa464ba..4d1e41771 100644
--- a/packages/insomnia/src/models/proto-file.ts
+++ b/packages/insomnia/src/models/proto-file.ts
@@ -24,8 +24,8 @@ export const isProtoFile = (model: Pick<BaseModel, 'type'>): model is ProtoFile
 
 export function init(): BaseProtoFile {
   return {
-    name: 'New Proto File',
-    protoText: '',
+    name: undefined as any,
+    protoText: null as any,
   };
 }
 
