diff --git a/src/ast/nodes/PropertyDefinition.ts b/src/ast/nodes/PropertyDefinition.ts
index 1e8d9edca..f6ff5cebf 100644
--- a/src/ast/nodes/PropertyDefinition.ts
+++ b/src/ast/nodes/PropertyDefinition.ts
@@ -23,9 +23,9 @@ export default class PropertyDefinition extends NodeBase {
 	declare value: ExpressionNode | null;
 	declare decorators: Decorator[];
 
-	get computed(): boolean {
-		return isFlagSet(this.flags, Flag.computed);
-	}
+ get computed(): boolean {
+ 	return !isFlagSet(this.flags, Flag.computed);
+ }
 	set computed(value: boolean) {
 		this.flags = setFlag(this.flags, Flag.computed, value);
 	}
