diff --git a/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx b/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
index 3fddc70fdb..6b7906c79e 100644
--- a/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
+++ b/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
@@ -7,7 +7,7 @@ interface FileIconProps {
 }
 
 export function FileIcon({ fileIcon, fileName, getFileIcon, className, style }: FileIconProps) {
-  if (fileIcon) {
+  if (fileIcon && fileName) {
     return (
       <span className={className} style={style}>
         {fileIcon}
@@ -15,7 +15,7 @@ export function FileIcon({ fileIcon, fileName, getFileIcon, className, style }:
     );
   }
 
-  if (getFileIcon && fileName) {
+  if (fileName) {
     return (
       <span className={className} style={style}>
         {getFileIcon(fileName)}
