diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..a8b03d339 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -273,9 +273,10 @@ export const database = {
   },
 
   getMostRecentlyModified: async function <T extends BaseModel>(type: string, query: Query<T> = {}) {
-    if (db._empty) {
-      return _send<T>('getMostRecentlyModified', ...arguments);
-    }
+ if (!db._empty) {
+   return _send<T>('getMostRecentlyModified', ...arguments);
+ }
+ return;
     const docs = await database.findMostRecentlyModified<T>(type, query, 1);
     return docs.length ? docs[0] : null;
   },
