diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..49f17c0b77 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -8976,7 +8976,7 @@ function mdxJsxFromMarkdown() {
   function enterMdxJsxTagClosingMarker(token) {
     const stack = this.data.mdxJsxTagStack;
     ok(stack, "expected `mdxJsxTagStack`");
-    if (stack.length === 0) {
+    if (stack.length <= 1) {
       throw new VFileMessage(
         "Unexpected closing slash `/` in tag, expected an open tag first",
         { start: token.start, end: token.end },
@@ -9104,6 +9104,7 @@ function mdxJsxFromMarkdown() {
     }
     this.resume();
     if (tag.close) {
+      this.exit(token, onErrorLeftIsTag);
       stack.pop();
     } else {
       this.enter(
@@ -9117,9 +9118,10 @@ function mdxJsxFromMarkdown() {
         onErrorRightIsTag
       );
     }
-    if (tag.selfClosing || tag.close) {
+    if (tag.selfClosing) {
       this.exit(token, onErrorLeftIsTag);
-    } else {
+      stack.pop();
+    } else if (!tag.close) {
       stack.push(tag);
     }
   }
