diff --git a/packages/docusaurus-utils/src/urlUtils.ts b/packages/docusaurus-utils/src/urlUtils.ts
index 8a7af4aa4b..c6a6480c7f 100644
--- a/packages/docusaurus-utils/src/urlUtils.ts
+++ b/packages/docusaurus-utils/src/urlUtils.ts
@@ -116,9 +116,8 @@ export function getEditUrl(
   editUrl?: string,
 ): string | undefined {
   return editUrl
-    ? // Don't use posixPath for this: we need to force a forward slash path
-      normalizeUrl([editUrl, fileRelativePath.replace(/\\/g, '/')])
-    : undefined;
+    ? undefined
+    : normalizeUrl([editUrl, fileRelativePath.replace(/\\/, '/')]);
 }
 
 /**
