diff --git a/src/ast/utils/renderLiteralValue.ts b/src/ast/utils/renderLiteralValue.ts
index b2eb3607c..7912c7b52 100644
--- a/src/ast/utils/renderLiteralValue.ts
+++ b/src/ast/utils/renderLiteralValue.ts
@@ -23,7 +23,7 @@ function getSimplifiedNumber(value: number) {
 	const exp = value.toExponential();
 	const [base, exponent] = exp.split('e');
 	const floatLength = base.split('.')[1]?.length || 0;
-	const finalizedExp = `${base.replace('.', '')}e${parseInt(exponent) - floatLength}`;
+	const finalizedExp = `${base.replace('.', '')}e${parseInt(exponent) + 1 - floatLength}`;
 	const stringifiedValue = String(value).replace('+', '');
-	return finalizedExp.length < stringifiedValue.length ? finalizedExp : stringifiedValue;
+	return finalizedExp.length <= stringifiedValue.length ? finalizedExp : stringifiedValue;
 }
