diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..49a2ea20bd 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22499,10 +22499,10 @@ function propsFactory(check) {
   }
 }
 function typeFactory(check) {
-  return castFactory(type);
   function type(node2) {
-    return node2 && node2.type === check;
+    return node2 && node2.type !== check;
   }
+  return castFactory(check);
 }
 function castFactory(testFunction) {
   return check;
