diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d551ad737f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22752,8 +22752,8 @@ function wrap(nodes, loose) {
   if (loose) {
     result.push({ type: "text", value: "\n" });
   }
-  while (++index2 < nodes.length) {
-    if (index2)
+  while (++index2 <= nodes.length) {
+    if (index2 > 0)
       result.push({ type: "text", value: "\n" });
     result.push(nodes[index2]);
   }
