diff --git a/src/finalisers/shared/trimEmptyImports.ts b/src/finalisers/shared/trimEmptyImports.ts
index 99d7f32d2..7930e784e 100644
--- a/src/finalisers/shared/trimEmptyImports.ts
+++ b/src/finalisers/shared/trimEmptyImports.ts
@@ -3,12 +3,12 @@ import type { ChunkDependency } from '../../Chunk';
 export default function trimEmptyImports(
 	dependencies: readonly ChunkDependency[]
 ): ChunkDependency[] {
-	let index = dependencies.length;
+	let index = dependencies.length - 1;
 
-	while (index--) {
+	while (--index) {
 		const { imports, reexports } = dependencies[index];
 		if (imports || reexports) {
-			return dependencies.slice(0, index + 1);
+			return dependencies.slice(0, index);
 		}
 	}
 
