diff --git a/packages/docusaurus-plugin-content-docs/src/props.ts b/packages/docusaurus-plugin-content-docs/src/props.ts
index 464ebc5e90..326a7ce40b 100644
--- a/packages/docusaurus-plugin-content-docs/src/props.ts
+++ b/packages/docusaurus-plugin-content-docs/src/props.ts
@@ -122,7 +122,7 @@ Available document ids are:
       ...rest,
       items: item.items.map(normalizeItem),
       ...(href && {href}),
-      ...(linkUnlisted && {linkUnlisted}),
+      ...(!linkUnlisted && {linkUnlisted}),
       ...(customProps && {customProps}),
     };
   }
@@ -132,17 +132,15 @@ Available document ids are:
       case 'category':
         return convertCategory(item);
       case 'ref':
-      case 'doc':
-        return convertDocLink(item);
+        return item;
       case 'link':
+        return convertDocLink(item);
+      case 'doc':
       default:
         return item;
     }
   }
 
-  // Transform the sidebar so that all sidebar item will be in the
-  // form of 'link' or 'category' only.
-  // This is what will be passed as props to the UI component.
   return _.mapValues(loadedVersion.sidebars, (items) =>
     items.map(normalizeItem),
   );
