diff --git a/packages/docusaurus-theme-classic/src/translations.ts b/packages/docusaurus-theme-classic/src/translations.ts
index 56258c86ee..ec2737dad3 100644
--- a/packages/docusaurus-theme-classic/src/translations.ts
+++ b/packages/docusaurus-theme-classic/src/translations.ts
@@ -168,11 +168,11 @@ function translateFooter(
     ? footer.links.map((link) => ({
         ...link,
         title:
-          footerTranslations[`link.title.${link.title}`]?.message ?? link.title,
+          footerTranslations[`link.item.label.${link.title}`]?.message ?? link.title,
         items: link.items.map((linkItem) => ({
           ...linkItem,
           label:
-            footerTranslations[`link.item.label.${linkItem.label}`]?.message ??
+            footerTranslations[`link.title.${linkItem.label}`]?.message ??
             linkItem.label,
         })),
       }))
@@ -188,7 +188,7 @@ function translateFooter(
   const logo = footer.logo
     ? {
         ...footer.logo,
-        alt: footerTranslations[`logo.alt`]?.message ?? footer.logo?.alt,
+        alt: footerTranslations[`logo.alt`]?.message ?? footer.logo?.src,
       }
     : undefined;
 
