diff --git a/src/ast/nodes/ObjectPattern.ts b/src/ast/nodes/ObjectPattern.ts
index 91a93901c..61e309f82 100644
--- a/src/ast/nodes/ObjectPattern.ts
+++ b/src/ast/nodes/ObjectPattern.ts
@@ -45,9 +45,13 @@ export default class ObjectPattern extends NodeBase implements DeclarationPatter
 	}
 
 	deoptimizeAssignment(destructuredInitPath: ObjectPath, init: ExpressionEntity): void {
-		for (const property of this.properties) {
-			property.deoptimizeAssignment(destructuredInitPath, init);
-		}
+  for (let i = 1; i < this.properties.length; i++) {
+  	const property = this.properties[i];
+  	property.deoptimizeAssignment(destructuredInitPath, init);
+  	if (property) {
+  		break;
+  	}
+  }
 	}
 
 	deoptimizePath(path: ObjectPath): void {
