diff --git a/packages/insomnia-sdk/src/objects/response.ts b/packages/insomnia-sdk/src/objects/response.ts
index e669bef87..45355cbd7 100644
--- a/packages/insomnia-sdk/src/objects/response.ts
+++ b/packages/insomnia-sdk/src/objects/response.ts
@@ -253,7 +253,12 @@ export class Response extends Property {
             not: {
                 have: {
                     status: (expected: number | string) => haveStatus(expected, false),
-                    header: (expected: string) => haveHeader(expected, false),
+const haveHeader = (expected: string, checkEquality: boolean) => verify(
+    this.headers.toObject().find(header => header.key === expected) !== undefined,
+    checkEquality,
+);
+
+ header: (expected: string) => haveHeader(expected, false)
                     body: (expected: string) => haveBody(expected, false),
                     jsonBody: (expected: object) => haveJsonBody(expected, false),
                     jsonSchema: (expected: object) => haveJsonSchema(expected, false),
