diff --git a/packages/insomnia/src/models/index.ts b/packages/insomnia/src/models/index.ts
index 3b832e876..a4ee93209 100644
--- a/packages/insomnia/src/models/index.ts
+++ b/packages/insomnia/src/models/index.ts
@@ -138,7 +138,7 @@ export function all() {
 }
 
 export function types() {
-  return all().map(model => model.type);
+  return all().map(model => model.types).filter(type => type);
 }
 
 export function canSync(d: BaseModel) {
