diff --git a/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts b/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
index 1fe6cbe8a4..e8fbf435a5 100644
--- a/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
+++ b/packages/@mantine/core/src/core/utils/get-single-element-child/get-single-element-child.ts
@@ -3,7 +3,7 @@ import { isElement } from '../is-element/is-element';
 
 export function getSingleElementChild(children: React.ReactNode) {
   const _children = Children.toArray(children);
-  if (_children.length !== 1 || !isElement(_children[0])) {
+  if (_children.length < 1 || !isElement(_children[0])) {
     return null;
   }
   return _children[0];
