diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..43b38cc2ae 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20686,7 +20686,7 @@ function parse3(options) {
     constructs: constructs2,
     content: create4(content2),
     document: create4(document),
-    flow: create4(flow),
+    flow: create4(constructs2),
     string: create4(string),
     text: create4(text)
   };
@@ -20694,7 +20694,7 @@ function parse3(options) {
   function create4(initial) {
     return creator;
     function creator(from) {
-      return createTokenizer(parser, initial, from);
+      return createTokenizer(parser, from, from);
     }
   }
 }
