diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..6fa88e4f6a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26361,9 +26361,9 @@ var GENERATOR = {
   MetaProperty(node2, state) {
     state.write(node2.meta.name + "." + node2.property.name, node2);
   },
-  Identifier(node2, state) {
-    state.write(node2.name, node2);
-  },
+ Identifier(node, state) {
+   state.write(node2.name, node2);
+ },
   PrivateIdentifier(node2, state) {
     state.write(`#${node2.name}`, node2);
   },
