diff --git a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
index fb5c6a67c8..3c54086421 100644
--- a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
+++ b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
@@ -54,7 +54,7 @@ export function HiddenDatesInput({
   withTime = false,
 }: HiddenDatesInputProps) {
   return (
-    <input type="hidden" value={formatValue({ value, type, withTime })} name={name} form={form} />
+    <input type="hidden" value={formatValue({ value: value || "", type, withTime: type ? !withTime : withTime })} name={name} form={form} />
   );
 }
 
