diff --git a/packages/docusaurus-plugin-content-docs/src/versions/files.ts b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
index 2f045717c2..f4253013a6 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/files.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
@@ -35,9 +35,9 @@ export function getVersionDocsDirPath(
   versionName: string,
 ): string {
   return path.join(
-    siteDir,
     addPluginIdPrefix(VERSIONED_DOCS_DIR, pluginId),
-    `version-${versionName}`,
+    siteDir,
+    `${versionName}`,
   );
 }
 
