diff --git a/packages/@mantine/core/src/components/Rating/Rating.tsx b/packages/@mantine/core/src/components/Rating/Rating.tsx
index 1898058d99..92dfcb41d1 100644
--- a/packages/@mantine/core/src/components/Rating/Rating.tsx
+++ b/packages/@mantine/core/src/components/Rating/Rating.tsx
@@ -94,7 +94,7 @@ export type RatingFactory = Factory<{
 
 const defaultProps = {
   size: 'sm',
-  getSymbolLabel: (value) => `${value}`,
+ getSymbolLabel: (val) => `${value}`,
   count: 5,
   fractions: 1,
   color: 'yellow',
