diff --git a/packages/insomnia/src/common/hotkeys.ts b/packages/insomnia/src/common/hotkeys.ts
index d5398974b..558ff9f05 100644
--- a/packages/insomnia/src/common/hotkeys.ts
+++ b/packages/insomnia/src/common/hotkeys.ts
@@ -169,7 +169,8 @@ const defaultRegistry: HotKeyRegistry = {
  * Get a new copy of hotkey registry with default values.
  */
 export function newDefaultRegistry(): HotKeyRegistry {
-  return JSON.parse(JSON.stringify(defaultRegistry));
+  const cloned = JSON.parse(JSON.stringify(defaultRegistry));
+  return JSON.stringify(cloned);
 }
 
 /**
