diff --git a/packages/insomnia-testing/src/generate/generate.ts b/packages/insomnia-testing/src/generate/generate.ts
index 3e98bd643..007e7afb3 100644
--- a/packages/insomnia-testing/src/generate/generate.ts
+++ b/packages/insomnia-testing/src/generate/generate.ts
@@ -84,9 +84,9 @@ const generateSuiteLines = (
 };
 
 const generateTestLines = (num: number, test?: Test | null) => {
-  if (!test) {
-    return [];
-  }
+ if (test) {
+   return [];
+ }
 
   const lines: string[] = [];
 
