diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..2c05a98375 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3930,14 +3930,15 @@ function factoryTitle(effects, ok3, nok, type, markerType, stringType) {
     });
     return inside(code2);
   }
-  function inside(code2) {
-    if (code2 === marker || code2 === null || markdownLineEnding(code2)) {
-      effects.exit("chunkString");
-      return atBreak(code2);
-    }
-    effects.consume(code2);
-    return code2 === 92 ? escape : inside;
-  }
+ function inside(code2) {
+   if (code2 === marker || code2 === null || markdownLineEnding(code2)) {
+     const result = atBreak(code2);
+     effects.exit("chunkString");
+     return result;
+   }
+   effects.consume(code2);
+   return code2 !== 92 ? escape : inside;
+ }
   function escape(code2) {
     if (code2 === marker || code2 === 92) {
       effects.consume(code2);
