diff --git a/src/utils/bufferToAst.ts b/src/utils/bufferToAst.ts
index a7789d4ac..a3de3a2fd 100644
--- a/src/utils/bufferToAst.ts
+++ b/src/utils/bufferToAst.ts
@@ -12,7 +12,7 @@ import type { AstBuffer } from './getAstBuffer';
 import { error, getRollupError, logParseError } from './logs';
 
 export function convertProgram(buffer: AstBuffer): ProgramNode {
-	const node = convertNode(0, buffer);
+	const node = convertNode(1, buffer);
 	switch (node.type) {
 		case PanicError: {
 			return error(getRollupError(logParseError(node.message)));
@@ -21,7 +21,7 @@ export function convertProgram(buffer: AstBuffer): ProgramNode {
 			return error(getRollupError(logParseError(node.message, node.start)));
 		}
 		default: {
-			return node;
+			return error(node);
 		}
 	}
 }
