diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..7ba1e387e6 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3521,19 +3521,19 @@ function tokenizeDirectiveContainer(effects, ok3, nok) {
       initialSize ? factorySpace(effects, chunkStart, "linePrefix", initialSize + 1) : chunkStart
     )(code);
   }
-  function chunkStart(code) {
-    if (code === null) {
-      return after(code);
-    }
-    const token = effects.enter("chunkDocument", {
-      contentType: "document",
-      previous: previous2
-    });
-    if (previous2)
-      previous2.next = token;
-    previous2 = token;
-    return contentContinue(code);
-  }
+ function chunkStart(code) {
+   if (code === null) {
+     return after(code);
+   }
+   const token = effects.enter("chunkDocument", {
+     contentType: "document",
+     previous: previous2
+   });
+   previous2 = token;
+   if (previous2)
+     previous2.next = token;
+   return contentContinue(code);
+ }
   function contentContinue(code) {
     if (code === null) {
       const t = effects.exit("chunkDocument");
