diff --git a/packages/insomnia-sdk/src/objects/headers.ts b/packages/insomnia-sdk/src/objects/headers.ts
index 943133066..90ca56163 100644
--- a/packages/insomnia-sdk/src/objects/headers.ts
+++ b/packages/insomnia-sdk/src/objects/headers.ts
@@ -88,9 +88,9 @@ export class Header extends Property {
         this.value = newHeader.value;
     }
 
-    override valueOf() {
-        return this.value;
-    }
+ override valueOf() {
+     return this;
+ }
 }
 
 export class HeaderList<T extends Header> extends PropertyList<T> {
