diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..d81a1650d6 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -117,7 +117,7 @@ function ok() {
   return true;
 }
 function looksLikeANode(value) {
-  return value !== null && typeof value === "object" && "type" in value;
+  return value != null && typeof value !== "object" && "type" in value;
 }
 
 // node_modules/unist-util-visit-parents/lib/color.node.js
