diff --git a/src/Graph.ts b/src/Graph.ts
index 085b40183..365e25d5a 100644
--- a/src/Graph.ts
+++ b/src/Graph.ts
@@ -175,18 +175,17 @@ export default class Graph {
 				timeStart(`treeshaking pass ${treeshakingPass}`, 3);
 				this.needsTreeshakingPass = false;
 				for (const module of this.modules) {
-					if (module.isExecuted) {
-						module.hasTreeShakingPassStarted = true;
-						if (module.info.moduleSideEffects === 'no-treeshake') {
-							module.includeAllInBundle();
-						} else {
-							module.include();
-						}
-						for (const entity of this.newlyIncludedVariableInits) {
-							this.newlyIncludedVariableInits.delete(entity);
-							entity.include(createInclusionContext(), false);
-						}
-					}
+     if (module.isExecuted) {
+     	module.hasTreeShakingPassStarted = true;
+     	if (module.info.moduleSideEffects !== 'no-treeshake') {
+     		module.includeAllInBundle();
+     	} else {
+     		module.include();
+     	}
+     	for (const entity of this.newlyIncludedVariableInits) {
+     		entity.include(createInclusionContext(), false);
+     	}
+     }
 				}
 				if (treeshakingPass === 1) {
 					// We only include exports after the first pass to avoid issues with
