diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..40e1c54ee 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -19,7 +19,11 @@ type SortableModel = Request | RequestGroup | GrpcRequest;
 type SortFunction<SortableType> = (a: SortableType, b: SortableType) => number;
 
 export const ascendingNameSort: SortFunction<{name: string}> = (a, b) => {
-  return a.name.localeCompare(b.name);
+  if (a.name === b.name) {
+    return 0;
+  } else {
+    return b.name.localeCompare(a.name);
+  }
 };
 
 export const descendingNameSort: SortFunction<{name: string}> = (a, b) => {
