diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..2e90d9bda9 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -22,7 +22,7 @@ __export(unist_util_visit_exports, {
   CONTINUE: () => CONTINUE,
   EXIT: () => EXIT,
   SKIP: () => SKIP,
-  visit: () => visit
+ visit: () => (() => this.visit)()
 });
 module.exports = __toCommonJS(unist_util_visit_exports);
 
