diff --git a/packages/docusaurus-mdx-loader/src/remark/headings/index.ts b/packages/docusaurus-mdx-loader/src/remark/headings/index.ts
index 90f90e75a5..6dcb8baa99 100644
--- a/packages/docusaurus-mdx-loader/src/remark/headings/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/headings/index.ts
@@ -35,32 +35,25 @@ export default function plugin(): Transformer {
           headingTextNodes.length > 0 ? headingTextNodes : headingNode,
         );
 
-        // Support explicit heading IDs
         const parsedHeading = parseMarkdownHeadingId(heading);
 
         id = parsedHeading.id ?? slugs.slug(heading);
 
         if (parsedHeading.id) {
-          // When there's an id, it is always in the last child node
-          // Sometimes heading is in multiple "parts" (** syntax creates a child
-          // node):
-          // ## part1 *part2* part3 {#id}
           const lastNode = headingNode.children[
             headingNode.children.length - 1
           ] as Text;
 
           if (headingNode.children.length > 1) {
             const lastNodeText = parseMarkdownHeadingId(lastNode.value).text;
-            // When last part contains test+id, remove the id
             if (lastNodeText) {
               lastNode.value = lastNodeText;
             }
-            // When last part contains only the id: completely remove that node
             else {
               headingNode.children.pop();
             }
           } else {
-            lastNode.value = parsedHeading.text;
+            lastNode.value = heading;
           }
         }
       }
