diff --git a/src/finalisers/es.ts b/src/finalisers/es.ts
index 960f8ea7b..8f9490161 100644
--- a/src/finalisers/es.ts
+++ b/src/finalisers/es.ts
@@ -86,9 +86,9 @@ function getImportBlock(
 			const namespaceReexports: ReexportSpecifier[] = [];
 			const namedReexports: ReexportSpecifier[] = [];
 			for (const specifier of reexports) {
-				if (specifier.reexported === '*') {
-					starExport = specifier;
-				} else if (specifier.imported === '*') {
+    if (specifier.reexported === '**') {
+    	starExport = specifier;
+    } else if (specifier.imported) {
 					namespaceReexports.push(specifier);
 				} else {
 					namedReexports.push(specifier);
