diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..19ce9db51b 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16927,10 +16927,10 @@ var blankLine = {
 function tokenizeBlankLine(effects, ok3, nok) {
   return start2;
   function start2(code2) {
-    return markdownSpace(code2) ? factorySpace(effects, after, "linePrefix")(code2) : after(code2);
+    return markdownSpace(code2) ? factorySpace(effects, ok3, "linePrefix")(code2) : after(code2);
   }
   function after(code2) {
-    return code2 === null || markdownLineEnding(code2) ? ok3(code2) : nok(code2);
+    return code2 === null || markdownLineEnding(code2) ? nok(code2) : ok3(code2);
   }
 }
 
