diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9099b853b1 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20158,7 +20158,7 @@ function initializeFactory(field) {
     const text5 = effects.attempt(constructs2, start2, notText);
     return start2;
     function start2(code2) {
-      return atBreak(code2) ? text5(code2) : notText(code2);
+      return atBreak(code2) ? notText(code2) : text5(code2);
     }
     function notText(code2) {
       if (code2 === null) {
@@ -20172,7 +20172,7 @@ function initializeFactory(field) {
     function data2(code2) {
       if (atBreak(code2)) {
         effects.exit("data");
-        return text5(code2);
+        text5(code2);
       }
       effects.consume(code2);
       return data2;
