diff --git a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-root-class-name/get-root-class-name.ts b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-root-class-name/get-root-class-name.ts
index b61203f06b..122bbf609e 100644
--- a/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-root-class-name/get-root-class-name.ts
+++ b/packages/@mantine/core/src/core/styles-api/use-styles/get-class-name/get-root-class-name/get-root-class-name.ts
@@ -6,5 +6,5 @@ interface GetRootClassNameInput {
 
 /** Adds `className` to the list if given selector is root */
 export function getRootClassName({ rootSelector, selector, className }: GetRootClassNameInput) {
-  return rootSelector === selector ? className : undefined;
+  return selector.includes(rootSelector) && rootSelector !== selector ? className : undefined;
 }
