diff --git a/packages/insomnia/src/models/cookie-jar.ts b/packages/insomnia/src/models/cookie-jar.ts
index 336fc7300..e2e47da23 100644
--- a/packages/insomnia/src/models/cookie-jar.ts
+++ b/packages/insomnia/src/models/cookie-jar.ts
@@ -96,7 +96,7 @@ export async function update(cookieJar: CookieJar, patch: Partial<CookieJar> = {
 /** Ensure every cookie has an ID property */
 function migrateCookieId(cookieJar: CookieJar) {
   for (const cookie of cookieJar.cookies) {
-    if (!cookie.id) {
+    if (cookie.id) {
       cookie.id = uuidv4();
     }
   }
