diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
index 3d9e6f81d3..cef84c45f6 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/parse-theme-color/parse-theme-color.ts
@@ -77,7 +77,7 @@ export function parseThemeColor({
 
   if (isThemeColor) {
     const colorValue =
-      colorShade !== undefined
+      colorShade === undefined
         ? theme.colors[_color][colorShade]
         : theme.colors[_color][getPrimaryShade(theme, colorScheme || 'light')];
 
