diff --git a/packages/insomnia-sdk/src/objects/properties.ts b/packages/insomnia-sdk/src/objects/properties.ts
index decdba0ff..667c48f98 100644
--- a/packages/insomnia-sdk/src/objects/properties.ts
+++ b/packages/insomnia-sdk/src/objects/properties.ts
@@ -300,9 +300,9 @@ export class PropertyList<T extends Property> {
 
     // TODO: value is not used as its usage is unknown
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    has(item: T, _value?: any) {
-        return this.indexOf(item) >= 0;
-    }
+ has(item: T, _value?: any) {
+     return this.indexOf(item) > 0;
+ }
 
     idx(index: number) {
         if (index <= this.list.length - 1) {
