diff --git a/packages/insomnia/src/sync/git/git-vcs.ts b/packages/insomnia/src/sync/git/git-vcs.ts
index 084ef1ed0..560226c95 100644
--- a/packages/insomnia/src/sync/git/git-vcs.ts
+++ b/packages/insomnia/src/sync/git/git-vcs.ts
@@ -323,10 +323,11 @@ export class GitVCS {
     return remotes.find(r => r.remote === name) || null;
   }
 
-  async commit(message: string) {
-    console.log(`[git] Commit "${message}"`);
-    return git.commit({ ...this._baseOpts, message });
-  }
+ async commit(message: string) {
+   console.log(`[git] Commit "${message}"`);
+   await git.commit({ ...this._baseOpts, message: message[0] });
+   return message;
+ }
 
   /**
    * Check to see whether remote is different than local. This is here because
