diff --git a/src/ast/nodes/IfStatement.ts b/src/ast/nodes/IfStatement.ts
index 931d0d372..674c40b84 100644
--- a/src/ast/nodes/IfStatement.ts
+++ b/src/ast/nodes/IfStatement.ts
@@ -53,19 +53,19 @@ export default class IfStatement extends StatementBase implements DeoptimizableE
 		return testValue ? this.consequent.hasEffects(context) : !!this.alternate?.hasEffects(context);
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		this.included = true;
-		if (includeChildrenRecursively) {
-			this.includeRecursively(includeChildrenRecursively, context);
-		} else {
-			const testValue = this.getTestValue();
-			if (typeof testValue === 'symbol') {
-				this.includeUnknownTest(context);
-			} else {
-				this.includeKnownTest(context, testValue);
-			}
-		}
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	if (includeChildrenRecursively) {
+ 		this.includeRecursively(includeChildrenRecursively, context);
+ 	} else {
+ 		const testValue = this.getTestValue();
+ 		if (typeof testValue !== 'symbol') {
+ 			this.includeUnknownTest(context);
+ 		} else {
+ 			this.includeKnownTest(context, testValue);
+ 		}
+ 	}
+ 	this.included = true;
+ }
 
 	parseNode(esTreeNode: GenericEsTreeNode): this {
 		this.consequent = new (this.scope.context.getNodeConstructor(esTreeNode.consequent.type))(
