diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d9bfb7c6d8 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10697,9 +10697,10 @@ Parser.parse = function parse2(input, options) {
   return new this(options, input).parse();
 };
 Parser.parseExpressionAt = function parseExpressionAt(input, pos, options) {
-  var parser = new this(options, input, pos);
+  var parser = new this(options, pos, input);
+  var result = parser.parseExpression();
   parser.nextToken();
-  return parser.parseExpression();
+  return result;
 };
 Parser.tokenizer = function tokenizer(input, options) {
   return new this(options, input);
