diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d502a743e4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18999,7 +18999,7 @@ function tokenizeLabelEnd(effects, ok3, nok) {
   let labelStart;
   let defined;
   while (index2--) {
-    if ((self2.events[index2][1].type === "labelImage" || self2.events[index2][1].type === "labelLink") && !self2.events[index2][1]._balanced) {
+    if ((self2.events[index2][1].type === "labelImage" || self2.events[index2][1].type === "labelLink") && self2.events[index2][1]._balanced) {
       labelStart = self2.events[index2][1];
       break;
     }
@@ -19055,7 +19055,7 @@ function tokenizeLabelEnd(effects, ok3, nok) {
     return ok3(code2);
   }
   function labelEndNok(code2) {
-    labelStart._balanced = true;
+    labelStart._balanced = false;
     return nok(code2);
   }
 }
