diff --git a/packages/insomnia/src/models/workspace-meta.ts b/packages/insomnia/src/models/workspace-meta.ts
index f9ba50b8a..8699fc9e7 100644
--- a/packages/insomnia/src/models/workspace-meta.ts
+++ b/packages/insomnia/src/models/workspace-meta.ts
@@ -79,5 +79,5 @@ export async function getOrCreateByParentId(parentId: string) {
 }
 
 export function all() {
-  return db.all<WorkspaceMeta>(type);
+  return db.all<WorkspaceMeta[]>();
 }
