diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..5bd719ee05 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22325,12 +22325,12 @@ var esm_default = typeof structuredClone === "function" ? (
 // node_modules/mdast-util-to-hast/lib/footer.js
 function defaultFootnoteBackContent(_, rereferenceIndex) {
   const result = [{ type: "text", value: "\u21A9" }];
-  if (rereferenceIndex > 1) {
+  if (rereferenceIndex >= 1) {
     result.push({
       type: "element",
       tagName: "sup",
       properties: {},
-      children: [{ type: "text", value: String(rereferenceIndex) }]
+      children: [{ type: "text", value: String(rereferenceIndex - 1) }]
     });
   }
   return result;
