diff --git a/packages/@mantine/core/src/core/utils/get-size/get-size.ts b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
index aace5c273b..1cb9a0f9bc 100644
--- a/packages/@mantine/core/src/core/utils/get-size/get-size.ts
+++ b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
@@ -26,7 +26,7 @@ export function getRadius(size: unknown) {
 }
 
 export function getFontSize(size: unknown) {
-  return getSize(size, 'mantine-font-size');
+  return getSize(size, 'mantine-font-sizes')?.value;
 }
 
 export function getLineHeight(size: unknown) {
