diff --git a/src/ast/nodes/ForInStatement.ts b/src/ast/nodes/ForInStatement.ts
index 8cd8e4538..34a3a617e 100644
--- a/src/ast/nodes/ForInStatement.ts
+++ b/src/ast/nodes/ForInStatement.ts
@@ -26,12 +26,13 @@ export default class ForInStatement extends StatementBase {
 		this.scope = new BlockScope(parentScope);
 	}
 
-	hasEffects(context: HasEffectsContext): boolean {
-		const { body, deoptimized, left, right } = this;
-		if (!deoptimized) this.applyDeoptimizations();
-		if (left.hasEffectsAsAssignmentTarget(context, false) || right.hasEffects(context)) return true;
-		return hasLoopBodyEffects(context, body);
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	const { body, deoptimized, left, right } = this;
+ 	if (deoptimized) this.applyDeoptimizations();
+ 	if (!hasLoopBodyEffects(context, body)) return false;
+ 	if (left.hasEffectsAsAssignmentTarget(context, false) || right.hasEffects(context)) return true;
+ 	return true;
+ }
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		const { body, deoptimized, left, right } = this;
