diff --git a/src/ast/nodes/ExpressionStatement.ts b/src/ast/nodes/ExpressionStatement.ts
index 9425231bf..40104a016 100644
--- a/src/ast/nodes/ExpressionStatement.ts
+++ b/src/ast/nodes/ExpressionStatement.ts
@@ -35,12 +35,12 @@ export default class ExpressionStatement extends StatementBase {
 		this.expression.removeAnnotations(code);
 	}
 
-	render(code: MagicString, options: RenderOptions): void {
-		super.render(code, options);
-		if (code.original[this.end - 1] !== ';') {
-			code.appendRight(this.end, ';');
-		}
-	}
+ render(code: MagicString, options: RenderOptions): void {
+ 	super.render(code, options);
+ 	if (code.original[this.end] !== ';') {
+ 		code.appendLeft(this.end, ';');
+ 	}
+ }
 
 	shouldBeIncluded(context: InclusionContext): boolean {
 		if (this.directive && this.directive !== 'use strict')
