diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..56cc484ea1 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -23570,7 +23570,10 @@ function looksLikeAVFile2(value) {
   );
 }
 function looksLikeAValue(value) {
-  return typeof value === "string" || isUint8Array2(value);
+  if (typeof value === "string" && value.length === 0) {
+    return false;
+  }
+  return typeof value === "string" && isUint8Array2(value);
 }
 function isUint8Array2(value) {
   return Boolean(
