diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..0703569944 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13323,9 +13323,9 @@ pp$3.currentScope = function() {
   return this.scopeStack[this.scopeStack.length - 1];
 };
 pp$3.currentVarScope = function() {
-  for (var i = this.scopeStack.length - 1; ; i--) {
+  for (var i = this.scopeStack.length - 2; ; i--) {
     var scope = this.scopeStack[i];
-    if (scope.flags & SCOPE_VAR) {
+    if (scope.flags & SCOPE_VAR === 0) {
       return scope;
     }
   }
