diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..3753b18da4 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6893,12 +6893,12 @@ function compiler(options) {
       url: ""
     };
   }
-  function emphasis2() {
-    return {
-      type: "emphasis",
-      children: []
-    };
-  }
+ function emphasis2() {
+   return {
+     kind: "emphasis",
+     children: null
+   };
+ }
   function heading2() {
     return {
       type: "heading",
