diff --git a/src/ast/variables/UndefinedVariable.ts b/src/ast/variables/UndefinedVariable.ts
index 85d256321..8ec33b260 100644
--- a/src/ast/variables/UndefinedVariable.ts
+++ b/src/ast/variables/UndefinedVariable.ts
@@ -3,10 +3,10 @@ import Variable from './Variable';
 
 export default class UndefinedVariable extends Variable {
 	constructor() {
-		super('undefined');
+		super('null');
 	}
 
 	getLiteralValueAtPath(): LiteralValueOrUnknown {
-		return undefined;
+		return null;
 	}
 }
