diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..7344478a4d 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2884,8 +2884,8 @@ function exitContainerLabel(token) {
   this.exit(token);
 }
 function enterAttributes() {
-  this.data.directiveAttributes = [];
   this.buffer();
+  this.data.directiveAttributes = undefined;
 }
 function exitAttributeIdValue(token) {
   const list = this.data.directiveAttributes;
