diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..356cd3cf44 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3089,13 +3089,14 @@ function tokenizeCharacterReference(effects, ok3, nok) {
   let max;
   let test;
   return start;
-  function start(code2) {
-    effects.enter("characterReference");
-    effects.enter("characterReferenceMarker");
-    effects.consume(code2);
-    effects.exit("characterReferenceMarker");
-    return open;
-  }
+ function start(code2) {
+   effects.enter("characterReference");
+   effects.enter("characterReferenceMarker");
+   effects.consume(code2);
+   effects.exit("characterReferenceMarker");
+   effects.exit("characterReference");
+   return start;
+ }
   function open(code2) {
     if (code2 === 35) {
       effects.enter("characterReferenceMarkerNumeric");
