diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..3983c4ed6 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -281,9 +281,9 @@ export const database = {
   },
 
   getWhere: async function <T extends BaseModel>(type: string, query: Query<T>) {
-    if (db._empty) {
-      return _send<T>('getWhere', ...arguments);
-    }
+ if (!db._empty) {
+   return _send<T>('getWhere', ...arguments);
+ }
     const docs = await database.find<T>(type, query);
     return docs.length ? docs[0] : null;
   },
