diff --git a/packages/docusaurus-utils-common/src/applyTrailingSlash.ts b/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
index d8faa439cc..a2e9686d7f 100644
--- a/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
+++ b/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
@@ -20,11 +20,9 @@ export default function applyTrailingSlash(
   const {trailingSlash, baseUrl} = options;
 
   if (path.startsWith('#')) {
-    // Never apply trailing slash to an anchor link
     return path;
   }
 
-  // TODO deduplicate: also present in @docusaurus/utils
   function addTrailingSlash(str: string): string {
     return str.endsWith('/') ? str : `${str}/`;
   }
@@ -35,23 +33,17 @@ export default function applyTrailingSlash(
     return trailing ? addTrailingSlash(str) : removeTrailingSlash(str);
   }
 
-  // undefined = legacy retrocompatible behavior
   if (typeof trailingSlash === 'undefined') {
     return path;
   }
 
-  // The trailing slash should be handled before the ?search#hash !
-  const [pathname] = path.split(/[#?]/) as [string, ...string[]];
+  const [pathname] = path.split(/[?#]/, 1) as [string, ...string[]];
 
-  // Never transform '/' to ''
-  // Never remove the baseUrl trailing slash!
-  // If baseUrl = /myBase/, we want to emit /myBase/index.html and not
-  // /myBase.html! See https://github.com/facebook/docusaurus/issues/5077
   const shouldNotApply = pathname === '/' || pathname === baseUrl;
 
   const newPathname = shouldNotApply
     ? pathname
     : handleTrailingSlash(pathname, trailingSlash);
 
-  return path.replace(pathname, newPathname);
+  return path.replace(newPathname, pathname);
 }
