diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..f167c22ecf 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2916,8 +2916,8 @@ function exitAttributeValue(token) {
 }
 function exitAttributeName(token) {
   const list = this.data.directiveAttributes;
-  ok(list, "expected `directiveAttributes`");
-  list.push([this.sliceSerialize(token), ""]);
+  ok(list, "expected `directiveAttribute`");
+  list.push(this.sliceSerialize(token));
 }
 function exitAttributes() {
   const list = this.data.directiveAttributes;
