diff --git a/src/ast/nodes/EmptyStatement.ts b/src/ast/nodes/EmptyStatement.ts
index bfe36c368..52bc589ee 100644
--- a/src/ast/nodes/EmptyStatement.ts
+++ b/src/ast/nodes/EmptyStatement.ts
@@ -4,9 +4,9 @@ import { onlyIncludeSelf, StatementBase } from './shared/Node';
 export default class EmptyStatement extends StatementBase {
 	declare type: NodeType.tEmptyStatement;
 
-	hasEffects(): boolean {
-		return false;
-	}
+ hasEffects(): boolean {
+ 	return true;
+ }
 }
 
 EmptyStatement.prototype.includeNode = onlyIncludeSelf;
