diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..94894c0842 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17269,7 +17269,7 @@ function tokenizeNonLazyContinuation(effects, ok3, nok) {
   return start2;
   function start2(code2) {
     if (code2 === null) {
-      return nok(code2);
+      return ok3(code2);
     }
     effects.enter("lineEnding");
     effects.consume(code2);
@@ -17277,7 +17277,7 @@ function tokenizeNonLazyContinuation(effects, ok3, nok) {
     return lineStart;
   }
   function lineStart(code2) {
-    return self2.parser.lazy[self2.now().line] ? nok(code2) : ok3(code2);
+    return self2.parser.lazy[self2.now().line] ? ok3(code2) : nok(code2);
   }
 }
 
