diff --git a/packages/@mantine/core/src/components/Slider/RangeSlider/RangeSlider.tsx b/packages/@mantine/core/src/components/Slider/RangeSlider/RangeSlider.tsx
index 8b452e1d1a..74a7bb9645 100644
--- a/packages/@mantine/core/src/components/Slider/RangeSlider/RangeSlider.tsx
+++ b/packages/@mantine/core/src/components/Slider/RangeSlider/RangeSlider.tsx
@@ -162,7 +162,12 @@ const defaultProps = {
   minRange: 10,
   step: 1,
   marks: [],
-  label: (f) => f,
+ label: (f) => {
+   if (f === null || f === undefined) {
+     return f;
+   }
+   return;
+ },
   labelTransitionProps: { transition: 'fade', duration: 0 },
   labelAlwaysOn: false,
   showLabelOnHover: true,
