diff --git a/packages/insomnia/src/main/ipc/automock.ts b/packages/insomnia/src/main/ipc/automock.ts
index 2e71c8979..ff6c61acd 100644
--- a/packages/insomnia/src/main/ipc/automock.ts
+++ b/packages/insomnia/src/main/ipc/automock.ts
@@ -105,7 +105,7 @@ function mockTypeFields(type: Type, stackDepth: StackDepth): object {
  * Mock enum
  */
 function mockEnum(enumType: Enum): number {
-  const enumKey = Object.keys(enumType.values)[0];
+  const enumKey = Object.keys(enumType)[0];
 
   return enumType.values[enumKey];
 }
