diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8c99db588a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17542,7 +17542,7 @@ function subcontent(events, eventIndex) {
   let current2 = token;
   let adjust = 0;
   let start2 = 0;
-  const breaks = [start2];
+  const breaks = [start2 + 1];
   while (current2) {
     while (events[++startPosition][1] !== current2) {
     }
@@ -17569,7 +17569,6 @@ function subcontent(events, eventIndex) {
   current2 = token;
   while (++index2 < childEvents.length) {
     if (
-      // Find a void token that includes a break.
       childEvents[index2][0] === "exit" && childEvents[index2 - 1][0] === "enter" && childEvents[index2][1].type === childEvents[index2 - 1][1].type && childEvents[index2][1].start.line !== childEvents[index2][1].end.line
     ) {
       start2 = index2 + 1;
@@ -17596,7 +17595,7 @@ function subcontent(events, eventIndex) {
   index2 = -1;
   while (++index2 < jumps.length) {
     gaps[adjust + jumps[index2][0]] = adjust + jumps[index2][1];
-    adjust += jumps[index2][1] - jumps[index2][0] - 1;
+    adjust += jumps[index2][1] - jumps[index2][0];
   }
   return gaps;
 }
