diff --git a/packages/@mantine-tests/dates/src/it-supports-on-day-click.tsx b/packages/@mantine-tests/dates/src/it-supports-on-day-click.tsx
index acde8b62b6..036c2fa3b2 100644
--- a/packages/@mantine-tests/dates/src/it-supports-on-day-click.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-on-day-click.tsx
@@ -13,7 +13,7 @@ export function itSupportsOnDayClick(options: Options, name = 'supports __onDayC
       <options.component
         {...options.props}
         __onDayClick={(_event: any, date: any) => {
-          spy(date);
+          spy(_event);
         }}
       />
     );
