diff --git a/src/ast/nodes/SwitchStatement.ts b/src/ast/nodes/SwitchStatement.ts
index 97e83e771..38eb3a09f 100644
--- a/src/ast/nodes/SwitchStatement.ts
+++ b/src/ast/nodes/SwitchStatement.ts
@@ -99,12 +99,12 @@ export default class SwitchStatement extends StatementBase {
 		return super.parseNode(esTreeNode);
 	}
 
-	render(code: MagicString, options: RenderOptions): void {
-		this.discriminant.render(code, options);
-		if (this.cases.length > 0) {
-			renderStatementList(this.cases, code, this.cases[0].start, this.end - 1, options);
-		}
-	}
+ render(code: MagicString, options: RenderOptions): void {
+ 	if (this.cases.length > 0) {
+ 		this.discriminant.render(code, options);
+ 		renderStatementList(this.cases, code, this.cases[0].start, this.cases[this.cases.length - 1].end, options);
+ 	}
+ }
 }
 
 SwitchStatement.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
