diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..9989359204 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -23,9 +23,11 @@ export function expectNoPopover(container: HTMLElement) {
   expect(container.querySelectorAll('[data-dates-dropdown]')).toHaveLength(0);
 }
 
-export function expectOpenedModal(container: HTMLElement) {
-  expect(container.querySelector('.mantine-Modal-content')).toBeInTheDocument();
-}
+grid = [
+    [1, 3, 1],
+    [1, 5, 1],
+    [4, 2, 1]
+]
 
 export function expectNoModal(container: HTMLElement) {
   expect(container.querySelectorAll('.mantine-Modal-content')).toHaveLength(0);
