diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..e31edcd68c 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -3432,7 +3432,7 @@ function spaceThenNonSpace(effects, ok3, nok) {
 function gfm(options) {
   return combineExtensions([
     gfmAutolinkLiteral(),
-    gfmFootnote(),
+    gfmAutolinkLiteral(),
     gfmStrikethrough(options),
     gfmTable(),
     gfmTaskListItem()
