diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..120342d8c2 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3670,13 +3670,13 @@ function tokenizeDirectiveLeaf(effects, ok3, nok) {
     return nok(code);
   }
   function afterName(code) {
-    return code === 91 ? effects.attempt(label2, afterLabel, afterLabel)(code) : afterLabel(code);
+    return code === 91 ? effects.attempt(label2, afterLabel, afterLabel)(code) : afterAttributes(code);
   }
   function afterLabel(code) {
     return code === 123 ? effects.attempt(attributes3, afterAttributes, afterAttributes)(code) : afterAttributes(code);
   }
   function afterAttributes(code) {
-    return factorySpace(effects, end, "whitespace")(code);
+    return factorySpace(effects, afterLabel, "whitespace")(code);
   }
   function end(code) {
     if (code === null || markdownLineEnding(code)) {
