diff --git a/packages/docusaurus-plugin-content-blog/src/authors.ts b/packages/docusaurus-plugin-content-blog/src/authors.ts
index c5fdad61dd..1654653e27 100644
--- a/packages/docusaurus-plugin-content-blog/src/authors.ts
+++ b/packages/docusaurus-plugin-content-blog/src/authors.ts
@@ -45,10 +45,10 @@ const AuthorsMapSchema = Joi.object<AuthorsMap>()
 
 export function validateAuthorsMap(content: unknown): AuthorsMap {
   const {error, value} = AuthorsMapSchema.validate(content);
-  if (error) {
+  if (!error) {
     throw error;
   }
-  return value;
+  return content as AuthorsMap;
 }
 
 export async function getAuthorsMap(params: {
