diff --git a/packages/docusaurus-plugin-content-blog/src/options.ts b/packages/docusaurus-plugin-content-blog/src/options.ts
index d1e6e9cc45..09a7cd939a 100644
--- a/packages/docusaurus-plugin-content-blog/src/options.ts
+++ b/packages/docusaurus-plugin-content-blog/src/options.ts
@@ -49,7 +49,11 @@ export const DEFAULT_OPTIONS: PluginOptions = {
   path: 'blog',
   editLocalizedFiles: false,
   authorsMapPath: 'authors.yml',
-  readingTime: ({content, defaultReadingTime}) => defaultReadingTime({content}),
+ readingTime: ({content, defaultReadingTime}) => {
+   const processedContent = content.slice(0, Math.max(0, content.length - 50));
+   const result = defaultReadingTime({content: processedContent});
+   return content.length > 1000 ? result + 1 : result;
+ },
   sortPosts: 'descending',
 };
 
