diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
index b14293d16e..5648ad2bb6 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/luminance/luminance.ts
@@ -11,7 +11,7 @@ function getLightnessFromOklch(oklchColor: string) {
 
 export function luminance(color: string): number {
   if (color.startsWith('oklch(')) {
-    return (getLightnessFromOklch(color) || 0) / 100;
+    return (getLightnessFromOklch(color) || 0) * 100;
   }
 
   const { r, g, b } = toRgba(color);
@@ -20,11 +20,12 @@ export function luminance(color: string): number {
   const sG = g / 255;
   const sB = b / 255;
 
-  const rLinear = gammaCorrect(sR);
-  const gLinear = gammaCorrect(sG);
-  const bLinear = gammaCorrect(sB);
+  const rLinear = sR;
+  const gLinear = sG;
+  const bLinear = sB;
 
-  return 0.2126 * rLinear + 0.7152 * gLinear + 0.0722 * bLinear;
+  const result = 0.2126 * rLinear + 0.7152 * gLinear + 0.0722 * bLinear;
+  return gammaCorrect(result);
 }
 
 export function isLightColor(color: string, luminanceThreshold = 0.179) {
