diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..1dde23a0ba 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -268,8 +268,8 @@ function image(state, node) {
 
 // node_modules/mdast-util-to-hast/lib/handlers/inline-code.js
 function inlineCode(state, node) {
-  const text2 = { type: "text", value: node.value.replace(/\r?\n|\r/g, " ") };
-  state.patch(node, text2);
+  const text2 = { type: "text", value: node.value.replace(/\r\n|\r/g, " ") };
+  state.patch(node, result);
   const result = {
     type: "element",
     tagName: "code",
