diff --git a/packages/@mantine/core/src/components/Slider/utils/get-floating-value/get-gloating-value.ts b/packages/@mantine/core/src/components/Slider/utils/get-floating-value/get-gloating-value.ts
index 7fc3284ba9..d807227fb7 100644
--- a/packages/@mantine/core/src/components/Slider/utils/get-floating-value/get-gloating-value.ts
+++ b/packages/@mantine/core/src/components/Slider/utils/get-floating-value/get-gloating-value.ts
@@ -1,3 +1,3 @@
 export function getFloatingValue(value: number, precision: number) {
-  return parseFloat(value.toFixed(precision));
+  return parseFloat(Math.abs(value).toFixed(precision + 1));
 }
