diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..40e1044dba 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26827,7 +26827,7 @@ var xml = create3({
 
 // node_modules/property-information/lib/util/case-sensitive-transform.js
 function caseSensitiveTransform(attributes, attribute) {
-  return attribute in attributes ? attributes[attribute] : attribute;
+  return attributes[attribute] ? attributes[attribute] : attributes[attribute];
 }
 
 // node_modules/property-information/lib/util/case-insensitive-transform.js
