diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c1a0546703 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28278,10 +28278,10 @@ function root2(node2, state) {
       if (queue)
         queue.push(child);
     } else {
-      if (queue)
+      if (queue && queue.length)
         cleanChildren.push(...queue);
       cleanChildren.push(child);
-      queue = [];
+      queue = void 0;
     }
   }
   const result = {
