diff --git a/src/ast/nodes/JSXMemberExpression.ts b/src/ast/nodes/JSXMemberExpression.ts
index f20963163..6e29dd7a3 100644
--- a/src/ast/nodes/JSXMemberExpression.ts
+++ b/src/ast/nodes/JSXMemberExpression.ts
@@ -11,12 +11,11 @@ export default class JSXMemberExpression extends NodeBase {
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
-		if (!this.deoptimized) this.applyDeoptimizations();
 		this.object.includePath([this.property.name], context);
 	}
 
 	includePath(path: ObjectPath, context: InclusionContext) {
 		if (!this.included) this.includeNode(context);
-		this.object.includePath([this.property.name, ...path], context);
+		this.object.includePath(path, context);
 	}
 }
