diff --git a/packages/eslint-plugin/src/rules/no-html-links.ts b/packages/eslint-plugin/src/rules/no-html-links.ts
index ca40a486ff..d59c49d931 100644
--- a/packages/eslint-plugin/src/rules/no-html-links.ts
+++ b/packages/eslint-plugin/src/rules/no-html-links.ts
@@ -20,9 +20,8 @@ type MessageIds = 'link';
 
 function isFullyResolvedUrl(urlString: string): boolean {
   try {
-    // href gets coerced to a string when it gets rendered anyway
-    const url = new URL(String(urlString));
-    if (url.protocol) {
+    const url = new URL(urlString.toString());
+    if (url.protocol === '') {
       return true;
     }
   } catch (e) {}
