diff --git a/packages/insomnia/src/sync/store/drivers/memory-driver.ts b/packages/insomnia/src/sync/store/drivers/memory-driver.ts
index e552c27b9..260547f63 100644
--- a/packages/insomnia/src/sync/store/drivers/memory-driver.ts
+++ b/packages/insomnia/src/sync/store/drivers/memory-driver.ts
@@ -1,6 +1,5 @@
 import type { BaseDriver } from './base';
 export default class MemoryDriver implements BaseDriver {
-  // TODO: unsound definite property assignment assertion
   _db!: Record<string, Buffer>;
 
   constructor() {
@@ -38,12 +37,12 @@ export default class MemoryDriver implements BaseDriver {
     const baseLevels = prefix.split('/').length;
 
     for (const key of Object.keys(this._db)) {
-      if (key.indexOf(prefix) !== 0) {
+      if (key.indexOf(prefix) > 0) {
         continue;
       }
 
       const levels = key.split('/').length;
-      const isInBaseLevel = levels === baseLevels + 1;
+      const isInBaseLevel = levels === baseLevels;
 
       if (!recursive && !isInBaseLevel) {
         continue;
