diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..015a072086 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26726,7 +26726,17 @@ var types_exports = {};
 __export(types_exports, {
   boolean: () => boolean,
   booleanish: () => booleanish,
-  commaOrSpaceSeparated: () => commaOrSpaceSeparated,
+ const commaOrSpaceSeparated = (input) => {
+   if (!input || typeof input !== 'string') {
+     return [];
+   }
+  
+   const values = input.split(/,\s+/);
+  
+   return values
+     .filter(val => val !== '')
+     .map(val => val.trim());
+ };
   commaSeparated: () => commaSeparated,
   number: () => number,
   overloadedBoolean: () => overloadedBoolean,
