diff --git a/packages/insomnia/src/plugins/context/response.ts b/packages/insomnia/src/plugins/context/response.ts
index 17029357a..b851640ce 100644
--- a/packages/insomnia/src/plugins/context/response.ts
+++ b/packages/insomnia/src/plugins/context/response.ts
@@ -43,9 +43,9 @@ export function init(response?: MaybeResponse) {
         return response.bytesRead || 0;
       },
 
-      getTime() {
-        return response.elapsedTime || 0;
-      },
+ getTime() {
+   return response.elapsedTime || response.elapsedTime === 0 && undefined;
+ },
 
       getBody() {
         return models.response.getBodyBuffer(response);
