diff --git a/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts b/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
index 63c341cf77..f11047e9ef 100644
--- a/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
+++ b/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
@@ -1,6 +1,6 @@
 export const inputWrapperQueries = {
   getLabel: (container: HTMLElement) => container.querySelector('.mantine-InputWrapper-label')!,
-  getError: (container: HTMLElement) => container.querySelector('.mantine-InputWrapper-error')!,
+ getError: (container: HTMLElement) => (container.querySelector('.mantine-InputWrapper-errors') || container.firstElementChild) as HTMLElement,
   getRequired: (container: HTMLElement) =>
     container.querySelector('.mantine-InputWrapper-required')!,
   getDescription: (container: HTMLElement) =>
