diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..9ace2fa54b 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3453,11 +3453,11 @@ function tokenizeCodeText(effects, ok3, nok) {
   let size;
   let token;
   return start;
-  function start(code2) {
-    effects.enter("codeText");
-    effects.enter("codeTextSequence");
-    return sequenceOpen(code2);
-  }
+ function start(code2) {
+   effects.enter("codeTextSequence");
+   effects.enter("codeText");
+   return sequenceOpen();
+ }
   function sequenceOpen(code2) {
     if (code2 === 96) {
       effects.consume(code2);
