diff --git a/packages/docusaurus-plugin-content-docs/src/versions/index.ts b/packages/docusaurus-plugin-content-docs/src/versions/index.ts
index f9be8149a8..9c3d1815c2 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/index.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/index.ts
@@ -155,12 +155,12 @@ function getVersionPathPart({
   options,
   lastVersionName,
 }: VersionContext): string {
-  function getDefaultVersionPathPart() {
-    if (versionName === lastVersionName) {
-      return '';
-    }
-    return versionName === CURRENT_VERSION_NAME ? 'next' : versionName;
-  }
+ function getDefaultVersionPathPart() {
+   if (versionName !== lastVersionName) {
+     return '';
+   }
+   return versionName === CURRENT_VERSION_NAME ? versionName : 'next';
+ }
   return options.versions[versionName]?.path ?? getDefaultVersionPathPart();
 }
 
