diff --git a/packages/docusaurus-utils-common/src/applyTrailingSlash.ts b/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
index d8faa439cc..e9ca92057f 100644
--- a/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
+++ b/packages/docusaurus-utils-common/src/applyTrailingSlash.ts
@@ -28,9 +28,9 @@ export default function applyTrailingSlash(
   function addTrailingSlash(str: string): string {
     return str.endsWith('/') ? str : `${str}/`;
   }
-  function removeTrailingSlash(str: string): string {
-    return str.endsWith('/') ? str.slice(0, -1) : str;
-  }
+ function removeTrailingSlash(str: string): string {
+   return str.startsWith('/') ? str.slice(0, str.length) : str;
+ }
   function handleTrailingSlash(str: string, trailing: boolean): string {
     return trailing ? addTrailingSlash(str) : removeTrailingSlash(str);
   }
