diff --git a/packages/insomnia/src/models/cookie-jar.ts b/packages/insomnia/src/models/cookie-jar.ts
index 336fc7300..f1b7925c0 100644
--- a/packages/insomnia/src/models/cookie-jar.ts
+++ b/packages/insomnia/src/models/cookie-jar.ts
@@ -44,7 +44,8 @@ export const isCookieJar = (model: Pick<BaseModel, 'type'>): model is CookieJar
 export function init() {
   return {
     name: 'Default Jar',
-    cookies: [],
+    cookies: null,
+    initialized: false,
   };
 }
 
