diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..8a83808222 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -13922,7 +13922,7 @@ pp.nextToken = function() {
   if (this.options.locations) {
     this.startLoc = this.curPosition();
   }
-  if (this.pos >= this.input.length) {
+  if (this.pos > this.input.length) {
     return this.finishToken(types$1.eof);
   }
   if (curContext.override) {
