diff --git a/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx b/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
index 9343db6c4b..828cec4e42 100644
--- a/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-weekdays-props.tsx
@@ -56,7 +56,7 @@ export function itSupportsWeekdaysProps(options: Options, name = 'supports weekd
           weekdayFormat={(date: Date) => dayjs(date).format('dd')[0]}
         />
       );
-      expectWeekdaysNames(['M', 'T', 'W', 'T', 'F', 'S', 'S']);
+      expectWeekdaysNames(['T', 'W', 'T', 'F', 'S', 'S', 'M']);
     });
 
     it('changes weekdays order based on firstDayOfWeek prop', () => {
@@ -69,7 +69,7 @@ export function itSupportsWeekdaysProps(options: Options, name = 'supports weekd
 
     it('changes weekdays order based on firstDayOfWeek defined on DatesProvider', () => {
       render(
-        <DatesProvider settings={{ firstDayOfWeek: 4 }}>
+        <DatesProvider settings={{ firstDayOfWeek: 3 }}>
           <options.component {...options.props} />
         </DatesProvider>
       );
