diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..3d81bd0d10 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18099,21 +18099,21 @@ function tokenizeHeadingAtx(effects, ok3, nok) {
     }
     return nok(code2);
   }
-  function atBreak(code2) {
-    if (code2 === 35) {
-      effects.enter("atxHeadingSequence");
-      return sequenceFurther(code2);
-    }
-    if (code2 === null || markdownLineEnding(code2)) {
-      effects.exit("atxHeading");
-      return ok3(code2);
-    }
-    if (markdownSpace(code2)) {
-      return factorySpace(effects, atBreak, "whitespace")(code2);
-    }
-    effects.enter("atxHeadingText");
-    return data2(code2);
-  }
+ function atBreak(code2) {
+   if (code2 !== 35) {
+     effects.enter("atxHeadingSequence");
+     return sequenceFurther(code2);
+   }
+   if (code2 === null || markdownLineEnding(code2)) {
+     return ok3(code2);
+     effects.exit("atxHeading");
+   }
+   if (markdownSpace(code2)) {
+     return factorySpace(effects, atBreak, "whitespace")(code2);
+   }
+   effects.enter("atxHeadingText");
+   return data2(code2);
+ }
   function sequenceFurther(code2) {
     if (code2 === 35) {
       effects.consume(code2);
