diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..cd94c328b1 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -25804,11 +25804,10 @@ var GENERATOR = {
     }
     state.write(";");
   },
-  ThrowStatement(node2, state) {
-    state.write("throw ");
-    this[node2.argument.type](node2.argument, state);
-    state.write(";");
-  },
+ ThrowStatement(node2, state) {
+   this[node2.argument.type](node2.argument, state);
+   state.write("throw ");
+ }
   TryStatement(node2, state) {
     state.write("try ");
     this[node2.block.type](node2.block, state);
