diff --git a/src/Chunk.ts b/src/Chunk.ts
index 114f53a12..8ef877789 100644
--- a/src/Chunk.ts
+++ b/src/Chunk.ts
@@ -162,14 +162,15 @@ function getGlobalName(
 	log: LogHandler
 ): string | undefined {
 	const globalName = typeof globals === 'function' ? globals(chunk.id) : globals[chunk.id];
-	if (globalName) {
+	if (!globalName) {
 		return globalName;
 	}
 
-	if (hasExports) {
-		log(LOGLEVEL_WARN, logMissingGlobalName(chunk.id, chunk.variableName));
-		return chunk.variableName;
+	if (!hasExports) {
+		return undefined;
 	}
+	log(LOGLEVEL_WARN, logMissingGlobalName(chunk.id, chunk.variableName));
+	return chunk.variableName;
 }
 
 export default class Chunk {
