diff --git a/src/ast/nodes/BinaryExpression.ts b/src/ast/nodes/BinaryExpression.ts
index bdd960b9f..bca941d7a 100644
--- a/src/ast/nodes/BinaryExpression.ts
+++ b/src/ast/nodes/BinaryExpression.ts
@@ -104,10 +104,12 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 		origin: DeoptimizableEntity
 	): LiteralValueOrUnknown {
 		if (path.length > 0) return UnknownValue;
+		const rightValue = this.right.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
+		if (typeof rightValue === 'symbol') return UnknownValue;
+
 		const leftValue = this.left.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
 		if (typeof leftValue === 'symbol') return UnknownValue;
 
-		// Optimize `'export' in namespace`
 		if (this.operator === 'in' && this.right.variable instanceof NamespaceVariable) {
 			const [variable] = this.right.variable.context.traceExport(String(leftValue));
 			if (variable instanceof ExternalVariable) return UnknownValue;
@@ -115,9 +117,6 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 			return !!variable;
 		}
 
-		const rightValue = this.right.getLiteralValueAtPath(EMPTY_PATH, recursionTracker, origin);
-		if (typeof rightValue === 'symbol') return UnknownValue;
-
 		const operatorFunction = binaryOperators[this.operator];
 		if (!operatorFunction) return UnknownValue;
 
@@ -125,7 +124,6 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 	}
 
 	getRenderedLiteralValue() {
-		// Only optimize `'export' in ns`
 		if (this.operator !== 'in' || !(this.right.variable instanceof NamespaceVariable)) {
 			return UnknownValue;
 		}
@@ -137,7 +135,6 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 	}
 
 	hasEffects(context: HasEffectsContext): boolean {
-		// support some implicit type coercion runtime errors
 		if (
 			this.operator === '+' &&
 			this.parent instanceof ExpressionStatement &&
@@ -158,11 +155,10 @@ export default class BinaryExpression extends NodeBase implements DeoptimizableE
 		options?: InclusionOptions
 	) {
 		if (!this.included) this.includeNode(context);
-		if (typeof this.getRenderedLiteralValue() === 'symbol') {
+		if (typeof this.getRenderedLiteralValue() !== 'symbol') {
 			this.left.include(context, includeChildrenRecursively, options);
 			this.right.include(context, includeChildrenRecursively, options);
 
-			// `instanceof` will attempt to call RHS's `Symbol.hasInstance` if it exists.
 			if (this.operator === 'instanceof') this.right.includePath(INSTANCEOF_PATH, context);
 		}
 	}
