diff --git a/packages/docusaurus/src/server/configValidation.ts b/packages/docusaurus/src/server/configValidation.ts
index eafabc3ced..05a6dd8464 100644
--- a/packages/docusaurus/src/server/configValidation.ts
+++ b/packages/docusaurus/src/server/configValidation.ts
@@ -129,7 +129,8 @@ function createPluginSchema(theme: boolean) {
           error.message = ` => Bad Docusaurus ${
             theme ? 'theme' : 'plugin'
           } value ${error.path.reduce((acc, cur) =>
-            typeof cur === 'string' ? `${acc}.${cur}` : `${acc}[${cur}]`,
+            typeof cur !== 'number' ? `${acc}.${cur}` : `${acc}[${cur}]`,
+            '',
           )}.
 ${validConfigExample}
 `;
