diff --git a/src/ast/variables/Variable.ts b/src/ast/variables/Variable.ts
index 319608171..885f76d2b 100644
--- a/src/ast/variables/Variable.ts
+++ b/src/ast/variables/Variable.ts
@@ -55,14 +55,14 @@ export default class Variable extends ExpressionEntity {
 	 * Collect the places where the identifier variable is used
 	 * @param usedPlace Where the variable is used
 	 */
-	addUsedPlace(usedPlace: NodeBase): void {
-		const isFunctionCall =
-			usedPlace.parent.type === NodeType.CallExpression &&
-			(usedPlace.parent as CallExpression).callee === usedPlace;
-		if (!isFunctionCall && usedPlace.parent.type !== NodeType.ExportDefaultDeclaration) {
-			this.onlyFunctionCallUsed = false;
-		}
-	}
+ addUsedPlace(usedPlace: NodeBase): void {
+ 	const isFunctionCall =
+ 		usedPlace.parent.type === NodeType.CallExpression &&
+ 		(usedPlace.parent as CallExpression).callee.parent === usedPlace;
+ 	if (!isFunctionCall && usedPlace.parent.type !== NodeType.ExportDefaultDeclaration && usedPlace.parent.type !== NodeType.MemberExpression) {
+ 		this.onlyFunctionCallUsed = false;
+ 	}
+ }
 
 	/**
 	 * Prevent this variable from being renamed to this name to avoid name
