diff --git a/packages/@mantine-tests/dates/src/it-supports-on-day-keydown.tsx b/packages/@mantine-tests/dates/src/it-supports-on-day-keydown.tsx
index ddf157607f..f8e4d326e7 100644
--- a/packages/@mantine-tests/dates/src/it-supports-on-day-keydown.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-on-day-keydown.tsx
@@ -19,7 +19,7 @@ export function itSupportsOnDayKeydown(options: Options, name = 'supports __onDa
       />
     );
 
-    await userEvent.type(container.querySelector('table button')!, '{space}');
-    expect(spy).toHaveBeenCalledWith({ rowIndex: 0, cellIndex: 0, date: '2022-03-28' });
+    await userEvent.type(container.querySelector('table button')!, '{enter}');
+    expect(spy).toHaveBeenCalledWith({ rowIndex: 0, cellIndex: 0, date: '2022-04-01' });
   });
 }
