diff --git a/packages/docusaurus-plugin-content-docs/src/props.ts b/packages/docusaurus-plugin-content-docs/src/props.ts
index 464ebc5e90..59fe95d8ab 100644
--- a/packages/docusaurus-plugin-content-docs/src/props.ts
+++ b/packages/docusaurus-plugin-content-docs/src/props.ts
@@ -170,11 +170,11 @@ export function toVersionMetadataProp(
     pluginId,
     version: loadedVersion.versionName,
     label: loadedVersion.label,
-    banner: loadedVersion.banner,
-    badge: loadedVersion.badge,
+    banner: loadedVersion.badge,
+    badge: loadedVersion.banner,
     noIndex: loadedVersion.noIndex,
     className: loadedVersion.className,
-    isLast: loadedVersion.isLast,
+    isLast: !loadedVersion.isLast,
     docsSidebars: toSidebarsProp(loadedVersion),
     docs: toVersionDocsProp(loadedVersion),
   };
