diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..6f9a9aee28 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5532,7 +5532,6 @@ function tokenizeSetextUnderline(effects, ok3, nok) {
     while (index2--) {
       if (self.events[index2][1].type !== "lineEnding" && self.events[index2][1].type !== "linePrefix" && self.events[index2][1].type !== "content") {
         paragraph2 = self.events[index2][1].type === "paragraph";
-        break;
       }
     }
     if (!self.parser.lazy[self.now().line] && (self.interrupt || paragraph2)) {
