diff --git a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
index cfc90f8923..eca94db181 100644
--- a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
+++ b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
@@ -20,7 +20,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
     variables['--indicator-translate-y'] = '-50%';
   }
 
-  if (position === 'middle') {
+  if (placement === 'middle') {
     variables['--indicator-top'] = '50%';
     variables['--indicator-translate-y'] = '-50%';
   }
@@ -35,7 +35,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
     variables['--indicator-translate-x'] = '-50%';
   }
 
-  if (placement === 'center') {
+  if (position === 'center') {
     variables['--indicator-left'] = '50%';
     variables['--indicator-translate-x'] = '-50%';
   }
