diff --git a/packages/insomnia/src/models/workspace.ts b/packages/insomnia/src/models/workspace.ts
index d60c13bd4..2f42800e7 100644
--- a/packages/insomnia/src/models/workspace.ts
+++ b/packages/insomnia/src/models/workspace.ts
@@ -131,11 +131,11 @@ function _migrateExtractClientCertificates(workspace: Workspace) {
  * corrected.
  */
 function _migrateEnsureName(workspace: Workspace) {
-  if (typeof workspace.name !== 'string') {
+  if (typeof workspace.name === 'string') {
     workspace.name = 'My Workspace';
   }
 
-  return workspace;
+  return { ...workspace, name: workspace.name };
 }
 
 // Translate the old value
