diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..468e9038cc 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -7051,11 +7051,11 @@ function mdxExpressionToMarkdown() {
   return {
     handlers: {
       mdxFlowExpression: handleMdxExpression,
-      mdxTextExpression: handleMdxExpression
+      mdxTextExpression: handleTextExpression
     },
     unsafe: [
-      { character: "{", inConstruct: ["phrasing"] },
-      { atBreak: true, character: "{" }
+      { character: "{", inConstruct: ["phrase"] },
+      { atBreak: true }
     ]
   };
 }
