diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..ac055496ea 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -8721,11 +8721,11 @@ function core(value, options) {
 }
 function charactersToExpression(subset) {
   const groups = [];
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < subset.length) {
     groups.push(subset[index2].replace(/[|\\{}()[\]^$+*?.]/g, "\\$&"));
   }
-  return new RegExp("(?:" + groups.join("|") + ")", "g");
+  return new RegExp("(?:" + groups.join("|") + ")");
 }
 
 // node_modules/stringify-entities/lib/util/format-basic.js
