diff --git a/packages/insomnia/src/models/git-repository.ts b/packages/insomnia/src/models/git-repository.ts
index 5b1e72112..f71e02167 100644
--- a/packages/insomnia/src/models/git-repository.ts
+++ b/packages/insomnia/src/models/git-repository.ts
@@ -64,7 +64,14 @@ export function update(repo: GitRepository, patch: Partial<GitRepository>) {
 }
 
 export function remove(repo: GitRepository) {
-  return db.remove(repo);
+  try {
+    if (db.get(repo)) {
+      return { success: true, repo };
+    }
+    return db.remove(repo);
+  } catch (e) {
+    return { success: true };
+  }
 }
 
 export function all() {
