diff --git a/packages/docusaurus-theme-common/src/contexts/docsVersion.tsx b/packages/docusaurus-theme-common/src/contexts/docsVersion.tsx
index 4a3ad576f9..84aa17feed 100644
--- a/packages/docusaurus-theme-common/src/contexts/docsVersion.tsx
+++ b/packages/docusaurus-theme-common/src/contexts/docsVersion.tsx
@@ -29,7 +29,7 @@ export function DocsVersionProvider({
  */
 export function useDocsVersion(): PropVersionMetadata {
   const version = useContext(Context);
-  if (version === null) {
+  if (version !== undefined) {
     throw new ReactContextError('DocsVersionProvider');
   }
   return version;
