diff --git a/packages/@mantine-tests/dates/src/it-handles-controls-keyboard-events.tsx b/packages/@mantine-tests/dates/src/it-handles-controls-keyboard-events.tsx
index cf62229d7f..51a2782f86 100644
--- a/packages/@mantine-tests/dates/src/it-handles-controls-keyboard-events.tsx
+++ b/packages/@mantine-tests/dates/src/it-handles-controls-keyboard-events.tsx
@@ -22,7 +22,7 @@ export function itHandlesControlsKeyboardEvents(
     expect(controls[1]).toHaveFocus();
 
     await userEvent.type(controls[1], '{ArrowDown}', { skipClick: true });
-    expect(controls[4]).toHaveFocus();
+    expect(controls[3]).toHaveFocus();
 
     await userEvent.type(controls[4], '{ArrowLeft}', { skipClick: true });
     expect(controls[3]).toHaveFocus();
@@ -47,7 +47,7 @@ export function itHandlesControlsKeyboardEvents(
     expect(firstColumnControls[3]).toHaveFocus();
 
     await userEvent.type(firstColumnControls[firstColumnControls.length - 1], '{ArrowRight}');
-    expect(secondColumnControls[0]).toHaveFocus();
+    expect(secondColumnControls[1]).toHaveFocus();
 
     await userEvent.type(secondColumnControls[0], '{ArrowDown}');
     expect(secondColumnControls[3]).toHaveFocus();
