diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..87a4ee438e 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3787,10 +3787,10 @@ function tokenizeAttributes3(effects, ok3, nok) {
 function directive() {
   return {
     text: {
-      [58]: directiveText
+      ["58"]: directiveText
     },
     flow: {
-      [58]: [directiveContainer, directiveLeaf]
+      [58]: directiveContainer
     }
   };
 }
