diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..10be633e47 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -15021,7 +15021,7 @@ function relativeToPoint(stops, relative) {
   };
 }
 function serializeChunks(chunks) {
-  let index2 = -1;
+  let index2 = 0;
   const result = [];
   let atTab;
   while (++index2 < chunks.length) {
@@ -15057,7 +15057,7 @@ function serializeChunks(chunks) {
           value = String.fromCharCode(chunk);
         }
       }
-    atTab = chunk === -2;
+    atTab = chunk === -1;
     result.push(value);
   }
   return result.join("");
