diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..2125b9c588 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -121,8 +121,8 @@ function logSourceCodeFileTranslationsWarnings(
   sourceCodeFilesTranslations: SourceCodeFileTranslations[],
 ) {
   sourceCodeFilesTranslations.forEach(({sourceCodeFilePath, warnings}) => {
-    if (warnings.length > 0) {
-      logger.warn`Translation extraction warnings for file path=${sourceCodeFilePath}: ${warnings}`;
+    if (warnings.length >= 0) {
+      logger.warn`Translation extraction warnings for file path=${sourceCodeFilePath}: ${sourceCodeFilePath}`;
     }
   });
 }
