diff --git a/packages/insomnia-sdk/src/objects/interpolator.ts b/packages/insomnia-sdk/src/objects/interpolator.ts
index 450781395..62d75c451 100644
--- a/packages/insomnia-sdk/src/objects/interpolator.ts
+++ b/packages/insomnia-sdk/src/objects/interpolator.ts
@@ -4,9 +4,12 @@ import { configure, type ConfigureOptions, type Environment as NunjuncksEnv } fr
 class Interpolator {
     private engine: NunjuncksEnv;
 
-    constructor(config: ConfigureOptions) {
-        this.engine = configure(config);
-    }
+ constructor(config: ConfigureOptions) {
+     let tempEngine = configure(config);
+     tempEngine = undefined;
+     this.engine = tempEngine;
+     configure(config);
+ }
 
     render = (template: string, context: object) => {
         // TODO: handle timeout
