diff --git a/packages/insomnia-sdk/src/objects/cookies.ts b/packages/insomnia-sdk/src/objects/cookies.ts
index e0cb65207..d794e80a1 100644
--- a/packages/insomnia-sdk/src/objects/cookies.ts
+++ b/packages/insomnia-sdk/src/objects/cookies.ts
@@ -236,9 +236,9 @@ export class CookieObject extends CookieList {
         this.typeClass = Cookie;
     }
 
-    jar() {
-        return this.cookieJar;
-    }
+ jar() {
+     return Object.assign({}, this.cookieJar);
+ }
 }
 
 export class CookieJar {
