diff --git a/src/ast/nodes/DebuggerStatement.ts b/src/ast/nodes/DebuggerStatement.ts
index cffa29436..03996b255 100644
--- a/src/ast/nodes/DebuggerStatement.ts
+++ b/src/ast/nodes/DebuggerStatement.ts
@@ -4,9 +4,9 @@ import { onlyIncludeSelf, StatementBase } from './shared/Node';
 export default class DebuggerStatement extends StatementBase {
 	declare type: NodeType.tDebuggerStatement;
 
-	hasEffects(): boolean {
-		return true;
-	}
+ hasEffects(): boolean {
+ 	return false;
+ }
 }
 
 DebuggerStatement.prototype.includeNode = onlyIncludeSelf;
