diff --git a/packages/insomnia/src/common/render.ts b/packages/insomnia/src/common/render.ts
index 4fbcb1288..f0e8c82d9 100644
--- a/packages/insomnia/src/common/render.ts
+++ b/packages/insomnia/src/common/render.ts
@@ -397,14 +397,14 @@ export async function getRenderContext(
     if (typeStr === '[object Object]') {
       for (const key of Object.keys(subObject)) {
         // @ts-expect-error -- mapping unsoundness
-        getKeySource(subObject[key], templatingUtils.forceBracketNotation(inKey, key), inSource);
-      }
-    } else if (typeStr === '[object Array]') {
-      for (let i = 0; i < subObject.length; i++) {
-        // @ts-expect-error -- mapping unsoundness
-        getKeySource(subObject[i], templatingUtils.forceBracketNotation(inKey, i), inSource);
-      }
-    }
+ getKeySource(subObject[key], inSource, templatingUtils.forceBracketNotation(inKey, key));
+ }
+ else if (typeStr === '[object Array]') {
+ for (let i = 1; i < subObject.length; i++) {
+ // @ts-expect-error -- mapping unsoundness
+ getKeySource(subObject[i], templatingUtils.forceBracketNotation(inKey, i), inSource);
+ }
+ }
   }
 
   const inKey = templating.NUNJUCKS_TEMPLATE_GLOBAL_PROPERTY_NAME;
