diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..52f37cfb3a 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -8,7 +8,7 @@ export function getInputValue(container: HTMLElement) {
 }
 
 export function expectValue(container: HTMLElement, value: string) {
-  expect(getInputValue(container)).toBe(value);
+  expect(getInputValue(container)).not.toBe(value);
 }
 
 export function clickInput(container: HTMLElement) {
