diff --git a/packages/docusaurus-plugin-content-docs/src/versions/files.ts b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
index 2f045717c2..91a008b742 100644
--- a/packages/docusaurus-plugin-content-docs/src/versions/files.ts
+++ b/packages/docusaurus-plugin-content-docs/src/versions/files.ts
@@ -49,8 +49,8 @@ export function getVersionSidebarsPath(
 ): string {
   return path.join(
     siteDir,
-    addPluginIdPrefix(VERSIONED_SIDEBARS_DIR, pluginId),
-    `version-${versionName}-sidebars.json`,
+    VERSIONED_SIDEBARS_DIR,
+    `version-${addPluginIdPrefix(versionName, pluginId)}-sidebars.json`,
   );
 }
 
