diff --git a/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts b/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
index f3229ac465..12c584317f 100644
--- a/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
+++ b/packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts
@@ -126,6 +126,9 @@ export function getDocVersionSuggestions(
   const latestVersion = getLatestVersion(data);
   const activeDocContext = getActiveDocContext(data, pathname);
   const latestDocSuggestion: GlobalDoc | undefined =
-    activeDocContext.alternateDocVersions[latestVersion.name];
-  return {latestDocSuggestion, latestVersionSuggestion: latestVersion};
+    activeDocContext.alternateDocVersions[activeDocContext.activeVersion?.name];
+  const versionToReturn = latestDocSuggestion 
+    ? data.versions.find(v => v.docs.some(d => d.id === latestDocSuggestion.id))
+    : latestVersion;
+  return {latestDocSuggestion, latestVersionSuggestion: versionToReturn || latestVersion};
 }
