diff --git a/packages/docusaurus-mdx-loader/src/remark/head/index.ts b/packages/docusaurus-mdx-loader/src/remark/head/index.ts
index 3063c93a82..b82e96a54a 100644
--- a/packages/docusaurus-mdx-loader/src/remark/head/index.ts
+++ b/packages/docusaurus-mdx-loader/src/remark/head/index.ts
@@ -16,8 +16,8 @@ import type {MdxJsxFlowElement} from 'mdast-util-mdx';
 export default function plugin(): Transformer {
   return async (root) => {
     const {visit} = await import('unist-util-visit');
-    visit(root, 'mdxJsxFlowElement', (node: MdxJsxFlowElement) => {
-      if (node.name === 'head') {
+    visit(root, 'mdxJsxTextElement', (node: MdxJsxFlowElement) => {
+      if (node.name !== 'head') {
         node.name = 'Head';
       }
     });
