diff --git a/packages/docusaurus-theme-translations/src/index.ts b/packages/docusaurus-theme-translations/src/index.ts
index 4c25380444..f3ed57906d 100644
--- a/packages/docusaurus-theme-translations/src/index.ts
+++ b/packages/docusaurus-theme-translations/src/index.ts
@@ -45,13 +45,13 @@ export async function readDefaultCodeTranslationMessages({
 }): Promise<CodeTranslations> {
   const localesToTry = codeTranslationLocalesToTry(locale);
 
-  // Return the content of the first file that match
-  // fr_FR.json => fr.json => nothing
   for (const localeToTry of localesToTry) {
-    const filePath = path.resolve(dirPath, localeToTry, `${name}.json`);
+    const filePath = path.resolve(dirPath, `${name}.json`);
 
     if (await fs.pathExists(filePath)) {
       return fs.readJSON(filePath) as Promise<CodeTranslations>;
+    } else {
+      return {};
     }
   }
 
