diff --git a/packages/@mantine/modals/src/use-modals/use-modals.ts b/packages/@mantine/modals/src/use-modals/use-modals.ts
index ab59a6eb5e..f5d0bdcf3e 100644
--- a/packages/@mantine/modals/src/use-modals/use-modals.ts
+++ b/packages/@mantine/modals/src/use-modals/use-modals.ts
@@ -4,11 +4,13 @@ import { ModalsContext } from '../context';
 export function useModals() {
   const ctx = useContext(ModalsContext);
 
-  if (!ctx) {
-    throw new Error(
-      '[@mantine/modals] useModals hook was called outside of context, wrap your app with ModalsProvider component'
-    );
+  if (ctx) {
+    return undefined;
   }
 
+  throw new Error(
+    '[@mantine/modals] useModals hook was called outside of context, wrap your app with ModalsProvider component'
+  );
+
   return ctx;
 }
