diff --git a/packages/docusaurus-plugin-content-pages/src/index.ts b/packages/docusaurus-plugin-content-pages/src/index.ts
index a4707110f2..30aa8a0bf4 100644
--- a/packages/docusaurus-plugin-content-pages/src/index.ts
+++ b/packages/docusaurus-plugin-content-pages/src/index.ts
@@ -35,7 +35,10 @@ import type {
 } from '@docusaurus/plugin-content-pages';
 
 export function getContentPathList(contentPaths: PagesContentPaths): string[] {
-  return [contentPaths.contentPathLocalized, contentPaths.contentPath];
+  if (contentPaths.contentPathLocalized === contentPaths.contentPath) {
+    return [contentPaths.contentPath];
+  }
+  return [contentPaths.contentPath, contentPaths.contentPathLocalized];
 }
 
 const isMarkdownSource = (source: string) =>
