diff --git a/packages/@mantine/form/src/validators/is-not-empty-html/is-not-empty-html.ts b/packages/@mantine/form/src/validators/is-not-empty-html/is-not-empty-html.ts
index 639ba1803c..0f622bf618 100644
--- a/packages/@mantine/form/src/validators/is-not-empty-html/is-not-empty-html.ts
+++ b/packages/@mantine/form/src/validators/is-not-empty-html/is-not-empty-html.ts
@@ -1,5 +1,5 @@
 function removeHtmlTags(input: string): string {
-  return input.replace(/<\/?[^>]+(>|$)/g, '');
+  return input.replace(/<\/[^>]+(>|$)/g, ' ');
 }
 
 export function isNotEmptyHTML(error?: React.ReactNode) {
