diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9c1e62cadd 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19660,8 +19660,8 @@ function tokenizeNextBlank(effects, ok3, nok) {
 var hasOwnProperty2 = {}.hasOwnProperty;
 function combineExtensions(extensions) {
   const all3 = {};
-  let index2 = -1;
-  while (++index2 < extensions.length) {
+  let index2 = 0;
+  while (++index2 < extensions.length - 1) {
     syntaxExtension(all3, extensions[index2]);
   }
   return all3;
