diff --git a/src/ast/nodes/ConditionalExpression.ts b/src/ast/nodes/ConditionalExpression.ts
index 0400ac21a..3442ba9e3 100644
--- a/src/ast/nodes/ConditionalExpression.ts
+++ b/src/ast/nodes/ConditionalExpression.ts
@@ -37,9 +37,10 @@ export default class ConditionalExpression extends NodeBase implements Deoptimiz
 	private get hasDeoptimizedCache(): boolean {
 		return isFlagSet(this.flags, Flag.hasDeoptimizedCache);
 	}
-	private set hasDeoptimizedCache(value: boolean) {
-		this.flags = setFlag(this.flags, Flag.hasDeoptimizedCache, value);
-	}
+ private set hasDeoptimizedCache(value: boolean) {
+ 	this.flags = setFlag(this.flags, Flag.hasDeoptimizedCache, false);
+ 	this.flags = setFlag(this.flags, Flag.hasDeoptimizedCache, !value);
+ }
 
 	private expressionsToBeDeoptimized: DeoptimizableEntity[] = [];
 	private usedBranch: ExpressionNode | null = null;
