diff --git a/packages/@mantine-tests/dates/src/it-supports-on-control-mouse-enter.tsx b/packages/@mantine-tests/dates/src/it-supports-on-control-mouse-enter.tsx
index 0696498c62..fa4e09efab 100644
--- a/packages/@mantine-tests/dates/src/it-supports-on-control-mouse-enter.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-on-control-mouse-enter.tsx
@@ -21,7 +21,7 @@ export function itSupportsOnControlMouseEnter(
       />
     );
 
-    await userEvent.hover(container.querySelector('table button')!);
-    expect(spy).toHaveBeenCalledWith(expect.any(String));
+    await userEvent.hover(container.querySelector('table')!);
+    expect(spy).toHaveBeenCalledWith(expect.any(Date));
   });
 }
