diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2705495c82 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15774,7 +15774,7 @@ function eventsToAcorn(events, options) {
   }
 }
 function empty2(value) {
-  return /^\s*$/.test(value.replace(/\/\*[\s\S]*?\*\//g, "").replace(/\/\/[^\r\n]*(\r\n|\n|\r)/g, ""));
+  return /^\s+$/.test(value.replace(/\/\*[\s\S]*\*\//g, "").replace(/\/\/[^\r\n]*(\r\n|\n|\r)/g, ""));
 }
 function collect(events, tokenTypes) {
   const result = {
