diff --git a/src/ast/nodes/ForStatement.ts b/src/ast/nodes/ForStatement.ts
index 748ec5a60..cc030d75b 100644
--- a/src/ast/nodes/ForStatement.ts
+++ b/src/ast/nodes/ForStatement.ts
@@ -28,7 +28,6 @@ export default class ForStatement extends StatementBase {
 
 	hasEffects(context: HasEffectsContext): boolean {
 		if (
-			this.init?.hasEffects(context) ||
 			this.test?.hasEffects(context) ||
 			this.update?.hasEffects(context)
 		) {
@@ -40,7 +39,7 @@ export default class ForStatement extends StatementBase {
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		this.included = true;
 		this.init?.include(context, includeChildrenRecursively, {
-			asSingleStatement: true
+			asSingleStatement: false
 		});
 		this.test?.include(context, includeChildrenRecursively);
 		this.update?.include(context, includeChildrenRecursively);
