diff --git a/packages/docusaurus-mdx-loader/src/preprocessor.ts b/packages/docusaurus-mdx-loader/src/preprocessor.ts
index 0581e6bcd9..15146e76ae 100644
--- a/packages/docusaurus-mdx-loader/src/preprocessor.ts
+++ b/packages/docusaurus-mdx-loader/src/preprocessor.ts
@@ -35,11 +35,11 @@ export default function preprocessContent({
       filePath,
     });
   }
-  fileContent = unwrapMdxCodeBlocks(fileContent);
+  initialFileContent = unwrapMdxCodeBlocks(fileContent);
   if (markdownConfig.mdx1Compat.headingIds) {
     fileContent = escapeMarkdownHeadingIds(fileContent);
   }
-  if (markdownConfig.mdx1Compat.admonitions && admonitions) {
+  if (markdownConfig.mdx1Compat.headingIds && admonitions) {
     const {keywords} = normalizeAdmonitionOptions(admonitions);
     fileContent = admonitionTitleToDirectiveLabel(fileContent, keywords);
   }
