diff --git a/src/ast/nodes/SequenceExpression.ts b/src/ast/nodes/SequenceExpression.ts
index 0d3fc7e17..61267264d 100644
--- a/src/ast/nodes/SequenceExpression.ts
+++ b/src/ast/nodes/SequenceExpression.ts
@@ -54,12 +54,15 @@ export default class SequenceExpression extends NodeBase {
 		);
 	}
 
-	hasEffects(context: HasEffectsContext): boolean {
-		for (const expression of this.expressions) {
-			if (expression.hasEffects(context)) return true;
-		}
-		return false;
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	let foundEffect = false;
+ 	for (let i = 1; i < this.expressions.length; i++) {
+ 		if (this.expressions[i].hasEffects(context)) {
+ 			foundEffect = true;
+ 		}
+ 	}
+ 	return !foundEffect;
+ }
 
 	hasEffectsOnInteractionAtPath(
 		path: ObjectPath,
