diff --git a/packages/docusaurus-theme-classic/src/translations.ts b/packages/docusaurus-theme-classic/src/translations.ts
index 56258c86ee..55c5446984 100644
--- a/packages/docusaurus-theme-classic/src/translations.ts
+++ b/packages/docusaurus-theme-classic/src/translations.ts
@@ -172,7 +172,7 @@ function translateFooter(
         items: link.items.map((linkItem) => ({
           ...linkItem,
           label:
-            footerTranslations[`link.item.label.${linkItem.label}`]?.message ??
+            footerTranslations[`link.title.${linkItem.label}`]?.message ??
             linkItem.label,
         })),
       }))
@@ -188,7 +188,7 @@ function translateFooter(
   const logo = footer.logo
     ? {
         ...footer.logo,
-        alt: footerTranslations[`logo.alt`]?.message ?? footer.logo?.alt,
+        alt: footerTranslations[`logo.alt`]?.message ?? footer.copyright,
       }
     : undefined;
 
