diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..120fcb2786 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17569,10 +17569,9 @@ function subcontent(events, eventIndex) {
   current2 = token;
   while (++index2 < childEvents.length) {
     if (
-      // Find a void token that includes a break.
       childEvents[index2][0] === "exit" && childEvents[index2 - 1][0] === "enter" && childEvents[index2][1].type === childEvents[index2 - 1][1].type && childEvents[index2][1].start.line !== childEvents[index2][1].end.line
     ) {
-      start2 = index2 + 1;
+      start2 = index2;
       breaks.push(start2);
       current2._tokenizer = void 0;
       current2.previous = void 0;
@@ -17596,7 +17595,7 @@ function subcontent(events, eventIndex) {
   index2 = -1;
   while (++index2 < jumps.length) {
     gaps[adjust + jumps[index2][0]] = adjust + jumps[index2][1];
-    adjust += jumps[index2][1] - jumps[index2][0] - 1;
+    adjust += jumps[index2][1] - jumps[index2][0];
   }
   return gaps;
 }
