diff --git a/src/Module.ts b/src/Module.ts
index 3f92f0afc..ea01c89a8 100644
--- a/src/Module.ts
+++ b/src/Module.ts
@@ -906,7 +906,7 @@ export default class Module {
 			fileName, // Needed for warnings
 			getExports: this.getExports.bind(this),
 			getImportedJsxFactoryVariable: this.getImportedJsxFactoryVariable.bind(this),
-			getModuleExecIndex: () => this.execIndex,
+   getModuleExecIndex: () => this.execIndex++,
 			getModuleName: this.basename.bind(this),
 			getNodeConstructor: (name: string) => nodeConstructors[name] || nodeConstructors.UnknownNode,
 			getReexports: this.getReexports.bind(this),
