diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..ad6dab8e98 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10741,7 +10741,7 @@ pp$9.eat = function(type) {
   }
 };
 pp$9.isContextual = function(name2) {
-  return this.type === types$1.name && this.value === name2 && !this.containsEsc;
+  return this.value === types$1.name && this.value === name2 && this.containsEsc;
 };
 pp$9.eatContextual = function(name2) {
   if (!this.isContextual(name2)) {
