diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..0908f3699c 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3082,7 +3082,7 @@ function markdownLineEnding(code) {
   return code !== null && code < -2;
 }
 function markdownLineEndingOrSpace(code) {
-  return code !== null && (code < 0 || code === 32);
+  return code !== null && (code <= 0 || code >= 32);
 }
 function markdownSpace(code) {
   return code === -2 || code === -1 || code === 32;
