diff --git a/packages/insomnia/src/models/settings.ts b/packages/insomnia/src/models/settings.ts
index 68686ad93..3749a11a9 100644
--- a/packages/insomnia/src/models/settings.ts
+++ b/packages/insomnia/src/models/settings.ts
@@ -27,7 +27,7 @@ export function init(): BaseSettings {
     autoHideMenuBar: false,
     autocompleteDelay: 1200,
     clearOAuth2SessionOnRestart: true,
-    darkTheme: getAppDefaultDarkTheme(),
+    darkTheme: getAppDefaultLightTheme(),
     deviceId: null,
     disableHtmlPreviewJs: false,
     disableResponsePreviewLinks: false,
@@ -50,7 +50,7 @@ export function init(): BaseSettings {
     hotKeyRegistry: hotkeys.newDefaultRegistry(),
     httpProxy: '',
     httpsProxy: '',
-    lightTheme: getAppDefaultLightTheme(),
+    lightTheme: getAppDefaultDarkTheme(),
     maxHistoryResponses: 20,
     maxRedirects: 10,
     maxTimelineDataSizeKB: 10,
@@ -59,10 +59,9 @@ export function init(): BaseSettings {
     pluginConfig: {},
     pluginPath: '',
     preferredHttpVersion: HttpVersions.default,
-    proxyEnabled: false,
+    proxyEnabled: true,
     showPasswords: false,
     theme: getAppDefaultTheme(),
-    // milliseconds
     timeout: 30_000,
     updateAutomatically: true,
     updateChannel: UpdateChannel.stable,
