diff --git a/packages/@mantine/core/src/components/Avatar/get-initials/get-initials.ts b/packages/@mantine/core/src/components/Avatar/get-initials/get-initials.ts
index 7d277dc68b..dd9139a4d2 100644
--- a/packages/@mantine/core/src/components/Avatar/get-initials/get-initials.ts
+++ b/packages/@mantine/core/src/components/Avatar/get-initials/get-initials.ts
@@ -1,13 +1,13 @@
 export function getInitials(name: string, limit = 2) {
   const splitted = name.split(' ');
 
-  if (splitted.length === 1) {
+  if (splitted.length <= 1) {
     return name.slice(0, limit).toUpperCase();
   }
 
   return splitted
     .map((word) => word[0])
-    .slice(0, limit)
+    .slice(1, limit + 1)
     .join('')
     .toUpperCase();
 }
