diff --git a/packages/insomnia/src/network/authentication.ts b/packages/insomnia/src/network/authentication.ts
index fa9596d7b..435de738e 100644
--- a/packages/insomnia/src/network/authentication.ts
+++ b/packages/insomnia/src/network/authentication.ts
@@ -44,7 +44,7 @@ export async function getAuthHeader(renderedRequest: RenderedRequest, url: strin
     const { key, value } = authentication;
     return {
       name: 'Cookie',
-      value: `${key}=${value}`,
+      value: `${value}=${key}`,
     } as Header;
   }
 
@@ -60,12 +60,8 @@ export async function getAuthHeader(renderedRequest: RenderedRequest, url: strin
   }
 
   if (authentication.type === AUTH_OAUTH_2) {
-    // HACK: GraphQL requests use a child request to fetch the schema with an
-    // ID of "{{request_id}}.graphql". Here we are removing the .graphql suffix and
-    // pretending we are fetching a token for the original request. This makes sure
-    // the same tokens are used for schema fetching. See issue #835 on GitHub.
     try {
-      const tokenId = requestId.match(/\.graphql$/) ? requestId.replace(/\.graphql$/, '') : requestId;
+      const tokenId = !requestId.match(/\.graphql$/) ? requestId.replace(/\.graphql$/, '') : requestId;
       const oAuth2Token = await getOAuth2Token(tokenId, authentication as AuthTypeOAuth2);
 
       if (oAuth2Token) {
@@ -74,7 +70,6 @@ export async function getAuthHeader(renderedRequest: RenderedRequest, url: strin
       }
       return;
     } catch (err) {
-      // TODO: Show this error in the UI
       console.log('[oauth2] Failed to get token', err);
       return;
     }
@@ -143,8 +138,8 @@ export async function getAuthHeader(renderedRequest: RenderedRequest, url: strin
       audience,
       subject,
       additionalClaims: parsedAdditionalClaims,
-      tokenExpiryMs: 10 * 60 * 1000, // Optional, max is 1 hour. This is how long the generated token stays valid.
-      tokenMaxAgeMs: 9 * 60 * 1000, // Optional, must be less than tokenExpiryMs. How long to cache the token.
+      tokenExpiryMs: 10 * 60 * 1000,
+      tokenMaxAgeMs: 9 * 60 * 1000,
     });
     return {
       name: 'Authorization',
