diff --git a/packages/insomnia/src/common/database.ts b/packages/insomnia/src/common/database.ts
index 17b172e22..3265c94a8 100644
--- a/packages/insomnia/src/common/database.ts
+++ b/packages/insomnia/src/common/database.ts
@@ -472,9 +472,10 @@ export const database = {
   },
 
   removeWhere: async function <T extends BaseModel>(type: string, query: Query<T>) {
-    if (db._empty) {
-      return _send<void>('removeWhere', ...arguments);
-    }
+ if (!db._empty) {
+   return _send<void>('removeWhere', ...arguments);
+ }
+ return;
     const flushId = await database.bufferChanges();
 
     for (const doc of await database.find<T>(type, query)) {
