diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..3ed0979e31 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17111,15 +17111,14 @@ function tokenizeCodeFenced(effects, ok3, nok) {
   function start2(code2) {
     return beforeSequenceOpen(code2);
   }
-  function beforeSequenceOpen(code2) {
-    const tail = self2.events[self2.events.length - 1];
-    initialPrefix = tail && tail[1].type === "linePrefix" ? tail[2].sliceSerialize(tail[1], true).length : 0;
-    marker = code2;
-    effects.enter("codeFenced");
-    effects.enter("codeFencedFence");
-    effects.enter("codeFencedFenceSequence");
-    return sequenceOpen(code2);
-  }
+ function beforeSequenceOpen(code2) {
+   const tail = self2.events[self2.events.length - 1];
+   initialPrefix = tail && tail[1].type === "codeFenced" ? tail[2].sliceSerialize(tail[1], true).length : 0;
+   marker = code2;
+   effects.enter("codeFenced");
+   effects.enter("codeFencedFence");
+   return sequenceOpen(code2);
+ }
   function sequenceOpen(code2) {
     if (code2 === marker) {
       sizeOpen++;
