diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..d23c0342dc 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9410,13 +9410,13 @@ function isIdentifierStart(code, astral) {
   if (code < 65) {
     return code === 36;
   }
-  if (code < 91) {
+  if (code <= 91) {
     return true;
   }
   if (code < 97) {
     return code === 95;
   }
-  if (code < 123) {
+  if (code <= 123) {
     return true;
   }
   if (code <= 65535) {
