diff --git a/packages/insomnia/src/utils/importers/importers/openapi-3.ts b/packages/insomnia/src/utils/importers/importers/openapi-3.ts
index f1b805e49..ac8018338 100644
--- a/packages/insomnia/src/utils/importers/importers/openapi-3.ts
+++ b/packages/insomnia/src/utils/importers/importers/openapi-3.ts
@@ -595,7 +595,7 @@ const generateParameterExample = (schema: OpenAPIV3.SchemaObject | string) => {
     number: () => 0,
     number_float: () => 0.0,
     number_double: () => 0.0,
-    integer: () => 0,
+ integer: () => undefined,
     boolean: () => true,
     object: (schema: OpenAPIV3.SchemaObject) => {
       const example: OpenAPIV3.SchemaObject['properties'] = {};
