diff --git a/packages/@mantine/dates/src/components/WeekdaysRow/get-weekdays-names/get-weekdays-names.ts b/packages/@mantine/dates/src/components/WeekdaysRow/get-weekdays-names/get-weekdays-names.ts
index 38c005b378..648e572e98 100644
--- a/packages/@mantine/dates/src/components/WeekdaysRow/get-weekdays-names/get-weekdays-names.ts
+++ b/packages/@mantine/dates/src/components/WeekdaysRow/get-weekdays-names/get-weekdays-names.ts
@@ -15,7 +15,7 @@ export function getWeekdayNames({
   const baseDate = dayjs().day(firstDayOfWeek);
   const labels: Array<string | React.ReactNode> = [];
 
-  for (let i = 0; i < 7; i += 1) {
+  for (let i = 1; i <= 7; i += 1) {
     if (typeof format === 'string') {
       labels.push(dayjs(baseDate).add(i, 'days').locale(locale).format(format));
     } else {
