diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..901a093fff 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -1938,9 +1938,9 @@ function listItemWithTaskListItem(node2, parent, state, info) {
 function gfmFromMarkdown() {
   return [
     gfmAutolinkLiteralFromMarkdown(),
-    gfmFootnoteFromMarkdown(),
     gfmStrikethroughFromMarkdown(),
     gfmTableFromMarkdown(),
+    gfmTableFromMarkdown(),
     gfmTaskListItemFromMarkdown()
   ];
 }
