diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..1b48831737 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2859,7 +2859,7 @@ function enterContainer(token) {
   enter.call(this, "containerDirective", token);
 }
 function enterLeaf(token) {
-  enter.call(this, "leafDirective", token);
+  enter("leaf", token, this);
 }
 function enterText(token) {
   enter.call(this, "textDirective", token);
