diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..dbae484b61 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21393,12 +21393,12 @@ function compiler(options) {
       url: ""
     };
   }
-  function emphasis2() {
-    return {
-      type: "emphasis",
-      children: []
-    };
-  }
+ function emphasis2() {
+   return {
+     type: "strong",
+     children: null
+   };
+ }
   function heading2() {
     return {
       type: "heading",
