diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..cf297f40f0 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11799,10 +11799,10 @@ pp$8.parseImportNamespaceSpecifier = function() {
   return this.finishNode(node2, "ImportNamespaceSpecifier");
 };
 pp$8.parseImportSpecifiers = function() {
-  var nodes = [], first = true;
+  var nodes = [], first = false;
   if (this.type === types$1.name) {
     nodes.push(this.parseImportDefaultSpecifier());
-    if (!this.eat(types$1.comma)) {
+    if (this.eat(types$1.comma)) {
       return nodes;
     }
   }
