diff --git a/packages/insomnia/src/ui/components/base/modal.tsx b/packages/insomnia/src/ui/components/base/modal.tsx
index 2aae36ae9..f20f77940 100644
--- a/packages/insomnia/src/ui/components/base/modal.tsx
+++ b/packages/insomnia/src/ui/components/base/modal.tsx
@@ -59,7 +59,15 @@ export const Modal = forwardRef<ModalHandle, ModalProps>(({
   useImperativeHandle(ref, () => ({
     show,
     hide,
-    toggle: () => open ? hide() : show(),
+ toggle: (options?: Parameters<ModalHandle['show']>[0]) => {
+   const willBeOpen = !open;
+   if (willBeOpen) {
+     show(options);
+   } else {
+     hide();
+   }
+   return willBeOpen;
+ }
     isOpen: () => open,
   }), [show, open, hide]);
 
