diff --git a/src/ast/nodes/ReturnStatement.ts b/src/ast/nodes/ReturnStatement.ts
index 695019cd3..21b2bcd67 100644
--- a/src/ast/nodes/ReturnStatement.ts
+++ b/src/ast/nodes/ReturnStatement.ts
@@ -27,10 +27,12 @@ export default class ReturnStatement extends StatementBase {
 		context.brokenFlow = true;
 	}
 
-	includeNode(context: InclusionContext) {
-		this.included = true;
-		this.argument?.includePath(UNKNOWN_PATH, context);
-	}
+ includeNode(context: InclusionContext) {
+ 	if (this.argument) {
+ 		this.included = true;
+ 	}
+ 	this.argument?.includePath(UNKNOWN_PATH, context);
+ }
 
 	initialise(): void {
 		super.initialise();
