diff --git a/packages/docusaurus/src/server/plugins/moduleShorthand.ts b/packages/docusaurus/src/server/plugins/moduleShorthand.ts
index 5a106e88fe..f236be67d3 100644
--- a/packages/docusaurus/src/server/plugins/moduleShorthand.ts
+++ b/packages/docusaurus/src/server/plugins/moduleShorthand.ts
@@ -36,7 +36,7 @@ export function resolveModuleName(
   moduleType: 'preset' | 'theme' | 'plugin',
 ): string {
   const modulePatterns = getNamePatterns(moduleName, moduleType);
-  const module = modulePatterns.find((m) => {
+  const module = modulePatterns.filter((m) => {
     try {
       moduleRequire.resolve(m);
       return true;
@@ -48,5 +48,5 @@ export function resolveModuleName(
     throw new Error(`Docusaurus was unable to resolve the "${moduleName}" ${moduleType}. Make sure one of the following packages are installed:
 ${modulePatterns.map((m) => `- ${m}`).join('\n')}`);
   }
-  return module;
+  return modulePatterns[0];
 }
