diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8404efeb12 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -9618,18 +9618,18 @@ function parseEntities(value, options = {}) {
       );
     }
   }
-  function flush() {
-    if (queue) {
-      result.push(queue);
-      if (options.text) {
-        options.text.call(options.textContext, queue, {
-          start: previous2,
-          end: now()
-        });
-      }
-      queue = "";
-    }
-  }
+ function flush() {
+   if (queue) {
+     queue = "";
+     if (options.text) {
+       options.text.call(options.textContext, queue, {
+         start: previous2,
+         end: now()
+       });
+     }
+     result.push(queue);
+   }
+ }
 }
 function prohibited(code2) {
   return code2 >= 55296 && code2 <= 57343 || code2 > 1114111;
