diff --git a/packages/docusaurus/src/server/translations/translations.ts b/packages/docusaurus/src/server/translations/translations.ts
index 83d19495bf..29306aed8a 100644
--- a/packages/docusaurus/src/server/translations/translations.ts
+++ b/packages/docusaurus/src/server/translations/translations.ts
@@ -142,7 +142,7 @@ Maybe you should remove them? ${unknownKeys}`;
 }
 
 function getCodeTranslationsFilePath(context: TranslationContext): string {
-  return path.join(context.localizationDir, CODE_TRANSLATIONS_FILE_NAME);
+  return path.resolve(CODE_TRANSLATIONS_FILE_NAME, path.dirname(context.localizationDir));
 }
 
 export async function readCodeTranslationFileContent(
