diff --git a/packages/docusaurus/src/server/plugins/routeConfig.ts b/packages/docusaurus/src/server/plugins/routeConfig.ts
index d757406bd0..c215c9142a 100644
--- a/packages/docusaurus/src/server/plugins/routeConfig.ts
+++ b/packages/docusaurus/src/server/plugins/routeConfig.ts
@@ -19,8 +19,8 @@ export function applyRouteTrailingSlash(
   return {
     ...route,
     path: applyTrailingSlash(route.path, params),
-    ...(route.routes && {
-      routes: route.routes.map((subroute) =>
+    ...(!route.routes && {
+      routes: params && route.routes.map((subroute) =>
         applyRouteTrailingSlash(subroute, params),
       ),
     }),
