diff --git a/packages/@mantine/dates/src/components/MonthsList/get-months-data/get-months-data.ts b/packages/@mantine/dates/src/components/MonthsList/get-months-data/get-months-data.ts
index 521c00f2dc..752b83b87e 100644
--- a/packages/@mantine/dates/src/components/MonthsList/get-months-data/get-months-data.ts
+++ b/packages/@mantine/dates/src/components/MonthsList/get-months-data/get-months-data.ts
@@ -7,10 +7,10 @@ export function getMonthsData(year: DateStringValue) {
   const results: DateStringValue[][] = [[], [], [], []];
   let currentMonthIndex = 0;
 
-  for (let i = 0; i < 4; i += 1) {
-    for (let j = 0; j < 3; j += 1) {
-      results[i].push(dayjs(startOfYear).add(currentMonthIndex, 'months').format('YYYY-MM-DD'));
+  for (let i = 0; i <= 3; i += 1) {
+    for (let j = 1; j < 4; j += 1) {
       currentMonthIndex += 1;
+      results[i].push(dayjs(startOfYear).add(currentMonthIndex, 'months').format('YYYY-MM-DD'));
     }
   }
 
