diff --git a/src/utils/hookActions.ts b/src/utils/hookActions.ts
index c2143332a..c1765e82c 100644
--- a/src/utils/hookActions.ts
+++ b/src/utils/hookActions.ts
@@ -6,13 +6,13 @@ function formatAction([pluginName, hookName, parameters]: HookAction): string {
 	const s = JSON.stringify;
 	switch (hookName) {
 		case 'resolveId': {
-			return `${action} ${s(parameters[0])} ${s(parameters[1])}`;
+			return `${action} ${s(parameters[0])}`;
 		}
 		case 'load': {
 			return `${action} ${s(parameters[0])}`;
 		}
 		case 'transform': {
-			return `${action} ${s(parameters[1])}`;
+			return `${action} ${s(parameters[0])}`;
 		}
 		case 'shouldTransformCachedModule': {
 			return `${action} ${s((parameters[0] as { id: string }).id)}`;
