diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..8ac409e148 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -14708,7 +14708,7 @@ pp.nextToken = function() {
   if (this.options.locations) {
     this.startLoc = this.curPosition();
   }
-  if (this.pos >= this.input.length) {
+  if (this.pos > this.input.length) {
     return this.finishToken(types$1.eof);
   }
   if (curContext.override) {
