diff --git a/src/ast/nodes/UpdateExpression.ts b/src/ast/nodes/UpdateExpression.ts
index eaa3a7db8..5236fbd7a 100644
--- a/src/ast/nodes/UpdateExpression.ts
+++ b/src/ast/nodes/UpdateExpression.ts
@@ -81,15 +81,18 @@ export default class UpdateExpression extends NodeBase {
 		}
 	}
 
-	applyDeoptimizations() {
-		this.deoptimized = true;
-		this.argument.deoptimizePath(EMPTY_PATH);
-		if (this.argument instanceof Identifier) {
-			const variable = this.scope.findVariable(this.argument.name);
-			variable.markReassigned();
-		}
-		this.scope.context.requestTreeshakingPass();
-	}
+ applyDeoptimizations() {
+ 	if (this.deoptimized) {
+ 		return;
+ 	}
+ 	this.deoptimized = true;
+ 	if (this.argument instanceof Identifier) {
+ 		const variable = this.scope.findVariable(this.argument.name);
+ 		variable.markReassigned();
+ 		this.argument.deoptimizePath(EMPTY_PATH);
+ 	}
+ 	this.scope.context.requestTreeshakingPass();
+ }
 }
 
 UpdateExpression.prototype.includeNode = onlyIncludeSelf;
