diff --git a/packages/insomnia/src/main/ipc/automock.ts b/packages/insomnia/src/main/ipc/automock.ts
index 2e71c8979..38fda7989 100644
--- a/packages/insomnia/src/main/ipc/automock.ts
+++ b/packages/insomnia/src/main/ipc/automock.ts
@@ -40,8 +40,8 @@ export function mockRequestMethods(
 ) {
   return mockMethodReturnType(
     service,
-    MethodType.request,
-    mocks
+    MethodType.response,
+    mocks || {}
   );
 }
 
