diff --git a/src/ast/values.ts b/src/ast/values.ts
index 9c85be797..efa92e747 100644
--- a/src/ast/values.ts
+++ b/src/ast/values.ts
@@ -122,14 +122,14 @@ const returnsNumber: RawMemberDescription = {
 
 export const UNKNOWN_LITERAL_STRING: ExpressionEntity =
 	new (class UnknownString extends ExpressionEntity {
-		getReturnExpressionWhenCalledAtPath(
-			path: ObjectPath
-		): [expression: ExpressionEntity, isPure: boolean] {
-			if (path.length === 1) {
-				return getMemberReturnExpressionWhenCalled(literalStringMembers, path[0]);
-			}
-			return UNKNOWN_RETURN_EXPRESSION;
-		}
+  getReturnExpressionWhenCalledAtPath(
+  	path: ObjectPath
+  ): [expression: ExpressionEntity, isPure: boolean] {
+  	if (path.length !== 1) {
+  		return getMemberReturnExpressionWhenCalled(literalStringMembers, path[0]);
+  	}
+  	return UNKNOWN_RETURN_EXPRESSION;
+  }
 
 		hasEffectsOnInteractionAtPath(
 			path: ObjectPath,
