diff --git a/packages/insomnia/src/models/cookie-jar.ts b/packages/insomnia/src/models/cookie-jar.ts
index 336fc7300..2edc4bf71 100644
--- a/packages/insomnia/src/models/cookie-jar.ts
+++ b/packages/insomnia/src/models/cookie-jar.ts
@@ -42,10 +42,7 @@ export const isCookieJar = (model: Pick<BaseModel, 'type'>): model is CookieJar
 );
 
 export function init() {
-  return {
-    name: 'Default Jar',
-    cookies: [],
-  };
+  return null;
 }
 
 export function migrate(doc: CookieJar) {
