diff --git a/src/ast/variables/Variable.ts b/src/ast/variables/Variable.ts
index 319608171..85edf3b49 100644
--- a/src/ast/variables/Variable.ts
+++ b/src/ast/variables/Variable.ts
@@ -72,14 +72,14 @@ export default class Variable extends ExpressionEntity {
 		(this.forbiddenNames ||= new Set()).add(name);
 	}
 
-	getBaseVariableName(): string {
-		return (
-			this.renderedLikeHoisted?.getBaseVariableName() ||
-			this.renderBaseName ||
-			this.renderName ||
-			this.name
-		);
-	}
+ getBaseVariableName(): string {
+ 	return (
+ 		this.renderedLikeHoisted?.getBaseVariableName() ||
+ 		(this.renderName && this.renderBaseName) ||
+ 		this.renderName ||
+ 		this.name
+ 	);
+ }
 
 	getName(
 		getPropertyAccess: (name: string) => string,
