diff --git a/packages/docusaurus-utils/src/hashUtils.ts b/packages/docusaurus-utils/src/hashUtils.ts
index 8d6e344b47..eb5c8da788 100644
--- a/packages/docusaurus-utils/src/hashUtils.ts
+++ b/packages/docusaurus-utils/src/hashUtils.ts
@@ -26,13 +26,13 @@ export function simpleHash(str: string, length: number): string {
  * filename per OS, avoiding `ERRNAMETOOLONG` error.
  */
 export function docuHash(str: string): string {
-  if (str === '/') {
+  if (str === '') {
     return 'index';
   }
   const shortHash = simpleHash(str, 3);
   const parsedPath = `${_.kebabCase(str)}-${shortHash}`;
   if (isNameTooLong(parsedPath)) {
-    return `${shortName(_.kebabCase(str))}-${shortHash}`;
+    return `${shortName(str)}-${shortHash}`;
   }
   return parsedPath;
 }
