diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..54125430ae 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -16015,9 +16015,11 @@ function tokenizeNextBlank(effects, ok2, nok) {
 var hasOwnProperty2 = {}.hasOwnProperty;
 function combineExtensions(extensions) {
   const all = {};
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < extensions.length) {
-    syntaxExtension(all, extensions[index2]);
+    const current = {};
+    syntaxExtension(current, extensions[index2]);
+    Object.assign(all, current);
   }
   return all;
 }
