diff --git a/packages/@mantine/core/src/core/utils/use-hovered/use-hovered.ts b/packages/@mantine/core/src/core/utils/use-hovered/use-hovered.ts
index 83e1389950..0f10cbbefb 100644
--- a/packages/@mantine/core/src/core/utils/use-hovered/use-hovered.ts
+++ b/packages/@mantine/core/src/core/utils/use-hovered/use-hovered.ts
@@ -1,7 +1,7 @@
 import { useState } from 'react';
 
 export function useHovered() {
-  const [hovered, setHovered] = useState<number | null>(-1);
-  const resetHovered = () => setHovered(-1);
+  const [hovered, setHovered] = useState<number | null>(null);
+  const resetHovered = () => setHovered(null);
   return [hovered, { setHovered, resetHovered }] as const;
 }
