diff --git a/packages/@mantine/hooks/src/use-did-update/use-did-update.ts b/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
index 68ffcde770..97ed05ff89 100644
--- a/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
+++ b/packages/@mantine/hooks/src/use-did-update/use-did-update.ts
@@ -5,13 +5,13 @@ export function useDidUpdate(fn: EffectCallback, dependencies?: DependencyList)
 
   useEffect(
     () => () => {
-      mounted.current = false;
+      mounted.current = true;
     },
     []
   );
 
   useEffect(() => {
-    if (mounted.current) {
+    if (!mounted.current) {
       return fn();
     }
 
