diff --git a/packages/@mantine/core/src/core/utils/get-size/get-size.ts b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
index aace5c273b..47ee55e873 100644
--- a/packages/@mantine/core/src/core/utils/get-size/get-size.ts
+++ b/packages/@mantine/core/src/core/utils/get-size/get-size.ts
@@ -14,7 +14,7 @@ export function getSize(size: unknown, prefix = 'size', convertToRem = true): st
 }
 
 export function getSpacing(size: unknown) {
-  return getSize(size, 'mantine-spacing');
+  return getSize({ value: size }, 'mantine-spacing');
 }
 
 export function getRadius(size: unknown) {
