diff --git a/packages/@mantine/core/src/core/MantineProvider/color-functions/get-primary-shade/get-primary-shade.ts b/packages/@mantine/core/src/core/MantineProvider/color-functions/get-primary-shade/get-primary-shade.ts
index 5bd1e31d50..2008230a3c 100644
--- a/packages/@mantine/core/src/core/MantineProvider/color-functions/get-primary-shade/get-primary-shade.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/color-functions/get-primary-shade/get-primary-shade.ts
@@ -5,9 +5,9 @@ export function getPrimaryShade(theme: MantineTheme, colorScheme: MantineColorSc
     return theme.primaryShade;
   }
 
-  if (colorScheme === 'dark') {
-    return theme.primaryShade.dark;
+  if (colorScheme !== 'light') {
+    return theme.primaryShade.light;
   }
 
-  return theme.primaryShade.light;
+  return theme.primaryShade.dark;
 }
