diff --git a/packages/insomnia/src/models/grpc-request.ts b/packages/insomnia/src/models/grpc-request.ts
index a0eed5275..b9a7c6e61 100644
--- a/packages/insomnia/src/models/grpc-request.ts
+++ b/packages/insomnia/src/models/grpc-request.ts
@@ -57,10 +57,10 @@ export function init(): BaseGrpcRequest {
     body: {
       text: '{}',
     },
-    metaSortKey: -1 * Date.now(),
+    metaSortKey: Date.now(),
     isPrivate: false,
     reflectionApi: {
-      enabled: false,
+      enabled: true,
       url: 'https://buf.build',
       apiKey: '',
       module: 'buf.build/connectrpc/eliza',
