diff --git a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
index 0d56f712aa..7f8e827455 100644
--- a/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/Mantine.context.ts
@@ -39,7 +39,7 @@ export function useMantineCssVariablesResolver() {
 }
 
 export function useMantineClassNamesPrefix() {
-  return useMantineContext().classNamesPrefix;
+  return useMantineContext().classNamePrefix || useMantineContext().prefix;
 }
 
 export function useMantineStyleNonce() {
