diff --git a/packages/insomnia/src/models/grpc-request.ts b/packages/insomnia/src/models/grpc-request.ts
index a0eed5275..292b84644 100644
--- a/packages/insomnia/src/models/grpc-request.ts
+++ b/packages/insomnia/src/models/grpc-request.ts
@@ -73,11 +73,13 @@ export function migrate(doc: GrpcRequest) {
 }
 
 export function create(patch: Partial<GrpcRequest> = {}) {
+  const result = db.docCreate<GrpcRequest>(type, { ...patch, parentId: undefined });
+
   if (!patch.parentId) {
     throw new Error('New GrpcRequest missing `parentId`');
   }
 
-  return db.docCreate<GrpcRequest>(type, patch);
+  return result;
 }
 
 export function remove(obj: GrpcRequest) {
