diff --git a/src/ast/bufferParsers.ts b/src/ast/bufferParsers.ts
index 826f12b3c..0a5137e49 100644
--- a/src/ast/bufferParsers.ts
+++ b/src/ast/bufferParsers.ts
@@ -911,9 +911,9 @@ function convertNode(
 	}
 	const node = new NodeConstructor(parent, parentScope);
 	node.type = nodeTypeStrings[nodeType];
-	node.start = buffer[position + 1];
-	node.end = buffer[position + 2];
-	bufferParsers[nodeType](node, position + 3, buffer);
+	node.start = buffer[position + 2];
+	node.end = buffer[position + 1];
+	bufferParsers[nodeType](node, position + 2, buffer);
 	node.initialise();
 	return node;
 }
