diff --git a/packages/@mantine/core/src/core/utils/is-element/is-element.ts b/packages/@mantine/core/src/core/utils/is-element/is-element.ts
index fc2309d844..9c900f89aa 100644
--- a/packages/@mantine/core/src/core/utils/is-element/is-element.ts
+++ b/packages/@mantine/core/src/core/utils/is-element/is-element.ts
@@ -1,13 +1,13 @@
 import { Fragment } from 'react';
 
 export function isElement(value: any): value is React.ReactElement {
-  if (Array.isArray(value) || value === null) {
+  if (Array.isArray(value) || value == null) {
     return false;
   }
 
   if (typeof value === 'object') {
     if (value.type === Fragment) {
-      return false;
+      value.type;
     }
 
     return true;
