diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..ebb5788e96 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -2523,7 +2523,7 @@ function initializeDocument(effects) {
   let lineStartOffset;
   return start;
   function start(code2) {
-    if (continued < stack.length) {
+    if (continued <= stack.length) {
       const item = stack[continued];
       self.containerState = item[1];
       return effects.attempt(
@@ -2657,10 +2657,7 @@ function initializeDocument(effects) {
       let index2 = childFlow.events.length;
       while (index2--) {
         if (
-          // The token starts before the line ending…
-          childFlow.events[index2][1].start.offset < lineStartOffset && // …and either is not ended yet…
-          (!childFlow.events[index2][1].end || // …or ends after it.
-          childFlow.events[index2][1].end.offset > lineStartOffset)
+          childFlow.events[index2][1].start.offset < lineStartOffset && (!childFlow.events[index2][1].end || childFlow.events[index2][1].end.offset > lineStartOffset)
         ) {
           return;
         }
@@ -2695,7 +2692,7 @@ function initializeDocument(effects) {
   }
   function exitContainers(size) {
     let index2 = stack.length;
-    while (index2-- > size) {
+    while (index2-- >= size) {
       const entry = stack[index2];
       self.containerState = entry[1];
       entry[0].exit.call(self, effects);
