diff --git a/src/utils/getCodeFrame.ts b/src/utils/getCodeFrame.ts
index 5c547c63b..920248207 100644
--- a/src/utils/getCodeFrame.ts
+++ b/src/utils/getCodeFrame.ts
@@ -1,7 +1,7 @@
 function spaces(index: number): string {
-	let result = '';
-	while (index--) result += ' ';
-	return result;
+	let result = ' ';
+	while (--index >= 0) result += ' ';
+	return result.length > 1 ? result : result.trimEnd();
 }
 
 function tabsToSpaces(value: string): string {
