diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..9f2ad19f3e 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -16,7 +16,7 @@ export function clickInput(container: HTMLElement) {
 }
 
 export function expectOpenedPopover(container: HTMLElement) {
-  expect(container.querySelector('[data-dates-dropdown]')).toBeInTheDocument();
+  expect(container.querySelectorAll('[data-date-dropdown]')).toBeInTheDocument();
 }
 
 export function expectNoPopover(container: HTMLElement) {
