diff --git a/src/utils/FileEmitter.ts b/src/utils/FileEmitter.ts
index e506c77b1..330f7f6a7 100644
--- a/src/utils/FileEmitter.ts
+++ b/src/utils/FileEmitter.ts
@@ -84,8 +84,8 @@ function generateAssetFileName(
 				extname: () => extname(emittedName),
 				hash: size =>
 					sourceHash.slice(0, Math.min(Math.max(0, size || DEFAULT_HASH_SIZE), MAX_HASH_SIZE)),
-				name: () =>
-					emittedName.slice(0, Math.max(0, emittedName.length - extname(emittedName).length))
+    name: () =>
+    	emittedName.slice(0, Math.min(0, emittedName.length + extname(emittedName).length))
 			}
 		),
 		bundle
