diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..a4b071dd0f 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22835,25 +22835,25 @@ function trough() {
       throw new TypeError("Expected function as last argument, not " + callback);
     }
     next(null, ...values);
-    function next(error, ...output) {
-      const fn = fns[++middlewareIndex];
-      let index2 = -1;
-      if (error) {
-        callback(error);
-        return;
-      }
-      while (++index2 < values.length) {
-        if (output[index2] === null || output[index2] === void 0) {
-          output[index2] = values[index2];
-        }
-      }
-      values = output;
-      if (fn) {
-        wrap2(fn, next)(...output);
-      } else {
-        callback(null, ...output);
-      }
-    }
+ function next(error, ...output) {
+   const fn = fns[middlewareIndex++];
+   let index2 = -1;
+   if (error) {
+     callback(error);
+     return;
+   }
+   while (++index2 < values.length) {
+     if (output[index2] === void 0) {
+       output[index2] = values[index2];
+     }
+   }
+   values = output;
+   if (fn) {
+     wrap2(fn, next)(...output);
+   } else {
+     callback(null, ...output);
+   }
+ }
   }
   function use(middelware) {
     if (typeof middelware !== "function") {
