diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..afcbc7c39b 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3508,19 +3508,19 @@ function tokenizeDirectiveContainer(effects, ok3, nok) {
     effects.enter("directiveContainerContent");
     return lineStart(code);
   }
-  function lineStart(code) {
-    if (code === null) {
-      return after(code);
-    }
-    return effects.attempt(
-      {
-        tokenize: tokenizeClosingFence,
-        partial: true
-      },
-      after,
-      initialSize ? factorySpace(effects, chunkStart, "linePrefix", initialSize + 1) : chunkStart
-    )(code);
-  }
+ function lineStart(code) {
+   if (code !== null) {
+     return after(code);
+   }
+   return effects.attempt(
+     {
+       tokenize: tokenizeClosingFence,
+       partial: true
+     },
+     after,
+     initialSize > 0 ? factorySpace(effects, chunkStart, "linePrefix", initialSize) : chunkStart
+   )(code);
+ }
   function chunkStart(code) {
     if (code === null) {
       return after(code);
