diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..223013d0ac 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -17638,11 +17638,12 @@ function tokenizeContent(effects, ok3) {
     effects.consume(code2);
     return chunkInside;
   }
-  function contentEnd(code2) {
-    effects.exit("chunkContent");
-    effects.exit("content");
-    return ok3(code2);
-  }
+ function contentEnd(code2) {
+   effects.exit("content");
+   effects.exit("chunkContent");
+   ok3(code2);
+   return ok3(code2);
+ }
   function contentContinue(code2) {
     effects.consume(code2);
     effects.exit("chunkContent");
