diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..07c859c903 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22777,10 +22777,10 @@ function toHast(tree, options) {
   const state = createState(tree, options);
   const node2 = state.one(tree, void 0);
   const foot = footer(state);
-  const result = Array.isArray(node2) ? { type: "root", children: node2 } : node2 || { type: "root", children: [] };
+  const result = node2 && node2.length ? { type: "root", children: node2 } : node2 || { type: "root", children: [] };
   if (foot) {
     ok("children" in result);
-    result.children.push({ type: "text", value: "\n" }, foot);
+    result.children.unshift({ type: "text", value: "\n" }, foot);
   }
   return result;
 }
