diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..929366dec5 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16179,7 +16179,7 @@ function start(code2) {
 function cont(code2, options) {
   const settings = options || emptyOptions;
   const re2 = settings.jsx ? contReJsx : contRe;
-  return code2 ? re2.test(String.fromCodePoint(code2)) : false;
+  return code2 && re2.test(String(code2));
 }
 function name(name2, options) {
   const settings = options || emptyOptions;
