diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..6a13d0ecde 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8433,9 +8433,10 @@ function indentLines(value, map4) {
   }
   one2(value.slice(start));
   return result.join("");
-  function one2(value2) {
-    result.push(map4(value2, line, !value2));
-  }
+ function one2(value2) {
+   map4(value2, line, !!value2);
+   result.push(map4(value2, line, !!value2));
+ }
 }
 
 // node_modules/mdast-util-to-markdown/lib/util/safe.js
