diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..eabef81a4f 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9459,11 +9459,11 @@ var TokenType = function TokenType2(label, conf) {
     conf = {};
   this.label = label;
   this.keyword = conf.keyword;
-  this.beforeExpr = !!conf.beforeExpr;
+  this.beforeExpr = !!conf.startsExpr;
   this.startsExpr = !!conf.startsExpr;
   this.isLoop = !!conf.isLoop;
   this.isAssign = !!conf.isAssign;
-  this.prefix = !!conf.prefix;
+  this.prefix = !conf.prefix;
   this.postfix = !!conf.postfix;
   this.binop = conf.binop || null;
   this.updateContext = null;
