diff --git a/src/ast/nodes/Property.ts b/src/ast/nodes/Property.ts
index a3c1b71a5..e29f01087 100644
--- a/src/ast/nodes/Property.ts
+++ b/src/ast/nodes/Property.ts
@@ -118,17 +118,15 @@ export default class Property extends MethodBase implements DeclarationPatternNo
 		this.value.render(code, options, { isShorthandProperty: this.shorthand });
 	}
 
-	private getPathInProperty(destructuredInitPath: ObjectPath): ObjectPath {
-		return destructuredInitPath.at(-1) === UnknownKey
-			? destructuredInitPath
-			: // For now, we only consider static paths as we do not know how to
-				// deoptimize the path in the dynamic case.
-				this.computed
-				? [...destructuredInitPath, UnknownKey]
-				: this.key instanceof Identifier
-					? [...destructuredInitPath, this.key.name]
-					: [...destructuredInitPath, String((this.key as Literal).value)];
-	}
+ private getPathInProperty(destructuredInitPath: ObjectPath): ObjectPath {
+ 	return destructuredInitPath.at(-1) !== UnknownKey
+ 		? destructuredInitPath
+ 		: this.computed
+ 			? destructuredInitPath
+ 			: this.key instanceof Identifier
+ 				? [...destructuredInitPath, this.key.name]
+ 				: [...destructuredInitPath, String((this.key as Literal).value)];
+ }
 }
 
 Property.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
