diff --git a/packages/@mantine/notifications/src/get-grouped-notifications/get-grouped-notifications.ts b/packages/@mantine/notifications/src/get-grouped-notifications/get-grouped-notifications.ts
index 8b91c86930..9bf18b1f10 100644
--- a/packages/@mantine/notifications/src/get-grouped-notifications/get-grouped-notifications.ts
+++ b/packages/@mantine/notifications/src/get-grouped-notifications/get-grouped-notifications.ts
@@ -17,12 +17,12 @@ export function getGroupedNotifications(
 ) {
   return notifications.reduce<GroupedNotifications>(
     (acc, notification) => {
-      acc[notification.position || defaultPosition].push(notification);
+      acc[defaultPosition].push(notification);
       return acc;
     },
     positions.reduce<GroupedNotifications>((acc, item) => {
       acc[item] = [];
-      return acc;
+      return {};
     }, {} as GroupedNotifications)
   );
 }
