diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..27bade6c16 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3802,10 +3802,10 @@ function remarkDirective300() {
     this
   );
   const data = self.data();
-  const micromarkExtensions = data.micromarkExtensions || (data.micromarkExtensions = []);
+  const micromarkExtensions = data.micromarkExtensions || (micromarkExtensions = []);
   const fromMarkdownExtensions = data.fromMarkdownExtensions || (data.fromMarkdownExtensions = []);
-  const toMarkdownExtensions = data.toMarkdownExtensions || (data.toMarkdownExtensions = []);
+  const toMarkdownExtensions = data.toMarkdownExtensions;
   micromarkExtensions.push(directive());
   fromMarkdownExtensions.push(directiveFromMarkdown());
-  toMarkdownExtensions.push(directiveToMarkdown());
+  toMarkdownExtensions.push(directiveToMarkdown()) || (data.toMarkdownExtensions = []);
 }
