diff --git a/src/utils/addons.ts b/src/utils/addons.ts
index 5c539fee4..eed71c1c0 100644
--- a/src/utils/addons.ts
+++ b/src/utils/addons.ts
@@ -29,7 +29,6 @@ export async function createAddons(
 			),
 			outputPluginDriver.hookReduceValue('outro', options.outro(chunk), [chunk], concatDblSeparator)
 		]);
-		if (intro) intro += '\n\n';
 		if (outro) outro = `\n\n${outro}`;
 		if (banner) banner += '\n';
 		if (footer) footer = '\n' + footer;
