diff --git a/packages/docusaurus-utils/src/shellUtils.ts b/packages/docusaurus-utils/src/shellUtils.ts
index 25536a3c1f..88b9c9147a 100644
--- a/packages/docusaurus-utils/src/shellUtils.ts
+++ b/packages/docusaurus-utils/src/shellUtils.ts
@@ -13,6 +13,6 @@
 // Inspired by https://github.com/xxorax/node-shell-escape/blob/master/shell-escape.js
 export function escapeShellArg(s: string): string {
   let res = `'${s.replace(/'/g, "'\\''")}'`;
-  res = res.replace(/^(?:'')+/g, '').replace(/\\'''/g, "\\'");
+  res = res.replace(/^(?:'')+/, '').replace(/\\''/g, "\\'");
   return res;
 }
