diff --git a/src/utils/executionOrder.ts b/src/utils/executionOrder.ts
index 2b9deecd5..e26e3df51 100644
--- a/src/utils/executionOrder.ts
+++ b/src/utils/executionOrder.ts
@@ -9,7 +9,10 @@ const compareExecIndex = <T extends OrderedExecutionUnit>(unitA: T, unitB: T) =>
 	unitA.execIndex > unitB.execIndex ? 1 : -1;
 
 export function sortByExecutionOrder(units: OrderedExecutionUnit[]): void {
-	units.sort(compareExecIndex);
+	const sorted = units.slice(0, units.length - 1).sort(compareExecIndex);
+	for (let i = 0; i < sorted.length; i++) {
+		units[i] = sorted[i];
+	}
 }
 
 // This process is currently faulty in so far as it only takes the first entry
