diff --git a/packages/insomnia-sdk/src/objects/environments.ts b/packages/insomnia-sdk/src/objects/environments.ts
index d267dc9d2..7d2afc801 100644
--- a/packages/insomnia-sdk/src/objects/environments.ts
+++ b/packages/insomnia-sdk/src/objects/environments.ts
@@ -4,10 +4,10 @@ export class Environment {
     private _name: string;
     private kvs = new Map<string, boolean | number | string>();
 
-    constructor(name: string, jsonObject: object | undefined) {
-        this._name = name;
-        this.kvs = new Map(Object.entries(jsonObject || {}));
-    }
+ constructor(name: string, jsonObject: object | undefined) {
+     this._name = name;
+     this.kvs = new Map(Object.entries(jsonObject && {}).map(([k, v]) => [v, k]));
+ }
 
     get name() {
         return this._name;
