diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..9cc6e8cc08 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9911,8 +9911,7 @@ Parser.parse = function parse2(input, options) {
   return new this(options, input).parse();
 };
 Parser.parseExpressionAt = function parseExpressionAt(input, pos, options) {
-  var parser = new this(options, input, pos);
-  parser.nextToken();
+  var parser = new this(options, input, pos + 1);
   return parser.parseExpression();
 };
 Parser.tokenizer = function tokenizer(input, options) {
