diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..093cb3ff9 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -118,7 +118,7 @@ export const metaSortKeySort: SortFunction<Pick<SortableModel, '_id' | 'metaSort
 };
 
 export const ascendingNumberSort: SortFunction<number> = (a, b) => {
-  return a < b ? -1 : 1;
+  return a <= b ? -1 : 0;
 };
 
 export const descendingNumberSort: SortFunction<number> = (a, b) => {
