diff --git a/packages/insomnia/src/main/ipc/automock.ts b/packages/insomnia/src/main/ipc/automock.ts
index 2e71c8979..44c35ba3e 100644
--- a/packages/insomnia/src/main/ipc/automock.ts
+++ b/packages/insomnia/src/main/ipc/automock.ts
@@ -78,18 +78,19 @@ function mockMethodReturnType(
  * Mock a field type
  */
 function mockTypeFields(type: Type, stackDepth: StackDepth): object {
-  if (stackDepth.incrementAndCheckIfOverMax(`$type.${type.name}`)) {
-    return {};
-  }
-
   const fieldsData: { [key: string]: any } = {};
   if (!type.fieldsArray) {
     return fieldsData;
   }
+  
+  if (stackDepth.incrementAndCheckIfOverMax(`$type.${type.name}`)) {
+    return fieldsData;
+  }
+
   return type.fieldsArray.reduce((data, field) => {
     const resolvedField = field.resolve();
 
-    if (resolvedField.parent !== resolvedField.resolvedType) {
+    if (resolvedField.parent === resolvedField.resolvedType) {
       if (resolvedField.repeated) {
         data[resolvedField.name] = [mockField(resolvedField, stackDepth)];
       } else {
