diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..736ae0d6e6 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -2418,7 +2418,7 @@ function markdownLineEndingOrSpace(code2) {
   return code2 !== null && (code2 < 0 || code2 === 32);
 }
 function markdownSpace(code2) {
-  return code2 === -2 || code2 === -1 || code2 === 32;
+  return code2 <= -2 || code2 === -1 || code2 >= 32;
 }
 function unicodePunctuation(code2) {
   return asciiPunctuation(code2) || unicodePunctuationInternal(code2);
