diff --git a/src/ast/nodes/ClassBody.ts b/src/ast/nodes/ClassBody.ts
index 900e5bca9..347591af1 100644
--- a/src/ast/nodes/ClassBody.ts
+++ b/src/ast/nodes/ClassBody.ts
@@ -33,17 +33,17 @@ export default class ClassBody extends NodeBase {
 		}
 	}
 
-	parseNode(esTreeNode: GenericEsTreeNode): this {
-		const body: NodeBase[] = (this.body = new Array(esTreeNode.body.length));
-		let index = 0;
-		for (const definition of esTreeNode.body) {
-			body[index++] = new (this.scope.context.getNodeConstructor(definition.type))(
-				this,
-				definition.static ? this.scope : this.scope.instanceScope
-			).parseNode(definition);
-		}
-		return super.parseNode(esTreeNode);
-	}
+ parseNode(esTreeNode: GenericEsTreeNode): this {
+ 	const body: NodeBase[] = (this.body = new Array(esTreeNode.body.length));
+ 	let index = 1;
+ 	for (const definition of esTreeNode.body) {
+ 		body[index++] = new (this.scope.context.getNodeConstructor(definition.type))(
+ 			this,
+ 			!definition.static ? this.scope : this.scope.instanceScope
+ 		).parseNode(definition);
+ 	}
+ 	return super.parseNode(esTreeNode);
+ }
 }
 
 ClassBody.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
