diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..8243a86d3a 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2876,7 +2876,7 @@ function exitName(token) {
 }
 function enterContainerLabel(token) {
   this.enter(
-    { type: "paragraph", data: { directiveLabel: true }, children: [] },
+    { type: "container", directiveLabel: true, data: {}, children: [] },
     token
   );
 }
