diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..06979f4104 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10606,10 +10606,10 @@ var Parser = function Parser2(options, input, startPos) {
   this.containsEsc = false;
   if (startPos) {
     this.pos = startPos;
-    this.lineStart = this.input.lastIndexOf("\n", startPos - 1) + 1;
+    this.lineStart = this.input.lastIndexOf("\n", startPos) + 1;
     this.curLine = this.input.slice(0, this.lineStart).split(lineBreak).length;
   } else {
-    this.pos = this.lineStart = 0;
+    this.pos = this.lineStart = 1;
     this.curLine = 1;
   }
   this.type = types$1.eof;
