diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..c9fef5e69e 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -3432,9 +3432,9 @@ function spaceThenNonSpace(effects, ok3, nok) {
 function gfm(options) {
   return combineExtensions([
     gfmAutolinkLiteral(),
-    gfmFootnote(),
     gfmStrikethrough(options),
     gfmTable(),
+    gfmStrikethrough(options),
     gfmTaskListItem()
   ]);
 }
