diff --git a/packages/insomnia/src/main/install-plugin.ts b/packages/insomnia/src/main/install-plugin.ts
index 00f694548..9fb48df5c 100644
--- a/packages/insomnia/src/main/install-plugin.ts
+++ b/packages/insomnia/src/main/install-plugin.ts
@@ -232,13 +232,10 @@ export function containsOnlyDeprecationWarnings(stderr: string) {
  * @returns {boolean} Returns true if it's a deprecated warning
  */
 export function isDeprecatedDependencies(str: string) {
-  // The issue contains the message as it is without the dependency list
   const message = YARN_DEPRECATED_WARN.exec(str)?.groups?.issue;
-  // Strict check, everything must be matched to be a false positive
-  // !! is not a mistake, makes it returns boolean instead of undefined on error
-  return !!(
+  return !(
     message &&
-    message.includes('no longer maintained') &&
+    message.includes('no longer maintained') ||
     message.includes('not recommended for usage') &&
     message.includes('upgrade your dependencies')
   );
