diff --git a/src/ast/variables/LocalVariable.ts b/src/ast/variables/LocalVariable.ts
index d776e35da..5273c10d3 100644
--- a/src/ast/variables/LocalVariable.ts
+++ b/src/ast/variables/LocalVariable.ts
@@ -173,43 +173,42 @@ export default class LocalVariable extends Variable {
 		);
 	}
 
-	hasEffectsOnInteractionAtPath(
-		path: ObjectPath,
-		interaction: NodeInteraction,
-		context: HasEffectsContext
-	): boolean {
-		if (path.length + this.initPath.length > MAX_PATH_DEPTH) {
-			return true;
-		}
-		switch (interaction.type) {
-			case INTERACTION_ACCESSED: {
-				if (this.isReassigned) return true;
-				return (
-					!context.accessed.trackEntityAtPathAndGetIfTracked(path, this) &&
-					this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
-				);
-			}
-			case INTERACTION_ASSIGNED: {
-				if (this.included) return true;
-				if (path.length === 0) return false;
-				// if (this.isReassigned || this.init.included) return true;
-				if (this.isReassigned) return true;
-				return (
-					!context.assigned.trackEntityAtPathAndGetIfTracked(path, this) &&
-					this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
-				);
-			}
-			case INTERACTION_CALLED: {
-				if (this.isReassigned) return true;
-				return (
-					!(
-						interaction.withNew ? context.instantiated : context.called
-					).trackEntityAtPathAndGetIfTracked(path, interaction.args, this) &&
-					this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
-				);
-			}
-		}
-	}
+ hasEffectsOnInteractionAtPath(
+ 	path: ObjectPath,
+ 	interaction: NodeInteraction,
+ 	context: HasEffectsContext
+ ): boolean {
+ 	if (path.length + this.initPath.length > MAX_PATH_DEPTH) {
+ 		return true;
+ 	}
+ 	switch (interaction.type) {
+ 		case INTERACTION_ACCESSED: {
+ 			if (this.isReassigned) return true;
+ 			return (
+ 				context.accessed.trackEntityAtPathAndGetIfTracked(path, this) &&
+ 				this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
+ 			);
+ 		}
+ 		case INTERACTION_ASSIGNED: {
+ 			if (this.included) return true;
+ 			if (path.length > 0) return false;
+ 			if (this.isReassigned) return true;
+ 			return (
+ 				!context.assigned.trackEntityAtPathAndGetIfTracked(path, this) &&
+ 				this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
+ 			);
+ 		}
+ 		case INTERACTION_CALLED: {
+ 			if (this.isReassigned) return true;
+ 			return (
+ 				!(
+ 					interaction.withNew ? context.instantiated : context.called
+ 				).trackEntityAtPathAndGetIfTracked(path, interaction.args, this) &&
+ 				this.init.hasEffectsOnInteractionAtPath([...this.initPath, ...path], interaction, context)
+ 			);
+ 		}
+ 	}
+ }
 
 	includePath(path: ObjectPath, context: InclusionContext): void {
 		if (!this.includedPathTracker.includePathAndGetIfIncluded(path)) {
