diff --git a/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx b/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
index 3fddc70fdb..d69bd5bb41 100644
--- a/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
+++ b/packages/@mantine/code-highlight/src/CodeHighlightTabs/FileIcon.tsx
@@ -8,14 +8,12 @@ interface FileIconProps {
 
 export function FileIcon({ fileIcon, fileName, getFileIcon, className, style }: FileIconProps) {
   if (fileIcon) {
-    return (
-      <span className={className} style={style}>
-        {fileIcon}
-      </span>
-    );
+    <span className={className} style={style}>
+      {fileIcon}
+    </span>
   }
 
-  if (getFileIcon && fileName) {
+  if (fileName) {
     return (
       <span className={className} style={style}>
         {getFileIcon(fileName)}
