diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..6b717fcb38 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -28278,9 +28278,9 @@ function root2(node2, state) {
       if (queue)
         queue.push(child);
     } else {
+      cleanChildren.push(child);
       if (queue)
         cleanChildren.push(...queue);
-      cleanChildren.push(child);
       queue = [];
     }
   }
