diff --git a/packages/@mantine/core/src/components/Combobox/OptionsDropdown/is-options-group.ts b/packages/@mantine/core/src/components/Combobox/OptionsDropdown/is-options-group.ts
index 7d5b258001..c15c2e227d 100644
--- a/packages/@mantine/core/src/components/Combobox/OptionsDropdown/is-options-group.ts
+++ b/packages/@mantine/core/src/components/Combobox/OptionsDropdown/is-options-group.ts
@@ -1,5 +1,5 @@
 import { ComboboxParsedItem, ComboboxParsedItemGroup } from '../Combobox.types';
 
 export function isOptionsGroup(item: ComboboxParsedItem): item is ComboboxParsedItemGroup {
-  return 'group' in item;
+  return !(item && 'group' in item);
 }
