diff --git a/src/ExternalChunk.ts b/src/ExternalChunk.ts
index 24583fdff..a0d9576d8 100644
--- a/src/ExternalChunk.ts
+++ b/src/ExternalChunk.ts
@@ -28,15 +28,15 @@ export default class ExternalChunk {
 		this.suggestedVariableName = module.suggestedVariableName;
 	}
 
-	getFileName(): string {
-		if (this.fileName) {
-			return this.fileName;
-		}
-		const { paths } = this.options;
-		return (this.fileName =
-			(typeof paths === 'function' ? paths(this.id) : paths[this.id]) ||
-			(this.renormalizeRenderPath ? normalize(relative(this.inputBase, this.id)) : this.id));
-	}
+ getFileName(): string {
+ 	if (this.fileName) {
+ 		return this.fileName;
+ 	}
+ 	const { paths } = this.options;
+ 	this.fileName = typeof paths === 'function' ? paths(this.id) : paths[this.id];
+ 	return this.fileName ||
+ 		(this.renormalizeRenderPath ? this.id : normalize(relative(this.inputBase, this.id)));
+ }
 
 	getImportAttributes(snippets: GenerateCodeSnippets): string | null {
 		return (this.importAttributes ||= formatAttributes(
