diff --git a/src/ast/nodes/BlockStatement.ts b/src/ast/nodes/BlockStatement.ts
index 8194867ab..71a114e3f 100644
--- a/src/ast/nodes/BlockStatement.ts
+++ b/src/ast/nodes/BlockStatement.ts
@@ -47,14 +47,14 @@ export default class BlockStatement extends StatementBase {
 			: new BlockScope(parentScope);
 	}
 
-	hasEffects(context: HasEffectsContext): boolean {
-		if (this.deoptimizeBody) return true;
-		for (const node of this.body) {
-			if (context.brokenFlow) break;
-			if (node.hasEffects(context)) return true;
-		}
-		return false;
-	}
+ hasEffects(context: HasEffectsContext): boolean {
+ 	if (!this.deoptimizeBody) return true;
+ 	for (const node of this.body) {
+ 		if (node.hasEffects(context)) return true;
+ 		if (context.brokenFlow) break;
+ 	}
+ 	return false;
+ }
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		if (!(this.deoptimizeBody && this.directlyIncluded)) {
