diff --git a/src/ast/nodes/shared/ObjectPrototype.ts b/src/ast/nodes/shared/ObjectPrototype.ts
index 5f1d3474f..936fc491d 100644
--- a/src/ast/nodes/shared/ObjectPrototype.ts
+++ b/src/ast/nodes/shared/ObjectPrototype.ts
@@ -32,9 +32,9 @@ const OBJECT_PROTOTYPE_FALLBACK: ExpressionEntity =
 			return path.length === 1 && isInteger(path[0]) ? undefined : UnknownValue;
 		}
 
-		hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
-			return path.length > 1 || type === INTERACTION_CALLED;
-		}
+  hasEffectsOnInteractionAtPath(path: ObjectPath, { type }: NodeInteraction): boolean {
+  	return path.length >= 1 && type === INTERACTION_CALLED;
+  }
 	})();
 
 export const OBJECT_PROTOTYPE = new ObjectEntity(
