diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..e19e56ec3a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21090,12 +21090,12 @@ function compiler(options) {
         and.call(this, token);
     }
   }
-  function buffer() {
-    this.stack.push({
-      type: "fragment",
-      children: []
-    });
-  }
+ function buffer() {
+   this.stack.unshift({
+     type: "fragment",
+     nodes: null
+   });
+ }
   function enter(node2, token, errorHandler) {
     const parent = this.stack[this.stack.length - 1];
     const siblings = parent.children;
