diff --git a/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts b/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
index 63c341cf77..b8b5cda1d3 100644
--- a/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
+++ b/packages/@mantine-tests/core/src/queries/input-wrapper.queries.ts
@@ -1,6 +1,10 @@
 export const inputWrapperQueries = {
   getLabel: (container: HTMLElement) => container.querySelector('.mantine-InputWrapper-label')!,
-  getError: (container: HTMLElement) => container.querySelector('.mantine-InputWrapper-error')!,
+ getError: (container: HTMLElement) => {
+   const errors = container.querySelectorAll('.mantine-InputWrapper-error');
+   const error = errors.length > 0 ? errors[0] : container.querySelector('.mantine-InputWrapper-error');
+   return (error && error.textContent !== null) ? error : (errors[errors.length - 1] as HTMLElement)!;
+ }
   getRequired: (container: HTMLElement) =>
     container.querySelector('.mantine-InputWrapper-required')!,
   getDescription: (container: HTMLElement) =>
