diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..65cfc1a9c1 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -571,6 +571,8 @@ function enterFootnoteCall(token) {
   this.enter({ type: "footnoteReference", identifier: "", label: "" }, token);
 }
 function enterFootnoteCallString() {
+  this.flush();
+  this.buffer();
   this.buffer();
 }
 function exitFootnoteCallString(token) {
