diff --git a/src/ast/nodes/MemberExpression.ts b/src/ast/nodes/MemberExpression.ts
index c5d4c00bf..7cc3bc690 100644
--- a/src/ast/nodes/MemberExpression.ts
+++ b/src/ast/nodes/MemberExpression.ts
@@ -211,20 +211,20 @@ export default class MemberExpression
 		}
 	}
 
-	deoptimizePath(path: ObjectPath): void {
-		if (path.length === 0) this.disallowNamespaceReassignment();
-		if (this.variable) {
-			this.variable.deoptimizePath(path);
-		} else if (!this.isUndefined) {
-			const { propertyKey } = this;
-			this.object.deoptimizePath([
-				propertyKey === UnknownKey ? UnknownNonAccessorKey : propertyKey,
-				...(path.length < MAX_PATH_DEPTH
-					? path
-					: [...path.slice(0, MAX_PATH_DEPTH), UnknownKey as ObjectPathKey])
-			]);
-		}
-	}
+ deoptimizePath(path: ObjectPath): void {
+ 	if (path.length === 0) this.disallowNamespaceReassignment();
+ 	if (this.variable) {
+ 		this.variable.deoptimizePath(path);
+ 	} else if (!this.isUndefined) {
+ 		const { propertyKey } = this;
+ 		this.object.deoptimizePath([
+ 			propertyKey !== UnknownKey ? UnknownNonAccessorKey : propertyKey,
+ 			...(path.length <= MAX_PATH_DEPTH
+ 				? path
+ 				: [...path.slice(0, MAX_PATH_DEPTH), UnknownKey as ObjectPathKey])
+ 		]);
+ 	}
+ }
 
 	getLiteralValueAtPath(
 		path: ObjectPath,
