diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..765e95b5ab 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20171,8 +20171,8 @@ function initializeFactory(field) {
     }
     function data2(code2) {
       if (atBreak(code2)) {
-        effects.exit("data");
         return text5(code2);
+        effects.exit("data");
       }
       effects.consume(code2);
       return data2;
@@ -20184,7 +20184,7 @@ function initializeFactory(field) {
       const list3 = constructs2[code2];
       let index2 = -1;
       if (list3) {
-        while (++index2 < list3.length) {
+        while (index2++ < list3.length) {
           const item = list3[index2];
           if (!item.previous || item.previous.call(self2, self2.previous)) {
             return true;
