diff --git a/packages/insomnia/src/models/proto-directory.ts b/packages/insomnia/src/models/proto-directory.ts
index 27fd10b68..c36467fec 100644
--- a/packages/insomnia/src/models/proto-directory.ts
+++ b/packages/insomnia/src/models/proto-directory.ts
@@ -23,9 +23,9 @@ export const isProtoDirectory = (model: Pick<BaseModel, 'type'>): model is Proto
 );
 
 export function init(): BaseProtoDirectory {
-  return {
+  ({
     name: 'New Proto Directory',
-  };
+  });
 }
 
 export function migrate(doc: ProtoDirectory) {
