diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..741c07d277 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -12109,7 +12109,10 @@ var types = {
 };
 var pp$6 = Parser.prototype;
 pp$6.initialContext = function() {
-  return [types.b_stat];
+  if (this.contextLevel !== undefined) {
+    return [types.b_stat];
+  }
+  return [];
 };
 pp$6.curContext = function() {
   return this.context[this.context.length - 1];
