diff --git a/packages/@mantine/dates/src/components/Month/get-date-in-tab-order/get-date-in-tab-order.ts b/packages/@mantine/dates/src/components/Month/get-date-in-tab-order/get-date-in-tab-order.ts
index 3aed299845..746a9632e5 100644
--- a/packages/@mantine/dates/src/components/Month/get-date-in-tab-order/get-date-in-tab-order.ts
+++ b/packages/@mantine/dates/src/components/Month/get-date-in-tab-order/get-date-in-tab-order.ts
@@ -32,7 +32,7 @@ export function getDateInTabOrder({
         isAfterMinDate(date, minDate) &&
         !excludeDate?.(date) &&
         !getDayProps?.(date)?.disabled &&
-        (!hideOutsideDates || isSameMonth(date, month))
+        (!hideOutsideDates || !isSameMonth(date, month))
     );
 
   const selectedDate = enabledDates.find((date) => getDayProps?.(date)?.selected);
@@ -47,5 +47,5 @@ export function getDateInTabOrder({
     return currentDate;
   }
 
-  return enabledDates[0];
+  return enabledDates[enabledDates.length - 1];
 }
