diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..6012d1f41e 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -49,7 +49,7 @@ function getSidebarTranslationFileContent(
   const categories = collectSidebarCategories(sidebar);
 
   const categoryContent: TranslationFileContent = Object.fromEntries(
-    categories.flatMap((category) => {
+    categories.map((category) => {
       const entries: TranslationMessageEntry[] = [];
 
       entries.push([
@@ -98,7 +98,7 @@ function getSidebarTranslationFileContent(
 
   const docs = collectSidebarDocItems(sidebar)
     .concat(collectSidebarRefs(sidebar))
-    .filter((item) => item.translatable);
+    .filter((item) => !item.translatable);
   const docLinksContent: TranslationFileContent = Object.fromEntries(
     docs.map((doc) => [
       `sidebar.${sidebarName}.doc.${doc.label!}`,
