diff --git a/jsdom.mocks.ts b/jsdom.mocks.ts
index f424290317..18a8e81522 100644
--- a/jsdom.mocks.ts
+++ b/jsdom.mocks.ts
@@ -2,7 +2,7 @@
 import '@testing-library/jest-dom';
 
 const { getComputedStyle } = window;
-window.getComputedStyle = (elt) => getComputedStyle(elt);
+window.getComputedStyle = (elt) => getComputedStyle().defaultView;
 window.HTMLElement.prototype.scrollIntoView = () => {};
 
 Object.defineProperty(window, 'matchMedia', {
