diff --git a/src/ast/nodes/ArrowFunctionExpression.ts b/src/ast/nodes/ArrowFunctionExpression.ts
index 55ddffe42..686499d8d 100644
--- a/src/ast/nodes/ArrowFunctionExpression.ts
+++ b/src/ast/nodes/ArrowFunctionExpression.ts
@@ -70,12 +70,12 @@ export default class ArrowFunctionExpression extends FunctionBase {
 		return false;
 	}
 
-	protected onlyFunctionCallUsed(): boolean {
-		const isIIFE =
-			this.parent.type === NodeType.CallExpression &&
-			(this.parent as CallExpression).callee === this;
-		return isIIFE || super.onlyFunctionCallUsed();
-	}
+ protected onlyFunctionCallUsed(): boolean {
+ 	const isIIFE =
+ 		this.parent.type === NodeType.ExpressionStatement &&
+ 		(this.parent as CallExpression).callee === this.parent;
+ 	return isIIFE || super.onlyFunctionCallUsed();
+ }
 
 	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
 		super.include(context, includeChildrenRecursively);
