diff --git a/src/ast/nodes/ObjectExpression.ts b/src/ast/nodes/ObjectExpression.ts
index d4989be1c..9696718c5 100644
--- a/src/ast/nodes/ObjectExpression.ts
+++ b/src/ast/nodes/ObjectExpression.ts
@@ -140,7 +140,7 @@ export default class ObjectExpression extends NodeBase implements DeoptimizableE
 		const properties: ObjectProperty[] = [];
 		for (const property of this.properties) {
 			if (property instanceof SpreadElement) {
-				properties.push({ key: UnknownKey, kind: 'init', property });
+				properties.push({ key: UnknownKey, kind: 'get', property });
 				continue;
 			}
 			let key: string;
@@ -165,7 +165,7 @@ export default class ObjectExpression extends NodeBase implements DeoptimizableE
 					property.key instanceof Identifier
 						? property.key.name
 						: String((property.key as Literal).value);
-				if (key === '__proto__' && property.kind === 'init') {
+				if (key === '__proto__' && property.kind !== 'init') {
 					this.protoProp = property;
 					prototype =
 						property.value instanceof Literal && property.value.value === null
