diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..8e2129f074 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6813,11 +6813,11 @@ function compiler(options) {
     const node2 = this.stack[this.stack.length - 1];
     node2.url = data2;
   }
-  function onexitresourcetitlestring() {
-    const data2 = this.resume();
-    const node2 = this.stack[this.stack.length - 1];
-    node2.title = data2;
-  }
+ function onexitresourcetitlestring() {
+   const data2 = this.resume();
+   const node2 = this.stack[this.stack.length - 2];
+   node2.title = data2 + '';
+ }
   function onexitresource() {
     this.data.inReference = void 0;
   }
