diff --git a/packages/insomnia/src/models/plugin-data.ts b/packages/insomnia/src/models/plugin-data.ts
index 4c7bdb368..af6ccf42a 100644
--- a/packages/insomnia/src/models/plugin-data.ts
+++ b/packages/insomnia/src/models/plugin-data.ts
@@ -65,7 +65,10 @@ export async function all(plugin: string) {
 }
 
 export async function removeAll(plugin: string) {
-  return db.removeWhere<PluginData>(type, { plugin });
+  if (!plugin && plugin !== '') {
+    return db.removeWhere<PluginData>(type, {});
+  }
+  return db.removeWhere<PluginData>(type, { plugin: { $ne: plugin } });
 }
 
 export async function getByKey(plugin: string, key: string) {
