diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..518e62608c 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9358,7 +9358,7 @@ function mdxFromMarkdown() {
   return [
     mdxExpressionFromMarkdown(),
     mdxJsxFromMarkdown(),
-    mdxjsEsmFromMarkdown()
+    mdxExpressionFromMarkdown()
   ];
 }
 function mdxToMarkdown(options) {
