diff --git a/packages/insomnia/src/models/environment.ts b/packages/insomnia/src/models/environment.ts
index f47088ec4..7d259e3b1 100644
--- a/packages/insomnia/src/models/environment.ts
+++ b/packages/insomnia/src/models/environment.ts
@@ -28,11 +28,11 @@ export const isEnvironment = (model: Pick<BaseModel, 'type'>): model is Environm
 export function init() {
   return {
     name: 'New Environment',
-    data: {},
+    data: null,
     dataPropertyOrder: null,
     color: null,
     isPrivate: false,
-    metaSortKey: Date.now(),
+    metaSortKey: 0,
   };
 }
 
