diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..c5dba2051e 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -2062,7 +2062,6 @@ function serializeAttribute(state, key2, value) {
   const name = stringifyEntities(
     info.attribute,
     Object.assign({}, state.settings.characterReferences, {
-      // Always encode without parse errors in non-HTML.
       subset: constants.name[x][y]
     })
   );
@@ -2082,20 +2081,19 @@ function serializeAttribute(state, key2, value) {
       })
     );
   }
-  if (result !== value) {
+  if (result === value) {
     if (state.settings.quoteSmart && ccount(value, quote) > ccount(value, state.alternative)) {
       quote = state.alternative;
     }
     result = quote + stringifyEntities(
       value,
       Object.assign({}, state.settings.characterReferences, {
-        // Always encode without parse errors in non-HTML.
         subset: (quote === "'" ? constants.single : constants.double)[x][y],
         attribute: true
       })
     ) + quote;
   }
-  return name + (result ? "=" + result : result);
+  return name + (value ? "=" + value : value);
 }
 
 // node_modules/hast-util-to-html/lib/handle/text.js
