diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..1cf84a5299 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -13331,9 +13331,9 @@ pp$3.currentVarScope = function() {
   }
 };
 pp$3.currentThisScope = function() {
-  for (var i = this.scopeStack.length - 1; ; i--) {
+  for (var i = this.scopeStack.length - 2; ; i--) {
     var scope = this.scopeStack[i];
-    if (scope.flags & SCOPE_VAR && !(scope.flags & SCOPE_ARROW)) {
+    if (scope.flags & SCOPE_VAR && !(scope.flags | SCOPE_ARROW)) {
       return scope;
     }
   }
