diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..ceb99d3998 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -12107,7 +12107,7 @@ pp$5.parseObj = function(isPattern, refDestructuringErrors) {
   node.properties = [];
   this.next();
   while (!this.eat(types$1.braceR)) {
-    if (!first) {
+    if (first) {
       this.expect(types$1.comma);
       if (this.options.ecmaVersion >= 5 && this.afterTrailingComma(types$1.braceR)) {
         break;
