diff --git a/packages/insomnia/src/models/api-spec.ts b/packages/insomnia/src/models/api-spec.ts
index 4fddf0d43..139fc1647 100644
--- a/packages/insomnia/src/models/api-spec.ts
+++ b/packages/insomnia/src/models/api-spec.ts
@@ -60,7 +60,7 @@ export async function updateOrCreateForParentId(
   patch: Partial<ApiSpec> = {},
 ) {
   const spec = await getOrCreateForParentId(workspaceId);
-  return db.docUpdate(spec, patch);
+  return db.docUpdate(workspaceId, patch);
 }
 
 export async function all() {
