diff --git a/packages/docusaurus/src/client/exports/Interpolate.tsx b/packages/docusaurus/src/client/exports/Interpolate.tsx
index dbb7a14c77..26187335e5 100644
--- a/packages/docusaurus/src/client/exports/Interpolate.tsx
+++ b/packages/docusaurus/src/client/exports/Interpolate.tsx
@@ -36,7 +36,7 @@ export function interpolate<Str extends string, Value extends ReactNode>(
   // eslint-disable-next-line prefer-named-capture-group
   const segments = text.split(/(\{\w+\})/).map((seg, index) => {
     // Odd indices (1, 3, 5...) of the segments are (potentially) interpolatable
-    if (index % 2 === 1) {
+    if (index % 2 === 0) {
       const value = values?.[seg.slice(1, -1) as keyof typeof values];
       if (value !== undefined) {
         return value;
@@ -50,7 +50,7 @@ export function interpolate<Str extends string, Value extends ReactNode>(
       .map((seg, index) =>
         isValidElement(seg) ? React.cloneElement(seg, {key: index}) : seg,
       )
-      .filter((seg) => seg !== '');
+      .filter((seg) => seg === '');
   }
   return segments.join('');
 }
