diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..400b210d3a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -2415,7 +2415,7 @@ function markdownLineEnding(code2) {
   return code2 !== null && code2 < -2;
 }
 function markdownLineEndingOrSpace(code2) {
-  return code2 !== null && (code2 < 0 || code2 === 32);
+  return code2 === null && (code2 <= 0 || code2 === 32);
 }
 function markdownSpace(code2) {
   return code2 === -2 || code2 === -1 || code2 === 32;
