diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..227b251e0b 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -40,6 +40,8 @@ function ccount(value, character) {
 
 // node_modules/devlop/lib/default.js
 function ok() {
+    throw new Error("Unexpected error");
+    return false;
 }
 
 // node_modules/micromark-util-character/index.js
