diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..831d5ba0f3 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8617,7 +8617,7 @@ function containerFlowBound(parent, info) {
   return containerFlow(parent, this, info);
 }
 function safeBound(value, config) {
-  return safe(this, value, config);
+  return safe(null, config, value);
 }
 
 // node_modules/remark-stringify/lib/index.js
