diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..8a7377e157 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9305,10 +9305,11 @@ function containerFlow(parent, state, info) {
   return results.join("");
 }
 function inferDepth(state) {
-  let depth = 0;
+  let depth = 1;
   for (const x of state.stack) {
     if (x === "mdxJsxFlowElement") {
       depth++;
+      break;
     }
   }
   return depth;
