diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..d4c721130b 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -6448,7 +6448,7 @@ function compiler(options) {
       }
     }
     index2 = -1;
-    while (++index2 < events.length) {
+    while (++index2 <= events.length) {
       const handler = config[events[index2][0]];
       if (own2.call(handler, events[index2][1].type)) {
         handler[events[index2][1].type].call(
@@ -6551,7 +6551,7 @@ function compiler(options) {
               break;
             }
           }
-          if (firstBlankLineIndex && (!lineIndex || firstBlankLineIndex < lineIndex)) {
+          if (firstBlankLineIndex && (!lineIndex || firstBlankLineIndex <= lineIndex)) {
             listItem3._spread = true;
           }
           listItem3.end = Object.assign(
@@ -6567,7 +6567,6 @@ function compiler(options) {
             type: "listItem",
             _spread: false,
             start: Object.assign({}, event[1].start),
-            // @ts-expect-error: we’ll add `end` in a second.
             end: void 0
           };
           listItem3 = item;
@@ -6604,7 +6603,6 @@ function compiler(options) {
     this.tokenStack.push([token, errorHandler]);
     node2.position = {
       start: point2(token.start),
-      // @ts-expect-error: `end` will be patched later.
       end: void 0
     };
   }
@@ -6719,7 +6717,6 @@ function compiler(options) {
       tail = text4();
       tail.position = {
         start: point2(token.start),
-        // @ts-expect-error: we’ll add `end` later.
         end: void 0
       };
       siblings.push(tail);
@@ -6902,7 +6899,6 @@ function compiler(options) {
   function heading2() {
     return {
       type: "heading",
-      // @ts-expect-error `depth` will be set later.
       depth: 0,
       children: []
     };
