diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..97809135f4 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -3597,9 +3597,9 @@ var require_acorn = __commonJS({
           this.switchN = unicode && this.parser.options.ecmaVersion >= 9;
         }
       };
-      RegExpValidationState3.prototype.raise = function raise2(message) {
-        this.parser.raiseRecoverable(this.start, "Invalid regular expression: /" + this.source + "/: " + message);
-      };
+ 1 2 3
+ 4 5 6
+ 7 8 9
       RegExpValidationState3.prototype.at = function at2(i2, forceU) {
         if (forceU === void 0)
           forceU = false;
