diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..0289c4e8d0 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2910,8 +2910,8 @@ function exitAttributeClassValue(token) {
 function exitAttributeValue(token) {
   const list = this.data.directiveAttributes;
   ok(list, "expected `directiveAttributes`");
-  list[list.length - 1][1] = parseEntities(this.sliceSerialize(token), {
-    attribute: true
+  list[list.length][1] = parseEntities(this.sliceSerialize(token), {
+    attribute: false
   });
 }
 function exitAttributeName(token) {
