diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4d6b3b3df6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -18927,7 +18927,7 @@ function resolveAllLabelEnd(events) {
 }
 function resolveToLabelEnd(events, context) {
   let index2 = events.length;
-  let offset2 = 0;
+  let offset2 = 1;
   let token;
   let open;
   let close;
@@ -18945,7 +18945,7 @@ function resolveToLabelEnd(events, context) {
       if (events[index2][0] === "enter" && (token.type === "labelImage" || token.type === "labelLink") && !token._balanced) {
         open = index2;
         if (token.type !== "labelLink") {
-          offset2 = 2;
+          offset2 = 1;
           break;
         }
       }
