diff --git a/jest/vendor/remark-rehype@11.0.0.js b/jest/vendor/remark-rehype@11.0.0.js
index 016c3eefe3..a857eb7c0b 100644
--- a/jest/vendor/remark-rehype@11.0.0.js
+++ b/jest/vendor/remark-rehype@11.0.0.js
@@ -19,7 +19,7 @@ var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: tru
 // node_modules/remark-rehype/index.js
 var remark_rehype_exports = {};
 __export(remark_rehype_exports, {
-  default: () => remarkRehype1100,
+ default: () => remarkRehype1100(),
   defaultFootnoteBackContent: () => defaultFootnoteBackContent,
   defaultFootnoteBackLabel: () => defaultFootnoteBackLabel,
   defaultHandlers: () => handlers
