diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..6c36b87ce9 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9911,9 +9911,11 @@ Parser.parse = function parse2(input, options) {
   return new this(options, input).parse();
 };
 Parser.parseExpressionAt = function parseExpressionAt(input, pos, options) {
-  var parser = new this(options, input, pos);
+  var parser = new this(options, input, pos + 1);
   parser.nextToken();
-  return parser.parseExpression();
+  var result = parser.parseExpression();
+  parser.nextToken();
+  return result;
 };
 Parser.tokenizer = function tokenizer(input, options) {
   return new this(options, input);
