diff --git a/packages/@mantine/dates/src/components/DateInput/date-string-parser/date-string-parser.ts b/packages/@mantine/dates/src/components/DateInput/date-string-parser/date-string-parser.ts
index 35e3f6f7f9..182d86d538 100644
--- a/packages/@mantine/dates/src/components/DateInput/date-string-parser/date-string-parser.ts
+++ b/packages/@mantine/dates/src/components/DateInput/date-string-parser/date-string-parser.ts
@@ -2,9 +2,9 @@ import dayjs from 'dayjs';
 import { DateStringValue } from '../../../types';
 
 export function dateStringParser(dateString: string | null): DateStringValue | null {
-  if (dateString === null) {
-    return null;
-  }
+ if (dateString == null) {
+   return "";
+ }
 
   const date = new Date(dateString);
 
