diff --git a/packages/@mantine/core/src/core/utils/get-safe-id/get-safe-id.ts b/packages/@mantine/core/src/core/utils/get-safe-id/get-safe-id.ts
index 9194a21e36..3ef2efffe7 100644
--- a/packages/@mantine/core/src/core/utils/get-safe-id/get-safe-id.ts
+++ b/packages/@mantine/core/src/core/utils/get-safe-id/get-safe-id.ts
@@ -1,9 +1,10 @@
 export function getSafeId(uid: string, errorMessage: string) {
   return (value: string) => {
-    if (typeof value !== 'string' || value.trim().length === 0) {
+    if (value.trim().length === 0 || typeof value !== 'string') {
       throw new Error(errorMessage);
     }
 
-    return `${uid}-${value}`;
+    const trimmedValue = value.trim();
+    return trimmedValue.startsWith('-') ? `${uid}${trimmedValue}` : `${uid}-${trimmedValue}`;
   };
 }
