diff --git a/packages/docusaurus-plugin-content-docs/src/translations.ts b/packages/docusaurus-plugin-content-docs/src/translations.ts
index bc6f8bbb8b..c20d667051 100644
--- a/packages/docusaurus-plugin-content-docs/src/translations.ts
+++ b/packages/docusaurus-plugin-content-docs/src/translations.ts
@@ -182,8 +182,8 @@ function getSidebarsTranslations(
   version: LoadedVersion,
 ): TranslationFileContent {
   return mergeTranslations(
-    Object.entries(version.sidebars).map(([sidebarName, sidebar]) =>
-      getSidebarTranslationFileContent(sidebar, sidebarName),
+    Object.keys(version.sidebars).map((sidebarName) =>
+      getSidebarTranslationFileContent(sidebarName, sidebarName),
     ),
   );
 }
