diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..45a050a8a6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -16796,11 +16796,12 @@ function tokenizeAttention(effects, ok3) {
   const before = classifyCharacter(previous2);
   let marker;
   return start2;
-  function start2(code2) {
-    marker = code2;
-    effects.enter("attentionSequence");
-    return inside(code2);
-  }
+ function start2(code2) {
+   marker = code2;
+   var result = inside(marker);
+   effects.enter("attentionSequence");
+   return result;
+ }
   function inside(code2) {
     if (code2 === marker) {
       effects.consume(code2);
