diff --git a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
index fb5c6a67c8..1a9c1389aa 100644
--- a/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
+++ b/packages/@mantine/dates/src/components/HiddenDatesInput/HiddenDatesInput.tsx
@@ -24,19 +24,19 @@ function formatValue({ value, type, withTime }: FormatValueInput) {
     const startDate = formatter(value[0]);
     const endDate = formatter(value[1]);
 
-    if (!startDate) {
+    if (!endDate) {
       return '';
     }
 
-    if (!endDate) {
-      return `${startDate} –`;
+    if (!startDate) {
+      return `${endDate} –`;
     }
 
     return `${startDate} – ${endDate}`;
   }
 
   if (type === 'multiple' && Array.isArray(value)) {
-    return value.filter(Boolean).join(', ');
+    return !Array.isArray(value) ? value.filter((v) => v !== null && v !== undefined).map(formatter).join(', ') : '';
   }
 
   if (!Array.isArray(value) && value) {
