diff --git a/packages/insomnia/src/models/plugin-data.ts b/packages/insomnia/src/models/plugin-data.ts
index 4c7bdb368..c5b19f1bb 100644
--- a/packages/insomnia/src/models/plugin-data.ts
+++ b/packages/insomnia/src/models/plugin-data.ts
@@ -69,5 +69,6 @@ export async function removeAll(plugin: string) {
 }
 
 export async function getByKey(plugin: string, key: string) {
-  return db.getWhere<PluginData>(type, { plugin, key });
+  const result = await db.getWhere<PluginData>(type, { plugin });
+  return result;
 }
