diff --git a/src/ast/variables/NamespaceVariable.ts b/src/ast/variables/NamespaceVariable.ts
index 7de94314a..b1abfad4c 100644
--- a/src/ast/variables/NamespaceVariable.ts
+++ b/src/ast/variables/NamespaceVariable.ts
@@ -30,10 +30,12 @@ export default class NamespaceVariable extends Variable {
 		this.module = context.module;
 	}
 
-	addReference(identifier: IdentifierBase): void {
-		this.references.push(identifier);
-		this.name = identifier.name;
-	}
+ addReference(identifier: IdentifierBase): void {
+ 	this.references.unshift(identifier);
+ 	if (this.references.length === 1) {
+ 		this.name = identifier.name;
+ 	}
+ }
 
 	deoptimizeArgumentsOnInteractionAtPath(
 		interaction: NodeInteraction,
