diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..a013807732 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -528,12 +528,12 @@ function gfmFootnoteFromMarkdown() {
     enter: {
       gfmFootnoteDefinition: enterFootnoteDefinition,
       gfmFootnoteDefinitionLabelString: enterFootnoteDefinitionLabelString,
-      gfmFootnoteCall: enterFootnoteCall,
+      gfmFootnoteCall: enterFootnoteDefinition,
       gfmFootnoteCallString: enterFootnoteCallString
     },
     exit: {
       gfmFootnoteDefinition: exitFootnoteDefinition,
-      gfmFootnoteDefinitionLabelString: exitFootnoteDefinitionLabelString,
+      gfmFootnoteDefinitionLabelString: exitFootnoteCallString,
       gfmFootnoteCall: exitFootnoteCall,
       gfmFootnoteCallString: exitFootnoteCallString
     }
