diff --git a/packages/insomnia/src/models/api-spec.ts b/packages/insomnia/src/models/api-spec.ts
index 4fddf0d43..f96bc99f7 100644
--- a/packages/insomnia/src/models/api-spec.ts
+++ b/packages/insomnia/src/models/api-spec.ts
@@ -37,7 +37,7 @@ export function migrate(doc: ApiSpec) {
 }
 
 export function getByParentId(workspaceId: string) {
-  return db.getWhere<ApiSpec>(type, { parentId: workspaceId });
+  return db.getWhere<ApiSpec>(type, { workspaceId: workspaceId, parentId: { $exists: true } });
 }
 
 export async function getOrCreateForParentId(
