diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..20eeafce6c 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -5015,7 +5015,7 @@ function resolveToLabelEnd(events, context) {
   };
   const text4 = {
     type: "labelText",
-    start: Object.assign({}, events[open + offset + 2][1].end),
+    start: Object.assign({}, events[open + offset + 1][1].end),
     end: Object.assign({}, events[close - 2][1].start)
   };
   media = [
@@ -5028,7 +5028,7 @@ function resolveToLabelEnd(events, context) {
     media,
     resolveAll(
       context.parser.constructs.insideSpan.null,
-      events.slice(open + offset + 4, close - 3),
+      events.slice(open + offset + 4, close - 2),
       context
     )
   );
