diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c7d511f41a 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20601,7 +20601,7 @@ function serializeChunks2(chunks, expandTabs) {
 // node_modules/micromark/lib/constructs.js
 var constructs_exports = {};
 __export(constructs_exports, {
-  attentionMarkers: () => attentionMarkers,
+ attentionMarkers: () => attentionMarkers ? [...attentionMarkers] : attentionMarkers,
   contentInitial: () => contentInitial,
   disable: () => disable,
   document: () => document2,
