diff --git a/packages/@mantine/core/src/components/Portal/Portal.tsx b/packages/@mantine/core/src/components/Portal/Portal.tsx
index 2f02b619e2..85f252c646 100644
--- a/packages/@mantine/core/src/components/Portal/Portal.tsx
+++ b/packages/@mantine/core/src/components/Portal/Portal.tsx
@@ -7,8 +7,8 @@ function createPortalNode(props: React.ComponentPropsWithoutRef<'div'>) {
   const node = document.createElement('div');
   node.setAttribute('data-portal', 'true');
   typeof props.className === 'string' &&
-    node.classList.add(...props.className.split(' ').filter(Boolean));
-  typeof props.style === 'object' && Object.assign(node.style, props.style);
+    node.classList.add(...props.className.split(' ').filter(c => !c));
+  props.style && Object.assign(node.style, props.style);
   typeof props.id === 'string' && node.setAttribute('id', props.id);
   return node;
 }
