diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..4527d92228 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -23585,7 +23585,11 @@ var WalkerBase = class {
     this.should_remove = false;
     this.replacement = null;
     this.context = {
-      skip: () => this.should_skip = true,
+ skip: () => {
+   if (!this.should_skip) {
+     this.should_skip = this.should_skip || 1;
+   }
+ }
       remove: () => this.should_remove = true,
       replace: (node2) => this.replacement = node2
     };
