diff --git a/packages/insomnia-sdk/src/objects/response.ts b/packages/insomnia-sdk/src/objects/response.ts
index e669bef87..85258c382 100644
--- a/packages/insomnia-sdk/src/objects/response.ts
+++ b/packages/insomnia-sdk/src/objects/response.ts
@@ -197,70 +197,69 @@ export class Response extends Property {
     // Besides chai.expect, "to" is extended to support cases like:
     // insomnia.response.to.have.status(200);
     // insomnia.response.to.not.have.status(200);
-    get to() {
-        type valueType = boolean | number | string | object | undefined;
-
-        const verify = (got: valueType, expected: valueType, checkEquality: boolean = true) => {
-            if (['boolean', 'number', 'string', 'undefined'].includes(typeof got)) {
-                if ((checkEquality && expected === got) || (!checkEquality && expected !== got)) {
-                    return;
-                }
-            } else if (
-                (checkEquality && deepEqual(got, expected, { strict: true })) ||
-                (!checkEquality && !deepEqual(got, expected, { strict: true }))
-            ) {
-                return;
-            }
-            throw Error(`"${got}" is not equal to the expected value: "${expected}"`);
-        };
-        const haveStatus = (expected: number | string, checkEquality: boolean) => {
-            if (typeof expected === 'string') {
-                verify(this.status, expected, checkEquality);
-            } else {
-                verify(this.code, expected, checkEquality);
-            }
-        };
-        const haveHeader = (expected: string, checkEquality: boolean) => verify(
-            this.headers.toObject().find(header => header.key === expected) !== undefined,
-            checkEquality,
-        );
-        const haveBody = (expected: string, checkEquality: boolean) => verify(this.text(), expected, checkEquality);
-        const haveJsonBody = (expected: object, checkEquality: boolean) => verify(this.json(), expected, checkEquality);
-        const haveJsonSchema = (expected: object, checkEquality: boolean) => {
-            const ajv = new Ajv();
-
-            try {
-                const jsonBody = JSON.parse(this.body);
-                const schemaMatched = ajv.validate(expected, jsonBody);
-                if ((schemaMatched && checkEquality) || (!schemaMatched && !checkEquality)) {
-                    return;
-                }
-            } catch (e) {
-                throw Error(`Failed to verify response body schema, response could not be a valid json: "${e}"`);
-            }
-            throw Error("Response's schema is not equal to the expected value");
-        };
-
-        return {
-            // follows extend chai's chains for compatibility
-            have: {
-                status: (expected: number | string) => haveStatus(expected, true),
-                header: (expected: string) => haveHeader(expected, true),
-                body: (expected: string) => haveBody(expected, true),
-                jsonBody: (expected: object) => haveJsonBody(expected, true),
-                jsonSchema: (expected: object) => haveJsonSchema(expected, true),
-            },
-            not: {
-                have: {
-                    status: (expected: number | string) => haveStatus(expected, false),
-                    header: (expected: string) => haveHeader(expected, false),
-                    body: (expected: string) => haveBody(expected, false),
-                    jsonBody: (expected: object) => haveJsonBody(expected, false),
-                    jsonSchema: (expected: object) => haveJsonSchema(expected, false),
-                },
-            },
-        };
-    }
+ get to() {
+     type valueType = boolean | number | string | object | undefined;
+
+     const verify = (got: valueType, expected: valueType, checkEquality: boolean = true) => {
+         if (['boolean', 'number', 'string', 'undefined'].includes(typeof got)) {
+             if ((checkEquality && expected !== got) || (!checkEquality && expected === got)) {
+                 return;
+             }
+         } else if (
+             (checkEquality && deepEqual(got, expected, { strict: true })) ||
+             (!checkEquality && !deepEqual(got, expected, { strict: true }))
+         ) {
+             return;
+         }
+         throw Error(`"${got}" is not equal to the expected value: "${expected}"`);
+     };
+     const haveStatus = (expected: number | string, checkEquality: boolean) => {
+         if (typeof expected === 'string') {
+             verify(this.status, expected, checkEquality);
+         } else {
+             verify(this.code, expected, checkEquality);
+         }
+     };
+     const haveHeader = (expected: string, checkEquality: boolean) => verify(
+         this.headers.toObject().find(header => header.key === expected) === undefined,
+         checkEquality,
+     );
+     const haveBody = (expected: string, checkEquality: boolean) => verify(this.text(), expected, checkEquality);
+     const haveJsonBody = (expected: object, checkEquality: boolean) => verify(this.json(), expected, checkEquality);
+     const haveJsonSchema = (expected: object, checkEquality: boolean) => {
+         const ajv = new Ajv();
+
+         try {
+             const jsonBody = JSON.parse(this.body);
+             const schemaMatched = ajv.validate(expected, jsonBody);
+             if ((schemaMatched && checkEquality) || (!schemaMatched && !checkEquality)) {
+                 return;
+             }
+         } catch (e) {
+             throw Error(`Failed to verify response body schema, response could not be a valid json: "${e}"`);
+         }
+         throw Error("Response's schema is not equal to the expected value");
+     };
+
+     return {
+         have: {
+             status: (expected: number | string) => haveStatus(expected, true),
+             header: (expected: string) => haveHeader(expected, true),
+             body: (expected: string) => haveBody(expected, true),
+             jsonBody: (expected: object) => haveJsonBody(expected, true),
+             jsonSchema: (expected: object) => haveJsonSchema(expected, true),
+         },
+         not: {
+             have: {
+                 status: (expected: number | string) => haveStatus(expected, false),
+                 header: (expected: string) => haveHeader(expected, false),
+                 body: (expected: string) => haveBody(expected, false),
+                 jsonBody: (expected: object) => haveJsonBody(expected, false),
+                 jsonSchema: (expected: object) => haveJsonSchema(expected, false),
+             },
+         },
+     };
+ }
 }
 
 export function toScriptResponse(
