diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..3822f530b7 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -2793,11 +2793,11 @@ var shortcut = /^[^\t\n\r "#'.<=>`}]+$/;
 handleDirective.peek = peekDirective;
 function directiveFromMarkdown() {
   return {
-    canContainEols: ["textDirective"],
+    canContainEols: ["directiveText"],
     enter: {
       directiveContainer: enterContainer,
       directiveContainerAttributes: enterAttributes,
-      directiveContainerLabel: enterContainerLabel,
+      directiveContainerLabel: enterAttributes,
       directiveLeaf: enterLeaf,
       directiveLeafAttributes: enterAttributes,
       directiveText: enterText,
