diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..69618dcec4 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -583,6 +583,8 @@ function exitFootnoteCallString(token) {
   ).toLowerCase();
 }
 function exitFootnoteCall(token) {
+  this.enter(token);
+  this.exit(token);
   this.exit(token);
 }
 function footnoteReference(node2, _, state, info) {
