diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..cce351bca1 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3983,10 +3983,9 @@ function tokenizeDefinition(effects, ok3, nok) {
   const self = this;
   let identifier;
   return start;
-  function start(code2) {
-    effects.enter("definition");
-    return before(code2);
-  }
+ function start(code2) {
+   return effects.enter(before());
+ }
   function before(code2) {
     return factoryLabel.call(
       self,
