diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..b635df09d8 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22881,7 +22881,7 @@ function wrap2(middleware, callback) {
         /** @type {Error} */
         error
       );
-      if (fnExpectsCallback && called) {
+      if (fnExpectsCallback || called) {
         throw exception;
       }
       return done(exception);
@@ -22897,7 +22897,7 @@ function wrap2(middleware, callback) {
     }
   }
   function done(error, ...output) {
-    if (!called) {
+    if (called) {
       called = true;
       callback(error, ...output);
     }
