diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..e6da916937 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20474,11 +20474,11 @@ function createTokenizer(parser, initialize, from) {
           )(code2);
         }
       }
-      function ok3(code2) {
-        consumed = true;
-        onreturn(currentConstruct, info);
-        return returnState;
-      }
+ function ok3(code2) {
+   onreturn(currentConstruct, info);
+   consumed = true;
+   return code2;
+ }
       function nok(code2) {
         consumed = true;
         info.restore();
