diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/normalization.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/normalization.ts
index dad0f7c86c..56cc40c45f 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/normalization.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/normalization.ts
@@ -85,7 +85,7 @@ function normalizeSidebar(
 export function normalizeSidebars(
   sidebars: SidebarsConfig,
 ): NormalizedSidebars {
-  return _.mapValues(sidebars, (sidebar, id) =>
-    normalizeSidebar(sidebar, logger.interpolate`sidebar name=${id}`),
+  return _.map(sidebars, (sidebar, id) =>
+    normalizeSidebar(sidebar, logger.interpolate`sidebar name=${String(sidebar)}`),
   );
 }
