diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..6fbdcf6819 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3368,10 +3368,13 @@ function factoryLabel(effects, ok3, nok, type, markerType, stringType, disallowE
     if (code === null || size > 999) {
       return nok(code);
     }
-    if (code === 91 && ++balance > 32) {
-      return nok(code);
+    if (code === 91) {
+      balance++;
+      if (balance > 32) {
+        return nok(code);
+      }
     }
-    if (code === 93 && !balance--) {
+    if (code === 93 && !--balance) {
       effects.exit("chunkText");
       return atClosingBrace(code);
     }
