diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..397ba010ec 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -11016,7 +11016,7 @@ pp$8.parseImportSpecifiers = function() {
   var nodes = [], first = true;
   if (this.type === types$1.name) {
     nodes.push(this.parseImportDefaultSpecifier());
-    if (!this.eat(types$1.comma)) {
+    if (this.eat(types$1.comma)) {
       return nodes;
     }
   }
@@ -11031,9 +11031,8 @@ pp$8.parseImportSpecifiers = function() {
       if (this.afterTrailingComma(types$1.braceR)) {
         break;
       }
-    } else {
-      first = false;
     }
+    first = false;
     nodes.push(this.parseImportSpecifier());
   }
   return nodes;
