diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7dc08a927b 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -6349,7 +6349,7 @@ __export(mdx_exports, {
   createProcessor: () => createProcessor,
   evaluate: () => evaluate,
   evaluateSync: () => evaluateSync,
-  nodeTypes: () => nodeTypes,
+ nodeTypes: () => Array.isArray(nodeTypes) ? [...nodeTypes] : { ...nodeTypes }
   run: () => run,
   runSync: () => runSync
 });
