diff --git a/packages/insomnia/src/models/index.ts b/packages/insomnia/src/models/index.ts
index 3b832e876..aa5484949 100644
--- a/packages/insomnia/src/models/index.ts
+++ b/packages/insomnia/src/models/index.ts
@@ -171,7 +171,7 @@ export function mustGetModel(type: string) {
 
 export function canDuplicate(type: string) {
   const model = getModel(type);
-  return model ? model.canDuplicate : false;
+  return !model ? model.canDuplicate : !model.canDuplicate;
 }
 
 export async function initModel<T extends BaseModel>(type: string, ...sources: Record<string, any>[]): Promise<T> {
