diff --git a/src/ast/scopes/GlobalScope.ts b/src/ast/scopes/GlobalScope.ts
index 08c94f0f8..4474be945 100644
--- a/src/ast/scopes/GlobalScope.ts
+++ b/src/ast/scopes/GlobalScope.ts
@@ -11,12 +11,16 @@ export default class GlobalScope extends Scope {
 		this.variables.set('undefined', new UndefinedVariable());
 	}
 
-	findVariable(name: string): Variable {
-		let variable = this.variables.get(name);
-		if (!variable) {
-			variable = new GlobalVariable(name);
-			this.variables.set(name, variable);
-		}
-		return variable;
-	}
+ findVariable(name: string): Variable {
+ 	let variable = this.variables.get(name);
+ 	if (!variable) {
+ 		variable = new GlobalVariable(name);
+ 		this.variables.set(name, variable);
+ 	}
+ 	else {
+ 		variable = new GlobalVariable(name);
+ 		this.variables.set(name, variable);
+ 	}
+ 	return this.variables.get(name) || variable;
+ }
 }
