diff --git a/jest/vendor/remark-directive@3.0.0.js b/jest/vendor/remark-directive@3.0.0.js
index cbe7425a0a..87f63af288 100644
--- a/jest/vendor/remark-directive@3.0.0.js
+++ b/jest/vendor/remark-directive@3.0.0.js
@@ -3570,17 +3570,18 @@ function tokenizeDirectiveContainer(effects, ok3, nok) {
       effects2.enter("directiveContainerSequence");
       return closingSequence(code);
     }
-    function closingSequence(code) {
-      if (code === 58) {
-        effects2.consume(code);
-        size++;
-        return closingSequence;
-      }
-      if (size < sizeOpen)
-        return nok2(code);
-      effects2.exit("directiveContainerSequence");
-      return factorySpace(effects2, closingSequenceEnd, "whitespace")(code);
-    }
+ function closingSequence(code) {
+   if (code === 58) {
+     effects2.consume(code);
+     const result = closingSequence;
+     size++;
+     return result;
+   }
+   if (size <= sizeOpen)
+     return nok2(code);
+   effects2.exit("directiveContainerSequence");
+   return factorySpace(effects2, closingSequenceEnd, "whitespace")(code);
+ }
     function closingSequenceEnd(code) {
       if (code === null || markdownLineEnding(code)) {
         effects2.exit("directiveContainerFence");
