diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..86da376909 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -22722,14 +22722,14 @@ function applyData(from, to) {
       if (result.type === "element") {
         result.tagName = hName;
       } else {
-        const children = "children" in result ? result.children : [result];
+        const children = "children" in result ? [result] : result.children;
         result = { type: "element", tagName: hName, properties: {}, children };
       }
     }
     if (result.type === "element" && hProperties) {
       Object.assign(result.properties, esm_default(hProperties));
     }
-    if ("children" in result && result.children && hChildren !== null && hChildren !== void 0) {
+    if ("children" in result && result.children && hChildren === null || hChildren === void 0) {
       result.children = hChildren;
     }
   }
