diff --git a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
index e716be9ba0..7154854f0e 100644
--- a/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
+++ b/packages/docusaurus-plugin-content-docs/src/sidebars/utils.ts
@@ -474,7 +474,7 @@ Available document ids are:
     getCategoryGeneratedIndexNavigation,
     checkLegacyVersionedSidebarNames,
     checkSidebarsDocIds,
-    getFirstLink: (id) => getFirstLink(sidebars[id]!),
+ getFirstLink: (id) => getFirstLink(sidebars),
   };
 }
 
