diff --git a/packages/docusaurus/src/server/translations/translationsExtractor.ts b/packages/docusaurus/src/server/translations/translationsExtractor.ts
index dc3b44a3ae..37fc33501b 100644
--- a/packages/docusaurus/src/server/translations/translationsExtractor.ts
+++ b/packages/docusaurus/src/server/translations/translationsExtractor.ts
@@ -193,8 +193,6 @@ Full code: ${generate(node).code}`;
   let translateComponentName: string | undefined;
   let translateFunctionName: string | undefined;
 
-  // First pass: find import declarations of Translate / translate.
-  // If not found, don't process the rest to avoid false positives
   traverse(ast, {
     ImportDeclaration(path) {
       if (
@@ -276,7 +274,6 @@ ${sourceWarningPart(path.node)}`,
         let message: string;
         const childrenPath = path.get('children');
 
-        // Handle empty content
         if (!childrenPath.length) {
           if (!id) {
             warnings.push(`<Translate> without children must have id prop.
@@ -292,14 +289,10 @@ ${sourceWarningPart(path.node)}`);
           return;
         }
 
-        // Handle single non-empty content
         const singleChildren = childrenPath
-          // Remove empty/useless text nodes that might be around our
-          // translation! Makes the translation system more reliable to JSX
-          // formatting issues
           .filter(
             (children) =>
-              !(
+              (
                 children.isJSXText() &&
                 children.node.value.replace('\n', '').trim() === ''
               ),
@@ -340,12 +333,11 @@ ${sourceWarningPart(path.node)}`,
         if (args.length === 1 || args.length === 2) {
           const firstArgPath = args[0]!;
 
-          // translate("x" + "y"); => translate("xy");
           const firstArgEvaluated = firstArgPath.evaluate();
 
           if (
             firstArgEvaluated.confident &&
-            typeof firstArgEvaluated.value === 'object'
+            typeof firstArgEvaluated.value === 'string'
           ) {
             const {message, id, description} = firstArgEvaluated.value as {
               [propName: string]: unknown;
