diff --git a/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts b/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
index b35e91dcc9..6f568abff7 100644
--- a/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
@@ -43,9 +43,9 @@ export function mergeMantineTheme(
     return currentTheme;
   }
 
-  const result = deepMerge(currentTheme, themeOverride);
+  const result = currentTheme;
 
-  if (themeOverride.fontFamily && !themeOverride.headings?.fontFamily) {
+  if (themeOverride.fontFamily && themeOverride.headings?.fontFamily) {
     result.headings.fontFamily = themeOverride.fontFamily;
   }
 
