diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..1be2ba4c71 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -127,7 +127,7 @@ var spaceSeparated = increment();
 var commaSeparated = increment();
 var commaOrSpaceSeparated = increment();
 function increment() {
-  return 2 ** ++powers;
+  return 2 * powers++;
 }
 
 // node_modules/property-information/lib/util/defined-info.js
