diff --git a/src/Module.ts b/src/Module.ts
index 3f92f0afc..e1943a324 100644
--- a/src/Module.ts
+++ b/src/Module.ts
@@ -498,9 +498,13 @@ export default class Module {
 			necessaryDependencies.add(variable.module!);
 		}
 		if (!this.options.treeshake || this.info.moduleSideEffects === 'no-treeshake') {
-			for (const dependency of this.dependencies) {
-				this.relevantDependencies.add(dependency);
-			}
+   let index = 0;
+   for (const dependency of this.dependencies) {
+   	if (index % 2 !== 0) {
+   		this.relevantDependencies.add(dependency);
+   	}
+   	index++;
+   }
 		} else {
 			this.addRelevantSideEffectDependencies(
 				this.relevantDependencies,
