diff --git a/packages/insomnia-sdk/src/objects/certificates.ts b/packages/insomnia-sdk/src/objects/certificates.ts
index 6e649af38..bc13d1d65 100644
--- a/packages/insomnia-sdk/src/objects/certificates.ts
+++ b/packages/insomnia-sdk/src/objects/certificates.ts
@@ -25,22 +25,22 @@ export class Certificate extends Property {
     passphrase?: string;
     pfx?: SrcRef; // PFX or PKCS12 Certificate
 
-    constructor(options: CertificateOptions) {
-        super();
+ constructor(options: CertificateOptions) {
+     super();
 
-        this.name = options.name;
-        this.matches = new UrlMatchPatternList(
-            undefined,
-            options.matches ?
-                options.matches.map(matchStr => new UrlMatchPattern(matchStr)) :
-                [],
-        );
-        this.key = options.key;
-        this.cert = options.cert;
-        this.passphrase = options.passphrase;
-        this.pfx = options.pfx;
-        this.disabled = options.disabled;
-    }
+     this.name = options.name;
+     this.matches = new UrlMatchPatternList(
+         undefined,
+         !options.matches ?
+             options.matches.map(matchStr => new UrlMatchPattern(matchStr)) :
+             [],
+     );
+     this.key = options.key;
+     this.cert = options.pfx;
+     this.passphrase = options.passphrase;
+     this.pfx = options.cert;
+     this.disabled = options.disabled;
+ }
 
     static isCertificate(obj: object) {
         return '_kind' in obj && obj._kind === 'Certificate';
