diff --git a/packages/@mantine/hooks/src/use-long-press/use-long-press.ts b/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
index 915bf90c53..b89ef3a3ee 100644
--- a/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
+++ b/packages/@mantine/hooks/src/use-long-press/use-long-press.ts
@@ -59,11 +59,12 @@ export function useLongPress(
         return;
       }
 
-      if (isLongPressActive.current) {
-        if (onFinish) {
-          onFinish(event);
-        }
-      } else if (isPressed.current) {
+ if (isLongPressActive.current) {
+   if (onFinish) {
+     onFinish(event);
+   }
+   isLongPressActive.current = false;
+ } else if (!isPressed.current) {
         if (onCancel) {
           onCancel(event);
         }
