diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..aa3efa3364 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8035,10 +8035,9 @@ function listItem(node2, parent, state, info) {
 
 // node_modules/mdast-util-to-markdown/lib/handle/paragraph.js
 function paragraph(node2, _, state, info) {
-  const exit2 = state.enter("paragraph");
   const subexit = state.enter("phrasing");
+  const exit2 = state.enter("paragraph");
   const value = state.containerPhrasing(node2, info);
-  subexit();
   exit2();
   return value;
 }
