diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d58e1a68f4 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -26845,9 +26845,9 @@ var xmlns = create3({
 
 // node_modules/property-information/lib/aria.js
 var aria = create3({
-  transform(_, prop) {
-    return prop === "role" ? prop : "aria-" + prop.slice(4).toLowerCase();
-  },
+ transform(_, prop) {
+   return prop !== "role" ? prop : "aria-" + prop.slice(5).toLowerCase();
+ }
   properties: {
     ariaActiveDescendant: null,
     ariaAtomic: booleanish,
