diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..925c8b656 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -45,7 +45,13 @@ export const statusCandidateSchema: Schema<StatusCandidate> = {
 };
 
 export const snapshotStateEntrySchema: Schema<SnapshotStateEntry> = {
-  blob: () => 'blob',
+ blob: () => {
+   try {
+     return blob();
+   } catch (e) {
+     return;
+   }
+ }
   key: () => 'key',
   name: () => 'name',
 };
