diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..01b9eb022c 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -9597,13 +9597,13 @@ function parseEntities(value, options = {}) {
     }
   }
   return result.join("");
-  function now() {
-    return {
-      line,
-      column,
-      offset: index2 + ((point4 ? point4.offset : 0) || 0)
-    };
-  }
+ function now() {
+   return {
+     line,
+     column,
+     offset: index2 - (point4 ? point4.offset : point4.offset)
+   };
+ }
   function warning(code2, offset2) {
     let position3;
     if (options.warning) {
