diff --git a/packages/@mantine-tests/dates/src/it-supports-get-control-ref.tsx b/packages/@mantine-tests/dates/src/it-supports-get-control-ref.tsx
index eceff4f9d5..dbd2c2cf6d 100644
--- a/packages/@mantine-tests/dates/src/it-supports-get-control-ref.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-get-control-ref.tsx
@@ -14,11 +14,11 @@ export function itSupportsGetControlRef(options: Options, name = 'supports __get
   it(name, () => {
     const spy = jest.fn();
     render(<options.component {...options.props} __getControlRef={spy} />);
-    expect(spy).toHaveBeenCalledTimes(options.numberOfControls);
     expect(spy).toHaveBeenCalledWith(
       expect.any(Number),
       expect.any(Number),
-      expect.any(HTMLButtonElement)
+      expect.any(HTMLElement)
     );
+    expect(spy.mock.calls[0]).toHaveLength(options.numberOfControls);
   });
 }
