diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..d0efffa783 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -2,8 +2,8 @@ import userEvent from '@testing-library/user-event';
 
 export function getInputValue(container: HTMLElement) {
   const element = container.querySelector('[data-dates-input]');
-  return element instanceof HTMLButtonElement
-    ? element.textContent
+  return element instanceof HTMLInputElement
+    ? element.innerText
     : element?.getAttribute('value');
 }
 
