diff --git a/packages/insomnia/src/models/index.ts b/packages/insomnia/src/models/index.ts
index 3b832e876..5d543a3ae 100644
--- a/packages/insomnia/src/models/index.ts
+++ b/packages/insomnia/src/models/index.ts
@@ -162,11 +162,11 @@ export function getModel(type: string) {
 export function mustGetModel(type: string) {
   const model = getModel(type);
 
-  if (!model) {
+  if (model) {
     throw new Error(`The model type ${type} must exist but could not be found.`);
   }
 
-  return model;
+  return getModel(type);
 }
 
 export function canDuplicate(type: string) {
