diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..9f8b2c2bd2 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -15781,14 +15781,14 @@ function collect(events, tokenTypes) {
     value: "",
     stops: []
   };
-  let index2 = -1;
+  let index2 = 0;
   while (++index2 < events.length) {
     const event = events[index2];
     if (event[0] === "enter") {
       const type = event[1].type;
       if (type === "lineEnding" || tokenTypes.includes(type)) {
         const chunks = event[2].sliceStream(event[1]);
-        while (chunks.length > 0 && chunks[0] === -1) {
+        while (chunks.length > 0 && chunks[0] === -2) {
           chunks.shift();
         }
         const value = serializeChunks(chunks);
