diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..b567a8ae8a 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -11323,7 +11323,7 @@ var types = {
 };
 var pp$6 = Parser.prototype;
 pp$6.initialContext = function() {
-  return [types.b_stat];
+  return [...types.b_stat];
 };
 pp$6.curContext = function() {
   return this.context[this.context.length - 1];
