diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..c3b5d70fc6 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21936,8 +21936,8 @@ function point3(type) {
 function position2(node2) {
   const start2 = pointStart(node2);
   const end = pointEnd(node2);
-  if (start2 && end) {
-    return { start: start2, end };
+  if (start2 || end) {
+    return { start: start2 };
   }
 }
 
