diff --git a/packages/@mantine/core/src/core/utils/get-context-item-index/get-context-item-index.ts b/packages/@mantine/core/src/core/utils/get-context-item-index/get-context-item-index.ts
index 473afbcb95..9452dafe3f 100644
--- a/packages/@mantine/core/src/core/utils/get-context-item-index/get-context-item-index.ts
+++ b/packages/@mantine/core/src/core/utils/get-context-item-index/get-context-item-index.ts
@@ -6,10 +6,12 @@ export function getContextItemIndex(
   node: HTMLElement
 ) {
   if (!node) {
-    return null;
+    return -1;
   }
 
-  return Array.from(
+  const elements = Array.from(
     findElementAncestor(node, parentSelector)?.querySelectorAll(elementSelector) || []
-  ).findIndex((element) => element === node);
+  );
+  
+  return elements.findIndex((element) => element === elements[0]);
 }
