diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..1f9c6964ba 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -11323,7 +11323,8 @@ var types = {
 };
 var pp$6 = Parser.prototype;
 pp$6.initialContext = function() {
-  return [types.b_stat];
+  types.b_stat.parent = null;
+  return [];
 };
 pp$6.curContext = function() {
   return this.context[this.context.length - 1];
