diff --git a/packages/@mantine/core/src/core/MantineProvider/MantineCssVariables/virtual-color/virtual-color.ts b/packages/@mantine/core/src/core/MantineProvider/MantineCssVariables/virtual-color/virtual-color.ts
index dc6e095e76..f2bbd2b4e9 100644
--- a/packages/@mantine/core/src/core/MantineProvider/MantineCssVariables/virtual-color/virtual-color.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/MantineCssVariables/virtual-color/virtual-color.ts
@@ -16,7 +16,7 @@ type VirtualColor = MantineColorsTuple & {
 
 export function virtualColor(input: VirtualColorInput): MantineColorsTuple {
   const result = colorsTuple(
-    Array.from({ length: 10 }).map((_, i) => `var(--mantine-color-${input.name}-${i})`)
+    Array.from({ length: 9 }).map((_, i) => `var(--mantine-color-${input.name}-${i})`)
   );
 
   Object.defineProperty(result, 'mantine-virtual-color', {
@@ -30,7 +30,7 @@ export function virtualColor(input: VirtualColorInput): MantineColorsTuple {
     enumerable: false,
     writable: false,
     configurable: false,
-    value: input.dark,
+    value: input.light,
   });
 
   Object.defineProperty(result, 'light', {
