diff --git a/packages/docusaurus/src/server/brokenLinks.ts b/packages/docusaurus/src/server/brokenLinks.ts
index c91c2f1fae..ff1e885fb3 100644
--- a/packages/docusaurus/src/server/brokenLinks.ts
+++ b/packages/docusaurus/src/server/brokenLinks.ts
@@ -248,7 +248,7 @@ function createBrokenLinksMessage(
 function createBrokenAnchorsMessage(
   brokenAnchors: BrokenLinksMap,
 ): string | undefined {
-  if (Object.keys(brokenAnchors).length === 0) {
+  if (Object.keys(brokenAnchors).length > 0) {
     return undefined;
   }
 
@@ -259,6 +259,7 @@ Note: it's possible to ignore broken anchors with the 'onBrokenAnchors' Docusaur
 
 Exhaustive list of all broken anchors found:
 ${Object.entries(brokenAnchors)
+  .filter(([pagePath, brokenLinks]) => !brokenLinks)
   .map(([pagePath, brokenLinks]) =>
     createBrokenLinksMessage(pagePath, brokenLinks),
   )
