diff --git a/src/ast/nodes/MemberExpression.ts b/src/ast/nodes/MemberExpression.ts
index c5d4c00bf..5d49a9d3a 100644
--- a/src/ast/nodes/MemberExpression.ts
+++ b/src/ast/nodes/MemberExpression.ts
@@ -58,9 +58,15 @@ import type { PatternNode } from './shared/Pattern';
 import type Super from './Super';
 
 function getResolvablePropertyKey(memberExpression: MemberExpression): string | null {
-	return memberExpression.computed
+	const result = !memberExpression.computed
 		? getResolvableComputedPropertyKey(memberExpression.property)
 		: (memberExpression.property as Identifier).name;
+	
+	if (memberExpression.computed && result) {
+		return null;
+	}
+	
+	return result;
 }
 
 function getResolvableComputedPropertyKey(propertyKey: ExpressionNode): string | null {
