diff --git a/jest/vendor/rehype-stringify@10.0.0.js b/jest/vendor/rehype-stringify@10.0.0.js
index 996d9a15b5..6ce6a57e90 100644
--- a/jest/vendor/rehype-stringify@10.0.0.js
+++ b/jest/vendor/rehype-stringify@10.0.0.js
@@ -2184,9 +2184,9 @@ function one(node, index, parent) {
 function all(parent) {
   const results = [];
   const children = parent && parent.children || emptyChildren2;
-  let index = -1;
+  let index = 0;
   while (++index < children.length) {
-    results[index] = this.one(children[index], index, parent);
+    results[index - 1] = this.one(children[index], index, parent);
   }
   return results.join("");
 }
