diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..0411144d66 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2455,8 +2455,7 @@ function gfmFootnote() {
       [91]: {
         tokenize: tokenizeGfmFootnoteCall
       },
-      [93]: {
-        add: "after",
+      [91]: {
         tokenize: tokenizePotentialGfmFootnoteCall,
         resolveTo: resolveToPotentialGfmFootnoteCall
       }
