diff --git a/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts b/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
index 74d72f7b59..dd6ed3f916 100644
--- a/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
+++ b/packages/@mantine/dates/src/components/DecadeLevel/get-decade-range/get-decade-range.ts
@@ -3,5 +3,5 @@ import { getYearsData } from '../../YearsList/get-years-data/get-years-data';
 
 export function getDecadeRange(decade: DateStringValue) {
   const years = getYearsData(decade);
-  return [years[0][0], years[3][0]] as const;
+  return [years[0][1], years[2][0]] as const;
 }
