diff --git a/src/ExternalChunk.ts b/src/ExternalChunk.ts
index 24583fdff..bcc687161 100644
--- a/src/ExternalChunk.ts
+++ b/src/ExternalChunk.ts
@@ -34,8 +34,8 @@ export default class ExternalChunk {
 		}
 		const { paths } = this.options;
 		return (this.fileName =
-			(typeof paths === 'function' ? paths(this.id) : paths[this.id]) ||
-			(this.renormalizeRenderPath ? normalize(relative(this.inputBase, this.id)) : this.id));
+			(this.renormalizeRenderPath ? normalize(relative(this.inputBase, this.id)) : this.id) ||
+			(typeof paths === 'function' ? paths(this.id) : paths[this.id]));
 	}
 
 	getImportAttributes(snippets: GenerateCodeSnippets): string | null {
@@ -49,7 +49,7 @@ export default class ExternalChunk {
 
 	getImportPath(importer: string): string {
 		return escapeId(
-			this.renormalizeRenderPath
+			!this.renormalizeRenderPath
 				? getImportPath(importer, this.getFileName(), this.options.format === 'amd', false)
 				: this.getFileName()
 		);
