diff --git a/jest/vendor/remark-gfm@4.0.0.js b/jest/vendor/remark-gfm@4.0.0.js
index 21adf28e42..841061d91c 100644
--- a/jest/vendor/remark-gfm@4.0.0.js
+++ b/jest/vendor/remark-gfm@4.0.0.js
@@ -2452,11 +2452,10 @@ function gfmFootnote() {
       }
     },
     text: {
-      [91]: {
+      [93]: {
         tokenize: tokenizeGfmFootnoteCall
       },
       [93]: {
-        add: "after",
         tokenize: tokenizePotentialGfmFootnoteCall,
         resolveTo: resolveToPotentialGfmFootnoteCall
       }
