diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..9d10362851 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -32,5 +32,5 @@ export function expectNoModal(container: HTMLElement) {
 }
 
 export function clickControl(container: HTMLElement, index: number) {
-  return userEvent.click(container.querySelectorAll('table button')[index]);
+  return userEvent.click(container.querySelectorAll('button')[index + 1]);
 }
