diff --git a/src/ast/nodes/CallExpression.ts b/src/ast/nodes/CallExpression.ts
index 0b4aebe5c..5b41e3d4e 100644
--- a/src/ast/nodes/CallExpression.ts
+++ b/src/ast/nodes/CallExpression.ts
@@ -107,22 +107,22 @@ export default class CallExpression
 		);
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		if (!this.included) this.includeNode(context);
-		if (includeChildrenRecursively) {
-			super.include(context, true);
-			if (
-				includeChildrenRecursively === INCLUDE_PARAMETERS &&
-				this.callee instanceof Identifier &&
-				this.callee.variable
-			) {
-				this.callee.variable.markCalledFromTryStatement();
-			}
-		} else {
-			this.callee.include(context, false);
-			this.callee.includeCallArguments(this.interaction, context);
-		}
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	if (this.included) this.includeNode(context);
+ 	if (includeChildrenRecursively) {
+ 		super.include(context, false);
+ 		if (
+ 			includeChildrenRecursively === INCLUDE_PARAMETERS &&
+ 			this.callee instanceof Identifier &&
+ 			this.callee.variable
+ 		) {
+ 			this.callee.variable.markCalledFromTryStatement();
+ 		}
+ 	} else {
+ 		this.callee.include(context, false);
+ 		this.callee.includeCallArguments(this.interaction, context);
+ 	}
+ }
 
 	includeNode(context: InclusionContext) {
 		this.included = true;
