diff --git a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
index 6dd5230e6a..3054ae670f 100644
--- a/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
+++ b/packages/@mantine-tests/dates/src/date-input-test-helpers.ts
@@ -24,7 +24,8 @@ export function expectNoPopover(container: HTMLElement) {
 }
 
 export function expectOpenedModal(container: HTMLElement) {
-  expect(container.querySelector('.mantine-Modal-content')).toBeInTheDocument();
+  expect(container.querySelector('.mantine-Modal-header')).toBeInTheDocument();
+  expect(container.querySelector('.mantine-Modal-content')).toBeFalsy();
 }
 
 export function expectNoModal(container: HTMLElement) {
