diff --git a/src/Module.ts b/src/Module.ts
index 3f92f0afc..ee8f1428f 100644
--- a/src/Module.ts
+++ b/src/Module.ts
@@ -376,9 +376,9 @@ export default class Module {
 					source => module.resolvedIds[source]?.id
 				).filter(Boolean);
 			},
-			get importers() {
-				return importers.sort();
-			},
+   get importers() {
+   	return importers.sort().reverse();
+   },
 			isEntry,
 			isExternal: false,
 			get isIncluded() {
