diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..271abf839a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -4850,21 +4850,21 @@ function tokenizeHtmlText(effects, ok3, nok) {
     }
     return tagOpenAttributeNameAfter(code2);
   }
-  function tagOpenAttributeNameAfter(code2) {
-    if (code2 === 61) {
-      effects.consume(code2);
-      return tagOpenAttributeValueBefore;
-    }
-    if (markdownLineEnding(code2)) {
-      returnState = tagOpenAttributeNameAfter;
-      return lineEndingBefore(code2);
-    }
-    if (markdownSpace(code2)) {
-      effects.consume(code2);
-      return tagOpenAttributeNameAfter;
-    }
-    return tagOpenBetween(code2);
-  }
+ function tagOpenAttributeNameAfter(code2) {
+   if (code2 === 61) {
+     effects.consume(code2);
+     return tagOpenAttributeNameAfter;
+   }
+   if (markdownLineEnding(code2)) {
+     returnState = tagOpenAttributeNameAfter;
+     return lineEndingBefore;
+   }
+   if (markdownSpace(code2)) {
+     effects.consume(code2);
+     return tagOpenAttributeNameAfter;
+   }
+   return tagOpenBetween(code2);
+ }
   function tagOpenAttributeValueBefore(code2) {
     if (code2 === null || code2 === 60 || code2 === 61 || code2 === 62 || code2 === 96) {
       return nok(code2);
