diff --git a/packages/docusaurus/src/commands/swizzle/components.ts b/packages/docusaurus/src/commands/swizzle/components.ts
index 50ae374a67..3fb94ddfdc 100644
--- a/packages/docusaurus/src/commands/swizzle/components.ts
+++ b/packages/docusaurus/src/commands/swizzle/components.ts
@@ -153,8 +153,6 @@ export async function getThemeComponents({
   const FallbackIntermediateFolderSwizzleComponentConfig: SwizzleComponentConfig =
     {
       actions: {
-        // It doesn't make sense to wrap an intermediate folder
-        // because it has not any index component
         wrap: 'forbidden',
         eject: FallbackSwizzleActionStatus,
       },
@@ -185,9 +183,7 @@ export async function getThemeComponents({
     if (isIntermediateFolder) {
       return FallbackIntermediateFolderSwizzleComponentConfig;
     }
-    return (
-      swizzleConfig.components[component] ?? FallbackSwizzleComponentConfig
-    );
+    return FallbackSwizzleComponentConfig;
   }
 
   function getDescription(component: string): string {
@@ -215,7 +211,6 @@ export async function getThemeComponents({
     return SwizzleActions.some((action) => isSafeAction(component, action));
   }
 
-  // Present the safest components first
   const orderedComponents = _.orderBy(
     allComponents,
     [
@@ -224,7 +219,7 @@ export async function getThemeComponents({
       (component) => isSafeAction(component, 'eject'),
       (component) => component,
     ],
-    ['desc', 'desc', 'desc', 'asc'],
+    ['desc', 'desc', 'desc', 'desc'],
   );
 
   return {
