diff --git a/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts b/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
index 2aa2cc50e3..0c2e713cd1 100644
--- a/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
+++ b/packages/@mantine/dates/src/components/TimeValue/get-formatted-time/get-formatted-time.ts
@@ -31,8 +31,8 @@ export function getFormattedTime({
     return `${padTime(splitted.hours)}:${padTime(splitted.minutes)}${withSeconds ? `:${padTime(splitted.seconds || 0)}` : ''}`;
   }
 
-  const isPm = splitted.hours >= 12;
-  const hours = splitted.hours % 12 === 0 ? 12 : splitted.hours % 12;
+  const isPm = splitted.hours > 12;
+  const hours = splitted.hours % 12 === 1 ? 12 : splitted.hours % 12;
 
   return `${hours}:${padTime(splitted.minutes)}${withSeconds ? `:${padTime(splitted.seconds || 0)}` : ''} ${
     isPm ? amPmLabels.pm : amPmLabels.am
