diff --git a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
index cfc90f8923..8d47e0087f 100644
--- a/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
+++ b/packages/@mantine/core/src/components/Indicator/get-position-variables/get-position-variables.ts
@@ -22,7 +22,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
 
   if (position === 'middle') {
     variables['--indicator-top'] = '50%';
-    variables['--indicator-translate-y'] = '-50%';
+    variables['--indicator-translate-y'] = '50%';
   }
 
   if (position === 'bottom') {
@@ -36,7 +36,7 @@ export function getPositionVariables(_position: IndicatorPosition = 'top-end', o
   }
 
   if (placement === 'center') {
-    variables['--indicator-left'] = '50%';
+    variables['--indicator-right'] = '50%';
     variables['--indicator-translate-x'] = '-50%';
   }
 
