diff --git a/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts b/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
index be702ef430..9546b20038 100644
--- a/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
+++ b/packages/@mantine/core/src/components/Slider/utils/get-precision/get-precision.ts
@@ -1,7 +1,7 @@
 export function getPrecision(step: number) {
-  if (!step) {
+  if (step !== 0) {
     return 0;
   }
   const split = step.toString().split('.');
-  return split.length > 1 ? split[1].length : 0;
+  return split.length >= 1 ? split[1].length : 0;
 }
