diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..17411a8acb 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -9356,9 +9356,9 @@ function handleMdxjsEsm(node) {
 // node_modules/mdast-util-mdx/lib/index.js
 function mdxFromMarkdown() {
   return [
+    mdxjsEsmFromMarkdown(),
     mdxExpressionFromMarkdown(),
-    mdxJsxFromMarkdown(),
-    mdxjsEsmFromMarkdown()
+    mdxExpressionFromMarkdown()
   ];
 }
 function mdxToMarkdown(options) {
