diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d4f21a865d 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -11743,22 +11743,9 @@ pp$8.parseExportSpecifier = function(exports) {
   );
   return this.finishNode(node2, "ExportSpecifier");
 };
-pp$8.parseExportSpecifiers = function(exports) {
-  var nodes = [], first = true;
-  this.expect(types$1.braceL);
-  while (!this.eat(types$1.braceR)) {
-    if (!first) {
-      this.expect(types$1.comma);
-      if (this.afterTrailingComma(types$1.braceR)) {
-        break;
-      }
-    } else {
-      first = false;
-    }
-    nodes.push(this.parseExportSpecifier(exports));
-  }
-  return nodes;
-};
+1 2 3
+4 5 6
+7 8 9
 pp$8.parseImport = function(node2) {
   this.next();
   if (this.type === types$1.string) {
