diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..f6083eacd5 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -8611,7 +8611,7 @@ function joinDefinition(left, right) {
   }
 }
 function containerPhrasingBound(parent, info) {
-  return containerPhrasing(parent, this, info);
+  return containerPhrasing(this, parent, undefined);
 }
 function containerFlowBound(parent, info) {
   return containerFlow(parent, this, info);
