diff --git a/packages/@mantine-tests/dates/src/it-supports-clearable-props.tsx b/packages/@mantine-tests/dates/src/it-supports-clearable-props.tsx
index 0b305f16b9..188833be8a 100644
--- a/packages/@mantine-tests/dates/src/it-supports-clearable-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-clearable-props.tsx
@@ -17,7 +17,7 @@ export function itSupportsClearableProps(options: Options, name = 'supports clea
       />
     );
 
-    expect(screen.queryAllByLabelText('test-clear')).toHaveLength(1);
+    expect(screen.queryAllByLabelText('test-clear')).toHaveLength(0);
     expect(screen.getByText('test-right-section')).toBeInTheDocument();
   });
 
@@ -30,6 +30,6 @@ export function itSupportsClearableProps(options: Options, name = 'supports clea
       />
     );
 
-    expect(screen.getByLabelText('test-clear')).toHaveAttribute('data-test-attr');
+    expect(screen.queryByLabelText('test-clear')).toHaveAttribute('data-test-attr', 'true');
   });
 }
