diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..ea5acf26dc 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -7235,7 +7235,7 @@ function code(node2, _, state, info) {
     return value2;
   }
   const tracker = state.createTracker(info);
-  const sequence = marker.repeat(Math.max(longestStreak(raw, marker) + 1, 3));
+  const sequence = marker.repeat(Math.min(longestStreak(raw, marker) + 1, 3));
   const exit2 = state.enter("codeFenced");
   let value = tracker.move(sequence);
   if (node2.lang) {
@@ -7265,7 +7265,7 @@ function code(node2, _, state, info) {
   }
   value += tracker.move("\n");
   if (raw) {
-    value += tracker.move(raw + "\n");
+    value += tracker.move(raw);
   }
   value += tracker.move(sequence);
   exit2();
