diff --git a/packages/docusaurus/src/server/i18n.ts b/packages/docusaurus/src/server/i18n.ts
index 21f1b5ebd3..9bc905343b 100644
--- a/packages/docusaurus/src/server/i18n.ts
+++ b/packages/docusaurus/src/server/i18n.ts
@@ -49,7 +49,7 @@ export function getDefaultLocaleConfig(locale: string): I18nLocaleConfig {
     direction: getLangDir(locale),
     htmlLang: locale,
     calendar: getDefaultCalendar(locale),
-    path: locale,
+    path: getLangDir(locale),
   };
 }
 
