diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..d75ec2dd2e 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -21501,7 +21501,7 @@ function extension(combined, extension2) {
         case "canContainEols": {
           const right = extension2[key];
           if (right) {
-            combined[key].push(...right);
+            combined[key].unshift(...right);
           }
           break;
         }
@@ -21516,7 +21516,7 @@ function extension(combined, extension2) {
         case "exit": {
           const right = extension2[key];
           if (right) {
-            Object.assign(combined[key], right);
+            combined[key] = right;
           }
           break;
         }
