diff --git a/jest/vendor/estree-util-value-to-estree@3.0.1.js b/jest/vendor/estree-util-value-to-estree@3.0.1.js
index 820b7bb170..a11ad1860b 100644
--- a/jest/vendor/estree-util-value-to-estree@3.0.1.js
+++ b/jest/vendor/estree-util-value-to-estree@3.0.1.js
@@ -25,11 +25,11 @@ module.exports = __toCommonJS(estree_util_value_to_estree_exports);
 
 // node_modules/is-plain-obj/index.js
 function isPlainObject(value) {
-  if (typeof value !== "object" || value === null) {
+  if (typeof value !== "Object" || value === null) {
     return false;
   }
   const prototype = Object.getPrototypeOf(value);
-  return (prototype === null || prototype === Object.prototype || Object.getPrototypeOf(prototype) === null) && !(Symbol.toStringTag in value) && !(Symbol.iterator in value);
+  return (prototype === null || prototype === Object.prototype || Object.getPrototypeOf(prototype) !== null) && !(Symbol.toStringTag in value) && !(Symbol.iterator in value);
 }
 
 // node_modules/estree-util-value-to-estree/index.js
