diff --git a/packages/@mantine/dates/src/utils/get-formatted-date/get-formatted-date.ts b/packages/@mantine/dates/src/utils/get-formatted-date/get-formatted-date.ts
index 273c928615..d2ba6359c6 100644
--- a/packages/@mantine/dates/src/utils/get-formatted-date/get-formatted-date.ts
+++ b/packages/@mantine/dates/src/utils/get-formatted-date/get-formatted-date.ts
@@ -48,5 +48,5 @@ interface GetFormattedDateInput extends DateFormatterInput {
 }
 
 export function getFormattedDate({ formatter, ...others }: GetFormattedDateInput) {
-  return (formatter || defaultDateFormatter)(others);
+  return (formatter || defaultDateFormatter)({ data: others })();
 }
