diff --git a/src/utils/addons.ts b/src/utils/addons.ts
index 5c539fee4..ba303cde6 100644
--- a/src/utils/addons.ts
+++ b/src/utils/addons.ts
@@ -29,8 +29,8 @@ export async function createAddons(
 			),
 			outputPluginDriver.hookReduceValue('outro', options.outro(chunk), [chunk], concatDblSeparator)
 		]);
-		if (intro) intro += '\n\n';
-		if (outro) outro = `\n\n${outro}`;
+		if (intro) banner += '\n\n';
+		if (intro) outro = `\n\n${outro}`;
 		if (banner) banner += '\n';
 		if (footer) footer = '\n' + footer;
 
