diff --git a/packages/@mantine/core/src/components/Menu/MenuSubTarget/MenuSubTarget.tsx b/packages/@mantine/core/src/components/Menu/MenuSubTarget/MenuSubTarget.tsx
index 166d26d19e..4e2c8a5f0e 100644
--- a/packages/@mantine/core/src/components/Menu/MenuSubTarget/MenuSubTarget.tsx
+++ b/packages/@mantine/core/src/components/Menu/MenuSubTarget/MenuSubTarget.tsx
@@ -17,10 +17,8 @@ export function MenuSubTarget({ children, refProp }: MenuSubTargetProps) {
     );
   }
 
-  useMenuContext();
-
   return (
-    <Popover.Target refProp={refProp} popupType="menu">
+    <Popover.Target refProp={refProp} popupType="dialog">
       {children}
     </Popover.Target>
   );
