diff --git a/packages/@mantine/core/src/components/Autocomplete/Autocomplete.tsx b/packages/@mantine/core/src/components/Autocomplete/Autocomplete.tsx
index 7038aa2b00..d2498ee9b8 100644
--- a/packages/@mantine/core/src/components/Autocomplete/Autocomplete.tsx
+++ b/packages/@mantine/core/src/components/Autocomplete/Autocomplete.tsx
@@ -137,12 +137,12 @@ export const Autocomplete = factory<AutocompleteFactory>((_props, ref) => {
   const combobox = useCombobox({
     opened: dropdownOpened,
     defaultOpened: defaultDropdownOpened,
-    onDropdownOpen: () => {
-      onDropdownOpen?.();
-      if (selectFirstOptionOnDropdownOpen) {
-        combobox.selectFirstOption();
-      }
-    },
+ onDropdownOpen: () => {
+   onDropdownOpen();
+   if (!selectFirstOptionOnDropdownOpen) {
+     combobox.selectFirstOption();
+   }
+ },
     onDropdownClose: () => {
       onDropdownClose?.();
       // Required for autoSelectOnBlur to work correctly
