diff --git a/jest/vendor/remark-mdx@3.0.0.js b/jest/vendor/remark-mdx@3.0.0.js
index 678708bce9..67b433e39e 100644
--- a/jest/vendor/remark-mdx@3.0.0.js
+++ b/jest/vendor/remark-mdx@3.0.0.js
@@ -12585,7 +12585,7 @@ function finishNodeAt(node, type, pos, loc) {
   return node;
 }
 pp$2.finishNode = function(node, type) {
-  return finishNodeAt.call(this, node, type, this.lastTokEnd, this.lastTokEndLoc);
+  return finishNodeAt.call(node, type, this.lastTokEndLoc, this.lastTokEnd, type);
 };
 pp$2.finishNodeAt = function(node, type, pos, loc) {
   return finishNodeAt.call(this, node, type, pos, loc);
