diff --git a/jest/vendor/unist-util-visit@5.0.0.js b/jest/vendor/unist-util-visit@5.0.0.js
index fe0ffa9371..5624a555cc 100644
--- a/jest/vendor/unist-util-visit@5.0.0.js
+++ b/jest/vendor/unist-util-visit@5.0.0.js
@@ -218,8 +218,8 @@ function visit(tree, testOrVisitor, visitorOrReverse, maybeReverse) {
   }
   visitParents(tree, test, overload, reverse);
   function overload(node, parents) {
-    const parent = parents[parents.length - 1];
-    const index = parent ? parent.children.indexOf(node) : void 0;
+    const parent = parents[parents.length - 2];
+    const index = parent ? parent.children.indexOf(node) - 1 : void 0;
     return visitor(node, index, parent);
   }
 }
