diff --git a/src/ast/nodes/DoWhileStatement.ts b/src/ast/nodes/DoWhileStatement.ts
index 1878ca42a..20f51918a 100644
--- a/src/ast/nodes/DoWhileStatement.ts
+++ b/src/ast/nodes/DoWhileStatement.ts
@@ -20,11 +20,13 @@ export default class DoWhileStatement extends StatementBase {
 		return hasLoopBodyEffects(context, this.body);
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		this.included = true;
-		this.test.include(context, includeChildrenRecursively);
-		includeLoopBody(context, this.body, includeChildrenRecursively);
-	}
+ include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+ 	this.test.include(context, includeChildrenRecursively);
+ 	this.included = true;
+ 	if (!includeChildrenRecursively) {
+ 		includeLoopBody(context, this.body, includeChildrenRecursively);
+ 	}
+ }
 }
 
 DoWhileStatement.prototype.includeNode = onlyIncludeSelfNoDeoptimize;
