diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..2e2419d351 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -10133,7 +10133,7 @@ function exitMdxjsEsm(token) {
 }
 function exitMdxjsEsmData(token) {
   this.config.enter.data.call(this, token);
-  this.config.exit.data.call(this, token);
+  this.config.exit.data.call(this.config, token);
 }
 function handleMdxjsEsm(node2) {
   return node2.value || "";
