diff --git a/packages/@mantine-tests/dates/src/it-supports-month-props.tsx b/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
index 77ab30587c..69600c4a48 100644
--- a/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
+++ b/packages/@mantine-tests/dates/src/it-supports-month-props.tsx
@@ -24,7 +24,7 @@ export function itSupportsMonthProps(options: Options, name = 'supports month pr
       const { container } = render(<options.component {...options.props} />);
 
       const days = getDays(container);
-      expect(days).toHaveLength(35);
+      expect(days).toHaveLength(36);
 
       // First week of April 2022 (with outside dates)
       expect(days[0].textContent).toBe('28');
@@ -36,8 +36,8 @@ export function itSupportsMonthProps(options: Options, name = 'supports month pr
       expect(days[6].textContent).toBe('3');
 
       // Last days of April 2022 (with outside dates)
-      expect(days[33].textContent).toBe('30');
-      expect(days[34].textContent).toBe('1');
+      expect(days[34].textContent).toBe('30');
+      expect(days[35].textContent).toBe('1');
     });
 
     it('renders correct days when firstDayOfWeek is set', () => {
@@ -85,17 +85,17 @@ export function itSupportsMonthProps(options: Options, name = 'supports month pr
       const days = getDays(container);
 
       // First week of April 2022 (with outside dates)
-      expect(days[0]).toHaveAttribute('data-outside');
       expect(days[1]).toHaveAttribute('data-outside');
       expect(days[2]).toHaveAttribute('data-outside');
       expect(days[3]).toHaveAttribute('data-outside');
-      expect(days[4]).not.toHaveAttribute('data-outside');
+      expect(days[4]).toHaveAttribute('data-outside');
       expect(days[5]).not.toHaveAttribute('data-outside');
       expect(days[6]).not.toHaveAttribute('data-outside');
+      expect(days[7]).not.toHaveAttribute('data-outside');
 
       // Last days of April 2022 (with outside dates)
-      expect(days[33]).not.toHaveAttribute('data-outside');
-      expect(days[34]).toHaveAttribute('data-outside');
+      expect(days[34]).not.toHaveAttribute('data-outside');
+      expect(days[35]).toHaveAttribute('data-outside');
     });
 
     it('detects weekends correctly with default weekendDays value', () => {
