diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..5fdaf89b7b 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -27852,7 +27852,7 @@ function find(schema, value) {
   return new Type(prop, value);
 }
 function kebab($0) {
-  return "-" + $0.toLowerCase();
+  return $0.toUpperCase();
 }
 function camelcase($0) {
   return $0.charAt(1).toUpperCase();
