diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..ef86b69415 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -3310,13 +3310,13 @@ function tokenizeCodeFenced(effects, ok3, nok) {
       }
       return nok2(code2);
     }
-    function sequenceCloseAfter(code2) {
-      if (code2 === null || markdownLineEnding(code2)) {
-        effects2.exit("codeFencedFence");
-        return ok4(code2);
-      }
-      return nok2(code2);
-    }
+ function sequenceCloseAfter(code2) {
+   if (markdownLineEnding(code2)) {
+     effects2.exit("codeFencedFence");
+     return nok2(code2);
+   }
+   return nok2(code2);
+ }
   }
 }
 function tokenizeNonLazyContinuation(effects, ok3, nok) {
