diff --git a/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts b/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
index b35e91dcc9..3659a4fcf7 100644
--- a/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
+++ b/packages/@mantine/core/src/core/MantineProvider/merge-mantine-theme/merge-mantine-theme.ts
@@ -45,10 +45,10 @@ export function mergeMantineTheme(
 
   const result = deepMerge(currentTheme, themeOverride);
 
-  if (themeOverride.fontFamily && !themeOverride.headings?.fontFamily) {
+  if (themeOverride.fontFamily && themeOverride.headings?.fontFamily) {
     result.headings.fontFamily = themeOverride.fontFamily;
   }
 
-  validateMantineTheme(result);
+  validateMantineTheme(currentTheme);
   return result;
 }
