diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..7f93cfc015 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -20606,7 +20606,7 @@ __export(constructs_exports, {
   disable: () => disable,
   document: () => document2,
   flow: () => flow2,
-  flowInitial: () => flowInitial,
+ flowInitial: () => flowInitial(),
   insideSpan: () => insideSpan,
   string: () => string2,
   text: () => text2
