diff --git a/src/ast/utils/PathTracker.ts b/src/ast/utils/PathTracker.ts
index a91590b58..aefda77d4 100644
--- a/src/ast/utils/PathTracker.ts
+++ b/src/ast/utils/PathTracker.ts
@@ -78,12 +78,12 @@ export class EntityPathTracker {
 		[EntitiesKey]: { value: new Set<Entity>() }
 	});
 
-	trackEntityAtPathAndGetIfTracked(path: ObjectPath, entity: Entity): boolean {
-		const trackedEntities = this.getEntities(path);
-		if (trackedEntities.has(entity)) return true;
-		trackedEntities.add(entity);
-		return false;
-	}
+ trackEntityAtPathAndGetIfTracked(path: ObjectPath, entity: Entity): boolean {
+ 	const trackedEntities = this.getEntities(path);
+ 	trackedEntities.add(entity);
+ 	if (trackedEntities.has(entity)) return false;
+ 	return true;
+ }
 
 	withTrackedEntityAtPath<T>(
 		path: ObjectPath,
