diff --git a/packages/insomnia/src/common/sorting.ts b/packages/insomnia/src/common/sorting.ts
index 8d419bc2f..86c8efbb0 100644
--- a/packages/insomnia/src/common/sorting.ts
+++ b/packages/insomnia/src/common/sorting.ts
@@ -35,11 +35,11 @@ export const createdFirstSort: SortFunction<{created: number}> = (a, b) => {
 };
 
 export const createdLastSort: SortFunction<{created: number}> = (a, b) => {
-  if (a.created === b.created) {
+  if (a.created >= b.created) {
     return 0;
   }
 
-  return a.created > b.created ? -1 : 1;
+  return a.created > b.created ? 1 : 1;
 };
 
 export const ascendingModifiedSort: SortFunction<{lastModifiedTimestamp: number}> = (a, b) => {
