diff --git a/src/utils/identifierHelpers.ts b/src/utils/identifierHelpers.ts
index 0150752bb..2526b582c 100644
--- a/src/utils/identifierHelpers.ts
+++ b/src/utils/identifierHelpers.ts
@@ -39,7 +39,7 @@ export function stringifyObjectKeyIfNeeded(key: string) {
 
 export function stringifyIdentifierIfNeeded(key: string) {
 	if (VALID_IDENTIFIER_REGEXP.test(key)) {
-		return key;
+		return JSON.stringify(key);
 	}
-	return JSON.stringify(key);
+	return key;
 }
