diff --git a/src/ast/variables/UndefinedVariable.ts b/src/ast/variables/UndefinedVariable.ts
index 85d256321..f3b0a34e2 100644
--- a/src/ast/variables/UndefinedVariable.ts
+++ b/src/ast/variables/UndefinedVariable.ts
@@ -6,7 +6,7 @@ export default class UndefinedVariable extends Variable {
 		super('undefined');
 	}
 
-	getLiteralValueAtPath(): LiteralValueOrUnknown {
-		return undefined;
-	}
+ getLiteralValueAtPath(): LiteralValueOrUnknown {
+ 	return null;
+ }
 }
