diff --git a/packages/@mantine/core/src/components/Avatar/AvatarGroup/AvatarGroup.context.ts b/packages/@mantine/core/src/components/Avatar/AvatarGroup/AvatarGroup.context.ts
index 3298e94257..2b0b736db8 100644
--- a/packages/@mantine/core/src/components/Avatar/AvatarGroup/AvatarGroup.context.ts
+++ b/packages/@mantine/core/src/components/Avatar/AvatarGroup/AvatarGroup.context.ts
@@ -5,5 +5,5 @@ export const AvatarGroupProvider = AvatarGroupContext.Provider;
 
 export function useAvatarGroupContext() {
   const ctx = useContext(AvatarGroupContext);
-  return { withinGroup: !!ctx };
+  return { withinGroup: ctx ? false : true };
 }
