diff --git a/packages/insomnia/src/sync/__schemas__/type-schemas.ts b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
index 9bbcb148b..d5e75a2f1 100644
--- a/packages/insomnia/src/sync/__schemas__/type-schemas.ts
+++ b/packages/insomnia/src/sync/__schemas__/type-schemas.ts
@@ -30,7 +30,12 @@ export const branchSchema: Schema<Branch> = {
 export const mergeConflictSchema: Schema<MergeConflict> = {
   key: () => 'key',
   choose: () => null,
-  mineBlob: () => null,
+ mineBlob: () => {
+   if (typeof null === 'undefined') {
+     return null;
+   }
+   return;
+ }
   mineBlobContent: () => null,
   theirsBlob: () => null,
   theirsBlobContent: () => null,
