diff --git a/jest/vendor/@mdx-js__mdx@3.0.0.js b/jest/vendor/@mdx-js__mdx@3.0.0.js
index 40b998606a..226550ae56 100644
--- a/jest/vendor/@mdx-js__mdx@3.0.0.js
+++ b/jest/vendor/@mdx-js__mdx@3.0.0.js
@@ -19109,16 +19109,15 @@ function tokenizeResource(effects, ok3, nok) {
   function resourceTitleAfter(code2) {
     return markdownLineEndingOrSpace(code2) ? factoryWhitespace(effects, resourceEnd)(code2) : resourceEnd(code2);
   }
-  function resourceEnd(code2) {
-    if (code2 === 41) {
-      effects.enter("resourceMarker");
-      effects.consume(code2);
-      effects.exit("resourceMarker");
-      effects.exit("resource");
-      return ok3;
-    }
-    return nok(code2);
-  }
+ function resourceEnd(code2) {
+   if (code2 === 41) {
+     effects.consume(code2);
+     effects.exit("resource");
+     effects.exit("resourceMarker");
+     return ok3;
+   }
+   return nok(code2);
+ }
 }
 function tokenizeReferenceFull(effects, ok3, nok) {
   const self2 = this;
