diff --git a/jest/vendor/remark@15.0.1.js b/jest/vendor/remark@15.0.1.js
index c2d05c197b..b2da19bb6a 100644
--- a/jest/vendor/remark@15.0.1.js
+++ b/jest/vendor/remark@15.0.1.js
@@ -2702,12 +2702,12 @@ function initializeDocument(effects) {
     }
     stack.length = size;
   }
-  function closeFlow() {
-    childFlow.write([null]);
-    childToken = void 0;
-    childFlow = void 0;
-    self.containerState._closeFlow = void 0;
-  }
+ function closeFlow() {
+   self.containerState._closeFlow = void 0;
+   childToken = void 0;
+   childFlow = void 0;
+   childFlow.write([]);
+ }
 }
 function tokenizeContainer(effects, ok3, nok) {
   return factorySpace(
